---
layout: post
title: "Structure of FFI prion protein"
author: ericminikel
deprecated: true
date: 2012-02-18 18:47:34
---
<p>A few weeks ago, @K posted Risener&#8217;s 2003 <a href="http://bmb.oxfordjournals.org/content/66/1/21.full">Biochemistry and structure of PrP<sup>c</sup> and PrP<sup>Sc</sup></a> , a review of what was then known and not known about how prion protein folds.  At that time, the structure of PrP<sup>Sc</sup> had not been fully elucidated and the author stated that &#8220;Most probably, the model shown in <a id="xref-fig-6-2" href="http://bmb.oxfordjournals.org/content/66/1/21.full#F6">Plate IV</a> is not the final description of the structure of PrP<sup>Sc</sup>, but it is the best model currently available&#8221;</p>
<p>It seems that our knowledge has moved forward since then: @a has just pointed out a 2009 article in EMBO which publishes the solved structures of several PrP variants including FFI prion protein: <a href="http://www.nature.com/emboj/journal/v29/n1/full/emboj2009333a.html">Conformational diversity in prion protein variants influences intermolecular β-sheet formation</a> by Lee et al.  This article publishes seven PrP structures&#8211; three wild type and four mutants.  The four mutants are all the possible permutations of D178N 129M and V, and F198S 129M and V.</p>
<p>Now, something a bit troubling is that the authors&#8211; not just once but throughout the article&#8211; refer to D178N 129V as being the genotype for FFI, and D178N 129M as being a genotype for fCJD.  They cite Goldfarb et al 1992 as follows:</p>
<blockquote><p>The D178N mutation is associated with two pathologically distinct inherited prion diseases, CJD and FFI, when the mutation colocalizes with M129 or V129, respectively (Goldfarb <em>et al</em>, 1992).</p></blockquote>
<p>But in fact, <a href="http://www.sciencemag.org/content/258/5083/806">Goldfarb et al 1992</a> actually state that:</p>
<blockquote><p>The Met129, Asn178 allele segregated with FFI in all 15 affected members of five kindreds whereas the Val129, Asn178 allele segregated with the familial CJD subtype in all 15 affected members of six kindreds.</p></blockquote>
<p>Indeed, every other source we&#8217;ve ever read, and at least one other reference that Lee et al cite (<a href="http://pubs.acs.org/doi/abs/10.1021/bi051455%2B">Apreti et al 2005</a>) state that D178N 129M = FFI and D178N 129V = fCJD.  So we are forced to conclude that this is simply a mistake that somehow made it through peer review, though so far we&#8217;ve not been able to find any correction published in EMBO.</p>
<p>But in any event this is an awesome article.  While it shows how much I&#8217;ve got to learn about protein structure and folding, here are a couple of highlights that are immediately accessible:</p>
<ul>
<li>The wonderfully clear graphics in <a href="http://www.nature.com/emboj/journal/v29/n1/fig_tab/emboj2009333a_F2.html">Figure 2</a> show how proximate the 178 and 129 positions are to one another in the folded structure and how the difference in a few bonds in this region affects the overall structure.</li>
<li><a href="http://www.nature.com/emboj/journal/v29/n1/fig_tab/emboj2009333a_F3.html">Figure 3</a> shows the interaction of PrP dimers depending on 178 and 129 polymorphisms.  Is this dimer the pathological agent in FFI?  Or can it help us understand the pathology?</li>
</ul>
<p>As for implications, the authors acknowledge there is still a lot we don&#8217;t know about all the conformational changes that occur in PrP:</p>
<blockquote><p>The crystal structures reported here represent a small sample of the conformational states that hPrP may adopt as a consequence of different pathogenic mutations or during the course of pathogenesis. These seven structures identify several conformationally variable features, which appear to be correlated and allow us to speculate on a sequence of related structural shifts, which may play a role in the pathogenic mechanism(s)</p></blockquote>
<p>Nevertheless this is an important step forward for curing prion diseases.  Happy reading.</p>
