---
layout: post
title: "Genomics"
author: ericminikel
deprecated: true
date: 2012-01-08 18:58:01
---
<div>
<p>“Prion protein repeat expansion results in increased aggregation and reveals phenotypic variability,” Tank et al., Mol. Cell Biol (2007)</p>
<p><a href="http://mcb.asm.org/content/27/15/5445.full">http://mcb.asm.org/content/27/15/5445.full</a></p>
<blockquote><p>Mammalian prion diseases are fatal neurodegenerative disorders dependent on the prion protein PrP. Expansion of the oligopeptide repeats (ORE) found in PrP is associated with inherited prion diseases. Patients with ORE frequently harbor PrP aggregates, but other factors may contribute to pathology, as they often present with unexplained phenotypic variability. We created chimeric yeast-mammalian prion proteins to examine the influence of the PrP ORE on prion properties in yeast. Remarkably, all chimeric proteins maintained prion characteristics. The largest repeat expansion chimera displayed a higher propensity to maintain a self-propagating aggregated state. Strikingly, the repeat expansion conferred increased conformational flexibility, as observed by enhanced phenotypic variation. Furthermore, the repeat expansion chimera displayed an increased rate of prion conversion, but only in the presence of another aggregate, the [RNQ+] prion. We suggest that the PrP ORE increases the conformational flexibility of the prion protein, thereby enhancing the formation of multiple distinct aggregate structures and allowing more frequent prion conversion. Both of these characteristics may contribute to the phenotypic variability associated with PrP repeat expansion diseases.</p></blockquote>
</div>
<p>&nbsp;</p>
<p>“Genotypic  and phenotypic studies of inherited prion disease,” Webb, T.E.F. (2010) Genotypic and phenotypic studies of inherited prion disease. Doctoral thesis, UCL (University College London).</p>
<p><a href="http://discovery.ucl.ac.uk/669507/" target="_blank">http://discovery.ucl.ac.uk/<wbr>669507/</wbr></a></p>
<blockquote><p>Background: Inherited prion diseases (IPD) show remarkable clinical heterogeneity, posing problems for clinicians in making an early diagnosis and raising questions about genetic or environmental modifiers. The P102L prion protein gene (PRNP) mutation, one of the most frequently identified causes of IPD, has been linked to a large English kindred for three decades. A series of further smaller kindreds did not share apparent ancestry with this large kindred, raising the possibility of distant common ancestry of a rare mutation or no ancestry and relatively common novel mutations occurring. Identification of the genetic modifiers of phenotype may have implications for sporadic and variant Creutzfeldt-Jakob disease (sCJD), and neurodegeneration more widely, while novel mutation rates in PRNP inform study of the unknown aetiology of sCJD. IPD, although rare, has unique advantages in terms of looking for genetic modifiers of prion disease. Methods: Genealogical work and microsatellite haplotyping was carried out on cases of IPD P102L. Clinical information on affected patients was collected retrospectively as well as prospectively. Heritability estimates of IPD were produced by comparing parental and offspring phenotypes. The expanded collection of IPD cases with reliable clinical information (composing P102L and other IPD associated mutations), allowed for the testing of candidate genetic modifiers of phenotype. Polymorphisms previously reported to have an impact on prion disease susceptibility or phenotype were tested, along with a panel of candidate polymorphisms selected from a genome-wide association study (GWAS) in vCJD. Findings: Common ancestry between the known large P102L kindred and a number of previously unlinked P102L kindreds was identified. However, a number of apparently unrelated IPD P102L kindreds remain, <strong>suggesting that multiple separate mutational events are responsible: PRNP codon 102 may be a mutation ‘hot- spot’.</strong> Microsatellite work on other IPD subtypes in the UK and from elsewhere in Europe also finds evidence of the existence of multiple unrelated kindreds.</p>
<p>&#8212;</p>
<p>Also identified was a <strong>possible effect of APOE genotype on IPD. APOE-E4 is associated with a significantly later age at onset in IPD, which has recently been identified in other categories of neurodegenerative disease.</strong> Interpretation: Overall, this work provides indirect support for the plausibility of the ‘somatic mutation hypothesis’ of sporadic CJD (sCJD), by suggesting a relatively frequent novel mutation rate in PRNP. The genetic contribution to phenotypic heterogeneity in IPD was estimated. This contribution is significant and may inform the search for genetic modifiers of susceptibility to acquired prion disease. The models established to analyse IPD as a quantitative trait will be used in future prion disease GWAS.</p></blockquote>
<p>&nbsp;</p>
<p>&#8220;Prion proteins from susceptible and resistance sheep exhibit some distinct cell biological features,&#8221; Sabuncu, et. a. (2005)<br/>
<span style="text-decoration: underline;"><a href="http://www.wikigenes.org/e/ref/e/16214113.html" target="_blank">http://www.wikigenes.org/e/<wbr>ref/e/16214113.html</wbr></a><br/>
</span></p>
<blockquote><p>It is well established that natural polymorphisms in coding sequence of the PrP protein can control the expression of prion disease.  Studies with a cell model of sheep prion infection have shown that ovine PrP allele associated with resistance to sheep scrapie may confer resistance by impairing the multiplication of the infectious agent.  To further explore the biochemical and cellular mechanisms underlying the genetic control of scrapie susceptibility, we established permissive cells expressing two different PrP variants.  In this study, we show that PrP variants with opposite effects on prion multiplication exhibit distinct cell biological features.  These findings indicate that <strong>cell biological properties of ovine PrP can vary with natural polymorphisms</strong> and raise the possibility that <strong>differential interactions of PrP variants with the cellular machinery may contribute to permissiveness or resistance to prion multiplication.</strong></p></blockquote>
