---
layout: post
title: "Screen reveals antiprion compounds that act indirectly"
author: ericminikel
deprecated: true
date: 2012-10-06 20:04:08
---
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3149362/">Poncet-Montagne 2011</a>, of UCSF, brings us the results of a 2,160-compound screen for antiprion molecules with a (perhaps) surprising conclusion: all the molecules found to exhibit antiprion properties do so through third parties.  None interact directly with PrP.</p>
<p>The methods for this screen are described in quite a bit of detail, which I took as a chance to learn how a high-throughput assay for prion therapeutics can be designed.  The screen, which included &#8220;known drugs, bioactives, and natural products&#8221; (presumably <a href="http://www.msdiscovery.com/spectrum.html">MSDI&#8217;s Spectrum Collection</a>), was conducted at the <a href="https://smdc.ucsf.edu/">Small Molecule Discovery Center</a>, UCSF&#8217;s high-throughput screening facility.  The initial screen used mouse neuroblastoma cells (N2a cells) infected with RML prions (this combination is called ScN2a cells) in 96-well plates, coincubated with candidate molecules for 5 days.  The<a href="http://en.wikipedia.org/wiki/Scientific_control#Negative"> negative control</a> for each compound was ScN2a cells treated with <a href="http://en.wikipedia.org/wiki/Dimethyl_sulfoxide">DMSO</a>, in which the tested compounds had been solubilized; the <a href="http://en.wikipedia.org/wiki/Scientific_control#Positive">positive control</a> was ScN2a cells treated with quinacrine.  The overall screening methodology is called <a href="http://en.wikipedia.org/wiki/ELISA">ELISA</a> and had been described for ScN2a cells in an earlier screen done by the same lab [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838138/">Ghaemmaghami 2010</a>]; the full workflow for the assay is in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2838138/figure/f1/">Fig 1A</a> from that 2010 study.  So it took some digging, but basically the readout from this assay is as follows:</p>
<blockquote><p>For each compound, cytotoxicity was measured by a calcein-AM assay, and the effect on PrP<sup>Sc</sup> levels was detected by ELISA following PK digestion and PTA precipitation.</p></blockquote>
<p>So basically, the ScN2a cells were coincubated with the compounds for 5 days, then subjected to <a href="http://en.wikipedia.org/wiki/Proteinase_K">proteinase K</a>, which should digest PrP<sup>C</sup> but not PrP<sup>Sc</sup>, and then immunoblotted with anti-PrP antibodies to get a visual readout as to whether PrP was still present.   The images were analyzed using ImageJ (they don&#8217;t say exactly how, but here&#8217;s <a href="http://lukemiller.org/index.php/2010/11/analyzing-gels-and-western-blots-with-image-j/">an example of using ImageJ on Western blots</a>).  So that&#8217;s the readout: the less PrP is still present after proteinase K digestion, the less PrP<sup>Sc</sup> there must have been, and so the better the compound&#8217;s antiprion properties.  So the outcomes of this screen are only translatable into therapeutics to the extent that proteinase K resistance correlates with infectivity and toxicity.  I understand that this correlation is fairly strong but not absolute&#8211; I&#8217;m still looking for a citation that discusses the issue in more detail.</p>
<p>As an aside, I found it curious that the screen used quinacrine as a <a href="http://en.wikipedia.org/wiki/Scientific_control#Positive">positive control</a>.  This implies that quinacrine&#8217;s antiprion properties are so well-established that they are actually the standard by which to judge other compounds. I don&#8217;t know how to square this with the fact that <a href="http://jvi.asm.org/content/77/15/8462.full">Barret 2003</a> showed that quinacrine&#8217;s antiprion activity was less than that of tetracyclines.  It is also peculiar that quinacrine works so well in vitro yet ultimately failed clinical trials [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2660392/">Collinge 2009</a>].  Hopefully more on that in a future post.</p>
<p>Now, what I just described above was just the initial discovery assay as best I understand it.  A &#8220;hit&#8221; was defined as &#8220;a compound that reduced PrP<sup>Sc</sup> load by 40% with less than 20% cytotoxicity&#8221; &#8212; i.e. it both works and doesn&#8217;t kill the cells.  The assay of 2,160 compounds yielded 206 hits, which largely overlapped with the hits from another screen by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC228499/">Kociscko 2003</a>.  206 was too many to follow up on, so the authors randomly chose 40 of the 206 compounds.</p>
<p>These 40 compounds then went through the same ELISA screening protocol again and 16 of them passed it again.</p>
<p>So then came the question, what exactly are these 16 compounds doing?  The authors had a whole battery of tests to try to assess this.  Here is an incomplete list:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Isothermal_titration_calorimetry">Isothermal titration calorimetry</a> to figure out if the compounds were interacting directly with PrP.  Answer: no.  None of them interacted directly.  The positive control was <a href="http://en.wikipedia.org/wiki/Suramin">suramin</a>, which does interact directly with PrP (in a bad way, inducing aggregation).  This part of the study used synthetic peptides of mouse PrP 89-230 rather than full-length PrP.  Not sure why.</li>
<li><a href="http://en.wikipedia.org/wiki/Circular_dichroism">Circular dichroism</a> to measure whether the compounds affected the melting point of MoPrP(89-230).  If they did, that would suggest they were binding to PrP.  Answer: no, they didn&#8217;t bind to PrP.  The positive control was <a href="http://en.wikipedia.org/wiki/Trimethylamine_N-oxide">TMAO</a>, which does bind to and stabilize PrP.</li>
<li>Differential scanning fluorimetry (DSF) to measure whether compounds interact directly with PrP.  After the above two experiements, the authors wondered if the fact that the 16 hits didn&#8217;t interact with PrP was because none of the original 2,160 compounds did so, or because interacting with PrP is a poor predictor of antiprion activity.  So they actually went back and screened all of the original compounds with DSF.  Answer: none of the original compounds interact with PrP (again, this is actually MoPrP(89-230), not full-length PrP).  The authors did not find this surprising, because:<br/>
<blockquote><p>NMR and crystallographic studies indicate that the three-dimensional structure of PrP lacks deep cavities normally required for high-affinity binding interactions with small molecules, and as such, few molecules within an unbiased diversity chemical library are expected to bind PrP</p></blockquote>
</li>
<li>The authors purified PrP<sup>Sc</sup> from mouse brains and tested by Western blot whether any of the 16 hits increased the proteinase K sensitivity of this PrP<sup>Sc </sup>in this purified chemical setting.  Answer: no.  They don&#8217;t.  Interesting.  So when you add these compounds to ScN2a cells, digest with proteinase K and then Western blot, you find that there is less PrP<sup>Sc</sup>, yet when you do the same experiment with purified PrP and no cell present, the effect goes away.  So if the compounds do increase the proteinase K sensitivity of PrP<sup>Sc</sup> in cells, they must do so via some indirect pathway dependent upon cellular process or other proteins or small molecules present in the cell.</li>
<li>Screening of the 16 hits on <em>uninfected</em> N2a cells followed by Western blotting (no proteinase K step here) to simply see if the compounds reduce the amount of PrP<sup>C</sup> present in the cells.  Answer: yes, a few of them do.  It makes perfect sense that reducing the overall amount of PrP would be one route to antiprion activity.  There&#8217;s plenty of evidence out there that PrP infectivity is concentration-dependent.  For instance, <a href="/wp-content/uploads/2012/02/jackson-et-al-2009-spontaneous-generation-of-prion-infectivity-in-ffi-knockin-mice.pdf">Jackson 2009</a> was able to achieve FFI prion infectivity only in mice overexpressing wild-type PrP and not in mice expressing wild-type PrP at normal levels.  <a href="/2012/08/03/how-tafamidis-was-born/">Jeffrey W. Kelly</a> also likes to talk about how &#8220;protein aggregation is a concentration-dependent process&#8221;.  So if a compound simply reduced the total amount of PrP present, that could in itself have therapeutic value even if the compound didn&#8217;t target  PrP<sup>Sc</sup> with any specificity.  That looks to be at least part of the story for some of these 16 compounds.</li>
<li>A flotation assay to see if any of the 16 compounds disrupt lipid rafts.  The authors offer that &#8220;an alternative mechanism for decreasing PrP<sup>Sc</sup> formation is to alter the integrity of lipid rafts, membrane microdomains where PrP<sup>C</sup> is known to localize&#8221;, which has been the rationale for some of the <a href="/2012/09/28/statins-as-prion-therapeutics/">studies on statins and prion disease</a>, though those studies ultimately concluded that cholesterol was not the statins&#8217; mechanism of action on prions.  Anyway, two of the 16 hits in this study were statins.    Result: of the 16, only <a href="http://en.wikipedia.org/wiki/Lovastatin">lovastatin</a> appeared to disrupt lipid rafts.  However, several of the compounds appeared to affect PrP localization.</li>
<li>Two more experiments on <a href="http://en.wikipedia.org/wiki/Amcinonide">amcinonide</a>, which was the compound which reduced the concentration of PrP<sup>C</sup> the most in the above experiment on the uninfected N2a cells. The authors wanted to know at what step amcinonide acted on PrP<sup>C</sup> levels: by reducing transcription? Nope, <a href="http://www.sigmaaldrich.com/life-science/molecular-biology/molecular-biology-products.html?TablePage=9620611">qPCR</a> showed no effect on PrP mRNA levels.  So then the authors stopped translation of PrP with <a href="http://en.wikipedia.org/wiki/Cycloheximide">cyclohexamide</a>, and then added amcinonide to see how it affected the degradation of the already-translated PrP.  The half life of PrP was reduced from the 12-18h range down to 6h.  So amcinonide acts by increasing PrP degradation.</li>
</ul>
<p>So those were the experiments.  Here are the 16 molecules:</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3149362/table/T1/"><img class="alignnone size-full wp-image-842" title="zou-2011-fig-1-16-antiprion-compounds" src="/wp-content/uploads/2012/10/zou-2011-fig-1-16-antiprion-compounds.jpg"/></a></p>
<p>The authors haven&#8217;t yet reported any in vivo validation of these 16 hits, so it&#8217;s on the early side to get hopeful that any of these compounds could be used as prion therapeutics.  But what&#8217;s really interesting from this screen is the finding that maybe direct interaction with PrP is  not the route to drug development.  In fact, the authors even question the idea that <em>any</em> compounds previously identified by other studies really directly interact with PrP in a way that is specific, reproducible, and highly affine.  They think this idea of direct interaction with PrP is a dead end and that assays focused on direct interaction hold little promise:</p>
<blockquote><p>The results are consistent with the reported crystal and NMR structures of PrP<sup>C</sup>, which indicate the lack of prominent structural clefts capable of ligand binding (<a id="__tag_228996331" href="http://www.ncbi.nlm.nih.gov/pubmed/8700211">32</a>–<a id="__tag_228996283" href="http://www.ncbi.nlm.nih.gov/pubmed/11524679">35</a>). Various compounds have been reported to bind PrP<sup>C</sup> <em>in vitro</em>. However, these interactions have been shown to be nonspecific (<em>e.g.</em> anionic tetrapyrroles (<a id="__tag_228996332" href="http://www.ncbi.nlm.nih.gov/pubmed/17709470">48</a>)), irreproducible in alternative binding assays (<em>e.g.</em> Gn8 (<a id="__tag_228996329" href="http://www.ncbi.nlm.nih.gov/pubmed/20876144">10</a>)), or have millimolar affinities (<em>e.g.</em> quinacrine (<a id="__tag_228996330" href="http://www.ncbi.nlm.nih.gov/pubmed/12904059">49</a>)). More recently, it was reported that a cationic tetrapyrrole (Fe(III)-TMPyP) can specifically interact with the folded domain of human PrP with micromolar affinities, although the large size of the compound may preclude it as an effective brain-permeable clinical therapeutic (<a id="__tag_228996279" href="http://www.ncbi.nlm.nih.gov/pubmed/11274893">50</a>). The results of our study suggest that a drug-discovery screen focused on the identification of specific PrP-interacting molecules, e.g. protein X, is unlikely to identify many efficacious compounds.</p></blockquote>
<p>Instead, antiprion compounds are likely to act through a third party molecule or &#8220;protein X&#8221;:</p>
<blockquote><p>In protein aggregation diseases, including prion diseases, a common target for pharmacological intervention has been the aggregating molecule itself (<a id="__tag_228996324" href="http://www.ncbi.nlm.nih.gov/pubmed/10742177">46</a>, <a id="__tag_228996276" href="http://www.ncbi.nlm.nih.gov/pubmed/20043671">47</a>).  Although we were able to identify 16 nontoxic compounds that were active against prion formation in cell culture, none of the 16 molecules interacted with PrP<sup>C</sup> or PrP<sup>Sc</sup> <em>in vitro</em>, as evaluated by various biophysical techniques. These observations imply that at least one other cellular target (protein X), likely several given the diversity of the active molecules, modulates PrP conversion within the cell.</p></blockquote>
<p>This idea of a &#8220;protein X&#8221; which is involved in prion formation interesting.  If true, then genetic variation in the gene coding for such a &#8220;protein X&#8221; should be a significant modifier of prion disease.  For instance, homozygous loss of function mutations in protein X might render someone resistant to genetic prion disease.  Or variants which increase the affinity of protein X for interaction with PrP<sup>Sc</sup> might be risk factors for sporadic prion diseases.  Yet as mentioned in <a href="/2012/08/06/genome-wide-association-studies/">the GWAS post</a>, genome-wide association studies have yet to identify any risk factors for prion disease other than, of course, loci within PRNP itself.</p>
<p>This also leads me to the question, would &#8220;protein X&#8221; be the same for different prion diseases?  Perhaps different proteins would have different affinities for different misfolded conformations of PrP, or perhaps the different strains have different levels of dependence on things like lipid rafts which protein X could be involved in indirectly.  CJD, FFI, and GSS each have different patterns of pathology across the various brain regions, different signatures in terms of plaque formation and vacuole formation, and so on.  Roles for multiple different &#8220;X&#8221; proteins could be one explanation.  If so, then antiprion therapeutics might not always translate well across prion strains, but</p>
<p>In concluding, the authors identify a couple of mechanisms for potential antiprion therapies based on their results.  They remind us that PrP knockout mice are resistant to prion disease and point out that compounds like amcinonide which increase degradation of PrP could have therapeutic value.  They also discuss the two statins &#8212; <a href="http://en.wikipedia.org/wiki/Lovastatin">lovastatin</a> (Mevacor) and <a href="http://en.wikipedia.org/wiki/Rosuvastatin">rosuvastatin</a> (Crestor) &#8212; in their hit list and the potential role of lipid rafts as necessary for prion formation.  This notion of statins disrupting lipid rafts and therefore prion formation is difficult to reconcile with the fact that the <a href="/2012/09/28/statins-as-prion-therapeutics/">four studies on statins</a> I reviewed last week generally seemed to conclude that the mechanism of action of statins on prion disease did not involve cholesterol and, moreover, that mice treated with statins actually had <em>more</em> PrP<sup>Sc</sup> than untreated mice at the time of death. <strong>update 2013-05-24: it&#8217;s actually more complicated and hard to draw conclusions about mechanism of action from those studies, see <a href="/2013/05/24/a-comparison-of-statin-studies-in-scrapie-infected-mice/">this new post</a>.  </strong>For what it&#8217;s worth, these are different statins than tested in those studies.  I dug up <a href="http://www.msdiscovery.com/download/SPEC_2320.xls">the compound list</a> for MSDI&#8217;s Spectrum Collection, which has since grown to 2,320 compounds, and it does include both simvastatin and pravastatin.  These are old enough drugs that I am assuming they were also in the 2,160 initially screened in this study.  I&#8217;d be curious whether they were some of the 206 initial hits that just didn&#8217;t get randomly selected for followup.</p>
<p>As for the other 13 compounds, they state that we really don&#8217;t know the mechanism of action, other than it&#8217;s probably not direct interaction with PrP.</p>
<p>All in all, the 16 hits validated here are a pretty interesting mixture of FDA-approved drugs, herbal compounds, and experimental compounds:</p>
<ul>
<li><a href="http://en.wikipedia.org/wiki/Isoliquiritigenin">Isoliquiritigenin</a>.  This is one of the &#8216;natural&#8217; compounds, a licorice phenol; under &#8220;source&#8221;, MSDI simply says &#8220;widespread in <a href="http://en.wikipedia.org/wiki/Fabaceae">Fabaceae</a>&#8220;.  It acts on <a href="http://en.wikipedia.org/wiki/Sirtuin">sirtuins</a>, those magical proteins which some people think are the key to immortality.  (Sirtuins are also the target of <a href="http://en.wikipedia.org/wiki/Resveratrol">resveratrol</a>, that hyped-up compound in red wine).</li>
<li><a href="http://en.wikipedia.org/wiki/Irigenin">Irigenin</a>, 7-benzyl ester.  A derivative of another natural compound from iris rhizomes.</li>
<li>3-deoxy-3-beta hydroxyangolensic methyl ester.  No idea what this compound&#8217;s deal is.</li>
<li><a href="http://en.wikipedia.org/wiki/Amcinonide">Amcinonide</a>.  This is actually an FDA-approved drug, though for topical use only.   It&#8217;s one of those corticosteroids they prescribe as a cream for itchy skin. Branded as <a href="http://www.drugs.com/cdi/cyclocort-cream.html">cyclocort</a>.</li>
<li><a href="http://www.rdchemicals.com/chemicals.php?mode=details&amp;mol_id=7910">4&#8242;-hydroxy chalcone</a>.  No idea.</li>
<li><a href="http://en.wikipedia.org/wiki/Genistein">Genistein</a>, another herbal compound, an isoflavone present in a lot of legumes as well as coffee.  Despite the &#8216;herbal&#8217; status, there seem to be a lot of references to molecular interactions that have been demonstrated for genistein.  Among them, <a href="http://cebp.aacrjournals.org/content/17/7/1596.full">Singletary 2008</a> cites evidence that it promotes autophagy and apoptosis in cancer (good and bad from the perspective of prion disease??)</li>
<li><a href="http://en.wikipedia.org/wiki/Daidzein">Daidezin</a>, another isoflavone present in soybeans among other things.  It&#8217;s a legal dietary supplement in the U.S. but not considered a drug.</li>
<li><a href="http://en.wikipedia.org/wiki/Digitoxin">Digitoxin</a>, true to the &#8216;toxin&#8217; in its name, was once the murder weapon (it&#8217;s present in foxglove) in an Agatha Christie novel. But in lower quantities it&#8217;s also a drug.  It interferes with sodium-potassium pumps in cardiac muscle, thus increasing intracellular calcium concentration in the ER, so that when the cell is stimulated it releases more calcium, resulting in a stronger contraction.  It was used to against heart failure for a while, though it seems to have fallen from favor.</li>
<li>Dehydrovariabilin.  No idea.</li>
<li><a href="http://en.wikipedia.org/wiki/Lovastatin">Lovastatin</a> (Mevacor), a statin.</li>
<li><a href="http://en.wikipedia.org/wiki/Rosuvastatin">Rosuvastatin</a> (Crestor), another statin.</li>
<li><a href="http://www.enzolifesciences.com/BML-FR119/triacetyl-resveratrol/">Triacetyl resveratrol</a>.  According to the link at left from Enzo Life Sciences, it&#8217;s regular old resveratrol plus a phenol group to increase its half life in the body.  As mentioned above, resveratrol is a sirtuin activator and much-hyped anti-aging compound.</li>
<li><a href="http://en.wikipedia.org/wiki/Tretinoin">Tretinoin</a>, the acid form of vitamin A, in use as a drug for acne and cancer.</li>
<li><a href="http://en.wikipedia.org/wiki/Profenamine">Ethopropazine</a>, aka profenamine.  A drug prescribed to control the symptoms of Parkinson&#8217;s disease.  It antagonizes <a href="http://en.wikipedia.org/wiki/Muscarinic_acetylcholine_receptor">muscarinic ACh receptors</a> and so controls the involuntary movement in Parkinson&#8217;s.</li>
<li><a href="http://en.wikipedia.org/wiki/Digitonin">Digitonin</a>, which is only one letter different from digitoxin simply because they both come from foxglove (<a href="http://en.wikipedia.org/wiki/Digitalis_purpurea">Digitalis purpurea</a>).  It water-solubilizes lipids.</li>
<li>Chrysanthellin A.  Couldn&#8217;t find much about this online, and it&#8217;s actually not in MSDI&#8217;s spreadsheet, at least under that name.  This <a href="http://www.clearsynth.com/docs/MSD-CS-E-00231.pdf">material safety data sheet</a> from a supplier in Mumbai suggests it is probably not ready for prime time as a drug.</li>
</ul>
<p>Again, it&#8217;s an interesting list but remember these were merely the hits from an in vitro screen with no in vivo validation yet.  <em>This blog does not provide opinions on the efficacy of any medical treatment and nothing said here should be construed as medical advice.</em></p>
