---
layout: post
title: "Glossary Part 2"
author: ericminikel
deprecated: true
date: 2012-01-08 18:40:45
---
<blockquote><p><strong>Recombinant DNA</strong> (rDNA) molecules are <a title="DNA" href="http://en.wikipedia.org/wiki/DNA" target="_blank">DNA</a> sequences that result from the use of laboratory methods (<a title="Molecular cloning" href="http://en.wikipedia.org/wiki/Molecular_cloning" target="_blank">molecular cloning</a>) to bring together genetic material from multiple sources, creating <a title="DNA sequence" href="http://en.wikipedia.org/wiki/DNA_sequence" target="_blank">sequences</a> that would not otherwise be found in biological organisms. Recombinant DNA is possible because DNA molecules from all organisms share the same chemical structure; they differ only in the sequence of nucleotides within that identical overall structure. Consequently, when DNA from a foreign source is linked to host sequences that can drive <a title="DNA replication" href="http://en.wikipedia.org/wiki/DNA_replication" target="_blank">DNA replication</a> and then introduced into a host organism, the foreign DNA is replicated along with the host DNA.</p>
<p>&nbsp;</p>
<p><strong>Polymerase Chain Reaction</strong></p>
<p>The <strong>polymerase chain reaction</strong> (<strong>PCR</strong>) is a <a title="Scientific technique" href="http://en.wikipedia.org/wiki/Scientific_technique" target="_blank">scientific technique</a> in <a title="Molecular biology" href="http://en.wikipedia.org/wiki/Molecular_biology" target="_blank">molecular biology</a> to <a title="DNA replication" href="http://en.wikipedia.org/wiki/DNA_replication" target="_blank">amplify</a> a single or a few copies of a piece of <a title="DNA" href="http://en.wikipedia.org/wiki/DNA" target="_blank">DNA</a> across several orders of magnitude, generating thousands to millions of copies of a particular <a title="DNA sequence" href="http://en.wikipedia.org/wiki/DNA_sequence" target="_blank">DNA sequence</a>.</p>
<p>Developed in 1983 by <a title="Kary Mullis" href="http://en.wikipedia.org/wiki/Kary_Mullis" target="_blank">Kary Mullis</a>,<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-Bartlett_.26_Stirling-0" target="_blank">[1]</a></sup> PCR is now a common and often indispensable technique used in medical and biological research labs for a variety of applications.<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-Saiki1-1" target="_blank">[2]</a><a href="http://en.wikipedia.org/wiki/PCR#cite_note-Saiki2-2" target="_blank">[3]</a></sup> These include <a title="DNA cloning" href="http://en.wikipedia.org/wiki/DNA_cloning" target="_blank">DNA cloning</a> for <a title="DNA sequencing" href="http://en.wikipedia.org/wiki/DNA_sequencing" target="_blank">sequencing</a>, DNA-based <a title="Phylogeny" href="http://en.wikipedia.org/wiki/Phylogeny" target="_blank">phylogeny</a>, or functional analysis of <a title="Gene" href="http://en.wikipedia.org/wiki/Gene" target="_blank">genes</a>; the diagnosis of <a title="Hereditary disease" href="http://en.wikipedia.org/wiki/Hereditary_disease" target="_blank">hereditary diseases</a>; the identification of <a title="Genetic fingerprint" href="http://en.wikipedia.org/wiki/Genetic_fingerprint" target="_blank">genetic fingerprints</a> (used in <a title="Forensic science" href="http://en.wikipedia.org/wiki/Forensic_science" target="_blank">forensic sciences</a> and <a title="Parental testing" href="http://en.wikipedia.org/wiki/Parental_testing" target="_blank">paternity testing</a>); and the detection and diagnosis of <a title="Infectious disease" href="http://en.wikipedia.org/wiki/Infectious_disease" target="_blank">infectious diseases</a>. In 1993, Mullis was awarded the <a title="Nobel Prize in Chemistry" href="http://en.wikipedia.org/wiki/Nobel_Prize_in_Chemistry" target="_blank">Nobel Prize in Chemistry</a> along with <a title="Michael Smith (chemist)" href="http://en.wikipedia.org/wiki/Michael_Smith_%28chemist%29" target="_blank">Michael Smith</a> for his work on PCR.<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-Kary_Mullis_Nobel_Lecture-3" target="_blank">[4]</a></sup></p>
<p>The method relies on <a title="Thermal cycler" href="http://en.wikipedia.org/wiki/Thermal_cycler" target="_blank">thermal cycling</a>, consisting of cycles of repeated heating and cooling of the reaction for <a title="DNA melting" href="http://en.wikipedia.org/wiki/DNA_melting" target="_blank">DNA melting</a> and <a title="Enzyme" href="http://en.wikipedia.org/wiki/Enzyme" target="_blank">enzymatic</a> <a title="DNA replication" href="http://en.wikipedia.org/wiki/DNA_replication" target="_blank">replication</a> of the DNA. <a title="Primer (molecular biology)" href="http://en.wikipedia.org/wiki/Primer_%28molecular_biology%29" target="_blank">Primers</a> (short DNA fragments) containing sequences complementary to the target region along with a <a title="DNA polymerase" href="http://en.wikipedia.org/wiki/DNA_polymerase" target="_blank">DNA polymerase</a> (after which the method is named) are key components to enable selective and repeated amplification. As PCR progresses, the DNA generated is itself used as a template for replication, setting in motion a <a title="Chain reaction" href="http://en.wikipedia.org/wiki/Chain_reaction" target="_blank">chain reaction</a> in which the DNA template is <a title="Exponential growth" href="http://en.wikipedia.org/wiki/Exponential_growth" target="_blank">exponentially</a> amplified. PCR can be extensively modified to perform a wide array of <a title="Genetic engineering" href="http://en.wikipedia.org/wiki/Genetic_engineering" target="_blank">genetic manipulations</a>.</p>
<p>Almost all PCR applications employ a heat-stable DNA polymerase, such as <a title="Taq polymerase" href="http://en.wikipedia.org/wiki/Taq_polymerase" target="_blank">Taq polymerase</a>, an enzyme originally isolated from the bacterium <em><a title="Thermus aquaticus" href="http://en.wikipedia.org/wiki/Thermus_aquaticus" target="_blank">Thermus aquaticus</a></em>. This DNA polymerase <a title="Enzyme" href="http://en.wikipedia.org/wiki/Enzyme" target="_blank">enzymatically</a> assembles a new DNA strand from DNA building-blocks, the <a title="Nucleotide" href="http://en.wikipedia.org/wiki/Nucleotide" target="_blank">nucleotides</a>, by using single-stranded DNA as a template and <a title="Oligonucleotide" href="http://en.wikipedia.org/wiki/Oligonucleotide" target="_blank">DNA oligonucleotides</a> (also called <a title="Primer (molecular biology)" href="http://en.wikipedia.org/wiki/Primer_%28molecular_biology%29" target="_blank">DNA primers</a>), which are required for initiation of DNA synthesis. The vast majority of PCR methods use <a title="Thermocycler" href="http://en.wikipedia.org/wiki/Thermocycler" target="_blank">thermal cycling</a>, i.e., alternately heating and cooling the PCR sample to a defined series of temperature steps. These thermal cycling steps are necessary first to physically separate the two strands in a DNA double helix at a high temperature in a process called <a title="DNA melting" href="http://en.wikipedia.org/wiki/DNA_melting" target="_blank">DNA melting</a>. At a lower temperature, each strand is then used as the <a title="DNA" href="http://en.wikipedia.org/wiki/DNA#Polymerases" target="_blank">template</a> in DNA synthesis by the DNA polymerase to selectively amplify the target DNA. The selectivity of PCR results from the use of <a title="Primer (molecular biology)" href="http://en.wikipedia.org/wiki/Primer_%28molecular_biology%29" target="_blank">primers</a> that are <a title="Complementary DNA" href="http://en.wikipedia.org/wiki/Complementary_DNA" target="_blank">complementary</a> to the DNA region targeted for amplification under specific thermal cycling conditions.</p>
<p>A basic PCR set up requires several components and reagents.<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-molecular_cloning-5" target="_blank">[6]</a></sup> These components include:</p>
<ul>
<li><em>DNA template</em> that contains the DNA region (target) to be amplified.</li>
<li>Two <em><a title="Primer (molecular biology)" href="http://en.wikipedia.org/wiki/Primer_%28molecular_biology%29" target="_blank">primers</a></em> that are <a title="Complementarity (molecular biology)" href="http://en.wikipedia.org/wiki/Complementarity_%28molecular_biology%29" target="_blank">complementary</a> to the <a title="Directionality (molecular biology)" href="http://en.wikipedia.org/wiki/Directionality_%28molecular_biology%29" target="_blank">3&#8242;</a> (three prime) ends of each of the <a title="DNA" href="http://en.wikipedia.org/wiki/DNA#Sense_and_antisense" target="_blank">sense and anti-sense</a> strand of the DNA target.</li>
<li><em><a title="Taq polymerase" href="http://en.wikipedia.org/wiki/Taq_polymerase" target="_blank">Taq polymerase</a></em> or another <a title="DNA polymerase" href="http://en.wikipedia.org/wiki/DNA_polymerase" target="_blank">DNA polymerase</a> with a temperature optimum at around 70 °C.</li>
<li><em>Deoxynucleoside triphosphates</em> (dNTPs; <a title="Nucleotide" href="http://en.wikipedia.org/wiki/Nucleotide" target="_blank">nucleotides</a> containing triphosphate groups), the building-blocks from which the DNA polymerase synthesizes a new DNA strand.</li>
<li><em><a title="Buffer solution" href="http://en.wikipedia.org/wiki/Buffer_solution" target="_blank">Buffer solution</a></em>, providing a suitable chemical environment for optimum activity and stability of the DNA polymerase.</li>
<li><em><a title="Divalent" href="http://en.wikipedia.org/wiki/Divalent" target="_blank">Divalent</a> <a title="Cations" href="http://en.wikipedia.org/wiki/Cations" target="_blank">cations</a></em>, <a title="Magnesium" href="http://en.wikipedia.org/wiki/Magnesium" target="_blank">magnesium</a> or <a title="Manganese" href="http://en.wikipedia.org/wiki/Manganese" target="_blank">manganese</a> ions; generally Mg<sup>2+</sup> is used, but Mn<sup>2+</sup> can be utilized for PCR-mediated DNA <a title="PCR mutagenesis" href="http://en.wikipedia.org/wiki/PCR_mutagenesis" target="_blank">mutagenesis</a>, as higher Mn<sup>2+</sup> concentration increases the error rate during DNA synthesis<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-6" target="_blank">[7]</a></sup></li>
<li><em>Monovalent cation</em> <a title="Potassium" href="http://en.wikipedia.org/wiki/Potassium" target="_blank">potassium</a> ions.</li>
</ul>
<p>Typically, PCR consists of a series of 20-40 repeated temperature changes, called cycles, with each cycle commonly consisting of 2-3 discrete temperature steps, usually three (Fig. 2). The cycling is often preceded by a single temperature step (called <em>hold</em>) at a high temperature (&gt;90°C), and followed by one hold at the end for final product extension or brief storage. The temperatures used and the length of time they are applied in each cycle depend on a variety of parameters. These include the enzyme used for DNA synthesis, the concentration of divalent ions and dNTPs in the reaction, and the melting temperature (<a title="DNA melting" href="http://en.wikipedia.org/wiki/DNA_melting" target="_blank">Tm</a>) of the primers.<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-7" target="_blank">[8]</a></sup></p>
<p>The PCR is commonly carried out in a reaction volume of 10–200 μl in small reaction tubes (0.2–0.5 ml volumes) in a <a title="Thermal cycler" href="http://en.wikipedia.org/wiki/Thermal_cycler" target="_blank">thermal cycler</a>. The thermal cycler heats and cools the reaction tubes to achieve the temperatures required at each step of the reaction (see below). Many modern thermal cyclers make use of the <a title="Thermoelectric cooling" href="http://en.wikipedia.org/wiki/Thermoelectric_cooling" target="_blank">Peltier effect</a>, which permits both heating and cooling of the block holding the PCR tubes simply by reversing the electric current. Thin-walled reaction tubes permit favorable thermal conductivity to allow for rapid thermal equilibration. Most thermal cyclers have heated lids to prevent condensation at the top of the reaction tube. Older thermocyclers lacking a heated lid require a layer of oil on top of the reaction mixture or a ball of wax inside the tube.</p>
<p>The PCR process can be divided into three stages:</p>
<p><em>Exponential amplification</em>: At every cycle, the amount of product is doubled (assuming 100% reaction efficiency). The reaction is very sensitive: only minute quantities of DNA need to be present.<sup><a href="http://en.wikipedia.org/wiki/PCR#cite_note-11" target="_blank">[12]</a></sup></p>
<p><em>Leveling off stage</em>: The reaction slows as the DNA polymerase loses activity and as consumption of reagents such as dNTPs and primers causes them to become limiting.</p>
<p><em>Plateau</em>: No more product accumulates due to exhaustion of reagents and enzyme.</p>
<p>&nbsp;</p>
<p><strong>Molecular cloning</strong> refers to a set of experimental methods in <a title="Molecular biology" href="http://en.wikipedia.org/wiki/Molecular_biology" target="_blank">molecular biology</a> that are used to assemble <a title="Recombinant DNA" href="http://en.wikipedia.org/wiki/Recombinant_DNA" target="_blank">recombinant DNA</a> molecules and to direct their <a title="DNA replication" href="http://en.wikipedia.org/wiki/DNA_replication" target="_blank">replication</a> within <a title="Host (biology)" href="http://en.wikipedia.org/wiki/Host_%28biology%29" target="_blank">host organisms</a>.<sup><a href="http://en.wikipedia.org/wiki/Molecular_cloning#cite_note-isbn0-7167-2866-4-0" target="_blank">[1]</a></sup> The use of the word <em><a title="Cloning" href="http://en.wikipedia.org/wiki/Cloning" target="_blank">cloning</a></em> refers to the fact that the method involves the replication of a single DNA molecule starting from a single living cell to generate a large population of cells containing identical DNA molecules. Molecular cloning generally uses DNA sequences from two different organisms: the species that is the source of the DNA to be cloned, and the species that will serve as the living <a title="Host (biology)" href="http://en.wikipedia.org/wiki/Host_%28biology%29" target="_blank">host</a> for replication of the <a title="Recombinant DNA" href="http://en.wikipedia.org/wiki/Recombinant_DNA" target="_blank">recombinant DNA</a>. Molecular cloning methods are central to many contemporary areas of modern biology and medicine.<sup><a href="http://en.wikipedia.org/wiki/Molecular_cloning#cite_note-isbn1-55581-498-0-1" target="_blank">[2]</a></sup></p>
<p>In a conventional molecular cloning experiment, the DNA to be cloned is obtained from an organism of interest, then treated with enzymes in the test tube to generate smaller DNA fragments. Subsequently, these fragments are then combined with <a title="Vector (molecular biology)" href="http://en.wikipedia.org/wiki/Vector_%28molecular_biology%29" target="_blank">vector DNA</a> to generate recombinant DNA molecules. The recombinant DNA is then introduced into a host organism (typically an easy-to-grow, benign, laboratory strain of <em><a title="E. coli" href="http://en.wikipedia.org/wiki/E._coli" target="_blank">E. coli</a></em> bacteria). This will generate a population of organisms in which recombinant DNA molecules are replicated along with the host DNA. Because they contain foreign DNA fragments, these are <a title="Transgenic" href="http://en.wikipedia.org/wiki/Transgenic" target="_blank">transgenic</a> or genetically-modified microorganisms (<a title="Genetically modified organism" href="http://en.wikipedia.org/wiki/Genetically_modified_organism" target="_blank">GMO</a>).<sup><a href="http://en.wikipedia.org/wiki/Molecular_cloning#cite_note-isbn1-4051-1121-6-2" target="_blank">[3]</a></sup> This process takes advantage of the fact that a single bacterial cell can be induced to take up and replicate a single recombinant DNA molecule. This single cell can then be expanded exponentially to generate a large amount of bacteria, each of which contain copies of the original recombinant molecule. Thus, both the resulting bacterial population, and the recombinant DNA molecule, are commonly referred to as &#8220;clones&#8221;. Strictly speaking, <em>recombinant DNA</em> refers to DNA molecules, while <em>molecular cloning</em> refers to the experimental methods used to assemble them.</p>
<p>&nbsp;</p>
<p><strong>Protein purification</strong></p>
<p>Choice of a starting material is key to the design of a purification process. In a plant or animal, a particular protein usually isn&#8217;t distributed homogeneously throughout the body; different organs or tissues have higher or lower concentrations of the protein. Use of only the tissues or organs with the highest concentration decreases the volumes needed to produce a given amount of purified protein. If the protein is present in low abundance, or if it has a high value, scientists may use <a title="Recombinant DNA" href="http://en.wikipedia.org/wiki/Recombinant_DNA" target="_blank">recombinant DNA</a> technology to develop cells that will produce large quantities of the desired protein (this is known as an <a title="Expression system" href="http://en.wikipedia.org/wiki/Expression_system" target="_blank">expression system</a>). Recombinant expression allows the protein to be tagged, e.g. by a <a title="His-tag" href="http://en.wikipedia.org/wiki/His-tag" target="_blank">His-tag</a>, to facilitate purification, which means that the purification can be done in fewer steps. In addition, recombinant expression usually starts with a higher fraction of the desired protein than is present in a natural source.</p>
<p>An analytical purification generally utilizes three properties to separate proteins. First, proteins may be purified according to their isoelectric points by running them through a pH graded gel or an ion exchange column. Second, proteins can be separated according to their size or molecular weight via <a title="Size exclusion chromatography" href="http://en.wikipedia.org/wiki/Size_exclusion_chromatography" target="_blank">size exclusion chromatography</a> or by <a title="SDS-PAGE" href="http://en.wikipedia.org/wiki/SDS-PAGE" target="_blank">SDS-PAGE</a> (sodium dodecyl sulfate-polyacrylamide gel electrophoresis) analysis. Proteins are often purified by using <a title="2D-PAGE" href="http://en.wikipedia.org/wiki/2D-PAGE" target="_blank">2D-PAGE</a> and are then analysed by <a title="Peptide mass fingerprinting" href="http://en.wikipedia.org/wiki/Peptide_mass_fingerprinting" target="_blank">peptide mass fingerprinting</a> to establish the protein identity. This is very useful for scientific purposes and the detection limits for protein are nowadays very low and nanogram amounts of protein are sufficient for their analysis. Thirdly, proteins may be separated by polarity/hydrophobicity via <a title="High performance liquid chromatography" href="http://en.wikipedia.org/wiki/High_performance_liquid_chromatography" target="_blank">high performance liquid chromatography</a> or <a title="Reversed-phase chromatography" href="http://en.wikipedia.org/wiki/Reversed-phase_chromatography" target="_blank">reversed-phase chromatography</a>.</p></blockquote>
