---
layout: post
title: "Forward and reverse reads in paired-end sequencing"
author: ericminikel
date: 2012-12-19 23:54:44
---
<p>This topic is incredibly easy to get confused about, so here is as clear an explanation as I can muster.   It will start out big picture and then get into the weeds.</p>
<p>Sequencing by synthesis, which is how most commercially available high-throughput sequencing technologies work as of December 2012 (see <a title="UAB day 1: intro to sequencing" href="/2012/12/11/uab-day-1-intro-to-sequencing/">notes on sequencing technologies</a>), always synthesizes the <em>new</em> strand (which becomes your read) in a 5&#8242;-to-3&#8242; direction.  That&#8217;s because this is how <a href="http://en.wikipedia.org/wiki/DNA_polymerase">DNA polymerase</a> works in our cells (indeed, in every living thing&#8217;s cells) and sequencing relies on DNA polymerase.  Since the new strand is synthesized 5&#8242;-to-3&#8242;, you are working your way up the template strand in a 3&#8242;-to-5&#8242; direction.</p>
<p>In any sequencing technology, you PCR amplify the individual DNA fragments once they have hybridized to flowcells or beads.  This means you end up with <em>both</em> strands of DNA.  If you were to read both of the strands from their respective 3&#8242; ends at once, you&#8217;d be getting two different sequences and your results would be uninterpretable.  To avoid this problem, sequencing technologies ligate non-complementary adapters to the 3&#8242; and 5&#8242; ends of DNA fragments so that the primer for one adapter only begins synthesis on one strand and not on its complement.</p>
<p>In conventional paired-end sequencing, you simply sequence using the adapter for one end, and then once you&#8217;re done you start over sequencing using the adapter for the other end.</p>
<p>This means your two reads are the reverse complement of the 100 3&#8242;-most bases of the Watson strand and the Crick strand; these reads are assumed to be identical to the 100 5&#8242;-most bases of the Crick strand and Watson strand respectively.</p>
<p>Here&#8217;s what your reads represent, then:</p>
<p><a href="/wp-content/uploads/2012/12/paired-end1.jpg"><img class="alignnone size-large wp-image-1339" title="paired-end" src="/wp-content/uploads/2012/12/paired-end1-1024x237.jpg"/></a></p>
<p>Therefore when you open your <a href="http://en.wikipedia.org/wiki/FASTQ_format">FASTQ</a> files and look at a pair of reads, the sequences you see are, conceptually, <em>pointing towards each other on opposite strands</em>.  When you align them to the genome, one read should align to the forward strand, and the other should align to the reverse strand, at a higher base pair position than the first one so that they are pointed towards one another.  This is known as an &#8220;FR&#8221; read &#8211; forward/reverse, in that order.</p>
<p>This is all for conventional paired-end sequencing.  Some specialized technologies, such as using circularized DNA fragments to create large insert jumping libraries [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3071919/">Talkowski 2011</a>], switch things around so that your reads <em>ought</em> to align in an &#8220;RF&#8221; position &#8211; reverse/forward, in that order.  This is different from FR because it means the reverse read aligned at a lower base pair position than the forward read, and thus that they are pointing away from another.</p>
<p>But if you&#8217;re just doing conventional paired-end sequencing (i.e. Illumina), your reads are supposed to align FR, and if they instead align RF, FF or RR, that&#8217;s a problem and often indicates the reads aligned incorrectly (though it could also mean they aligned correctly and that a real inversion or translocation exists in the sample&#8217;s genome &#8211; see <a href="/2012/12/11/uab-day-1-intro-to-sequencing/">notes from Devin Absher&#8217;s talk on calling structural variants</a>).  If read pairs <em>don&#8217;t</em> align FR, most aligners will flag them as &#8220;not a proper pair&#8221; in the SAM/BAM file by zeroing the FLAG 0&#215;02 bit (proper pair flag) (<a href="http://samtools.sourceforge.net/SAM1.pdf">see SAM spec</a>).   Heng Li, author of BWA, <a href="http://sourceforge.net/mailarchive/forum.php?thread_name=20100108144013.GD489%40dhc025321.med.harvard.edu&amp;forum_name=bio-bwa-help">states here</a> that BWA will only set the &#8216;proper pair flag&#8217; to 1 for Illumina reads aligned FR (for <a href="http://en.wikipedia.org/wiki/ABI_Solid_Sequencing">SOLiD</a> it allows FF or RR).</p>
<p>Therefore if you look at a SAM/BAM file (for Illumina data at least), it should be the case that in any pair of reads with the 0&#215;02 bit set (i.e. considered a proper pair), exactly one of the two reads will have the 0&#215;10 bit set as well (i.e. it is reverse-complemented; again, see the <a href="http://samtools.sourceforge.net/SAM1.pdf">SAM file spec</a>).  For the read with its 0&#215;10 bit set, the &#8220;SEQ&#8221; listed in the SAM file will be the reverse complement of the original read as seen in the FASTQ.  That means that in the SAM file, the SEQs for a pair of reads are now both being presented in forward orientation even though the &#8220;FR&#8221; orientation information is stored in the FLAG.</p>
<p>For reads that don&#8217;t form a proper pair, or aren&#8217;t mapped at all, (almost) all bets are off.  Pairs might be FF, RR or RF, and one might be mapped and the other not.  Moreover, an unmapped read might have the 0&#215;10 flag set, or not.</p>
<p>Why bother reverse-complementing the read if it doesn&#8217;t align anywhere anyway?  I don&#8217;t know; I can&#8217;t find any information in the BWA documentation about why this might occur.  But I spot checked the FLAGs of unmapped reads in one of my BAMs:</p>
<pre>cut -f 2 1_unmapped_sorted.sam | sort | uniq</pre>
<p>And found that a variety of FLAG values occur: <code>[101,103,117,133,141,151,157,165,167,181,69,77,87]</code>, several of which have the 0&#215;10 bit set, for instance 117.  When I go back and pull out a sampling of the reads with flag value 117:</p>
<pre>grep $'\t117\t' 1_unmapped_sorted.sam | head | less</pre>
<p>And then compare the reads in those to my original FASTQs, I find that they are indeed reverse complemented, for instance:</p>
<pre>grep $'\t117\t' 1_unmapped_sorted.sam | head | less
FCC0CHTACXX:6:1101:10003:55579#GCCAATAT 117 chr2 130057568 0 * = 130057568 0 GGGACACACTGAGCTCAGGGATAGGGTGGAGGTGGACTGGACTGAGAGCAGCGTCAGAGGGGAAGGCACTGCAGCAGGGGCCCGACATAGGCAGGGGTAC

grep FCC0CHTACXX:6:1101:10003:55579 -m 1 -A 3 1_1.fq
@FCC0CHTACXX:6:1101:10003:55579#GCCAATAT/1
GTACCCCTGCCTATGTCGGGCCCCTGCTGCAGTGCCTTCCCCTCTGACGCTGCTCTCAGTCCAGTCCACCTCCACCCTATCCCTGAGCTCAGTGTGTCCC
+
_b_eeeeegggggiihiiiiiiiiihhfhhiifhhifffhihhiiihhhfhghdgdg`cdeeeR]bdddbcccbbcca^bccc`bbccccccbcd_`b_b</pre>
<p>(You&#8217;ll notice that RNAME for this read is chr2, but remember from the <a href="http://samtools.sourceforge.net/SAM1.pdf">SAM spec</a> that if 0&#215;04 is set, no assumptions can be made about RNAME).</p>
<p>I conclude that even if 0&#215;04 is set, it <em>is</em> still safe to assume that for any individual read that has its 0&#215;10 flag set, the &#8220;SEQ&#8221; shown in the BAM file is actually the reverse complement of the original read from the FASTQ.</p>
<p>This came up because recently I was iterating through BAMs using <a href="http://www.cgat.org/~andreas/documentation/pysam/api.html">pysam</a>, trying to re-align unmapped reads, and for my particular purpose I wanted to have both of their sequences in the same orientation, i.e. both forward or both reverse.  This was confusing at first because zero, one or both of them might <em>already</em> be reverse-complemented in the SEQ field.  The most conceptually straightforward way is just to reverse complement whichever (neither, one or both) have  <code>.is_reverse = True</code>, so that now you&#8217;re back to baseline, and then you reverse complement exactly one of them.   To avoid the extra step, though, logically you could use an <a href="http://en.wikipedia.org/wiki/XNOR_gate">XNOR</a>, so in Python:</p>
<pre>if(read.is_reverse == mate.is_reverse):
    read = reversecomplement(read)</pre>
<p>&nbsp;</p>
<p><strong>Addendum</strong>: for BWA, at least, the proper pair flag depends not only on the FR orientation but also on insert size being within a certain range (from <a href="http://bio-bwa.sourceforge.net/bwa.shtml">BWA manual</a>):</p>
<blockquote><p>The maximum distance x for a pair considered to be properly paired (SAM flag 0&#215;2) is calculated by solving equation Phi((x-mu)/sigma)=x/L*p0, where mu is the mean, sigma is the standard error of the insert size distribution, L is the length of the genome, p0 is prior of anomalous pair and Phi() is the standard cumulative distribution function. For mapping Illumina short-insert reads to the human genome, x is about 6-7 sigma away from the mean. Quartiles, mean, variance and x will be printed to the standard error output.</p></blockquote>
<p>&nbsp;</p>
