---
layout: post
title: "Huntington's disease reading list"
author: ericminikel
deprecated: true
date: 2012-12-10 23:41:42
---
<p>A couple of months ago a co-worker provided me with a &#8220;must read&#8221; list for new people getting oriented on Huntington&#8217;s disease.  It was some heavy reading, but after working my way through it I&#8217;m ready to check my understanding by blogging about it.  The goals of this post will be to share the reading list along with a summary of what each paper is about, and then discuss the (long) list of things I learned and how they relate to each other.</p>
<p>Basically all of these papers open with approximately the same paragraph or two introducing Huntington&#8217;s Disease.  I&#8217;ve read this enough times I&#8217;m going to write my own short version of it here:  HD is caused by a dominant mutation in exon 1 of the HTT gene located on chromosome 4 (at 4p16.3), consisting of an expanded CAG repeat tract which translates into an expanded polyglutamine tract.  CAG repeat length is inversely correlated with age of onset of the disease.  This repeat exhibits germline instability leading to a tendency toward longer repeats (thus earlier onset aka <a href="http://en.wikipedia.org/wiki/Anticipation_(genetics)">anticipation</a>) in subsequent generations.  The disease leads to progressive cognitive decline and motor impairment / involuntary movement over 15 or 20 years, is fatal and at present the root causes of the disease are not treatable.</p>
<p>With that background out of the way, here&#8217;s the list, in chronological order&#8211; I have added a couple of &#8220;classics&#8221; to the beginning of the list for historical reasons.  Most of the later papers are reviews.</p>
<p><strong><a href="http://en.wikisource.org/wiki/On_Chorea">Huntington 1872</a> &#8220;On Chorea&#8221;.</strong>  I&#8217;m told that people had noticed this disease passed down through families even earlier than 1872, but George Huntington earned the dubious honor of the disease&#8217;s name by being the first to thoroughly characterize its symptoms and hereditary nature.  Most of the paper is actually about a supposedly non-hereditary form of chorea and available treatments such as &#8220;proto-carbon of iron&#8221; and &#8220;zinci sulph.&#8221;  Near the end he elaborates upon a hereditary form &#8221;which exists, so far as I know, almost exclusively on the east end of Long Island&#8221;.  He notes that it never skips a generation: &#8220;if by any chance these children go through life without it, the thread is broken and the grandchildren and great-grandchildren of the original shakers may rest assured that they are free from the disease&#8221;, in contrast to phthisis (tuberculosis) and syphilis, which apparently were then believed to be hereditary diseases capable of skipping generations.  This was an interesting historical read.</p>
<p><strong><a href="http://deepblue.lib.umich.edu/bitstream/2027.42/30901/3/0000570.pdf">MacDonald 1993</a> &#8220;A Novel Gene Containing a Trinucleotide Repeat That Is Expanded and Unstable on Huntington’s Disease Chromosomes&#8221;.</strong>  Alternately cited as Huntington&#8217;s Disease Collaborative Research Group 1993.  121 years after Huntington characterized the disease, this group discovered the HTT gene (then called IT15) and its CAG repeat expansion responsible for the disease.  According to Google Scholar this paper has been cited 3498 times.</p>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/7618107">Duyao 1995</a>. &#8221;Inactivation of the Mouse Huntington&#8217;s Disease Gene Homolog Hdh&#8221;.</strong> This paper established that HD is a gain-of-function disease and that HTT knockout is embryonic lethal.   Prior to this paper, it had already been shown that some people have one copy of HTT effectively knocked out by a translocation, yet these people exhibited no HD phenotype&#8211; in fact, they were perfectly healthy. So HTT was thought to be haplosufficient: you could live healthily with just one working allele, at presumably 50% the quantity of the protein that other people had. Based on this evidence, it did not appear that HD was a straightforward loss-of-function disease. However, there was still the possibility that it was a loss-of-function acting through <em>dominant negative inhibition</em>. Dominant negative inhibition is when one defective allele can dominantly silence the other one, leading to neither allele being able to perform its normal function. For instance, many proteins need to form oligomers in order to accomplish their job. Say it takes four copies of one protein to form a tetramer to accomplish some physiologic purpose. It&#8217;s possible that having just one of the four constituent copies of the protein be mutant will change the tetramer&#8217;s conformation so that it cannot achieve its purpose. If tetramers form randomly, then 15/16 of tetramers will contain at least one mutant protein and be unable to function. The mutant protein will have almost completely wiped out the functionality of the wild type protein. That&#8217;s dominant negative inhibition. <a href="http://www.pnas.org/content/93/25/14452.abstract">Michaels 1996</a> puts it more concisely:</p>
<blockquote><p>Dominant negative inhibition is most commonly seen when a mutant subunit of a multisubunit protein is coexpressed with the wild-type protein so that assembly of a functional oligomer is impaired.</p></blockquote>
<p>(Aside: dominant negative inhibition is also the mechanism by which some polymorphisms in PRNP (such as <a href="/2012/02/07/kuru-and-the-g127v-polymorphism/">G127V</a>) confer resistance to prion disease. See for instance <a href="http://jvi.asm.org/content/74/9/4351">Zulianello 2000</a>.  The Prusiner lab used this as a starting point for drug design 12 years ago [<a href="http://www.cmpharm.ucsf.edu/cohen/research/images/PNAS1.pdf">Perrier 2000</a>] )</p>
<p>So one hypothetical possibility was that CAG-expanded HTT acted through dominant negative inhibition to silence the the healthy allele, thus effectively knocking out huntingtin and causing the disease state.  But Duyao showed that&#8217;s not the case. Duyao used a vector to knock out HTT exons 4 and 5 in mouse embryonic stem cells and showed by immunoblot that HTT was expressed normally in wild type, at 50% level in knockout heterozygotes and not at all in knockout homozygotes. Duyao then raised heterozygous HTT knockout mice and showed that they are phenotypically normal, but when you cross them, the offspring are 1/3 wild type and 2/3 heterozygous: the homozygous knockouts never get born. The homozygous embryos begin development but exhibit abnormal morphology by E7.5 and are being reabsorbed by E8.5.</p>
<p>Apparently this is pretty rare: most genes are not embryonic lethal if knocked out. &#8220;Indeed, few other gene knock-outs produce death at gastrulation, and these display significant differences&#8221;</p>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/14600292">MacDonald 2003</a> &#8221;Huntingtin: Alive and Well and Working in Middle Management&#8221;.</strong>  This paper opens by reviewing the (large amount of progress) made in understanding huntingtin in the 10 years since the gene was discovered: its rough structure had been determined, and scientists had identified about 20 proteins it interacts with.  The view that had already begun to emerge&#8211; and to my knowledge largely stands to this day&#8211; is that huntingtin acts as a &#8220;scaffold&#8221; which facilitates the interaction of a wide variety of other proteins with one another.  Hence the &#8220;middle management&#8221; theme of the article.  The types of interactions facilitated by huntingtin are summarized as &#8220;neuronal cell signaling&#8221; but take a variety of specific forms: regulating signal transduction by interacting with <a href="http://en.wikipedia.org/wiki/NMDA_receptor">NMDA receptors</a>; regulating transcription of other genes through interaction with <a href="http://en.wikipedia.org/wiki/RE1-silencing_transcription_factor">REST</a>; and thereby regulating cell survival through downstream effects on <a href="http://en.wikipedia.org/wiki/Brain-derived_neurotrophic_factor">BDNF</a>.</p>
<p>These are all purported native functions of wild-type huntingtin.  But you might ask why we even care about the native function if our goal is to cure the disease.  After all, eight years earlier, Duyao had already demonstrated pretty conclusively that HD was a gain-of-function disease: the pathology is not caused by mutant huntingtin abdicating its native functions, but rather by mutant huntingtin taking on some entirely new function.  Well, it seems that in those eight years, several authors had provided evidence that HD might be a <em>partial</em> loss-of-function disease&#8211; while the primary cause might be gain-of-function, the disease might be hastened or worstened by the fact that mutant huntingtin loses some or all of its physiologic function. Naturally this led to a lot of interest in identifying the native function.  But in this article MacDonald actually takes a step back and makes an eloquent argument as to why the gained function is more important.  She points to two main pieces of evidence:</p>
<ol>
<li>If the loss of huntingtin&#8217;s managerial function in facilitating the work of its partner proteins were a disease trigger for HD, then we&#8217;d also expect HD-like phenotypes to be caused by mutations or deficiencies in those partner proteins.  That&#8217;s not what we see.</li>
<li>Even mutant huntingtin with (CAG)150 is capable of rescuing the embryonic survival of mice with <em>Hdh</em> knocked out.  Therefore even this mutant huntingtin is still capable of fulfilling its native function.</li>
</ol>
<div>This leads MacDonald to conclude that the disease is indeed caused by gain-of-function and that any consequences of partial loss-of-function are downstream, i.e. they reduce the cell&#8217;s ability to cope with the negative effects of the gained function.</div>
<blockquote><p>Therefore, the polyglutamine expansion in mutant huntingtin does not greatly abrogate the protein’s intrinsic activity, but instead confers a novel dominant property that triggers the disease process. Thus, even though polyglutamine expansion can alter the associations of huntingtin’s N terminus with dozens of partner proteins tested, the evidence does not favor the popular idea that such changes are sufficient to trigger the disease cascade.</p>
<p>If polyglutamine expansion does not inhibit the native function of the mutant huntingtin molecule, why then is huntingtin-facilitated signaling often altered in cells expressing mutant huntingtin? One reasonable scenario is that these huntingtin-mediated pathways are downstream of the mutant huntingtin trigger mechanism. In this view, huntingtin signaling would participate, along with other homeostatic processes, in the response of the mutant cells to events in the ongoing disease cascade. These changes would not occur in wild-type cells, which are not responding to the consequences of mutant huntingtin. Indeed, from experiments with HD cell death models, huntingtin has been proposed as an important prosurvival factor for neuronal cells (19). Thus, once the disease process is under way, the loss of huntingtin signaling may well play a role in the downstream disease cascade, further weakening the already compromised neuronal cells.</p></blockquote>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/17417937">Orr &amp; Zoghbi 2007</a> &#8221;Trinucleotide Repeat Disorders.&#8221; </strong>The main thing I got from this review was a perspective on the different ways that trinucleotide repeat expansion can cause disease. I&#8217;m used to thinking about HD, where the expanded CAG in the DNA means expanded polyQ in the protein, and a probable gain of function (though loss of function may also be part of the story). Not so for all trinucleotide repeat diseases. <a href="http://en.wikipedia.org/wiki/Fragile_X_syndrome">Fragile X syndrome</a> is caused by a large (&gt;200 repeat) CCG tract in the <a href="http://en.wikipedia.org/wiki/Five_prime_untranslated_region">5&#8242;UTR</a> of the <a href="http://en.wikipedia.org/wiki/FMR1">FMR1</a> gene which leads to no gene expression. Some people have an allele which is only somewhat expanded (55 to 200 repeats) and get tremor/ataxia or ovarian failure only later in life, apparently also due to lowered FMR1 expression. So that&#8217;s an RNA-mediated loss of function. There is also RNA-mediated gain of function: <a href="http://en.wikipedia.org/wiki/Myotonic_dystrophy">myotonic dystrophy</a> (DM1) is caused by CTG repeat expansion in the <a href="http://en.wikipedia.org/wiki/Three_prime_untranslated_region">3&#8242;UTR</a> of <a href="http://en.wikipedia.org/wiki/Myotonin-protein_kinase">DMPK</a>. The mechanism of this gain of function is believed to involve the expanded CUG in DMPK&#8217;s RNA binding to other proteins, causing a gain of function in <a href="http://en.wikipedia.org/wiki/CUGBP1">CUG-BP1</a> and a loss of function in <a href="http://en.wikipedia.org/wiki/MBNL1">MBNL</a>, leading ultimately to splicing abnormalities. Then you have the protein-coding trinucleotide repeats, such as the polyQ in HD, leading to a presumed gain of function. There are yet other trinucleuotide repeat disorders where it&#8217;s not yet known how the repeat leads to disease.</p>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/18588526">Gil &amp; Rego 2008</a>  &#8221;Mechanisms of neurodegeneration in Huntington&#8217;s Disease&#8221;</strong>.    This is a review of what we know (or knew, as of 2008) about the pathways by which mutant HTT causes neurodegeneration.</p>
<p>It opens with some anatomy of the disease. By far the most heavily affected brain region in HD is the striatum (caudate nucleus and putamen). In the overall scheme of brain regions, this falls under cerebral hemisphere &gt; basal ganglia &gt; striatum. Upon looking up <a href="http://en.wikipedia.org/wiki/Striatum">striatum on Wikipedia</a> I learned that it&#8217;s 96% composed of medium spiny neurons, which are the cell type most affected by HD. (I had lamented before that the <a href="http://www.alleninstitute.org/">Allen Institute</a> has <a href="http://human.brain-map.org/microarray/search">gene expression data by brain region</a> but not by cell type; but for the striatum, the observed expression levels must surely be dominated by medium spiny neurons, so perhaps that can act as a proxy.)  Several other regions including the cerebral cortex are later/lesser affected.</p>
<p>Huntingtin is expressed in every tissue in the body.  People have worked to &#8220;localize&#8221; wild-type huntingtin, meaning figure out where in the cell it is normally, and it appears to spend most of its time in the cytoplasm though it&#8217;s been seen in the nucleus from time to time.  A lot of effort has gone into figuring out what huntingtin protein&#8217;s native function is, i.e. what the normal healthy protein does for a living.  It&#8217;s a matter of controversy whether this even matters: some people think HD is caused purely by a gain of toxic function (in which case who cares what the native function is), while others think it&#8217;s purely or mostly loss-of-function.  Still others say it could be both: a toxic gain of function made worse by the partial loss of native function.  The frustrating thing about looking for HTT native function is that it appears to do just about everything.  That&#8217;s an exaggeration, of course, but there are a lot of different proposed native functions, each with good evidence behind them, so it&#8217;s likely it has several complex roles.  Gil and Rego categorize these broadly into the following categories:</p>
<blockquote><p>(i) protein trafficking; (ii) vesicle transport and anchoring to the cytoskeleton; (iii) clathrinmediated endocytosis; (iv) postsynaptic signaling; (v) transcriptional<br/>
regulation; and (vi) anti-apoptotic function.</p></blockquote>
<p>Besides neuronal loss and reduction in brain weight, another feature seen in HD pathology is &#8220;neuronal nuclear inclusions&#8221; which based on the wiki on <a href="http://en.wikipedia.org/wiki/Inclusion_body">Inclusion bodies</a> I think are simply protein aggregates in the nucleus. This is HD&#8217;s closest analogue of PrP plaques in CJD or Aβ plaques in Alzheimer&#8217;s.  Just like in those diseases, no one knows for sure if these aggregates are actually toxic, protective, or neutral.  But they&#8217;re certainly correlated in some way, and in HD they correlate with CAG length and they appear before any reductions in brain weight, which in turn precede overt symptoms.</p>
<p>Gil and Rego address the question &#8220;how do aggregates form?&#8221; and discuss two main theories: either expanded polyQ destabilizes huntingtin, allowing <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2142615/">polar zippers</a> (a particular form of beta sheet hydrogen bonding) to form between different huntingtin molecules / between huntingtin and other polyglutamine proteins, or <a href="http://en.wikipedia.org/wiki/Transglutaminase">transglutaminases</a> actively sew the huntingtin molecules to each other.</p>
<p>As for whether the aggregates are bad, Gil and Rego seem to come down on the side that they are fine or even protective.  There&#8217;s a variety of evidence here: an accidentally-created mouse model that expresses a huntingtin fragment and gets aggregates but no neurodegeneration or other symptoms; in vitro experiments where aggregation was suppressed and cell death heightened, or aggregation promoted and cell death lessened.</p>
<p>Gil and Rego then survey the proposed molecular mechanisms for how mutant huntingtin causes pathogenesis.  As with the proposed native functions of huntingtin, there are an unfathomable number of different theories, one or more of which may be correct.  The four main theories seem to be these:</p>
<ol>
<li><a href="http://en.wikipedia.org/wiki/Protease">Proteases</a>.   The theory is, the expanded polyQ tract attracts <a href="http://en.wikipedia.org/wiki/Caspase_3">caspase-3</a> and/or <a href="http://en.wikipedia.org/wiki/Calpain">calpains</a> to cleave huntingtin, creating toxic fragments and disabling huntingtin&#8217;s purported native function of suppressing caspase-3.  Caspase-3 is a trigger for <a href="http://en.wikipedia.org/wiki/Apoptosis">apoptosis</a> (programmed cell death), so this cascade can lead to cell death.</li>
<li>The <a href="http://en.wikipedia.org/wiki/Ubiquitin">ubiquitin</a>-<a href="http://en.wikipedia.org/wiki/Proteasome">proteasome</a> system (UPS), which is one of the cell&#8217;s ways of disposing of cellular waste / misfolded proteins.  Mutant huntingtin is so demanding of attention from the <a href="http://en.wikipedia.org/wiki/Heat_shock_protein">heat shock proteins</a> Hsp70 and Hsp40 that it makes them unavailable to help refold other proteins, increasing the amount of misfolded proteins in the cell.  Then huntingtin itself and these other proteins eventually overwhelm this disposal system, pulling heat shock proteins and parts of the proteasome into aggregates.</li>
<li>Dysregulation of <a href="http://en.wikipedia.org/wiki/Transcription_(genetics)">transcription</a>.  Part of the purported native function of huntingtin is to regulate gene expression by interacting with a bunch of other transcription factors.  So mutant huntingtin might lose this function, or it might still interact with the same transcription factors but in a new and toxic way.</li>
<li>Disruption of <a href="http://en.wikipedia.org/wiki/Axoplasmic_transport">axonal transport</a>.  As background, a bit of introductory neurobio: neurons have huge long axons but the DNA -&gt; RNA -&gt; protein machinery exists only in the nucleus, so proteins have to travel a long way to get to and from the tips of the axon terminals and dendrites.  Neurons have specialized machinery to move proteins up and down the axon, where &#8220;anterograde&#8221; transport means nucleus -&gt; terminals and &#8220;retrograde&#8221; means terminals -&gt; nucleus.  Some people think huntingtin is natively involved in facilitating this process, and loses this function, and/or that huntingtin aggregates physically obstruct transport along the narrow axon.</li>
<li>Synaptic dysfunction.  This is an incredibly general category.  The synapse is where one neuron&#8217;s axon releases neurotransmitters onto the next neuron&#8217;s dendrite.  These two neurons are called the presynaptic and postsynaptic neuron respectively.  Various reasons have been proposed why mutant huntingtin might mess things up: making required proteins unavailable, increasing the postsynaptic cell&#8217;s sensitivity to <a href="http://en.wikipedia.org/wiki/NMDA_receptor">NMDA</a> stimulus, or disrupting <a href="http://en.wikipedia.org/wiki/Endocytosis">endocytosis</a>, which is how the presynaptic neuron reuptakes neurotransmitters.</li>
</ol>
<div>Next, moving away from the precise molecular mechanism of HD pathogenesis, Gil &amp; Rego address the issue at more of a systems level: how does mutant huntingtin cause neurodegeneration?  A few theories:</div>
<div></div>
<div>
<ol>
<li>Excitotoxicity.  This is when neurons get overstimulated and fire action potentials too often.  This is implicated as a mechanism in some other neurodegenerative diseases too and is the reason why people have been interested in <a title="Memantine" href="/2012/12/01/memantine/">memantine</a> as a therapeutic for prion diseases and Alzheimer&#8217;s . (By the way, they&#8217;re trying it for Huntington&#8217;s too &#8212; see <a href="http://www.clinicaltrials.gov/ct2/show/NCT00652457">clinical trial</a> and <a href="http://hdbuzz.net/4">commentary on HDBuzz</a>.)   If HD neurodegeneration is caused by excitotoxicity, then interestingly, that would mean that the problem actually originates not in the <a href="http://en.wikipedia.org/wiki/Striatum">striatum</a> (which is most affected) but in the <a href="http://en.wikipedia.org/wiki/Cerebral_cortex">cerebral cortex</a>, which is the source of excitatory <a href="http://en.wikipedia.org/wiki/Glutamic_acid">glutamatergic</a> inputs <em>into </em>the striatum.</li>
<li>Dopamine toxicity.  The striatum also gets <a href="http://en.wikipedia.org/wiki/Dopaminergic">dopaminergic</a> inputs from the <a href="http://en.wikipedia.org/wiki/Substantia_nigra">substantia nigra</a>.  The substantia nigra is severely affected in Parkinson&#8217;s disease, leading to dopamine-producing neurons dying there and thus a lack of dopamine.  That&#8217;s why patients with Parkinson&#8217;s are often prescribed dopamine agonists (make no mistake, this is a band-aid and not a cure for the fact that neurons are dying).  HD is the opposite: dopamine-receiving cells in the striatum are dying, leading to an excess of dopamine which can cause toxicity.</li>
<li>Mitochondria and oxidative stress.  I don&#8217;t totally understand why these two are being grouped together.  HD is associated with mitochondrial dysfunction and lack of cellular energy/ATP (that&#8217;s why <a href="http://en.wikipedia.org/wiki/Creatine">creatine</a> <a href="http://www.clinicaltrials.gov/ct2/show/NCT00026988">has been tested as a therapeutic</a>); it&#8217;s also associated with increased oxidative stress (hence antioxidants as therapeutics).  There&#8217;s some connection here, hence the idea of antioxidants targeted to mitochondria as a therapeutic [<a href="http://download.cell.com/cell-reports/pdf/PIIS221112471200335X.pdf">Xun 2012</a>], but I got a bit lost in Gil &amp; Rego&#8217;s explanation.</li>
<li><a href="http://en.wikipedia.org/wiki/Apoptosis">Apoptosis</a> and <a href="http://en.wikipedia.org/wiki/Autophagy">autophagy</a>.  It&#8217;s not actually clear how neurons die in HD, but there&#8217;s some evidence of apoptotic death.  In neurodegenerative disease, I normally think of autophagy as a good thing &#8211; a process of cleaning up misfolded proteins and protecting cellular health &#8211; and that seems to be mostly the case in HD as well, but the authors also point to the possibility of autophagy eventually becoming &#8220;overloaded, insufficient and dysfunctional, leading to cell degradation.&#8221;</li>
</ol>
</div>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/19650870">Bauer &amp; Nukina 2009</a> &#8221;The pathogenic mechanisms of polyglutamine diseases and current therapeutic strategies.&#8221;</strong></p>
<p>This provides some overview of what HD has in common with other polyQ diseases, such as spinal bulbar muscular atrophy (SBMA), dentatorubropallidoluysian<br/>
atrophy (DRPLA), and spinocerebellar ataxia. Here are some of the commonalities:</p>
<ol>
<li>Protein aggregates. All of these diseases result in protein aggregation, and in all of them, it&#8217;s not agreed upon whether the aggregates are toxic, protective or neutral. More specifically, it is not agreed what, if any, <em>size</em> of aggregates are good or bad &#8211; perhaps small oligmers are toxic but large aggregates are cytoprotective, the cell&#8217;s way of sequestering the oligomers into something less harmful.</li>
<li>Protein cleavage. Many or all of the proteins involved in these diseases get cleaved into smaller fragments by <a href="http://en.wikipedia.org/wiki/Protease">proteases</a>, so much research focuses on figuring out whether the smaller protein fragments are the toxic element causing disease.</li>
<li><a href="http://en.wikipedia.org/wiki/Protein_phosphorylation">Phosphorylation</a>. Many of these proteins get phosphorylated, which might or might not affect the protein&#8217;s propensity toward aggregation, cleavage or other toxic behaviors.</li>
<li>Sumoylation. This is when a <a href="http://en.wikipedia.org/wiki/SUMO_protein">small ubiquitin-like modifier</a> (SUMO) forms a covalent bond with a lysine in the protein. Again, apparently not clear if this is good, bad, part of the disease or part of the cell&#8217;s response to disease.</li>
<li>Nuclear localization. In general, proteins need to have a particular <a href="http://en.wikipedia.org/wiki/Nuclear_localization_sequence">nuclear localization sequence</a> of amino acids to get into the nucleus, and a <a href="http://en.wikipedia.org/wiki/Nuclear_export_signal">nuclear export sequence</a> to get out. Many of the polyQ proteins have these sequences, and it&#8217;s possible that an imbalance in transport in and out of the nucleus is part of what gives rise to toxicity. For example huntingtin is usually mostly in the cytoplasm but in HD more of it is found in the nucleus than in healthy individuals. PolyQ expansion and cleavage may make it harder for huntingtin to get out of the nucleus once it&#8217;s in.</li>
<li>Role of <a href="http://en.wikipedia.org/wiki/Heat_shock_protein">heat shock proteins</a>. HSPs can re-fold misfolded proteins, reduce aggregation and are suspected to have a protective role in polyQ diseases.</li>
<li><a href="http://en.wikipedia.org/wiki/Apoptosis">Apoptosis</a>. One route to cell death in polyQ diseases is apoptosis (programmed cell death). There are many mechanisms by which expanded polyQ might lead to apoptosis: activating <a href="http://en.wikipedia.org/wiki/Caspase">caspases</a>, dysregulating mitochondria and thus leading to release of <a href="http://en.wikipedia.org/wiki/Cytochrome_c">cytochrome c</a>, reduced inhibition of pro-apoptotic factors, and so on.</li>
<li>Mitochondrial disfunction. Expanded polyQ proteins appear to mess up mitochondria, whether directly or through an increase in <a href="http://en.wikipedia.org/wiki/Reactive_oxygen_species">reactive oxygen species</a> (ROS)</li>
<li><a href="http://en.wikipedia.org/wiki/Ubiquitin">Ubiquitin</a>-<a href="http://en.wikipedia.org/wiki/Proteasome">proteasome</a> system (UPS) impairment. This system is supposed to clean up and dispose of misfolded proteins. Ubiquitin is a &#8220;tag&#8221; which targets the misfolded proteins to be broken down by the proteasome which chops them up into individual amino acids. This system often gets impaired in polyQ diseases. Evidence shows that the UPS is tagging and destroying the mutant polyQ proteins and that if the UPS is blocked, toxicity is heightened. It looks like the UPS may get overwhelmed with all the expanded polyQ proteins, and/or get drawn into aggregates.</li>
</ol>
<p>Next the paper describes models of these diseases and the challenges associated therewith. Usually the CAG length that causes disease in humans does <em>not</em> cause disease in mice, but you can model the diseases by introducing an even longer CAG tract. The paper provides a review mostly of HD mouse models, which I&#8217;ll cover below under Kim&#8217;s paper in this post. Finally Bauer &amp; Nukina discuss therapeutic strategies for these diseases along with listing drugs that have been tested.  Just as there are a multitude of theories as to huntingtin&#8217;s native function and and a multitude of theories as to how mutant huntingtin causes pathogenesis, so there are also an accordingly large number of different therapeutic strategies and specific molecules that have been tested.  Bauer and Nukina roughly divide the approaches as follows:</p>
<ol>
<li><a href="http://en.wikipedia.org/wiki/Antisense_therapy">Antisense gene silencing</a>.  The theoretical utility of this approach has been demonstrated by using <a href="http://en.wikipedia.org/wiki/Tetracycline-controlled_transcriptional_activation">Tet-on or Tet-off switches</a> to show that silencing mutant huntingtin does alleviate the disease.  The trouble in practice is that it&#8217;s hard to make antisense be specific to the disease allele, and you can&#8217;t afford to knock down both alleles because huntingtin is too important for survival.  One strategy is to look for concentrations of SNPs, MNPs and indels that differ (in a particular individual) between the disease allele and normal allele and target those with antisense or <a href="http://en.wikipedia.org/wiki/MicroRNA">miRNA</a>s.  This has been successful in cell culture for spinocerebellar ataxia by targeting polymorphisms in the 3&#8242;UTR of <a href="http://en.wikipedia.org/wiki/Ataxin_1">ATXN1</a> but the authors don&#8217;t mention any examples in HD, and it seems this approach hasn&#8217;t been tested in vivo yet.</li>
<li>Speeding up protein degradation through drugs that promote autophagy -  <a href="http://en.wikipedia.org/wiki/Sirolimus">rapamycin</a>, <a href="http://en.wikipedia.org/wiki/Lithium_pharmacology">lithium</a> - or seem to increase UPS activity - <a href="http://en.wikipedia.org/wiki/Amiloride">amiloride</a>, <a href="http://en.wikipedia.org/wiki/Benzamil">benzamil</a>, <a href="http://en.wikipedia.org/wiki/Y-27632">Y-27632</a> though it&#8217;s not totally clear what those latter set of compounds do.</li>
<li>Inhibiting aggregation. Recall that in prion disease, Congo red was found to help dissolve PrP plaques, leading to other ideas for therapeutics. Same in polyQ: Congo red seems to reduce aggregation. This and other compounds that directly interfere with aggregation have been explored in therapeutics, along with compounds that work indirectly, say by promoting heat shock protein activity (ex. cystamine, geranylgeranylacetone).</li>
<li>Adjusting transcription. PolyQ diseases can lead to dysregulation of transcription of other genes, so one therapeutic strategy is to target gene expression via histone deacetylase inhibitors (HDACi) or histone methylation inhibitors.</li>
<li>Mitochondria / energy / oxidative stress. Cellular metabolism and energy are disrupted in these diseases; people have tested drugs to increase ATP production (creatine), increase mitochondrial function (triacetyluridine) or reduce oxidative stress (CoQ10)</li>
<li>Reducing excitotoxicity by antagonizing NMDA receptors, e.g. with memantine.</li>
<li>Inhibiting apoptosis. The main drug tested here has been minocycline, and it&#8217;s not clear that it works.</li>
</ol>
<div>Bauer and Nukina list a lot of specific drugs and small molecules that have been tested, but these are reviewed in greater detail by Kim 2011, so I&#8217;ll cover those below in this post.  Observe that all of the small molecule approaches correspond to a particular hypothesis about the pathogenic mechanism of mutant huntingtin or the native function of normal huntingtin.  Those approaches will only work to the extent that those hypotheses are correct.  So of all the approaches listed above, only the antisense gene silencing is neutral with regards to how HD really works.  On the flipside, antisense gene silencing would need to be personalized for each HD patient and might not work at all for people whose diseased and mutant alleles don&#8217;t have enough differences.</div>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2675790/">Zheng &amp; Joinnides 2009</a>. &#8220;Hunting for the function of huntingtin.&#8221;</strong> This is an introduction to <a href="http://en.wikipedia.org/wiki/Drosophila_melanogaster">Drosophila</a> models of HD. Duyao 1995 showed that HTT knockout is embryonic lethal in mice. Not so with dHtt, the Drosophila homologue of HTT: knockout flies appear pretty much phenotypicall normal. This means the native function of dHtt in flies is clearly not quite the same as HTT in mammals&#8211; a huge caveat to any Drosophila research&#8211; but then it&#8217;s not as if we&#8217;re certain that the function is the same in mice as in humans anyway. The advantage of Drosophila is that because the flies actually live, you can study the (subtle) ways in which they might be different. Indeed, the knockout flies do get impaired mobility late in life, neuron defects visible in histology, and impaired responses to stresses. Flies with a transgenic expanded HTT allele get a phenotype, and this is worse in knockout flies than in otherwise-normal flies. This provides more clues as to what normal and mutant huntingtin are doing. And of course the advantage of working with Drosophila is they are incredibly quick and cheap compared to rodents.</p>
<p>&nbsp;</p>
<p><strong><a href="http://physrev.physiology.org/content/90/3/905.full">Zuccato 2010</a> &#8220;Molecular Mechanisms and Potential Therapeutical Targets in Huntington’s Disease&#8221;  </strong>This is the longest and probably best-written of all these reviews. A lot of the content of this review has been covered elsewhere in this post, but the discussion of HTT phylogenetics is unique. HTT homologs are seen in all vertebrates, with 80% conservation even between the most distant vertebrate species such as Homo sapiens vs. <a href="http://en.wikipedia.org/wiki/Fugu">Fugu</a> (pufferfish). There are homologs in invertebrates like <a href="http://en.wikipedia.org/wiki/Drosophila_melanogaster">Drosophila</a>, <a href="http://en.wikipedia.org/wiki/Dictyostelium_discoideum">Dictyostelium</a>, and <a href="http://en.wikipedia.org/wiki/Caenorhabditis_elegans">C. elegans</a> too, with more like 20-50% of amino acids conserved vs. humans, indicating HTT is quite an ancient gene. This paper takes the position that there are no homologs in yeast. Invertebrates have one or zero Qs in the position where vertebrates have a polyQ repeat. Humans (even without HD) have the longest polyQ repeat of all (avg. 17 Qs in non-HD chromosomes). Has evolution favored the polyQ repeat in vertebrates, and if so, why? One theory goes back to the &#8220;polar zipper&#8221; mechanism: perhaps huntingtin&#8217;s function is to form polar zipper structures with transcription factors (proteins that regulate expression of other genes) which also have polyQ tracts. To see if the polyQ tract is important in mammals, someone created mice with the polyQ (normally length 7 in mice) deleted, and the mice developed pretty normally but had defects in learning and memory.  HTT also has ~36 separate HEAT repeat regions. HEAT repeats are a motif involved in protein-protein interactions. These are among the most evolutionarily conserved parts of HTT, and are therefore believed to be critical to HTT function.</p>
<p>See also the very clear side-by-side tabular comparison of different rodent models of HD in <a href="http://physrev.physiology.org/content/90/3/905/T1.expansion.html">Table 1</a></p>
<p><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/21907096">Kim 2011</a> &#8221;Experimental Models of HD and Reflection on Therapeutic Strategies&#8221;.</strong></p>
<p>This is an exceptionally <em>long</em> review paper that reviews existing animal models of HD, proposed (and tested) therapeutic approaches, and also provides guidance on how to translate animal findings into success in human clinical trials. I found that all of these aspects gave a nice introduction to key concepts in the HD field as well as a lot that is relevant to prion diseases.</p>
<p>In the introduction, I was impressed to learn that as early as 1985, pathologists had been &#8220;topographically grading&#8221; the degree of neuronal loss across the striatum. It seems there has been a lot of work by pathologists in mapping the pattern of neurodegeneration across brain tissues [<a href="http://www.ncbi.nlm.nih.gov/pubmed/2932539">Vonsattel 1985</a>]. To my knowledge the study of prion disease is still a bit behind on this point, though this has motivated me to dig up some more papers on prion pathology as well.</p>
<p>Mouse models</p>
<p>Decades before MacDonald 1993 showed the world that polyQ expansion in HTT causes Huntington&#8217;s Disease, scientists had already created sort-of mouse models of HD by injecting NMDA agonists (<a href="http://en.wikipedia.org/wiki/Kainic_acid">kainic acid</a> and <a href="http://en.wikipedia.org/wiki/Quinolinic_acid">quinolinic acid</a>) into the brains of mice. Excitotoxicity &#8212; which I understand to mean &#8220;too much excitation, leading to toxicity&#8221; &#8212; is a feature of HD, and remarkably, these injections actually recapitulated many features of HD, most notably the differential effect on different cell types. Kainic acid caused degeneration of striatal <a href="http://en.wikipedia.org/wiki/Gamma-Aminobutyric_acid">GABA</a> neurons but not striatal afferents (not sure what this means in this context&#8211; afferents are inbound nerve fibers, efferents are outbound nerve fibers, where the center is the CNS); quinolinic acid caused degeneration of GABA neurons and substance P-containing neurons but not NADPH-diaphorase neurons and cholinergic neurons. Both of these patterns reflect what happens in HD. These early researchers also showed that the (toxically) excitatory impulses to the striatum were coming from the cortex. There&#8217;s also evidence that dopaminergic inputs from the substantia nigra cause excitotoxicity in HD. Years later, researchers tried severing the inputs to the striatum from the cortex and substantia nigra in an HD mouse model (R6/2, to be introduced shortly) and found that this significantly improved phenotype and survival.</p>
<p>There&#8217;s also evidence that energy deficits are part of HD pathology, and so people have also experimented with inducing energy deficits in mice as an HD model via 3-nitropropionic acid (3-NP), which is &#8220;a naturally occurring plant toxin and mycotoxin&#8230; an irreversible inhibitor of succinate dehydrogenase that inhibits both the Krebs cycle and Complex II activity of the electron transport chain.&#8221; Ingestion causes damage to the striatum, though apparently it is also toxic to the heart, so it&#8217;s hard to tease out what part of the patholgy is strictly neurological.</p>
<p>These biochemical models of HD are obviously less faithful to the disease than genetic models, and in 1996, the world finally got its first genetic mouse model of HD: the <a href="http://jaxmice.jax.org/strain/006494.html">R6/2 mouse</a>, otherwise known as the &#8220;<a href="http://en.wikipedia.org/wiki/Gillian_Bates">Gill Bates</a> mouse model&#8221;.</p>
<p>Kim defines three categories of genetic mouse models:</p>
<ol>
<li>mice expressing a fragment of mutant human HTT</li>
<li>mice with expanded CAG knocked into the murine Hdh gene</li>
<li>mice with the full-length mutant human HTT gene</li>
</ol>
<p>R6/2 falls into the first category, fragment models. R6/2 mice still express both of their copies of wild-type Hdh (mouse huntingtin), but also express an N-terminal fragment of expanded polyQ human HTT. This fragment consists of ~1 kb of 5&#8242;UTR, all of exon 1 including CAG<sub>~130</sub> and the first 262bp of intron 1. To this day, these mice are still the most widely used model and most of the drugs that have been tested for HD in mice, have been tested in R6/2 mice. Kim explains why this model is so popular. It basically boils down to robust phenotype, short lifespan, low variance in lifespan, and &#8220;well-defined neurobehavioral and neuropathological findings&#8221;. The mice reliably die at age 100 &#8211; 150 days, and have obvious behavioral phenotypes such as &#8220;age-related impairments in dystonic movements, motor performance, grip strength and body weight&#8221; and obvious pathological hallmarks such as decreased brain weight, brain volume, striatal neuronal loss, and so on &#8212; all of which are shared with human HD.</p>
<p>To editorialize a bit: if you&#8217;re doing mouse studies, you&#8217;re looking at something like $1.50 to $3.00 per mouse per day for cage space (a commodity service which includes food, cleaning etc.) Let&#8217;s take $2.00 for the sake of example. A quick mouse model might exhibit phenotype in 150 days, a longer mouse model might take 450 days. If your mice exhibit low variance in age of onset, you may only need 10 control and 10 test mice in order to have statistical power to tell if a candidate therapy had any effect; if variance is high, you might need 20 control and 20 test. So consider $2 * (10+10) * 150 = $6,000 versus $2 * (20+20) * 450 = $36,000 for your cage space. Besides cage space, there&#8217;s also the labor and technology costs to do your study, and the easier it is to detect a phenotype the less you&#8217;ll spend on those. Dollars and cents aside, would you (whether you be a scientist looking to publish or a patient hoping for a therapeutic) rather have results in 5 months or 15 months? Would you rather have a decisive phenotypic readout or more of a subjective one?</p>
<p>So there are good reasons why R6/2 is the most widely used HD mouse model. But it comes with sacrifices. R6/2 doesn&#8217;t have the full human HD gene &#8212; just the exon containing the polyQ expansion. The only phenotypic difference that Kim points out between R6/2 mice and human HD patients is that the mice get &#8220;inclusions&#8221; (huntingtin aggregates) throughout their brain (humans only get them in the striatum). But at the molecular level there are at least three major differences between R6/2 and human HD:</p>
<ol>
<li>R6/2 express only the N-terminal fragment of mutant huntingtin, not the whole gene</li>
<li>This fragment is under a strong promoter so expression is very high</li>
<li>The number of CAG repeats (~130) is far more than most human HD cases&#8211; in humans, 130 CAGs would lead to juvenile onset which is a fairly different phenotype than adult onset HD</li>
</ol>
<p>And these differences can mean that findings from R6/2 mice, whether basic science or therapeutic success, do not always translate to humans.</p>
<p>Kim also describes two other models in the fragment category which don&#8217;t seem to have attracted quite as much attention. The R6/1 mouse model is like R6/2 but contains only human exon 1 and only 116 CAG repeats; the mice exhibit behavioral phenotype and some <em>atrophy</em> of neurons but no <em>loss</em> of neurons. N171-82Q mice express human HTT exons 1 and 2 with CAG<sub>82</sub> under the mouse prion promoter so that within the CNS it is expressed only in neurons.  These mice can have a fairly rich phenotype but exhibit more phenotypic variance than the R6/2 mice and live more like 130-180 days.</p>
<p>In category (2), there are the <a href="http://jaxmice.jax.org/strain/003456.html">HdhQ111</a>, HdhQ48, HdhQ89, etc. lines of mice, which express murine Hdh with a knocked-in CAG repeat expansion in exon 1. So: not a transgene, no human genetic material here, but the repeat is part of the full-length huntingtin protein. Depending on polyQ length, these mice may get some kind of phenotype at the age of a couple months, but most phenotypic readouts take a year or two, and although the mice get cognitive and motor impairments, they don&#8217;t necessarily die sooner than wild-type mice.</p>
<p>Finally, in category (3), mice are made to express the full-length, mutant human HTT gene from a yeast artificial chromosome (YAC) or bacterial artificial chromosome (BAC). These models get names like <a href="http://jaxmice.jax.org/strain/004938.html">YAC128</a> (for 128 CAGs). Phenotypically, Kim seems to think these are the best model of all: &#8220;The neuropathology in the YAC mice has excellent fidelity with human HD&#8221;. Some phenotypes are visible as early as 2 or 3 months, while others take a year.</p>
<p>Kim seems to suggest that the tradeoff between the quick-and-reliable mouse models (R6/2) and the highly faithful models is inherent for a complex, age-onset disease like HD:</p>
<blockquote><p>Although all of these models share features with human HD, the phenotype of fulllength huntingtin mutation models develops gradually over many months and may not have a sufficient expression of disease to use progressive morbidity and survival as endpoints. HD progresses over many years in patients and the exact expression of the clinical and pathological phenomena observed in patients may not be present in short-lived animal. While the full-length models are genetically more accurate, the fragment models have a rapidly coursing robust phenotype, well-defined neurobehavioral and neuropathological findings, and die between 3 and 5 months of age. It has been a common practice to use the fragment models for therapeutics research because the outcomes are more clearly established and trials are more easily conducted.</p></blockquote>
<p>After reading about all the candidate therapeutics that have been tested for HD, though, I started to think maybe HD research needs to shift a bit towards slower, more costly but more accurate models. As described above, the list of drugs that have been tested &#8212; <em>and succeeded</em> &#8212; in mouse trials for HD is staggeringly long, and the number of those drugs that have then succeeded when they went to human clinical trials is staggeringly zero. I have asked several other scientists for their interpretation of this disconcerting fact. Is it simply that mouse biology is so different from human biology? Is it regression to the mean and we just aren&#8217;t seeing the troves of <em>un</em>successful mouse drug studies because no one bothered to publish them? Is it something about how the human clinical trials are set up (e.g. the patients are starting the drugs after onset of symptoms whereas the mice got them from very early in life)? While all of these could play a role, a number of scientists now seem to think that a lot of the problem lies with the mouse models&#8211; perhaps R6/2 just isn&#8217;t accurate enough and what works in R6/2 is a poor predictor of what will work in humans.</p>
<p>One possibility is to use R6/2 for initial discovery (actually, more like step 2 or 3 after high-throughput screening, in-vitro validation experiments, etc.) and then move to more faithful models as a next step before human clinical trials. Kim also notes the development of HD transgenic rhesus monkeys as a possible final model for testing therapeutics before drugs move into human trials.</p>
<p>Methodological Considerations</p>
<p>On pp. 436-439, Kim provides thoughts on how to best plan and execute a mouse study and how to best translate those results into human therapeutics. Kim&#8217;s major principles in this regard seem to be:</p>
<ul>
<li>Before starting the mouse trial, study the pharmacokinetics of the property in wild-type mice. Figure out what the maximum tolerated dose by increasing the dose every day until you hit LD50. Check levels of the compound in blood and brain after administration &#8212; make sure it&#8217;s getting through the BBB and see how quickly it is degraded or excreted, and therefore how frequent the doses will need to be.</li>
<li>Do a power analysis before starting, to see how many mice you will need in order to detect what effect size, given the variance in age of onset (or survival, etc.) observed in your mouse model.</li>
<li>The people running the study should be blinded as to which animals are control and which are treatment.</li>
<li>Decide on inclusion and exclusion criteria before starting &#8212; for instance, excluding &#8220;runts&#8221;, injured mice, etc.</li>
<li>Genotype the mice before starting. Make sure they&#8217;re all what you think they are.</li>
<li>Make sure the mice&#8217;s environment is uniform, because &#8220;environmental enrichment&#8221; (e.g. wheels to play on) affects phenotype in HD mice.</li>
<li>Decide upon the study endpoint and outcome measures. Kim asserts the following:<br/>
<blockquote><p>in neurodegenerative disease research neuroprotection is of the greatest importance, which at its most basic level is the preservation of neuronal processes, somata, and function. Measures that assess these directly (brain weight, gross atrophy, cellular atrophy, neuronal counts, gliosis, and volumetric imaging) should be considered the primary outcome measures. While it may also be possible to model the treatment of clinical symptoms in genetic models and it may also happen that improving behavioral symptoms corresponds to neuroprotection, assessments of clinical symptoms should be considered secondary outcome measures. These include survival, body weight, performance on motor and cognitive tasks, and laboratory studies examining toxicity or putative mechanisms of action. Clinical symptoms can be modified without affecting neurodegeneration. This distinction is important to keep in mind when considering how informative the results of a mouse therapeutic trial might be for translation to humans. Of those secondary measures that have been most useful, survival, or extension of life, is the most meaningful.</p></blockquote>
</li>
<li>Get quantitative, rather than just observational, measures from pathological study. (so basically: CellProfiler!)</li>
<li>Benchmark by using as positive control a compound you know works. Kim suggests cystamine for HD.</li>
</ul>
<p>I was a bit surprised by this point about &#8220;neuroprotection&#8221; being the key outcome, and wish that Kim had provided more of the reasoning behind it. I would have said that onset of cognitive impairment is the most important readout&#8211; I want to know, if this compound works in humans, will it let you live more good years? I am always a bit disappointed when studies use survival as the outcome, because that doesn&#8217;t distinguish between extending good quality years and extending disease course. And here Kim seems to say that <em>neither</em> of these is most important, and that pathological examination is most important. I could be convinced, but would need to hear some more of the reasoning and evidence. Perhaps the explanation is that pathological measures translate better between mouse and human or that behavioral phenotypes are too subjective, or something. But for now I remain skeptical.</p>
<p>I am also skeptical about this idea of using a known compound as a &#8220;positive control.&#8221; This seems to be the tack that UCSF took with its assay last year, using quinacrine as a positive control, and I&#8217;m skeptical. If you have a compound you know works, why isn&#8217;t that the therapeutic advancing to human trials? Or if it has failed human trials (as quinacrine has), why is it still considered a positive control?</p>
<p>Despite these two points of skepticism, I find Kim&#8217;s list to be a very helpful guide to designing mouse studies.</p>
<p>Next there is a brief section on clinical management of HD. Doctors prescribe antidepressants, antipsychotics, anxiotylics, and benzodiazepenes for the psychiatric disturbances of HD, and there&#8217;s a drug called tetrabenzine which is FDA approved specifically for treating chorea in HD. To be quite clear: this is all just management of symptoms, there is no drug known to impact disease progression in humans.</p>
<p>The next section discusses different biological targets for potential HD therapeutics. First off, people have tried to find compounds that prevent aggregation or clear aggregates. This is dodgy since no one actually knows if the aggregates are toxic, though Kim argues they must be at least somewhat bad due to:</p>
<blockquote><p>factors as the mass effect of cytosolic and nuclear huntingtin aggregate burden, the sequestration of critical transcription factors and neuronal proteins that are essential for neuronal survival by huntingtin aggregates and their subsequent reduced activity, altered proteosomal function, and the localization of mutant huntingtin aggregates to cellular organelles such as mitochondria</p></blockquote>
<p>Apparently cystamine and cysteamine are compounds that seem to inhibit aggregation and have shown some promise for HD.</p>
<p>Another candidate therapeutic route is autophagy. Now, huntingtin is a bit confusing because there is some disagreement on how it gets broken down; it appears that it can be degraded by <a href="http://en.wikipedia.org/wiki/Calpain">calpains</a>, <a href="http://en.wikipedia.org/wiki/Caspase">caspases</a> and other proteases, or the <a href="http://en.wikipedia.org/wiki/Proteasome">ubiquitin-proteasome system</a>. But apparently the lysosome is one of the many potential fates or huntingtin, and so kicking autophagy up a notch is perceived as a potential therapeutic strategy. A modified version of rapamycin has shown some success in N171-82Q mice. And apparently another path to promoting autophagy, completely independent of mTOR, is <a href="http://en.wikipedia.org/wiki/IRS2">insulin receptor substrate-2</a>. I can&#8217;t figure out if any drugs target this. And lithium provides yet a third independent route to autophagy induction, through <a href="http://en.wikipedia.org/wiki/Inositol_monophosphatase">inositol monophosphatase 1</a> (IMPase) &#8212; this must be why lithium keeps showing up in neurodegenerative disease studies (ALS, Parkinson&#8217;s). Lithium and a related drug <a href="http://en.wikipedia.org/wiki/Carbamazepine">carbamazepine</a>, an inhibitor of voltage-gated sodium channels, have both had some initial success against in vitro models of HD.</p>
<p>The conversation quickly gets weirder. As noted in the other review papers, huntingtin is believed to have important roles in both transcriptional and epigenetic regulation of other genes&#8217; expression. Gene regulation, especially in the brain, is an intensely fine-tuned process and I&#8217;d have said that to directly correct the effects of dysregulation would be far beyond the pale of any small molecule to do. Yet apparently histone deacetylase inhibitors (HDACi) (<a href="http://en.wikipedia.org/wiki/Valproic_acid">valproic acid</a> aka depakote is one of these) and <a href="http://en.wikipedia.org/wiki/Anthracycline">anthracyclines</a> (compounds which are better known as antibiotics and as chemotherapy but, importantly, bind to pyrimidine-rich gene promoters in genomic DNA) have both shown some therapeutic value in N171-82Q mice. This ties into something else I was just reading, which is that, in development, differentiation into neurons is achieved by the deactivation of PTB (polypyrimidine tract binding protein) leading to the activation of the subtly different nPTB (neuronal PTB) which is unique to neurons [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1899473/">Boutz &amp; Stoilov 2007</a>]. This is starting to be how people differentiate stem cells into neurons. But I think PTB / nPTB are recognizing polypyrimidine tracts in mRNA whereas anthracyclines are binding to DNA.</p>
<p>Mutant huntingtin causes heightened oxidative stress in neurons, probably by causing mitochondrial disfunction. This is of interest since oxidative stress seems to be a feature of the other neurodegenerative diseases, including prion disease, as well. In the case of HD, evidence for oxidative stress comes from several avenues. <a href="http://en.wikipedia.org/wiki/Lipofuscin">Lipofuscin</a> is a product of oxidized fatty acids and, I learned recently from the &#8220;aging&#8221; chapter in The Machinery of Life, is responsible for the &#8220;liver spots&#8221; on elderly peoples&#8217; skin. Lipofucsin also gets produced in the brain under conditions of oxidative stress, and is seen to be elevated in the striatum in HD patients. It may be not only a symptom but also have its own downstream pathological effects, interfering with lysosome function. Aside from lipofucsin, evidence of oxidative stress can also be seen by the heightened amount of DNA strand breakage in HD brains (detected by in situ end labeling) and the production of 8-hydroxy-2&#8242;-deoxyguanosine or OH<sup>8</sup>dG.</p>
<p>Because mitochondria are upstream of most of the oxidative stress problems, researchers have investigated compounds to aid mitochondrial function. First and foremost: <a href="http://en.wikipedia.org/wiki/Creatine">creatine</a>. Prior to getting into the HD field, my only exposure to the word &#8220;creatine&#8221; was from my youth, when controversy was stirred by local high school football players taking it as a performance-enhancing drug &#8212; it was purported to help you put on muscle faster, while being more benign than steroids, and was (is) sold in <a href="http://www.gnc.com/product/index.jsp?productId=2133642">big tubs of powder</a> at nutritional stores. For HD, creatine seems to have several potentially benefical properties. It is beleived to &#8220;buffer&#8221; energy reserves by forming phosphocreatine (PCr) which can then donate a phosphoryl group to ADP to make ATP. It is also an antioxidant and &#8220;stabilizes intracellular calcium, and inhibits activation of the mitochondrial pore.&#8221; To date, creatine seems to have generated more attention as a candidate HD therapeutic than any other compound. At HD2012 a few months ago, I received a handout with a bar chart of different compounds&#8217; results from human clincal trials, with creatine having the largest bar i.e. the most positive results, though still not statistically significant. It&#8217;s been shown to reduce patients&#8217; levels of OH<sup>8</sup>dG and appeared to stabilize, but not improve, patients&#8217; scores on the <a href="http://www.huntington-study-group.org/Resources/UHDRS/tabid/67/Default.aspx">UHDRS</a> (Unified Huntington Disease Rating Scale), a combined measure of cognitive and motor performance, though without reaching statistical significance. Apparently today another clinical trial with creatine at a higher dose is ongoing, but skeptics have begun to think creatine&#8217;s success in preclinical studies may just be an artifact of imperfect animal models. <a href="http://en.wikipedia.org/wiki/Coenzyme_Q10">Coenzyme Q<sub>10</sub></a> or CoQ<sub>10</sub>, another one of these hyped-up nutritional store supplements, has also been tested for therapeutic value against HD due to its powerful antioxidant properties. Like creatine, it has minimal side effects and some suggestion of therapeutic value but does not reach statistical significance. Other potential compounds to improve mitochondrial function are <a href="http://en.wikipedia.org/wiki/Ethyl_eicosapentaenoic_acid">ethyl-eicosapentaenoic acid</a> (ethyl-EPA), a fatty acid derivative, and <a href="http://en.wikipedia.org/wiki/Latrepirdine">latrepirdine</a> (Dimebon), an antihistamine that seems to prevent the mitochondrial pore from opening in response to calcium. Both made it to human clinical trials but showed no significant improvements.</p>
<p>Alternately, to target the excitotoxic aspect of HD, researchers have tried <a href="http://en.wikipedia.org/wiki/Riluzole">riluzole</a>, a weak blocker of voltage-gated sodium channels, and <a href="http://en.wikipedia.org/wiki/Amantadine">amantadine</a> and <a href="http://en.wikipedia.org/wiki/Memantine">memantine</a>, both NMDA antagonists, thus all three action potential inhibitors. Unlike creatine and CoQ<sub>10</sub>, these drugs have pretty bad side effects; nonetheless, some human clinical trials are ongoing.</p>
<p>For me given my interest in prion diseases, one of the most interesting therapeutic strategies for HD is to try to block apoptosis. Cell suicide is regulated through a careful balance of pro-apoptotic and anti-apoptotic proteins, with PrP and huntingtin both falling in the anti camp. While these are both primarily gain of function diseases, a partial loss of anti-apoptotic function may be one part of what causes neuronal death. In the case of HD, there are a host of both gain-of-function and loss-of-function explanations for why the disease leads to apoptosis. Wild type huntingtin is believed to prevent the formation of an &#8220;apoptosome complex&#8221;, or alternately to directly inhibit <a href="http://en.wikipedia.org/wiki/Caspase_3">caspase-3</a>, an apoptotic &#8220;executioner&#8221; protein, from starting the apoptotic cascade by cleaving its substrates. Evidence for huntingtin&#8217;s native antiapoptotic role comes partly from experiments where cells overexpressing wild-type huntingtin were less vulnerable to apoptosis than regular cells. But it&#8217;s not all loss-of-function: mutant huntingtin is also thought to overactivate NMDA receptors (via a direct, gain-of-function interaction) thus letting too much calcium into the cytosol, to the point that mitochondria swell and release pro-apoptotic factors including <a href="http://en.wikipedia.org/wiki/Cytochrome_c#Role_in_apoptosis">cytochrome c</a>. Fascinatingly, <a href="http://en.wikipedia.org/wiki/Minocycline">minocycline</a> turns out to be an anti-apoptotic compound. You&#8217;ll recall from the <a href="/2012/09/14/the-history-of-tetracyclines-as-treatments-for-prion-disease/">tetracycline post</a> that minocycline has been tested for prion disease&#8211; in fact, <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0001888">Luigi 2009</a> found it extended survival in scrapie hamsters by 81%, far more than doxycycline or tetracycline. Yet that study mainly played up tetracyclines&#8217; proposed role as direct binders of PrP, inhibiting infectivity and/or aggregation &#8211; minocycline&#8217;s anti-apoptotic properties earned only the briefest mention, even though according to Kim these properties are rather well-established. To quote some of the relevant facts from Kim:</p>
<blockquote><p>minocycline [is] a second-generation tetracycline antiapoptotic compound that inhibits caspase-1 and caspase-3 activity and expression levels, the release of apoptogenic factors from mitochondria, and caspase-independent neuronal cell death pathways. Additionally, it may inhibit iNOS activity and reactive microgliosis, both of which have been found to be present in HD patients and HD mice and have been implicated in disease pathogenesis (Friedlander, 2003). Previous research has found minocycline to be neuroprotective in multiple experimental models of neurodegeneration including brain trauma, cerebral ischemia, amyotropic lateral sclerosis, and Parkinson’s disease (Chen et al., 2000; Du et al., 2001; Sanchez et al., 2003; Tikka et al., 2001; Wu et al., 2002; Yrjanheikki et al., 1998; Zhu et al., 2002). Importantly, it is able to cross the blood–brain barrier and has been found to be safe for chronic administration. [In animal studies,] Minocycline also significantly reduced mHtt cleavage. In addition to their role in apoptotic signaling cascades, caspases also play a role in cleaving mHtt, yielding the toxic fragment</p></blockquote>
<p>Unfortunately it seems that after a strong performance in animal models of HD, minocycline had no effect in human clinical trials and by 2010 &#8220;futility&#8221; was declared, meaning there was deemed to be no point in further study. But to my knowledge minocycline hasn&#8217;t yet been tested in humans for prion disease. It is interesting to know that its success in animal models may be due to blocking apoptosis through interference with caspases. I wonder if PrP, like huntingtin, gets cleaved by caspases &#8212; I can&#8217;t find any articles about this online.</p>
<p>Kim provides an introduction to RNA-based gene therapy approaches for HD, which I found very helpful. For months I have been wondering what (if anything) was the difference between RNAi and antisense oligonucleotides (ASO). After all, the most basic principle is the same: design a complementary sequence that will hybridize to the undesired mRNA, reducing the disease gene&#8217;s expression level. But from there the two paths diverge. RNAi takes advantage of the genome&#8217;s existing micro RNA (miRNA), short interfering RNA (siRNA) or virally introduced <a href="http://en.wikipedia.org/wiki/Small_hairpin_RNA">short hairpin RNA</a> (shRNA). These types of RNA will form a complex with <a href="http://www.uniprot.org/uniprot/Q9UKV8">Argonaute-2</a> protein whereby, when they hybridize with an mRNA, Argonaute-2 then cleaves that mRNA. So RNAi works by cleaving targeted transcripts. In experiments for HD they&#8217;ve introduced the desired RNAi transcripts by <a href="http://en.wikipedia.org/wiki/Adeno-associated_virus">AAV</a>, so that the sequence ends up getting transcribed by the cells&#8217; own molecular machinery. ASO, by contrast, refers to using synthetic 15-25 bp transcripts to hybridize to the target mRNA and, depending on their sequence (which in turn depends on what part of the mRNA you&#8217;re targeting) can either just sterically interfere with translation or can recruit RNAse H to degrade the mRNA. So ASO works by steric hindrance or degradation. There are people working on both approaches for HD, and we saw some preliminary results at HD2012, but for HD at least, these approaches still seem to be a long way from the clinic. Promisingly, gene therapy has had its first clinical success with <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2839440/">Leber&#8217;s congenital amaurosis</a> but that was different in two ways &#8212; first, LCA2 is a loss-of-function disease so the therapy consisted of <em>introducing</em> a gene via AAV to replace the lost protein, and second, the AAV was administered to the retina, not CNS. Still, ASO is coming along and HD2012 featured some updates on progress in that area from <a href="http://www.cmmt.ubc.ca/directory/staff/amber-southwell">Amber Southwell</a>.</p>
<p>The most surprising therapeutic approach that Kim discusses is striatal neuron transplants. I was pretty shocked by this at first, but the striatum is involved in coordinating complex motor activity and <a href="http://en.wikipedia.org/wiki/Working_memory">working memory</a> so perhaps you could actually still be the same person after receiving a neuron transplant in this area. Animal trials have shown that the neurons actually do reform connections and seem to take on the duties of the neurons they have replaced. Human trials have apparently shown impressive improvement in UHDRS, with at least two huge caveats: (1) you need immunosuppressants and a lot of people had complications from that, and (2) the studies only looked at performance one year after surgery and so can&#8217;t assess the longer-term success which is presumably less since in time HD affects the cortex as well. But to me, the fact that a transplant can even conceivably work at all is promising because it means the issue is worth revisiting in a few years once the science of <a href="http://www.nytimes.com/2012/09/16/health/research/scientists-make-progress-in-tailor-made-organs.html?pagewanted=all&amp;_r=0">own-tissue transplants</a> using induced pluripotent stem cells has advanced enough. You can imagine generating stem cell lines from someone&#8217;s skin or blood or epithelial cells, correcting the disease gene via a <a href="http://en.wikipedia.org/wiki/Zinc_finger#Zinc_finger_nucleases">zinc finger nuclease</a>, differentiating into neuroprogenitor cells and ultimately neurons, and then transplanting these neurons back into the same person.</p>
<p>It sounds like sci fi but most of the technology is already there; the remaining issues seem to be that (1) differentiation into neurons is possible but not perfected yet, (2) making iPSC from adult cells involves turning on some oncogenes, potentially putting the patient at elevated risk of cancer post-transplantation, and (3) iPSC are still relatively new territory and their behavior isn&#8217;t well understood. Whether this approach could work for HD or any other neurodegenerative disease is an open question, but I do think it could be tested within the next several years.</p>
<p>There is also a quick note about immunotherapy, i.e. vaccinating people to make them produce antibodies against mutant huntingtin. As touched upon in the <a title="Anti-PrP vaccines" href="/2012/11/18/anti-prp-vaccines/">anti-PrP vaccines</a> post, some people have been scared off of the antibody approach by the fact that about 6% of Alzheimer&#8217;s patients treated with anti-Aβ vaccines got encephalitis as a result.</p>
<p><strong>Conclusions</strong></p>
<p>The above reviews illuminate several of the key themes, divisions, and debates within the HD field today. Probably the biggest divide is gain of function versus loss of function.  HTT knockout is embryonic lethal in mice, and can be rescued even by mutant huntingtin.  That&#8217;s a pretty strong argument that HD is <em>not </em>a loss of function disease.  But HD is incredibly complicated, and so over the years evidence has also accumulated that a loss of HD normal function could be part of the disease mechanism of HD.  There could be a mix: perhaps mutant huntingtin has a toxic gain of function, and the partial loss of its native function makes it harder to cope with that.  At this time, different investigators have different beliefs about the relative contribution of gain of function versus loss of function.  This makes a predictably huge difference in what you think is worth studying.  If you&#8217;re more in the loss of function camp, you want to research native function of huntingtin; if you&#8217;re purely gain of function, you don&#8217;t care about native function, you just want to know what that gained function is.</p>
<p>The diversity of hypotheses, opinions and approaches in the HD field is, in a way, a sign of how little we know: if we had clear evidence of what&#8217;s going on in the disease, then people would agree on it.  But more than one of these approaches may be correct &#8211; it&#8217;s a complex disease, and so all of this knowledge may prove valuable.  Moreover, even if some of these approaches ultimately prove not to lead to a cure, they may still lead to therapeutic approaches that at least alleviate or delay symptoms.  And if nothing else, the volume of different kinds of research going on in HD shows you how many bright people are working hard on this disease, which can only be a good sign for things to come.</p>
