---
layout: post
title: "Prion protein folding"
author: ericminikel
date: 2012-01-08 18:51:46
---
<p><span style="text-decoration: underline;">PROTEIN STRUCTURE AND FOLDING<br/>
</span></p>
<p>&nbsp;</p>
<p>“Hot spots in prion protein for pathogenic conversion,” Kuwata et al., PNAS (2007)</p>
<p><a href="http://www.pnas.org/content/104/29/11921.full" target="_blank">http://www.pnas.org/content/<wbr>104/29/11921.full</wbr></a></p>
<blockquote><p>Prion proteins are key molecules in transmissible spongiform encephalopathies (TSEs), but the precise mechanism of the conversion from the cellular form (PrP<sup>C</sup>) to the scrapie form (PrP<sup>Sc</sup>) is still unknown. Here we discovered a chemical chaperone to stabilize the PrP<sup>C</sup> conformation and <strong>identified the hot spots to stop the pathogenic conversion</strong>. We conducted <em>in silico</em> screening to <strong>find compounds that fitted into a “pocket” created by residues undergoing the conformational rearrangements between the native and the sparsely populated high-energy states (PrP*) and that directly bind to those residues.</strong> Forty-four selected compounds were tested in a TSE-infected cell culture model, among which one, <strong>2-pyrrolidin-1-yl-<em>N</em>-[4-[4-(2-<wbr>pyrrolidin-1-yl-acetylamino)-<wbr>benzyl]-phenyl]-acetamide, termed GN8, efficiently reduced PrP<sup>Sc</sup></wbr></wbr></strong>. Subsequently, administration of GN8 was found to prolong the survival of TSE-infected mice. Heteronuclear NMR and computer simulation showed that <strong>the specific binding sites are the A-S2 loop (N159) and the region from helix B (V189, T192, and K194) to B-C loop (E196), indicating that the intercalation of these distant regions (hot spots) hampers the pathogenic conversion process.</strong> Dynamics-based drug discovery strategy, demonstrated here focusing on the hot spots of PrP<sup>C</sup>, will open the way to the development of novel anti-prion drugs.</p></blockquote>
<p>Strengthening alpha-helical PrP(c) confirmation:</p>
<p>&nbsp;</p>
<p>“The Prion Diseases,”  Stanley Prusiner, Scientific American (1995)</p>
<p><a href="http://mvc.bioweb.dcccd.edu/weblinks/scbsesp.htm" target="_blank">http://mvc.bioweb.dcccd.edu/<wbr>weblinks/scbsesp.htm</wbr></a></p>
<blockquote><p>Fred Cohen and I think we might be able to explain why the various mutations that have been noted in PrP genes could facilitate folding into the beta-sheet form. Many of the human mutations give rise to the substitution of one amino acid for another within the four putative helices or at their borders.<strong> Insertion of incorrect amino acids at those positions might destabilize a helix, thus increasing the likelihood that the affected helix and its neighbors will refold into a beta-sheet conformation.</strong> Conversely, Hermann Sch&#8230;tzel in my laboratory finds that the harmless differences distinguishing the PrP gene of humans from those of apes and monkeys affect amino acids lying outside of the proposed helical domains-where the divergent amino acids probably would not profoundly influence the stability of the helical regions.</p>
<p>&#8212;</p>
<p>Even though we do not yet know much about how PrP scrapie harms brain tissue, we can foresee that an understanding of the three-dimensional structure of the PrP protein will lead to therapies. If, for example, the four-helix-bundle model of PrP is correct, drug developers might be able to design<strong> a compound that would bind to a central pocket that could be formed by the four helices. So bound, the drug would stabilize these helices and prevent their conversion into beta sheets.</strong></p></blockquote>
<p>&nbsp;</p>
<p>&#8220;Influence of pH on the human prion protein: insights into the early steps of misfolding,&#8221; van der Kamp et al., Biophys J (2010)<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/20923664" target="_blank">http://www.ncbi.nlm.nih.gov/<wbr>pubmed/20923664</wbr></a></p>
<blockquote><p>Transmissible spongiform encephalopathies, or prion diseases, are caused by misfolding and aggregation of the prion protein PrP. <strong>Conversion from the normal cellular form (PrP(C)) or recombinant PrP (recPrP) to a misfolded form is pH-sensitive, in that misfolding and aggregation occur more readily at lower pH.</strong> To gain more insight into the influence of pH on the dynamics of PrP and its potential to misfold, we performed extensive molecular-dynamics simulations of the recombinant PrP protein (residues 90-230) in water at three different pH regimes: neutral (or cytoplasmic) pH (∼7.4), middle (or endosomal) pH (∼5), and low pH (&lt;4). We present five different simulations of 50 ns each for each pH regime, amounting to a total of 750 ns of simulation time. A detailed analysis and comparison with experiment validate the simulations and lead to new insights into the mechanism of pH-induced misfolding. <strong>The mobility of the globular domain increases with decreasing pH, through displacement of the first helix and instability of the hydrophobic core. At middle pH, conversion to a misfolded (PrP(Sc)-like) conformation is observed.</strong> The observed changes in conformation and stability are consistent with experimental data and thus provide a molecular basis for the initial steps in the misfolding process.</p></blockquote>
<p>&nbsp;</p>
<p>&#8220;Prion protein misfolding,&#8221; Kupfer et al., Curr Mol Med (2009)<br/>
<a href="http://www.ncbi.nlm.nih.gov/pubmed/19860662" target="_blank">http://www.ncbi.nlm.nih.gov/<wbr>pubmed/19860662</wbr></a></p>
<blockquote><p>The crucial event in the development of transmissible spongiform encephalopathies (TSEs) is the conformational change of a host-encoded membrane protein &#8211; the cellular PrP(C) &#8211; into a disease associated, fibril-forming isoform PrP(Sc). This conformational transition from the alpha-helix-rich cellular form into the mainly beta-sheet containing counterpart initiates an &#8216;autocatalytic&#8217; reaction which leads to the accumulation of amyloid fibrils in the central nervous system (CNS) and to neurodegeneration, a hallmark of TSEs. The exact molecular mechanisms which lead to the conformational change are still unknown. It also remains to be brought to light how a polypeptide chain can adopt at least two stable conformations. This review focuses on structural aspects of the prion protein with regard to protein-protein interactions and the initiation of prion protein misfolding. It therefore<strong> highlights parts of the protein which might play a notable role in the conformational transition from PrP(C) to PrP(Sc)</strong> and consequently in inducing a fatal chain reaction of protein misfolding. Furthermore, features of different proteins, which are able to adopt insoluble fibrillar states under certain circumstances, are compared to PrP in an attempt to understand the unique characteristics of prion diseases.</p></blockquote>
<p>&nbsp;</p>
<p>“Molecular architecture of human prion protein amyloid: a parallel, in-register beta-structure,” Cobb et al., Proc Natl Acad Sci USA (2007)</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025469" target="_blank">http://www.ncbi.nlm.nih.gov/<wbr>pubmed/18025469</wbr></a></p>
<blockquote><p>Transmissible spongiform encephalopathies (TSEs) represent a group of fatal neurodegenerative diseases that are associated with conformational conversion of the normally monomeric and alpha-helical prion protein, PrP(C), to the beta-sheet-rich PrP(Sc). This latter conformer is believed to constitute the main component of the infectious TSE agent. In contrast to high-resolution data for the PrP(C) monomer, structures of the pathogenic PrP(Sc) or synthetic PrP(Sc)-like aggregates remain elusive. Here we have used site-directed spin labeling and EPR spectroscopy to probe the molecular architecture of the recombinant PrP amyloid, a misfolded form recently reported to induce transmissible disease in mice overexpressing an N-terminally truncated form of PrP(C). Our data show that, in contrast to earlier, largely theoretical models, <strong>the conformational conversion of PrP(C) involves major refolding of the C-terminal alpha-helical region. The core of the amyloid maps to C-terminal residues from approximately 160-220, and these residues form single-molecule layers that stack on top of one another with parallel, in-register alignment of beta-strands.</strong> This structural insight has important implications for understanding the molecular basis of prion propagation, as well as hereditary prion diseases, most of which are associated with point mutations in the region found to undergo a refolding to beta-structure.</p></blockquote>
<p>&nbsp;</p>
<p>“Dynamic diagnosis of familial prion diseases supports the beta2-alpha2 loop as a universal interference target,” Meli et al., Plos One (2011)</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/21552571" target="_blank">http://www.ncbi.nlm.nih.gov/<wbr>pubmed/21552571</wbr></a></p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3084259/?tool=pubmed" target="_blank">http://www.ncbi.nlm.nih.gov/<wbr>pmc/articles/PMC3084259/?tool=<wbr>pubmed</wbr></wbr></a></p>
<blockquote><p><strong>BACKGROUND:</strong></p>
<p>Mutations in the cellular prion protein associated to familial prion disorders severely increase the likelihood of its misfolding into pathogenic conformers. Despite their postulation as incompatible elements with the native fold, these mutations rarely modify the native state structure. However they variably have impact on the thermodynamic stability and metabolism of PrP(C) and on the properties of PrP(Sc) aggregates. To investigate whether the pathogenic mutations affect the dynamic properties of the HuPrP(125-229) α-fold and find possible common patterns of effects that could help in prophylaxis we performed a dynamic diagnosis of ten point substitutions.</p>
<p><strong>METHODOLOGY/PRINCIPAL FINDINGS:</strong></p>
<p>Using all-atom molecular dynamics simulations and novel analytical tools we have explored the effect of D178N, V180I, T183A, T188K, E196K, F198S, E200K, R208H, V210I and E211Q mutations on the dynamics of HuPrP(125-228) α-fold. We have found that while preserving the native state, all mutations produce dynamic changes which perturb the coordination of the α2-α3 hairpin to the rest of the molecule and cause the reorganization of the patches for intermolecular recognition, as the disappearance of those for conversion inhibitors and the emergence of an interaction site at the β2-α2 loop region.</p>
<p><strong>CONCLUSIONS/SIGNIFICANCE:</strong></p>
<p>Our results suggest that <strong>pathogenic mutations share a common pattern of dynamical alterations that converge to the conversion of the β2-α2 loop into an interacting region that can be used as target for interference treatments in genetic diseases.</strong></p>
<p>&#8211;</p>
<p><strong>NMR studies have shown that PrP<sup>C</sup> native state is featured by an N-terminal flexible tail and a C-terminal globular domain (residues 125–228, human sequence numbering), with a fold containing three α-helices and an anti-parallel β-sheet</strong> <a href="http://www.ncbi.nlm.nih.gov/pubmed/8700211" target="_blank">[10]</a>–<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3084259/?tool=pubmed#pone.0019093-Gossert1" target="_blank">[13]</a>.</p>
<p>&#8211;</p>
<p>Despite important advances in the last decade, the role that pathogenic mutations identified in the human PrP play in favoring the production of misfolded forms and of disease remains an open issue. Answering these mechanistic questions is essential for the <strong>design and development of prophylactic strategies for delaying the disease onset in asymptomatic carriers.</strong> Using the native state as the prevalent form and focusing on the early events, we have shown here that combining the observations from all-atom simulations of ten PrP pathogenic mutants contained in the α2-α3 region, depicts a common perturbation pathway that is supported by previous unrelated experimental findings. The obtained results show that mutations alter the flexibility and the coordination properties of the native state, with a specific impact on the dynamics of the α2-α3 helical hairpin, which in turn modify the protein interactive surface impinging on the binding sites for conversion inhibitors and activating the interacting properties of the β2-α2 loop\</p>
<p>&#8211;</p>
<p>While the protein is still in the native state and therefore functional, the studied mutations perturb the conformational dynamics. As a general effect and in agreement with previous reports, <strong>most mutations behave as flexibility amplifiers which favors the access to otherwise impeded states </strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/20932979" target="_blank">[19]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20949975" target="_blank">[30]</a>–<a href="http://www.ncbi.nlm.nih.gov/pubmed/19172188" target="_blank">[32]</a>. Importantly, <strong>the detected perturbations transmit through the protein chain to sites distant from the mutation position</strong>. In particular, the alterations are sensed by the β2-α2 loop, the C-terminal region of α2, part of the α2-α3 loop, and the C-terminal part of α3. These effects are reminiscent of those caused by the polar perturbations of α3 methionines <a href="http://www.ncbi.nlm.nih.gov/pubmed/19172188" target="_blank">[32]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20454997" target="_blank">[33]</a>. Moreover, the long range effects of sequence variation have emerged as a general property of many proteins <a href="http://www.ncbi.nlm.nih.gov/pubmed/20133562" target="_blank">[51]</a>–<a href="http://www.ncbi.nlm.nih.gov/pubmed/19841628" target="_blank">[53]</a>. <strong>All the affected regions are endowed with proven importance in driving fibrillation processes</strong> <a href="http://www.ncbi.nlm.nih.gov/pubmed/1438300" target="_blank">[4]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/19602569" target="_blank">[8]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20410442" target="_blank">[14]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20375014" target="_blank">[15]</a>. <strong>The β2-α2 loop, which links the two halves of the native fold, contains a motif that drives amyloid formation when inserted in an unrelated protein</strong> <a href="http://www.ncbi.nlm.nih.gov/pubmed/19602569" target="_blank">[8]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/17468747" target="_blank">[9]</a>. The other two regions are contained in the α2-α3 subdomain, which participates in the backbone of recombinant PrP fibrils <a href="http://www.ncbi.nlm.nih.gov/pubmed/18025469" target="_blank">[16]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20925423" target="_blank">[17]</a>. Interestingly, the α2-α3 subdomain was also identified as the most relevant region for PrP oligomerization. Despite the presence of two sequences undergoing fast fibrillation, the α2-α3 hairpin behaves as a stable, independently structurally stable folded unit when isolated from the context of the whole protein <a href="http://www.ncbi.nlm.nih.gov/pubmed/1438300" target="_blank">[4]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20410442" target="_blank">[14]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/20375014" target="_blank">[15]</a>. Furthermore, chromatography and light scattering experiments showed that the α2-α3 oligomerization pattern recapitulate that of the full-length protein <a href="http://www.ncbi.nlm.nih.gov/pubmed/20410442" target="_blank">[14]</a>. Moreover, except for T188K, mutations also increase the average solvent exposure of M213 compared to the wt, a factor that may facilitate the oxidation of its side chain by external agents. In turn, M213 oxidation facilitates the oxidation of M205 and M206, all immunologically proven as covalent features of aggregated human PrP chains <a href="http://www.ncbi.nlm.nih.gov/pubmed/20625387" target="_blank">[34]</a>–<a href="http://www.ncbi.nlm.nih.gov/pubmed/19416900" target="_blank">[36]</a>. The quantitative determination of the amounts of oxidized protein, the relevance of the in vitro experimental conditions to the in vivo situation and of this factor as a trigger for pathogenic conversion, are still open issues that need to be further investigated at the mechanistic, biochemical and structural levels.</p>
<p>&#8211;</p>
<p>All the changes in the native state dynamics induced by the mutations reverberate as a reorganization of reactive regions, sites entailing the recognition of small molecules, antibodies or self-association events. In the wt form the interaction hot-spots appear at the 141–147 and 187–198 regions, both proven sites for the binding of conversion inhibitors <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3084259/?tool=pubmed#pone.0019093-Antonyuk1" target="_blank">[38]</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/11507642" target="_blank">[40]</a><a href="http://www.ncbi.nlm.nih.gov/pubmed/17616582" target="_blank">[42]</a>–<a href="http://www.ncbi.nlm.nih.gov/pubmed/19725511" target="_blank">[43]</a>. In contrast, <strong>in the mutants</strong>, either directly or as consequence of a facilitated putative M213 oxidation as for V210I and R208H, <strong>the interacting regions change. This change is featured by the disappearance of the 141–147 inhibitor binding site and the emergence of the 166–179 stretch (β2-α2 loop) as interacting site.</strong> The capacity of the β2-α2 loop to participate in both heterologous and homologous intermolecular interaction has been experimentally shown validating the site reactivity <a href="http://www.ncbi.nlm.nih.gov/pubmed/17468747" target="_blank">[9]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/9363892" target="_blank">[39]</a>,<a href="http://www.ncbi.nlm.nih.gov/pubmed/9294164" target="_blank">[44]</a>. On the other hand, the inhibition of wt PrP conversion reactions by synthetic peptides overlapping the β2-α2 loop suggests that the acquisition of binding properties by this loop is a general step in the prion formation and validates its behavior as pharmacological target <a href="http://www.ncbi.nlm.nih.gov/pubmed/11279046" target="_blank">[45]</a>.</p>
<p>&#8211;</p>
<p><strong>In order to apply our approach to PrP<sup>Sc</sup>, reliable models of the respective structures should be available.</strong> A step into this direction may be represented by the study of Chakroun <em>et al</em>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/20410442" target="_blank">[14]</a>, whereby the structural properties of the isolate α2-α3 region, which recapitulates the amyloidogenic features of the whole protein and contains the mutations herein studied, have been elucidated by means of MD, generating a as the isolate β-sheet rich conformation consistent with experimental observations. In any case, <strong>the generation and validation of yet unavailable reliable structural models is an absolute requirement.</strong></p></blockquote>
<p>&nbsp;</p>
<p>Structural Biochemistry/ Proteins/ Structures<br/>
<a href="http://en.wikibooks.org/wiki/Structural_Biochemistry/Proteins/Structures" target="_blank">http://en.wikibooks.org/wiki/<wbr>Structural_Biochemistry/<wbr>Proteins/Structures</wbr></wbr></a></p>
<blockquote><p>Proteins are either folded, or not. There <strong>does not exist</strong> a stage where a protein is &#8220;<strong>half-folded</strong>&#8220;. This can be observed by slowly adding denaturant to a protein. This will result in a <strong>sharp transition</strong>, from the folded state to the unfolded state, suggesting there only exist these two forms. This is a result of <strong>cooperative transition</strong>.</p>
<p>For instance, if a protein is put in a denaturant where only one part of the protein is unstable, <strong>the entire protein will unfold</strong>. This is due to the domino effect where <strong>destabilizing one part</strong> of the protein will in turn <strong>destabilize the remainder of the structure</strong>. When a protein is in conditions which correspond to the middle of the transition between folded and unfolded, there is a <strong>50/50</strong> mixture of folded and unfolded protein, instead of &#8216;half-folded&#8217; protein.</p>
<p>After all is said about being in one structure or the other, there must be something in between them on an atomic level. Unfortunately, this is an area that is still under development, and much research is still being done. Theories such as the condensation Nucleation Principle are concerned with this area of protein folding.</p>
<p>&nbsp;</p></blockquote>
<p>&nbsp;</p>
