---
layout: post
title: "Exome sequencing and analysis"
author: ericminikel
deprecated: true
date: 2012-08-02 20:53:59
---
<p>This post will aim to answer the following questions:</p>
<ul>
<li>How is exome sequencing accomplished?</li>
<li>Why is exome sequencing still a commonly used alternative to increasingly cheap whole genome sequencing?</li>
<li>What are the particular challenges of working with exome data and how can these be mitigated?</li>
<li>What can be achieved with exome data?</li>
</ul>
<p><strong>How is exome sequencing accomplished?</strong></p>
<p>The most popular method is in-solution capture, also known as liquid hybridization. This technique relies on the fact that, at this point in human history, we already have a pretty good idea of what parts of the genome are exomic.  A few companies (Agilent, Roche NimbleGen &#8212; see <a href="http://metapress.com/content/x1172n1m87276538/fulltext.pdf">Sulonen 2011</a>) offer solution-based exome capture kits which contain probes complementary to subsequences of (almost) every known exomic region of the human genome.  That&#8217;s right, we&#8217;re going in with an <em>a priori</em> idea of what regions are exomic and we&#8217;re only going to capture those for sequencing.  So this technique would never, for instance, discover new exomic regions not previously known to be expressed.</p>
<p>Moreover, it&#8217;s not necessarily perfectly known or agreed which regions are exomic, and the answer also depends on if you only want protein-coding sequences or also miRNA sequences.  <a href="http://hmg.oxfordjournals.org/content/19/R2/R145.long">Teer &amp; Mullikin (2010</a>) provide the follwoing discussion of this issue:</p>
<blockquote><p>The term ‘whole human exome’ can be defined in many different ways. Two companies offer commercial kits for exome capture and have targeted the human consensus coding sequence regions (28), which cover ∼29 Mb of the genome. This is a more conservative set of genes and includes only protein-coding sequence. It covers ∼83% of the RefSeq coding exon bases. Both companies also target selected miRNAs, and extra regions can sometimes be added (Agilent). Although still a subset of the genome, exome capture allows the investigation of a more complete set of human genes with the cost and time advantages of genome capture.</p></blockquote>
<p>The probes are added to a solution of fragmented genomic DNA and they hybridize to their complement sequences.  The primers themselves might be RNA or DNA (again see <a href="http://metapress.com/content/x1172n1m87276538/fulltext.pdf">Sulonen 2011</a>).  The primers are <a href="http://en.wikipedia.org/wiki/Biotinylation">biotinylated</a>, and biotin binds to <a href="http://www.piercenet.com/browse.cfm?fldID=01021103">streptavidin beads</a>, so once you&#8217;ve got the primers hybridized to the genomic DNA fragments of interest, you add the beads, then wash the bead-bait complexes to remove the undesired (non-exomic) DNA, and <a href="http://en.wikipedia.org/wiki/Elution">elute</a> the DNA. <a href="http://hmg.oxfordjournals.org/content/19/R2/R145.long">Teer &amp; Mullikin (2010)</a> offer <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953745/figure/DDQ333F1/">the best diagram I&#8217;ve seen of this process</a>.</p>
<p>At this point, you have a solution full of <em>only</em> the exomic DNA you&#8217;re interested in sequencing.  Now you just sequence it the same way you would if you were doing whole-genome sequencing.  Most likely using the <a href="http://www.illumina.com/technology/sequencing_technology.ilmn">Illumina method</a>, which Illumina summarizes as follows:</p>
<blockquote><p>TruSeq technology supports massively parallel sequencing using a proprietary reversible terminator-based method that enables detection of single bases as they are incorporated into growing DNA strands. A fluorescently-labeled terminator is imaged as each dNTP is added and then cleaved to allow incorporation of the next base. Since all four reversible terminator-bound dNTPs are present during each sequencing cycle, natural competition minimizes incorporation bias. The end result is true base-by-base sequencing that enables the industry’s most accurate data for a broad range of applications.</p></blockquote>
<p>Illumina&#8217;s video explains this a lot more clearly, particularly the part from 5:13 &#8211; 6:06:</p>
<p><iframe width="560" height="315" src="http://www.youtube.com/embed/45vNetkGspo" frameborder="0" allowfullscreen></iframe></p>
<p>An important thing to keep track of is your <em>coverage</em> or <em>call depth</em>.  This means how many reads per base pair sequenced.  Typical numbers seem to range from 20x to 80x (for instance, <a href="http://web.archive.org/web/20130129132346/https://www.23andme.com/exome/">23andme&#8217;s personal exome product</a> boasts 80x coverage) (update: <a href="https://www.23andme.com/compare-dna-tests/?">it is no longer offered</a>).  But the figure won&#8217;t be uniform across the genome, so you end up with sentences like this one: &#8220;Exome capture produced 1x coverage of 97% of the exome; 30x coverage was 74%&#8221; <a href="http://www.neurology.org/cgi/content/meeting_abstract/78/1_MeetingAbstracts/P04.034"></a>.</p>
<p>By the way, there do exist other methods for exome capture.  <a href="http://hmg.oxfordjournals.org/content/19/R2/R145.long">Teer &amp; Mullikin (2010)</a> discuss some of them, as does <a href="http://en.wikipedia.org/wiki/Exome_sequencing#Target-enrichment_strategies">Wikipedia</a>, but word on the street is that no one uses the other methods anymore; as of August 2012, it&#8217;s all in-solution capture.</p>
<p><strong>Why is exome sequencing still a commonly used alternative to increasingly cheap whole genome sequencing?</strong></p>
<p>tl;dr: It&#8217;s still cheaper, both in terms of direct cost to sequence, and in terms of your time and computing power to deal with the data.</p>
<p><a href="http://genomebiology.com/2011/12/9/128/">Genome Biology asked experts Biesecker, Shianna and Mullikin</a> to discuss this among other issues.  As of September 2011, <a href="http://genomebiology.com/2011/12/9/128/#sec2">all three experts stated</a> that whole exome sequencing is still a lot cheaper (~1/6) than whole genome sequencing, and that cost is still a limiting factor in terms of how much sequence data they can generate.  They also said you wait longer for your sequence data to be ready if you are getting a whole genome than just an exome, and then you also have loads more data to store and process (the exome is about 30 Megabases, the (haploid) genome is about 3.2 Gigabases)</p>
<p>Meanwhile, while we are no longer so ignorant as to call non-exomic DNA &#8220;<a href="http://en.wikipedia.org/wiki/Junk_DNA">junk DNA</a>&#8220;, the exome is still highly enriched for Stuff That Matters.  To wit: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2768590/">Choi 2009</a> says:</p>
<blockquote><p>Protein coding genes constitute only approximately 1% of the human genome but harbor 85% of the mutations with large effects on disease-related traits.</p></blockquote>
<p><strong>What are the particular challenges of working with exome data and how can these be mitigated?</strong></p>
<p>Most obviously, the exome isn&#8217;t everything.  Mutations in splice sites, promoter regions and other non-exomic regions matter too.  The fact that exome sequencing misses non-coding regions is clearly a limitation, but that&#8217;s what exome sequencing is; this is sort of a &#8220;known unknown.&#8221;  Perhaps more troubling is the fact that there are probably also some coding regions that we haven&#8217;t yet identified as coding regions and so we&#8217;re missing them.  Because this is an &#8220;unknown unknown&#8221; it bothers me a bit more. <a href="http://genomebiology.com/2011/12/9/128/#sec3">Biesecker and Mullikin</a> touch on this issue:</p>
<blockquote><p>Also, our current understanding of the genome limits our exome interrogation &#8211; nucleotides in regions of the genome not currently recognized to be a gene will be missed by exome approaches.</p></blockquote>
<p>The blog at Scientific American lists <a href="http://blogs.scientificamerican.com/guest-blog/2012/05/16/10-things-exome-sequencing-cant-do-but-why-its-still-powerful/">ten things exome sequencing doesn&#8217;t capture</a>.  This was a nice, concise introduction to the issue.  I&#8217;d take a few things with a grain of salt&#8211; the list includes &#8220;gene-gene (epistatic) interactions&#8221;, but that&#8217;s just a limitation of our scientific understanding, not of exome sequence data, and moreover, it&#8217;s precisely the sort of science that exome sequencing will help to advance.  But this list was useful and it did alert me to an important issue: exome sequencing (whole genome sequencing, too) cannot detect the length of CAG repeats in HTT.  This is because standard read lengths from, say, an <a href="http://www.illumina.com/systems/hiseq_systems.ilmn">Illumina HiSeq</a> machine, are about 150bp long.  To determine the length of CAG repeats in HTT, you&#8217;d need a read that starts with about 20bp that you can uniquely align to HTT, then the entire CAG repeat region, then at least 3 or 4 bases at the end that are <em>not</em> CAG, to show you that the repeat region has ended.  Huntington&#8217;s disease is caused by repeat lengths from about 40 on up, with cases in the 100s.  At the low end of 40, you could just barely fit the entire CAG repeat region into one read: 20 + 40*3 +4 = 144 &lt; 150.  Longer repeats won&#8217;t fit in one read length.</p>
<p>You might think you could overcome this, if you had enough resolution, by measuring the <em>quantity</em> of all-CAG-repeat reads, but other genes besides HTT also have CAG repeat regions, so you would not know where those reads fit into the genome.</p>
<p>An added problem is that, just as CAG repeats have a tendency to lengthen due to <a href="http://en.wikipedia.org/wiki/Trinucleotide_repeat_disorder#Trinucleotide_repeat_expansion">slippage</a> during replication in gamete production, particularly spermatogenesis (giving rise to <a href="http://en.wikipedia.org/wiki/Anticipation_(genetics)">anticipation</a>), they also have a tendency to lengthen during <a href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction">PCR</a>, which means you can&#8217;t accurately amplify HTT DNA.</p>
<p>I asked around and apparently the standard way that clinical genetic diagnostics of Huntington&#8217;s are done is to use restriction enzymes to cleave DNA immediately on either side of the expanded repeat region and then perform gel electrophoresis on the cleaved region and simply measure how long it is (based on how far it travels on the gel).</p>
<p><strong>What can be achieved with exome data?</strong></p>
<p>The web is littered with references to what seem to be the two most famous examples of successful applications of exome data: one in which genome-wide association study was used to find the mutation responsible for a familial syndrome (Miller syndrome) which had already been characterized but whose cause was unknown, and one in which the exome data was used as a diagnostic tool, scanning a patient&#8217;s exome for any known deleterious mutations in order to identify the cause of the patient&#8217;s symptoms (revealed to be congenital chloride diarrhea).</p>
<p><span style="text-decoration: underline;">1. Identifying the mutation responsible for a familial syndrome.</span>  <a href="http://www.nature.com/ng/journal/v42/n1/full/ng.499.html">Ng &amp; Buckingham 2009</a> in Nature Genetics used exome data from just 4 cases and 8 controls to pinpoint the gene (DHODH) in which mutations are responsible for Miller syndrome.  The filtering and modeling they did was pretty remarkable in its elegance and power.  Two of the four cases were siblings, so they reasoned that the two siblings must share the <em>same</em> mutation and, because Miller syndrome is so rare, it must be a variant not observed in any of the controls nor in dbSNP129 (a database of SNPs, as it sounds like).  They also hypothesized the variant would be recessive, so that meant that each case had to be homozygous for the given mutation.  These few filters alone reduced the pool of candidates to just 9 genes.  Comparison to just 1 of the other unrelated cases reduced the pool to 1 gene, DHODH.  Even with the <a href="http://en.wikipedia.org/wiki/Bonferroni_correction">Bonferroni correction</a>, this result was highly statistically significant.  The researchers then sequenced DHODH in several other Miller syndrome cases and found mutations there as well, thus validating their finding.</p>
<p><span style="text-decoration: underline;">2. Diagnosing a patient&#8217;s by discovering that they have a mutation already known to cause a particular syndrome.</span>  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2768590/">Choi 2009</a> performed exome sequencing on a patient with an unidentified ailment.  Due to <a href="http://en.wikipedia.org/wiki/Consanguinity">consanguinity</a> the researchers hypothesized a homozygous recessive mutation.  They found 2,405 homozygous variants, including 668 non-synonymous substitutions (i.e. <a href="http://en.wikipedia.org/wiki/Missense_mutation">missense</a>), of which 29 were novel.  Reasoning that mutations in highly conserved loci are more likely to cause disease, they ranked the novel missense variants by phyloP score (a measure of how highly conserved that locus is across species) and found 10 variants with a score above their chosen threshold of 2.  It appears the final step was simply to look at these 10 and reason about which one made the most sense as an explanation for the patient&#8217;s symptoms.  A mutation in SLC26A3 stood out because the locus was very highly conserved, the read quality was very good, and the gene was known to be related to congenital chloride-losing diarrhea, which would match the patient&#8217;s symptoms.</p>
<p>A third application is the use of exome data for <a href="http://en.wikipedia.org/wiki/Genome-wide_association_study">genome-wide association studies</a> &#8211; in other words, searching the whole exome for variants correlated with a given phenotype.  These types of studies will be the subject of my next post.</p>
