---
layout: post
title: "Lindquist lab follows up on yeast prion epigenetics"
author: ericminikel
deprecated: true
date: 2012-02-28 23:28:38
---
<p>Halfmann and Jarosz et al of Susan Lindquist&#8217;s lab have published in Nature a significant followup on the lab&#8217;s earlier work establishing the epigenetic role of prions in yeast: <a href="http://www.nature.com/nature/journal/v482/n7385/full/nature10875.html">Prions are a common mechanism for phenotypic inheritance in wild yeast</a>.</p>
<p>In the <a href="/2012/01/22/prion-lectures-by-susan-lindquist/">videos we blogged about earlier</a>, Lindquist shows how prion amyloid formation can take translation termination factors out of commission, thus causing read-through and leading to the expression of a huge number of new traits.  This amyloid formation happens randomly (at a rate of 10<sup>-6</sup>) and can also be triggered in response to stresses.  Many of the new traits prove to be adaptive and help yeast to survive these stresses.  Lindquist speculates that these yeast prions have been selected by evolution precisely for this reason.</p>
<p>The new article provides a strong new body of evidence for this conclusion by demonstrating that (1) such prions are found in a large number of wild yeast strains (not only in laboratory strains), (2) proteins with &#8220;meta&#8221; functionality such as transcription factors, translation factors, etc. are over-represented among these prions, and (3) the conversion of these prions into amyloids frequently provides adaptive benefits.</p>
<p>It&#8217;s an excellent article, and the conclusion reminds us how much of science is devoted towards clarifying the KDKs (things you know you don&#8217;t know) at the expense of overlooking DKDKs (things we don&#8217;t know we don&#8217;t know):</p>
<blockquote><p>Saccharomyces cerevisiae is perhaps the most thoroughly characterized organism in experimental science. How, then, could this pervasive influence on the inheritance of biological traits have been missed for so long? The frequency of [PSI<sup>+</sup>] in wild strains suggests that previous efforts to find it simply did not examine enough strains (see Supplementary Information for further discussion). But we suspect that standard practices in yeast genetics provide a far more general explanation. Phenotypic analysis of new traits typically begins by testing for 2:2 segregation in crosses and discarding variants that do not follow this Mendelian pattern. It is equally common to discard variants that prove to be restricted to individual  trains. Thus, prion-based phenotypes may largely have been ignored because investigations  were strongly biased by the known rules of nucleic-acid based inheritance and because of a pragmatism that neglected the biological significance of strain-to-strain variation.</p></blockquote>
<p>A note about vocabulary: I&#8217;ve realized that it seems to be customary to use the term &#8220;prion&#8221; generally to refer to proteins that can misfold and infectiously transmit the misfolded state.  As <a href="http://en.wikipedia.org/wiki/Prion">Wikipedia</a> has it: &#8220;A prion is an infectious agent composed of protein in a misfolded form&#8221;.  The term PrP refers to the &#8220;Major prion protein&#8221; found in mammals, and PRNP refers to the human gene for PrP (<a href="http://en.wikipedia.org/wiki/PRNP">Wikipedia</a>).  So when this article refers to &#8220;prions&#8221; it just means any protein of this nature (and there are many, they find).  As I understand, the yeast prions discussed in this article are not of any common evolutionary origin with human PrP and are not particularly similar in function or in amino acid sequence (though of course they are probably similarly rich in β-sheets, hence the ability to form amyloid fibers).</p>
<p>This article seems to be a breakthrough because of its implications for our understanding of evolution and epigenetics rather than due to any direct implications for human health.  But hey, a big advance in scientific knowledge is still a good thing.</p>
<p>&nbsp;</p>
