---
layout: post
title: "Allele-specific gene silencing using siRNA"
author: s
deprecated: true
date: 2012-01-18 17:51:14
---
<p>“Enhancement of allele discrimination by introduction of nucleotide mismatches into siRNA in allele-specific gene silencing by RNAi,” Ohnishi et al., PLoS One (2008)</p>
<p>www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248</p>
<blockquote><p>Allele-specific gene silencing by RNA interference (RNAi) is therapeutically useful for specifically inhibiting the expression of disease-associated alleles without suppressing the expression of corresponding wild-type alleles. To realize such allele-specific RNAi (ASP-RNAi), the design and assessment of small interfering RNA (siRNA) duplexes conferring ASP-RNAi is vital; however, it is also difficult. In a previous study, we developed an assay system to assess ASP-RNAi with mutant and wild-type reporter alleles encoding the <em>Photinus</em> and <em>Renilla luciferase</em> genes. In line with experiments using the system, we realized that it is necessary and important to enhance allele discrimination between mutant and corresponding wild-type alleles. Here, we describe the improvement of ASP-RNAi against mutant alleles carrying single nucleotide variations by introducing base substitutions into siRNA sequences, where original variations are present in the central position. Artificially mismatched siRNAs or short-hairpin RNAs (shRNAs) against mutant alleles of the human <em>Prion Protein</em> (<em>PRNP</em>) gene, which appear to be associated with susceptibility to prion diseases, were examined using this assessment system. The data indicates that <strong>introduction of a one-base mismatch into the siRNAs and shRNAs was able to enhance discrimination between the mutant and wild-type alleles. Interestingly, the introduced mismatches that conferred marked improvement in ASP-RNAi, appeared to be largely present in the guide siRNA elements, corresponding to the ‘seed region’ of microRNAs. </strong>Due to the essential role of the ‘seed region’ of microRNAs in their association with target RNAs, it is conceivable that disruption of the base-pairing interactions in the corresponding seed region, as well as the central position (involved in cleavage of target RNAs), of guide siRNA elements could influence allele discrimination. In addition, we also suggest that nucleotide mismatches at the 3′-ends of sense-strand siRNA elements, which possibly increase the assembly of antisense-strand (guide) siRNAs into RNA-induced silencing complexes (RISCs), may enhance ASP-RNAi in the case of inert siRNA duplexes. Therefore, the data presented here suggest that structural modification of functional portions of an siRNA duplex by base substitution could greatly influence allele discrimination and gene silencing, thereby contributing to enhancement of ASP-RNAi.</p>
<p>In this study, we focused on the <strong>human <em>Prion Protein (PRNP)</em> gene,</strong> which is known to possess a number of single nucleotide variations <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248-Collinge1">[20]</a>, <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248-Jackson1">[21]</a>. We selected three <em>PRNP</em> variants, which are also followed by amino acid substitutions (<em>P102L, P105L</em>, and <em><strong>D178N</strong></em>) and appear to be associated with susceptibility to various prion diseases such as Gerstmann-Sträussler-Scheinker disease (GSS) and <strong>fatal familial insomnia (FFI)</strong> <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248-Prusiner1">[22]</a>–<a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248-Chen1">[25]</a>. We constructed three mutant reporter alleles, designated the <em>PRNP-P102L, PRNP-P105L</em>, and <em>PRNP-D178N</em> alleles, and their corresponding wild-type reporter alleles (<a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone-0002248-g001">Figure 1A</a>). The reporter alleles, synthetic siRNA duplexes against the mutant alleles (supplementary <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248.s004">Table s1</a> and supplementary <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone.0002248.s001">Figure s1</a>), and the <em>beta-galactosidase</em> gene (control), were cotransfected into HeLa cells; thus, the transfected cells were artificially heterozygous with the mutant and wild-type reporter alleles. The effects of the designed siRNA duplexes on suppression of both the mutant and wild-type alleles were then simultaneously examined. As shown in <a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone-0002248-g001">Figure 1</a>, the siRNA duplexes other than siPrnp102(T7), siPrnp102(T8), siPrnp105(T7) and siPrnp105(T9) were not able to induce significant ASP-RNAi. Of the four siRNAs just listed, the siPrnp105(T9) duplex appears to confer ASP-RNAi.</p>
<p>In the present study, we observed an improvement in ASP-RNAi when siRNAs induced double knockdown of mutant and wild-type alleles: <strong>introduction of a one-base substitution into such siRNAs carrying the original variations around the central position appeared to influence allele discrimination and inhibition of target mutant alleles, although different base substitutions conferred different levels of discrimination and inhibition</strong> (<a href="http://www.plosone.org/article/info:doi%2F10.1371%2Fjournal.pone.0002248#pone-0002248-g006">Figure 6B</a>). This phenomenon may be associated with the thermodynamic properties of the modified siRNA duplexes. Interestingly, <strong>the base substitutions conferring marked ASP-RNAi appeared to be largely present in the region of guide siRNAs, corresponding to the seed region of microRNAs. Since such siRNAs exhibit one and two mismatches against mutant and wild-type alleles, respectively, we suggest that disruption of base-pairing interaction in the seed region, as well as the central position, of the guide siRNAs reduces recognition and/or silencing activity against wild-type alleles, and that a one-base mismatch in the seed region of the guide siRNAs against the target mutant alleles hardly affects gene silencing, i.e., potent RNAi activity against the mutant alleles may remain unchanged.</strong></p>
<p>In conclusion, in order to realize ASP-RNAi against target mutant alleles carrying nucleotide variations, the design and evaluation of competent siRNA duplexes conferring ASP-RNAi is vital; but designed siRNAs do not always confer potent ASP-RNAi activity. The evidence presented here suggests that structural modification of siRNA duplexes by base substitutions may improve ASP-RNAi. <strong>The key regions in an siRNA duplex for such modifications are the central position, the seed region and the 3′-end of the sense-strand siRNA element, which appear to be related to target RNA cleavage, target RNA recognition and assembly of the antisense-strand (guide) siRNA element into RISCs, respectively.</strong> Therefore, structural modification of such functional portions of siRNA duplexes may greatly influence allele discrimination and gene silencing activity, thereby conferring improvement of ASP-RNAi.</p></blockquote>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>Miller VM, Xia H, Marrs GL, Gouvion CM, Lee G, et al. (2003) Allele-specific silencing of dominant disease genes. Proc Natl Acad Sci U S A 100: 7195–7200.</p>
<p>http://www.pnas.org/content/100/12/7195.full</p>
<blockquote><p>Small interfering RNA (siRNA) holds therapeutic promise for silencing dominantly acting disease genes, particularly if mutant alleles can be targeted selectively. <strong>In mammalian cell models we demonstrate that allele-specific silencing of disease genes with siRNA can be achieved by targeting either a linked single-nucleotide polymorphism (SNP) or the disease mutation directly.</strong> For a polyglutamine neurodegenerative disorder in which we first determined that selective targeting of the disease-causing CAG repeat is not possible, we took advantage of an associated SNP to generate siRNA that exclusively silenced the mutant Machado–Joseph disease/spinocerebellar ataxia type 3 allele while sparing expression of the WT allele. <strong>Allele-specific suppression was accomplished with all three approaches currently used to deliver siRNA: <em>in vitro</em>-synthesized duplexes as well as plasmid and viral expression of short hairpin RNA.</strong> We further optimized siRNA to specifically target a missense Tau mutation, V337M, that causes frontotemporal dementia. These studies establish that siRNA can be engineered to silence disease genes differing by a single nucleotide and highlight a key role for SNPs in extending the utility of siRNA in dominantly inherited disorders.</p>
<p>Based on our findings, we conclude that allele-specific silencing should be possible for many dominant disease genes. Issues of <em>in vivo</em> delivery and efficacy remain to be resolved, of course. Notably, the long-term consequences of chronically triggering the RNAi pathway <em>in vivo</em>, as may be required to treat neurodegenerative conditions, are unknown.</p>
<p>Our data indicate that <strong>genes can differ widely in their susceptibility to inhibition, with no obvious sequence features predicting the success or failure of a given siRNA</strong>. For example, every siRNA we designed against ataxin-3 displayed significant activity (7/7, 100%), whereas Tau proved more difficult to inhibit, with only a single region centered on the V337M mutation yielding effective siRNAs (3/7, 43%). Moreover, although silencing of a specific gene is readily achieved once an accessible target is found, preferential silencing of a particular allele of that gene requires careful design and engineering of the siRNA. Our results indicate, for example, that <strong>a single nucleotide difference between two alleles may not be sufficient to confer allele specificity unless it is placed centrally in the siRNA</strong>, which suggests that at least two factors contribute to siRNA specificity: (<em>i</em>) the overall efficiency of base-pairing between siRNA and mRNA and (<em>ii</em>) the presence of Watson–Crick base-pairing between siRNA and mRNA at the central position across from the RNA-induced silencing complex cleavage site. In this study we have demonstrated that these factors can be manipulated by introducing peripheral mismatches to alter the silencing specificity of siRNA. Beginning with simply and economically prepared RNA duplexes, we have shown that <strong>multiple targets and siRNA designs can be screened systematically to optimize allele-specific siRNA for a disease gene. Once the best siRNA has been identified by this approach, it can be incorporated into shRNA expressed from plasmids or viral vectors that retain the efficacy and allele specificity of the original duplex. </strong></p>
<p>In summary, we conclude that siRNA can be engineered to silence expression of disease alleles differing from WT alleles by as little as a single nucleotide. We have established that this approach can target missense mutations directly, as in frontotemporal dementia, or associated SNPs, as in MJD/SCA3. Our stepwise strategy for optimizing allele-specific targeting, together with recent advances in viral delivery (<a href="http://www.pnas.org/content/100/12/7195.full#ref-5">5</a>) and the demonstration of RNAi in primary neurons (<a href="http://www.pnas.org/content/100/12/7195.full#ref-32">32</a>), should extend the utility of siRNA to a wide range of dominant diseases in which the disease gene normally plays an important or essential role. One such example is another polyQ disease, Huntington disease (HD), in which normal HD protein levels are developmentally essential (<a href="http://www.pnas.org/content/100/12/7195.full#ref-12">12</a>). The <strong>availability of mouse models </strong>for many dominant disorders including MJD/SCA3 (<a href="http://www.pnas.org/content/100/12/7195.full#ref-33">33</a>), HD (<a href="http://www.pnas.org/content/100/12/7195.full#ref-34">34</a>), and FTDP-17 (<a href="http://www.pnas.org/content/100/12/7195.full#ref-35">35</a>) should speed the <em>in vivo</em> testing of siRNA-based therapy for these and other human diseases.</p></blockquote>
