---
layout: post
title: "Resources for systems biology in prion disease"
author: ericminikel
date: 2012-12-14 03:23:31
---
<p>Systems biology is the science of using large datasets to infer what molecular interactions are happening and what biological networks are getting disturbed at the root of the disease or biological phenomenon you&#8217;re studying.  When I first studied bioinformatics, my instructor told us a story about a drunk looking for his lost keys under a street light &#8211; not because that&#8217;s where he had lost them, but just because that was the only place with good enough light to bother looking for anything.  This was an analogy for much of what we do in science.  We&#8217;d love to watch PrP interact with the proteins it interacts with, see exactly what happens to convert it to PrP<sup>Sc</sup>, and witness what that isoform does to cause trouble in the cell.   But since everything in biology is too small to see, we look where the light is better.</p>
<p>The good news is that there are a lot of bright lights these days.  In particular, arrays and sequencing have made everything to do with nucleic acids pretty high throughput: genotypes, expression levels, epigenetic markers, protein-DNA interactions.  These things aren&#8217;t always what we actually want to measure: for instance, in many cases, protein levels would be much more informative than RNA levels, but proteomics is still much more expensive than expression arrays or RNA-seq.  But they&#8217;re cheap enough to let us generate heaps of data that we can sift through to look for hints as to what&#8217;s really going on.</p>
<p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2671916/">Hwang 2009</a> provides us with a landmark demonstration of this systems approach applied to prion disease.  At the core is the use of <a href="http://en.wikipedia.org/wiki/DNA_microarray">microarrays</a> to measure gene expression levels in brain tissue of prion-infected and control mice and thus determine which genes are expressed at higher or lower levels as a result of prion disease.  Importantly, Hwang varied the prion strain (RML, 301V), time of measurement (60 to 350 days post infection), and the mouse strain: three unmodified inbred lines - B6, B6.I, FVB and three genetically modified lines &#8211;  Tg4053 (which overexpress PrP 8-fold), heterozygous and homozygous PrP knockout mice.  The large number of different combinations made it possible not only to cancel out biological and technical noise but also to examine which effects were prion strain-specific and which were shared, and which effects were time-dependent (or PrPSc concentration-dependent).  In all, 333 genes were found to be differentially expressed in all combinations tested.  These were then assembled &#8211; using a wealth of publicly available annotations on function, cell-specific expression, interactomics, and so on &#8211; into hypothetical functional networks to explain what is going on.</p>
<p>This all ends up producing giant graphs of gene names in bubbles pointing to one another, as depicted in Fig 4:</p>
<p><a href="http://www.nature.com/msb/journal/v5/n1/fig_tab/msb200910_F4.html"><img class="alignnone  wp-image-1319" title="hwang-2009-f4-small" src="/wp-content/uploads/2012/12/hwang-2009-f4-small.jpg"/></a></p>
<p>Which isn&#8217;t always the most accessible thing in the world, although Hwang&#8217;s is perhaps the most elegant I&#8217;ve seen.  But even if the pictures don&#8217;t speak to you, <a href="http://www.nature.com/msb/journal/v5/n1/full/msb200912.html">Omenn 2009</a> explains why this study is a big deal: instead of getting flummoxed by the variation in prion strains, genetic factors in the host that influence susceptibility, and so forth, this study uses all that information to its advantage, finding the core of what&#8217;s always involved in prion disease biology across all those factors.  Omenn also converts some of what&#8217;s in the graphs to more of a story:</p>
<blockquote><p>Visualization&#8230;provides a dynamic scheme for the processes that characterize the molecular conversion of benign prion protein (PrP<sup>C</sup>) to disease-causing PrP<sup>Sc</sup> isoforms accumulating in lipid rafts, followed by the three stages of neuropathology: synaptic degeneration, activation of microglia and astrocytes, and neuronal cell death.</p></blockquote>
<p>But if you don&#8217;t agree with that narrative, you&#8217;re welcome to spin your own: all the data from Hwang&#8217;s study are freely open to the public in the <a href="http://prion.systemsbiology.net">Prion Disease Database</a>.</p>
