---
layout: post
title: "Mitochondrial heteroplasmy"
author: ericminikel
date: 2013-01-31 15:59:52
---
<p>One topic that arose at yesterday&#8217;s <a title="Center for Human Genetic Research 2013 Retreat" href="/2013/01/30/center-for-human-genetic-research-2013-retreat/">CHGR retreat</a> was mitochondrial <a href="http://en.wikipedia.org/wiki/Heteroplasmy">heteroplasmy</a>, i.e. having more than one mitochondrial chromosome in the same cell.  If you get genotyped by <a href="https://www.23andme.com/">23andMe</a>, they&#8217;ll tell you your &#8216;maternal haplogroup&#8217; based on the overwhelming majority of your mitochondria, which you inherited from your mother.  But there are (at least) thousands of mitochondria in each cell in your body, and a handful may have a different mitochondrial chromosome.</p>
<p>One source of different mtDNA haplotypes is simply mutations in maternal mtDNA over time.  Another source is paternal mtDNA.  Sperm <em>do </em>have mitochondria.  It is generally held that after fertilization of the egg, the paternal mitochondria get killed off.  Or maybe they just get massively diluted because the sperm has so few mitochondria compared to the egg (<a href="http://en.wikipedia.org/wiki/Mitochondrial_DNA#Mitochondrial_inheritance">perhaps 1000-fold fewer</a>).  But not always.  Famously, <a href="http://www.nejm.org/doi/full/10.1056/NEJMoa020350">Schwartz &amp; Vissing 2002</a> reported on a patient with a mitochondrial disease owing to a mutated version of his father&#8217;s mitochondrial chromosome.  To make matters more complicated, the situation was <a title="different in different cells of the same person's body" href="http://en.wikipedia.org/wiki/Mosaic_(genetics)">mosaic</a>: he had maternal mtDNA in most tissues of his body but for some reason this mutated paternal mtDNA had become about 90% of the mitochondria in his blood.  It is speculated that this situation could only have arisen because the mutation made the paternal mitochondria multiply faster than normal mitochondria:</p>
<blockquote><p>The present case could be the result of the survival of one or a few sperm mitochondria that probably would have been diluted out and never have been recognized had the pathogenic mutation not conferred a selective proliferative advantage on the mitochondria.</p></blockquote>
<p>It was mentioned at the <a title="Center for Human Genetic Research 2013 Retreat" href="/2013/01/30/center-for-human-genetic-research-2013-retreat/">CHGR retreat</a> that many researchers believe heteroplasmy is toxic &#8211; but I couldn&#8217;t find any good citations for this viewpoint online.  Instead I found one small study (n=100 cases vs. 114 controls) that found that heteroplasmy was, on the contrary, associated with longevity [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2014781/">Rose &amp; Passarino 2007</a>].  But it seems that at this point we know relatively little about either the mechanisms or the consequences of heteroplasmy.</p>
<p>All this raises the interesting possibility of examining heteroplasmy using sequence data &#8211; see <a href="http://www.sciencedirect.com/science/article/pii/S0002929710003708">Li 2010</a> for an example of doing so.  As mentioned in my <a title="Exome sequencing pipeline using GATK" href="/2012/09/19/exome-sequencing-pipeline-using-gatk/">GATK pipeline</a>, even in exome data you can get plenty of mitochondrial depth even with an exome capture kit that doesn&#8217;t target mitochondria, simply because there are so many mitochondrial chromosomes in each cell.  I was surprised to find that the GATK / Queue default settings that I use in that pipeline <em>do </em>call heterozygous variants in mitochondria, and with a pretty low threshold too: browsing over my variants I see that GATK will call a heterozygous variant even there are 13 reads covering a base and just 1 read disagrees with the other 12.</p>
