---
layout: post
title: "How synonymous are synonymous SNPs?"
author: ericminikel
deprecated: true
date: 2013-01-24 17:22:58
---
<p>Synonymous SNPs are SNPs in protein-coding exons that don&#8217;t change the amino acid thanks to the redundancy of the <a href="http://en.wikipedia.org/wiki/Genetic_code#RNA_codon_table">genetic code</a>.  When researchers analyze sequence data, synonymous SNPs do figure into various <a title="Descriptive statistics and quality control on variants" href="/2012/10/17/descriptive-statistics-and-quality-control-on-variants/">quality control metrics</a>, but are often thrown out when people search for the causal mutation behind a disease or a trait.  A typical example of this is <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2930028/">Ng 2010</a>&#8216;s search for the causal mutation of Kabuki syndrome:</p>
<blockquote><p>our analyses focused primarily on nonsynonymous (NS) variants, splice acceptor and donor site mutations (SS) and coding indels (I), anticipating that synonymous variants were far less likely to be pathogenic.</p></blockquote>
<p>This is everyday practice when working with sequence data.  Throwing synonymous SNPs out seems less common in SNP array studies, because there you don&#8217;t sequence every possible genetic variation, so a synonymous SNP might well be tagging something else unseen-but-important via <a href="http://en.wikipedia.org/wiki/Linkage_disequilibrium">linkage disequilibrium</a>.</p>
<p>I started to wonder, are there <em>any</em> reasons why a synonymous SNP might actually be causal for a trait?  Yes, there are.  Here are a few possible reasons.</p>
<ol>
<li><strong>microRNAs</strong>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/21278745">Brest 2011</a>, studying <a href="http://en.wikipedia.org/wiki/Crohn's_disease">Crohn&#8217;s disease</a>, finds that a synonymous SNP (c.313C&gt;T in <a href="http://ghr.nlm.nih.gov/gene/IRGM">IGRM</a>) makes an enormous difference in whether <a href="http://en.wikipedia.org/wiki/Mir-196_microRNA_precursor_family">miR-196</a> binds to IGRM.  Specifically, miR-196 binds well to the C allele, downregulating IGRM, but not to the T allele.  The T allele thus leads to higher risk of Crohn&#8217;s disease because overexpression of IGRM reduces the cell&#8217;s ability to autophagize invading bacteria.</li>
<li><strong>Translation efficiency under amino acid starved conditions.  </strong>Different codons bind to different tRNAs, and different tRNAs have different affinity for amino acids.  Under conditions of amino acid abundance, there&#8217;s no difference in translation levels between codons, but a few investigators have shown that when bacteria are starved of a certain amino acid, proteins with the more affine codon will continue to be expressed while translation of proteins with the less affine codon will be suppressed.  As far as I can tell, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1299251/">Dittmar 2005</a> was the first to show this; there continues to be work on this issue e.g. <a href="http://arxiv.org/pdf/1212.1537v1.pdf">Subramaniam 2012</a>.  As far as I can tell all of this work has been done in bacteria and I don&#8217;t know whether it&#8217;s been shown in animals or if such specific amino acid-starved conditions ever even arise in humans.</li>
<li><strong>mRNA <a href="http://en.wikipedia.org/wiki/Nucleic_acid_secondary_structure">secondary structure</a></strong>.  This in turn can impact two things.   (3a) The efficiency of translation and thus the amount of protein produced.  See <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2761059/">Wang 2006</a> and <a href="http://www.ncbi.nlm.nih.gov/pubmed/17185601">Nackley 2006</a> for two examples. Or (3b) the speed at which translation proceeds along the mRNA, and since proteins begin to fold while they are still being translated, this can lead to totally different protein folding.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/17185560">Kimchi-Sarfaty 2007</a> seems to have discovered one such example, where a synonymous SNP leads to identical mRNA and protein levels but different protein folding and function in MDR1.  For a review of both of these mechanisms see <a href="http://www.ncbi.nlm.nih.gov/pubmed/17508390">Parmley 2007</a>.</li>
<li><strong>Alternative splicing.</strong>  In theory, a synonymous mutation might affect the way an RNA gets spliced.  A <a href="http://www.nature.com/nm/journal/v17/n12/full/nm1211-1536.html">2011 news feature in Nature</a> hints at this possibility but does not directly cite any studies that found such a case.  When I Googled it I found <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2860476/">Faa 2010</a> who found a mild form of cystic fibrosis caused by different splice products due to a synonymous mutation. However I hesitate to include this in the list here, because mutations that create a splice site cannot truly be called synonymous to begin with.  For instance, if you use <a href="http://snpeff.sourceforge.net/">snpEff</a> to annotate sequence variants, its <a href="http://snpeff.sourceforge.net/faq.html#How_is_impact_categorized?_(VCF_output)">classification scheme</a> will classify splice site mutations as &#8216;high impact&#8217;, as opposed to synonymous SNPs which are classified as &#8216;low impact&#8217;.  It seems imaginable that some synonymous SNP might have some subtle effect on splicing <em>without</em> actually creating or destroying a splice site, and if so, then I would consider this as a fourth mechanism, but so far I haven&#8217;t found any examples of it.</li>
</ol>
<p>There may yet be other mechanisms as well.  The <a href="http://www.nature.com/nm/journal/v17/n12/full/nm1211-1536.html">Nature article</a> points out that some synonymous sites are strongly conserved, and perhaps we don&#8217;t yet know all the reasons why this might be.</p>
<p>PS. Thanks to <a href="http://www.biostars.org/p/5242/#5245">Larry Parnell</a>, <a href="http://varigenome.blogspot.com/2011/02/synonymous-snps-are-not-so-synonymous.html">Variable Genome</a>, and <a href="http://phys.org/news/2013-01-hidden-genetic-code-key-differences.html">phys.org</a> for helping point me toward the relevant papers.</p>
