---
layout: post
title: "Tissue-specific gene expression data based on Human BodyMap 2.0"
author: ericminikel
date: 2013-07-11 13:23:05
---
<p><strong>motivation</strong></p>
<p>Often I find myself needing a quick reference on how highly expressed a particular gene is across different tissues, either to browse visually or to combine with other data in an analysis.  There are a few resources for this online:</p>
<ul>
<li><span style="line-height: 15px;"><a href="http://bioinfo.wilmer.jhu.edu/tiger/">TiGER</a> [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2438328/">Liu 2008</a>] has gene expression levels across 30 human tissues based on <a href="http://www.ncbi.nlm.nih.gov/nucest">NCBI EST</a> data, which can be browsed visually or downloaded, but the method by which the data were created has never been clear to me even after reading the paper, and the <a href="http://bioinfo.wilmer.jhu.edu/tiger/document.html">documentation</a> is sparse.</span></li>
<li><a href="http://genecards.org/">GeneCards.org</a> has <a href="http://www.genecards.org/info.shtml#protein_expression">protein expression levels</a> in a little graphic for each gene across several tissues (e.g. <a href="http://www.genecards.org/cgi-bin/carddisp.pl?gene=PRNP#proteins">PRNP</a>) based on MOPED and PaxDB data, but they aren&#8217;t downloadable.</li>
<li><a href="http://biogps.org/">BioGPS</a> has microarray gene expression data for 79 human and 61 mouse tissues, including several cancers, which can be browsed visually or downloaded.</li>
<li>The Allen Institute <a href="http://human.brain-map.org/">Human Brain Map</a> has microarray for hundreds of human brain regions, along with several tools for browsing visually, comparing regions, and downloading subsets of data.</li>
<li><a href="http://www.broadinstitute.org/gtex/">GTEx</a> has RNA-seq expression levels for about 40 postmortem human tissues from hundreds of donors (and counting).  Sadly, most of the data is under lock and key at <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v1.p1">dbGaP</a>, and as of today only the called <a href="http://en.wikipedia.org/wiki/Expression_quantitative_trait_loci">eQTLs</a> have been <a href="http://www.ncbi.nlm.nih.gov/gtex/GTEX2/gtex.cgi">made public</a>.  They recently added a &#8220;Search genes&#8230;&#8221; feature where you can type in a gene symbol such as PRNP and view a pretty graphic of average gene expression across tissues (grouped across all samples), but so far there is no option to download the underlying expression values.</li>
</ul>
<p>These are some pretty cool tools, but they don&#8217;t always meet my needs.  For instance, I analyze RNA-seq data, so it&#8217;s nice to have RNA-seq data to compare to.  Sure, RNA-seq protocols and computational pipelines can differ a lot between one another, but at least it&#8217;s better than comparing RNA-seq to microarray.  I also like to have something I can download to merge with other datasets rather than picking out individual genes or looking at a graphic.</p>
<p>I was excited, therefore, when I found out about <a href="http://www.ensembl.info/blog/2011/05/24/human-bodymap-2-0-data-from-illumina/">Human BodyMap 2.0</a>, a fully public RNA-seq dataset across 16 human tissues brought to us by Ensembl and Illumina.  It doesn&#8217;t divide the brain into regions, but other than that, it&#8217;s a pretty awesome dataset.  Ensembl has provided <a href="ftp://ftp.ensembl.org/pub/release-70/bam/homo_sapiens/genebuild/">aligned BAMs</a> (a huge convenience) but no gene expression results.  So I set out to call gene expression levels from the BAMs for my own future use, and figured I might as well put the results online while I&#8217;m at it.</p>
<p><strong>methods &amp; caveats</strong></p>
<p>I downloaded the <a href="ftp://ftp.ensembl.org/pub/release-70/bam/homo_sapiens/genebuild/">Ensembl BAMs</a>, then used <a href="http://cufflinks.cbcb.umd.edu/">Cufflinks</a> to call fragments per kilobase of exon per million reads (FPKMs), a measure of gene expression normalized to gene size and RNA-seq library size, and then used several Unix command line tools and R to combine and reshape the data.  Here is <a href="https://gist.github.com/ericminikel/7533289">the full bash script</a> I used.</p>
<p>A few notes and caveats:</p>
<ul>
<li><span style="line-height: 15px;">I only called <em>known</em> transcripts using the <a href="ftp://ftp.ensembl.org/pub/release-70/gtf/homo_sapiens/">Ensembl GRCh37 gtf file</a> and did not ask Cufflinks to discover novel transcripts.</span></li>
<li>Since I used Ensembl&#8217;s gtf file I ended up with ENST transcript IDs instead of RefSeq transcript IDs.  I couldn&#8217;t find a really good simple tool for mapping between these ID sets, so I didn&#8217;t bother and am only using the gene symbol results.</li>
<li>For genes with more than one Ensembl <code>gene_id</code> mapping to the same <code>gene_short_name</code> (i.e. gene symbol), I took the <em>sum</em> the FPKMs for the multiple <code>gene_id</code> values.</li>
<li>I discovered that Cufflinks throws a <code>Segmentation fault (core dumped)</code> <em>instantly</em> upon trying to open any of the Human BodyMap BAMs, a phenotype which was rescued by deleting the <code>@RG</code> lines from the BAMs.  Not sure why that worked.  Cufflinks could handle one <code>@RG</code> line but not the other, despite that viewing the raw characters in the single offending line with <code>od -c</code> didn&#8217;t reveal anything untoward.</li>
<li>I didn&#8217;t run Cuffcompare or Cuffdiff to test whether differences in expression between tissues were significant, and didn&#8217;t limit results to transcripts with some minimum number of FPKMs.  In other words the data are pretty raw.</li>
</ul>
<p><strong>results</strong></p>
<p><a href="http://docs.google.com/uc?export=download&#038;id=0BxObyChB_2VXUTRyczRITEVHTG8">Here is a 6 MB csv file</a> of expression by gene symbol (52686 rows) and tissue (16 columns).   You can open it with Excel, but be warned, Excel will autoformat the gene symbols, corrupting any of them that look like they could be numbers or dates.  I prefer to work with it in R.  Here&#8217;s a trivial bit of R code to get you started opening up the file and plotting the FPKMs for one gene:</p>
<div class="highlight" style="background: #f8f8f8;">
<pre style="line-height: 125%;">gene.df <span style="color: #666666;">=</span> read.table(<span style="color: #ba2121;">'gene.matrix.csv'</span>,sep<span style="color: #666666;">=</span><span style="color: #ba2121;">','</span>,header<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">TRUE</span>,stringsAsFactors<span style="color: #666666;">=</span><span style="color: #008000; font-weight: bold;">FALSE</span>) <span style="color: #408080; font-style: italic;"># load table</span>
gene.matrix <span style="color: #666666;">=</span> as.matrix(gene.df[,<span style="color: #666666;">2</span>:<span style="color: #666666;">17</span>]) <span style="color: #408080; font-style: italic;"># cast the FPKM columns to a matrix</span>
rownames(gene.matrix) <span style="color: #666666;">=</span> gene.df$X <span style="color: #408080; font-style: italic;"># reassign the gene symbols as the row names for the matrix</span>
barplot(gene.matrix[<span style="color: #ba2121;">'PRNP'</span>,],cex.names<span style="color: #666666;">=.6</span>,ylab<span style="color: #666666;">=</span><span style="color: #ba2121;">'FPKMs'</span>,main<span style="color: #666666;">=</span><span style="color: #ba2121;">'PRNP'</span>,col<span style="color: #666666;">=</span><span style="color: #ba2121;">'yellow'</span>,las<span style="color: #666666;">=3</span>) <span style="color: #408080; font-style: italic;"># cex.names shrinks tissue labels, las=3 makes them perpendicular</span></pre>
</div>
<p>Result:</p>
<p><a href="/wp-content/uploads/2013/07/human-bodymap-2.0-fpkms-prnp.png"><img class="alignnone size-full wp-image-2583" alt="human-bodymap-2.0-fpkms-prnp" src="/wp-content/uploads/2013/07/human-bodymap-2.0-fpkms-prnp.png"/></a></p>
