---
layout: post
title: "Cell Biology 03: Targeting proteins to organelles"
author: ericminikel
date: 2013-02-17 19:01:02
---
<p>These are notes from <a href="http://www.extension.harvard.edu/courses/22958">Harvard Extension&#8217;s Cell Biology course</a>.  This lecture and the next one will focus on the organelles (nucleus and mitochondria today, and the &#8216;<a href="http://en.wikipedia.org/wiki/Secretory_pathway">secretory pathway</a>&#8216; of ER and Golgi next week).  There exist stains specific to each organelle, and you can use these to see where each organelle is within the cell.  A major focus of today will be how proteins (and other molecules) move between the cytosol and organelles.</p>
<p><strong>The Nucleus</strong></p>
<p><a href="http://en.wikipedia.org/wiki/File:Diagram_human_cell_nucleus.svg"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Diagram_human_cell_nucleus.svg/462px-Diagram_human_cell_nucleus.svg.png" alt="File:Diagram human cell nucleus.svg"/></a></p>
<p>DNA replication and DNA -&gt; RNA transcription occur in the nucleus.  The nucleus holds the DNA and keeps it organized throughout the cell&#8217;s life cycle.  Its organization has some analogues to the whole cell&#8217;s organization (a membrane, skeleton(s), pores).  Here are some components of the nucleus:</p>
<ul>
<li><strong><a href="http://en.wikipedia.org/wiki/Nuclear_membrane">Nuclear envelope</a></strong>.  This is a double membrane &#8211; two lipid bilayers &#8211; surrounding the nucleus.  It is relatively impermeable &#8211; all movement in and out of the nucleus is tightly regulated.  The nuclear pores (see below) that regulate this movement must span the inner and outer nuclear memrbanes.  The outer membrane is continuous with the <a href="http://en.wikipedia.org/wiki/Endoplasmic_reticulum">ER</a> membrane.  The inner membrane has specialized surface proteins that allow attachment of lamina and matrix proteins.</li>
<li><strong><a href="http://en.wikipedia.org/wiki/Nuclear_lamina">Nuclear lamina</a></strong>.  A supportive &#8216;mesh&#8217; inside the nucleus made of fibrillar proteins.  It has roles in organizing chromatin, regulating DNA replication and cell division, and anchoring the nuclear pore complexes (see below).</li>
<li><strong><a href="http://en.wikipedia.org/wiki/Nuclear_matrix">Nuclear matrix</a></strong>.  This is another supportive fibrous network found inside the nucleus, distinct from the lamina (above).  Its function is less well understood.</li>
<li><strong>Chromosomal domains (<a href="http://en.wikipedia.org/wiki/Chromatin">chromatin</a>)</strong>.  DNA comes in the form of chromatin: condensed DNA wrapped tightly around histones.  The most actively transcribed chromosomes tend to be located towards the center of the nucleus, with less active chromosomes near the periphery.</li>
<li><strong><a href="http://en.wikipedia.org/wiki/Nuclear_pore">Nuclear pore complex</a> (NPC)</strong>.  This is a huge complex of many proteins that make up a pore spanning the inner and outer nuclear membranes.  Bottom (on the nuclear side) is called the nuclear basket, and it actually looks like a basket on electron micrographs.</li>
</ul>
<p>Let&#8217;s learn more about the NPC (image thanks to Wikimedia Commons user Mike Jones):</p>
<p><a href="http://en.wikipedia.org/wiki/File:NuclearPore_crop.svg"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/NuclearPore_crop.svg/500px-NuclearPore_crop.svg.png" alt=""/></a></p>
<p>The average vertebrate cell has 2000 NPCs.  NPCs weigh in among the cell&#8217;s largest protein complexes, at 60-80 MDa &#8211; 16x larger than a <a href="http://en.wikipedia.org/wiki/Ribosome">ribosome</a>.  It is made of 30 different proteins called nucleoporins (nups).  The central aperture is aqueous and large enough for ions, metabolites and proteins up to 40 kDa to simply pass through by passive diffusion.  Proteins too large to diffuse through passively must move through via a Brownian gate model, which is like facilitated diffusion plus gated transport dependent on ATP.</p>
<p>The NPC&#8217;s structure includes the &#8216;basket&#8217; portion on the nucleoplasm side, which is joined by filaments and attached to the lamina.  There are also cytoplasmic filaments on the outside.  Some of the nucleoporins have hydrophobic parts rich in phenylalanine (F) and glycine (G), accordingly called FG nups.  The FG nups help to regulate import and export as we&#8217;ll see shortly.  There are also structural (aka &#8216;symmetric&#8217;) nups embedded in the nuclear envelope which form a scaffold for the whole complex.  Y-complexes (16 of them per pore) form the basic structure scaffold</p>
<p><strong>Nuclear import and export</strong></p>
<p>Obviously, many proteins are needed in the nucleus &#8211; <a href="http://en.wikipedia.org/wiki/Polymerase">polymerases</a>, <a href="http://en.wikipedia.org/wiki/Helicase">helicases</a> (used in DNA replication), <a href="http://en.wikipedia.org/wiki/Histone">histones</a>, <a href="http://en.wikipedia.org/wiki/Transcription_factor">transcription factors</a>, the list goes on.  All of these are translated <em>outside</em> of the nucleus and therefore have to get back in somehow.  And of course, mRNAs and tRNAs and some ribosomal subunits are transcribed in the nucleus and must exit into the cytosol.  Still other proteins go back and forth in both directions.</p>
<p>Nuclear import and export depend on a host of different actors: <a href="http://en.wikipedia.org/wiki/Importin">importins</a> or exportins respectively; <a href="http://en.wikipedia.org/wiki/Ran_(biology)">Ran</a>; Ran helper proteins GEF and GAP; <a href="http://en.wikipedia.org/wiki/Guanosine_triphosphate">GTP</a> as an energy source; and the FG nups in the nuclear pore complex.  Let&#8217;s learn how it works.</p>
<p>The entire import and export processes are simple and elegant as long as you keep your eye on the energy and not on the horrid acronyms.  Nuclear import/export is powered by <a href="http://en.wikipedia.org/wiki/Guanosine_triphosphate">GTP</a>, the &#8220;other&#8221; energy currency of the cell.  If ATP is $1 of value on a prepaid debit card, GTP is sort of like a $1 Starbucks gift card: same nominal value, but only usable in certain transactions.  Like ATP to ADP, GTP is hydrolyzed to GDP to release energy.</p>
<p>Nuclear import relies on an <a href="http://en.wikipedia.org/wiki/Importin">importin</a> protein.  Importin floats around in the cytosol and binds to a <a href="http://en.wikipedia.org/wiki/Nuclear_localization_signal">nuclear localization signal</a> (NLSs) on a &#8216;cargo&#8217; protein to be imported.  There are a whole bunch of different importins that recognize a whole bunch of different NLSs.  NLS tend to be rich in K &amp; R, and one popular NLS motif looks like PKKKRKV.  This step of importin binding the NLS is energetically downhill &#8211; think of importin as a set mouse trap, ready to spring shut.  Importins are (usually) actually heterodimeric complexes of two different proteins &#8211; an alpha subunit and a beta subunit, with alpha being the one that binds the NLS and beta being the one that interacts with the FG nups to get through the nuclear pore.   (It is believed that beta can also act alone, though we don&#8217;t understand how.)</p>
<p>Once importin has bound to its cargo, it interacts with the FG nups in the pore in order to get through into the nucleus.  Once there it encounters <a href="http://en.wikipedia.org/wiki/Ran_(biology)">Ran</a> bound to a molecule of GTP (often called Ran-GTP).  When, and only when, Ran is &#8216;loaded&#8217; with GTP, it has an affinity for binding to importin.  This, too, is energetically downhill &#8211; like a mouse trap snapping shut.  When Ran-GTP binds to importin, it causes a conformational change that makes importin release its cargo.  Voila: the cargo has been imported.  But we&#8217;ve only engaged in downhill reactions thus far, so clearly for Ran and importin to be able to do this again, energy has to be expended somewhere.</p>
<p>Ran-GTP bound to importin has a tendency to diffuse <em></em>back<em> out</em> of the nucleus, where it encounters <a href="http://en.wikipedia.org/wiki/GTPase_activating_protein">GAP</a> (<span style="text-decoration: underline;">G</span>TPase-<span style="text-decoration: underline;">a</span>ctivating <span style="text-decoration: underline;">p</span>rotein).  Ran is a GTPase &#8211; it is capable of hydrolyzing GTP &#8211; but it doesn&#8217;t have the inclination to do so on its own; it needs GAP to stimulate it.  GAP causes Ran to &#8216;spend&#8217; its GTP, using the energy to induce the energetically uphill conformational change that releases importin.  Now importin is free in the cytosol again and ready to import another cargo molecule.  Having spent its GTP, Ran is now Ran-GDP, which moves back into the nucleus where it encounters GEF (<span style="text-decoration: underline;">g</span>uanine <a href="http://en.wikipedia.org/wiki/Nucleotide_exchange_factor">nucleotide <span style="text-decoration: underline;">e</span>xchange <span style="text-decoration: underline;">f</span>actor</a>), which removes the GDP and re-loads Ran with a new GTP.  Now Ran is ready to bind to the next importin/cargo complex that wanders in.</p>
<p>Note that this whole process keeps concentrations of importin high in the cytosol and low in the nucleus, and concentrations of Ran-GTP (as opposed to Ran-GDP) high in the nucleus and low in the cytosol.  This gradient allows passive diffusion to power the largely unidirectional travel of the import cycle.  (Although that may be a slight simplification: <a href="http://en.wikipedia.org/wiki/Nuclear_transport#Nuclear_import">Wikipedia says</a> there are some other proteins involved in the transport process).</p>
<p>Export is incredibly similar to import.  Exportin binds proteins inside the nucleus that have a <a href="http://en.wikipedia.org/wiki/Nuclear_export_signal">nuclear export signal</a> (NES) &#8211; apparently a popular sequence motif for an NES is LXXXLXXLXL.  Exportin has an affinity for Ran-GTP and forms a trimolecular complex of exportin/Ran-GTP/cargo.  The trimolecular complex has a higher affinity for FG nups than exportin alone does, thus driving export.  The complex moves through the pore and Ran-GTP encounters GAP which promotes burning GTP which provides the energy to dissociate the whole trimolecular complex.  Now the Ran-GDP wanders back into the nucleus, where GEF re-loads it with GTP making it Ran-GTP.</p>
<p>Notice that the energetics of export are identical to those of import.  Binding to cargo is downhill, association with Ran-GTP is downhill, and the dissociation from Ran-GTP is uphill, requiring the GTP to be spent.  The difference that makes importin import and exportin export is that importin releases its cargo when bound by Ran-GTP, and exportin releases its cargo when released by Ran-GTP.</p>
<p>Aside: the above explanation of exportin is only for export of proteins. mRNA export does not use Ran-GTP, instead it uses NXF1 and NXT1 and is powered by ATP using an ATPase called Dbn5.</p>
<p>This video shows a protein NFAT whose import/export is regulated by calcium concentrations:<br/>
<iframe src="http://www.youtube.com/embed/N6AF1C6GxF0" frameborder="0" width="420" height="315"></iframe><br/>
Above we talked about nuclear localization signals and nuclear export signals.  These are just two examples of signal peptides.  Proteins can also have localization signals specific to mitochondria or to the secretory pathway (ER + Golgi).  This signal is likened to a &#8216;zip code&#8217;.   Signal sequences are stretches of amino acids usually found in the beginning (i.e. N terminus) of proteins.  Signal peptides work by having chemical properties determined simply by their amino acid sequence which offer binding specificity for transport receptors such as importin.   For NLS, the signals stay as part of the protein, whereas, say, secretory pathway signals are cleaved off in post-translational modification.  (The first 22 amino acids of <a href="http://en.wikipedia.org/wiki/PRNP">PrP</a>: MANLGCWMLVLFVATWSDLGLC are a secretory pathway localization sequence which is cleaved off in post-translational modification).</p>
<p>Proteins found in <em>both</em> the nucleus and cytoplasm may need both an NLS and an NES &#8211; maybe.  The proteins are translated in the cytoplasm so depending on the relative concentrations needed, it may spend enough time in the cytoplasm before nuclear import and could meet its needs without an NES.</p>
<p><a href="http://en.wikipedia.org/wiki/Digitonin">Digitonin</a> (which, randomly, was a hit in <a href="/2012/10/06/screen-reveals-antiprion-compounds-that-act-indirectly/">UCSF&#8217;s antiprion compound screen</a>) is a detergent that permabilizes the cytoplasmic membrane but leaves the nuclear envelope intact.  This makes it possible to isolate the nucleus in the lab.  Once you&#8217;ve done that you can test to see if a protein of interest gets imported into the nucleus.  (Though, obviously, the import machinery such as importin still need to be available).</p>
<p><strong>Mitochondria</strong></p>
<p><a href="http://en.wikipedia.org/wiki/File:Mitochondrie.svg"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/3/39/Mitochondrie.svg/500px-Mitochondrie.svg.png" alt=""/></a></p>
<p>(Image thanks to Wikimedia Commons user Wikigraphists)</p>
<p>Mitochondria also have a double bilayer membrane.  The <a href="http://en.wikipedia.org/wiki/Outer_mitochondrial_membrane">outer membrane</a> (2) is a simple bilayer permeable to ions and small molecules.  The <a href="http://en.wikipedia.org/wiki/Inner_mitochondrial_membrane">inner membrane</a> (1) is totally impermeable to all molecules of any size, except through carrier channels.  The inner membrane is convoluted &#8211; it has a complex shape kind of like the surface of the cerebral cortex, which gives it extra surface area.  That extra surface area allows more capacity for the electron transport chain.  The space in between the two membranes is called the intermembrane space (the deep grooves are cristae (3)) and is where ATP is produced.  The space inside the inner membrane is the mitochondrial matrix (4).</p>
<p>Mitochondria harness energy via oxidation of glucose to pyruvate to produce ATP using the <a href="http://en.wikipedia.org/wiki/Citric_acid_cycle">citric acid cycle (Krebs cycle)</a> and <a href="http://en.wikipedia.org/wiki/Electron_transport_chain">electron transport chain</a>. This process consumes oxygen and releases carbon dioxide, thus it&#8217;s often called cellular respiration.  Here is a video of it:<br/>
<iframe src="http://www.youtube.com/embed/lRlTBRPv6xM" frameborder="0" width="420" height="315"></iframe><br/>
Inside the inner membrane, in the mitochondrial matirx, is where the mitochondrial chromosome lives, along with proteins for mitochondrial division and a few other things.  Although the mitochondria have their own ribosomes and tRNAs, they aren&#8217;t totally independent. In fact, the majority of proteins required for mitochondrial replication are encoded in the nucleus.  These nuclear-encoded proteins are translated in the cytoplasm and shipped to the mitochondria.  This requires an import/export system.</p>
<p>The signal for mitochondrial localization is part sequence, part structure.  It&#8217;s an <a href="http://en.wikipedia.org/wiki/Amphiphile">amphipathic</a> alpha helix, with R &amp; K (charged amino acids) on one side and hydrophobic amino acids on the other.  This general characteristic is more important than any specific sequence. The signal sequence is cleaved off after entry into the mitochondrial matrix &#8211; until this happens, the proteins are considered &#8216;precursor proteins&#8217;.</p>
<p>This requires membrane receptors in the mitochondria, called translocons.  There are <span style="text-decoration: underline;">t</span>ranslocons of the <span style="text-decoration: underline;">o</span>uter <span style="text-decoration: underline;">m</span>embrane (Toms) and <span style="text-decoration: underline;">t</span>ranslocons of the <span style="text-decoration: underline;">i</span>nner <span style="text-decoration: underline;">m</span>embrane (Tims).  Transport into the mitochondrial matrix occurs only in places where the inner and outer membranes are close together.  Chaperones like Hsc70 (gene: <a href="http://en.wikipedia.org/wiki/HSPA8">HSPA8</a>) keep the precursor proteins in an unfolded state &#8211; i.e. just a linear chain of amino acids &#8211; which is energetically unfavorable and requires expenditure of ATP &#8211; but makes them small enough, radius-wise, to fit through the import pores.  Hsc70 is often depicted as a Pac-man with the mouth as a generic binding pocket for polypeptides.  ATP closes the mouth, which clinches the precursor protein to keep it from folding.  Of course, it can still form some simple secondary structures &#8211; such as the signal alpha helix required for mitochondrial localization.  But Hsc70 keeps it from forming large bulky tertiary or quaternary structures.</p>
<p>Tom20/22 or 70/22 receptors in the outer membrane receive Hsc70, and Tom40 comprises the actual pore itself.  Tim 23/17 and 44 in the inner membrane.  These stand for translocon of inner/outer membrane.  These co-located at &#8216;contact sites&#8217; where the inner and outer membrane are close together, and the transport through both layers occurs simultaneously, i.e. in one swoop.  The matrix has its own Hsc70 proteins which help pull the incoming precursor protein in and keep it unfolded until import is complete, which again requires ATP.  Once import is complete, a matrix protease (&#8216;signal peptidase&#8217;) cleaves off the signal sequence and then the finished protein folds.  We watched this video in class:<br/>
<iframe src="http://www.youtube.com/embed/hTFaBS8D_w0" frameborder="0" width="420" height="315"></iframe><br/>
But I&#8217;m actually more fond of this video:</p>
<p><iframe src="http://www.youtube.com/embed/vNOmS0j662w" frameborder="0" width="420" height="315"></iframe><br/>
That description (and both of those videos) depicted the life of a protein destined to exist inside the mitochondrial matrix.  But there are three other places a protein might need to end up: embedded in the inner membrane, floating in the intermembrane space, or embedded in the outer membrane.  (For instance, the Toms themselves need to follow a path to get embedded in the outer membrane).</p>
<p>To get embedded in the inner membrane, there are three paths available.</p>
<p>Path A: Some proteins have a matrix targeting sequence <em>and </em>an internal stop-transfer sequence &#8211; a hydrophobic sequence recognized by the Tim23/17 channel &#8211; which causes Tim23/17 to move that sequence out sideways into the membrane (mechanism unknown), where it becomes a transmembrane protein lodged in the inner membrane.</p>
<p>Path B: Other proteins have a matrix targeting sequence plus a different hydrophobic domain recognized by Oxa1.  These proteins make it all the way into the matrix but then find their way <em>back</em> into the inner membrane to become transmembrane proteins.</p>
<p>Path C: Still other proteins do not have a matrix targeting sequence, yet are somehow recognized by a bunch of other Tims, and end up transmembrane.  (tl;dr: basically we know nothing about Path C).</p>
<p>There are also 2 different paths to the intermembrane space.</p>
<p>Path A requires a matrix targeting sequence and an intermembrane-space-targeting sequence; the latter is recognized by Tim23/17 which move it laterally into the membrane just like in the trans-inner-membrane Path A, but then the protein is cleaved, leaving only the part upstream of the intermembrane-space-targeting sequence floating in the intermembrane space.</p>
<p>Path B&#8217; is simple &#8211; some proteins just have a sequence which is recognized by the Toms but not the Tims.</p>
<p>We don&#8217;t know very much about the path to becoming an outer membrane transmembrane protein such as Tom40 itself, and its interacting proteins.  We know that proteins bound for the outer membrane interact with Tom40 which transfers them to the SAM (sorting and assembling machinery) complex comprised of 3 proteins which (by an unknown mechanism) insert them into the membrane.  The outermembrane proteins are usually quite hydrophobic, which probably drives the transfer from Tom40 into the membrane, but the exact mechanism is not known.</p>
<p><strong>Nuclear Pore Complex Deterioriation</strong></p>
<p>In section we discussed <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2805151/">D&#8217;Angelo 2009</a>.  This paper showed that the scaffold proteins &#8211; a crucial part of the nuclear pore complex discussed above &#8211; are produced only when cells are dividing and then have to last the entire lifetime of a cell &#8211; indeed, the entire lifetime of an organism, in post-mitotic cells.  So in your neurons, these critical components of the nuclear pore complex never get replaced or refreshed as long as you live.  They&#8217;re pretty durable, but they do undergo oxidative damage over time, and eventually this leads to the deterioration of the whole pore complex, to the point where the pore loses its selectivity and all sorts of large molecules in the cytosol can just diffuse into the nucleus.</p>
<p>This might have some pretty bad implications.  Cytosolic proteins getting into the nucleus through damaged pores might then contribute to more oxidative damage, creating a sort of vicious cycle of aging.  The authors speculate that evolution simply never figured out a way to disassemble and replace the pore complexes &#8211; or just replace their scaffold constituents &#8211; without opening up the nuclear envelope.</p>
<p>I finally got straight on one distinction of methods in molecular biology.  When people put <a href="http://en.wikipedia.org/wiki/Green_fluorescent_protein">GFP</a> under the promoter of a gene of interest, that&#8217;s a proxy for whether the gene of interest gets <em>transcribed</em>.  When people put the original gene + GFP (a GFP fusion protein) under that promoter, they&#8217;re checking whether the gene of interest gets <em>translated</em>.</p>
<p>There are some crazy methods in this paper too &#8211; new to me at least.  They use RNAi against RNAi.  They put GFP under the promoter of their gene of interest and then used RNAi against GFP until the GFP was gone to wipe out the existing GFP.  Then to see if GFP was still being actively expressed they had to stop RNAi, which they did by using RNAi against Dicer, a protein required for RNAi.</p>
<p><strong>Huntingtin</strong></p>
<p>Huntingtin doesn&#8217;t have a nuclear localization signal, but it does have an evolutionarily conserved nuclear export signal [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12783847">Xia 2003</a> (<a href="http://hmg.oxfordjournals.org/content/12/12/1393.full.pdf">ft</a>)].  That&#8217;s kind of peculiar, right?  It&#8217;s translated in the cytosol, so why would it need an export signal if it has no way of getting into the nucleus in the first place?  Ah, but somehow polyQ proteins <em>do </em>have a way of getting into the nucleus, especially when the polyQ tract is extra long [see for instance <a href="http://www.ncbi.nlm.nih.gov/pubmed/10699173">Wheeler 2000</a> (<a href="http://hmg.oxfordjournals.org/content/9/4/503.full">ft</a>)].  We don&#8217;t understand it well, and it&#8217;s a big subject of research because mutant huntingtin fragments are often found in the nucleus and some people think this nuclear localization might be required for Huntington&#8217;s Disease to cause its toxicity.</p>
<p><strong>PrP</strong></p>
<p>PrP goes through the <a href="http://en.wikipedia.org/wiki/Secretory_pathway">secretory pathway</a> so next week&#8217;s lecture should be more relevant to it.  When I Googled &#8216;PrP nuclear localization&#8217; and &#8216;PrP mitochondrial localization&#8217; I did find a couple of reports of PrP<sup>Sc</sup> being able to get into the nucleus and maybe having a &#8216;cryptic&#8217; NLS [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12667468">Gu 2003</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9647691">Jaegly 1998</a>] or being associated with mitochondrial membrane lipid rafts [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22842913">Sorice 2012</a>] but I am not sure how well-accepted either of these ideas are.</p>
