---
layout: post
title: "Regulation of prion protein transcription"
author: ericminikel
date: 2013-05-15 02:44:21
---
<p>Recently I&#8217;ve become interested in <a title="The case for depleting PrP as a therapeutic strategy against prion diseases" href="/2013/04/08/the-case-for-depleting-prp-as-a-therapeutic-strategy-against-prion-diseases/">depleting PrP</a> as a strategy for treating prion diseases. There are several different places to try to intervene in PrP&#8217;s life cycle, the earliest of which is <a href="http://en.wikipedia.org/wiki/Transcription_(genetics)">transcription</a>.  Therefore this post aims to summarize what is known about how PrP&#8217;s transcription is regulated, and to assess whether there are opportunities for therapeutic intervention to disrupt transcription of PrP.</p>
<p><strong>defining the prion promoter</strong></p>
<p>The first effort to characterize the promoter region of PrP concluded that a 273bp region was necessary for regulating PrP transcription [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11368906">Mahal 2001</a>].  Mahal undertook a series of deletion experiments, transfecting HeLa cells with plasmids that expressed luciferase under various deletion mutants of the sequence near the PrP transcription start site (TSS), in order to identify the crucial region without which luciferase was not expressed.   The resulting 273bp region was defined as &#8220;-148 to +125, relative to the cap site&#8221;.  &#8221;Cap&#8221; appears to refer to the <a href="http://en.wikipedia.org/wiki/5%27_cap">5&#8242; cap</a> of the PRNP transcript, i.e. the transcription start site.  In human genome hg19 coordinates, <a href="http://www.ncbi.nlm.nih.gov/nuccore/NM_000311.3">PRNP RefSeq transcript variant 1</a> is transcribed from <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?position=chr20:4666797-4682234&amp;hgsid=335849879&amp;knownGene=pack&amp;hgFind.matches=uc002wkw.3,">chr20:4666797-4682234</a>.  (Note the RefSeq link gives length and sequence for the mRNA, while the UCSC link shows the entire pre-mRNA.)   If chr20:4666797 were indeed the  TSS to which Mahal referred, then the promoter would extend from <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg19&amp;position=chr20%3A4%2C666%2C649-4%2C666%2C922&amp;hgt.positionInput=chr20%3A4%2C666%2C797-4%2C682%2C233&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=335850283">chr20:4666649-4666922</a>, but when I get DNA for that region it doesn&#8217;t quite match Mahal&#8217;s DNA, so Mahal must have been working from a different transcription start site.  Instead, the 273bp promoter identified by Mahal appears to be located at precisely <a href="http://genome.ucsc.edu/cgi-bin/hgTracks?hgHubConnect.destUrl=..%2Fcgi-bin%2FhgTracks&amp;clade=mammal&amp;org=Human&amp;db=hg19&amp;position=chr20%3A4%2C666%2C877-4%2C667%2C149&amp;hgt.suggestTrack=knownGene&amp;Submit=submit&amp;hgsid=335850283">chr20:4,666,877-4,667,149</a>.  Here&#8217;s the sequence:</p>
<pre>&gt;hg19_dna range=chr20:4666877-4667150 5'pad=0 3'pad=0 strand=+ repeatMasking=none
CAAGCGAATCTCAACTCGTTTTTTCCGGTGACTCATTCCCGGCCCTGCTT
GGCAGCGCTGCACCCTTTAACTTAAACCTCGGCCGGCCGCCCGCCGGGGG
CACAGAGTGTGCGCCGGGCCGCGCGGCAATTGGTCCCCGCGCCGACCTCC
GCCCGCGAGCGCCGCCGCTTCCCTTCCCCGCCCCGCGTCCCTCCCCCTCG
GCCCCGCGCGTCGCCTGTCCTCCGAGCCAGTCGCTGACAGCCGCGGCGCC
GCGAGCTTCTCCTCTCCTCACGAC</pre>
<p>As far as I can tell, the term &#8216;prion promoter&#8217; is used loosely, and may refer to exactly this 273bp, or to a larger or smaller region.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/9117892">Borchelt 1996</a> pioneered the use of a much larger swath of PrP regulatory elements as a promoter to drive expression of transgenes in mouse models of other human diseases (unrelated to PrP), for instance the <a href="http://jaxmice.jax.org/strain/003627.html">N171-82Q Huntington&#8217;s Disease mice</a>.   And <a href="http://www.ncbi.nlm.nih.gov/pubmed/12079398">Asante 2002</a> reported using a yet smaller, 214bp prion promoter to drive transgenes.  Transcriptional regulation is complex: regulatory elements in DNA can exist far upstream or downstream of the transcription start site, and we don&#8217;t always understand how they play their regulatory role, so it&#8217;s unlikely that Mahal&#8217;s critical region is the <em>only</em> region that matters.</p>
<p>Mahal&#8217;s bioinformatic analysis predicted binding sites in the prion promoter for <a href="http://en.wikipedia.org/wiki/Sp1_transcription_factor">Sp1</a>, a transcription factor involved in embryonic development, along with <a href="http://en.wikipedia.org/wiki/AP-1_transcription_factor">AP-1</a> and <a href="http://en.wikipedia.org/wiki/Activating_protein_2">AP-2</a>.  But as far as I can tell, no one actually tried to determine experimentally which transcription factors bind, or how PrP transcription is regulated, until a few years later.</p>
<p><strong>stress response</strong></p>
<p>The earliest study I could find that addressed PrP transcription [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12392052">Shyu 2002</a>] found that PrP protein was upregulated in response to heat shock (cells placed at 42 C).  PrP mRNA was never measured.  Instead, a reporter construct with luciferase under the prion promoter was used to confirm transcription as the mechanism of upregulation.  A predicted binding site for <a href="http://en.wikipedia.org/wiki/HSF1">heat shock transcription factor 1</a> (HSF1) was found in the promoter, and when this binding site was deleted, the luciferase response was greatly reduced.  Shyu then looked for HSF1 binding to the prion promoter using an electrophoretic mobility shift assay (EMSA) [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc2757439/">Hellman &amp; Fried 2007</a>], and confirmed binding.</p>
<p>Years later, <a href="http://www.ncbi.nlm.nih.gov/pubmed/18757733">Steele 2008</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2533240/">ft</a>) infected HSF1 knockout mice with prions and found that they succumbed to prion disease about 20% sooner than wild-type mice.  Curiously, they did not have elevated PrP<sup>Sc</sup> production nor infectivity nor earlier onset of symptoms &#8211; only fatality was accelerated, implying perhaps a reduced ability to cope with the PrP<sup>Sc</sup>.  In any event, Steele found no difference in baseline PrP levels in the brains of the HSF1 knockout mice vs. wild-type.  That doesn&#8217;t rule out a role for HSF1 in regulating PrP under heat shock conditions, but it does suggest HSF1 is not important for PrP transcription under normal conditions <em>in vivo</em>.</p>
<p>Another study [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15936714">Wang 2005</a>] found slightly elevated PrP protein and mRNA following exposure of N2a cells to oxidative stress agent nitric oxide (NO) and the inflammatory agent lipopolysaccharide (LPS).  NO was hypothesized to act through a guanlyl cyclase → MEK → p38 MAPK pathway, and pharmacological inhibitors of these proteins reduced PrP levels. The authors were not super rigorous about proving that this pathway regulates PrP: there was no knockdown and rescue, no confirmation of DNA binding, and many of the effects were of marginal statistical significance.  Another study reported that pharmacological <a href="http://en.wikipedia.org/wiki/MEK_inhibitor">MEK inhibitors</a> abolished PrP-res, in cell culture [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16162927">Nordstrom 2005</a> (<a href="http://www.jneurosci.org/content/25/37/8451.full">ft</a>)].  In Nordstrom&#8217;s <a href="http://www.jneurosci.org/content/25/37/8451/F3.expansion.html">Fig 3</a> you can see that both PrP-res and total PrP are reduced after treatment.   But there was no deeper investigation as to whether the MEK inhibitors acted through a transcriptional mechanism.</p>
<p>Another study hypothesized that the unfolded protein response (UPR) transcription factor XBP-1 might be involved in prion pathogenesis, but found no difference in disease course nor PrP levels in XBP-1 knockout mice [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2206609/">Hetz 2008</a>].  A later study suggested that, at least in breast cancer, ER stress does induce PrP transcription via XBP-1 and BiP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23497519">Dery 2013</a> (<a href="http://breast-cancer-research.com/content/pdf/bcr3398.pdf">ft</a>)].  PrP levels are elevated in several cancers, and high PrP levels are associated with a poorer prognosis, reportedly because PrP promotes cell survival.  Dery found that ER stress-inducing agents caused an increase in XBP-1 mRNA and prion protein levels, and one experiment with siRNA against XBP-1 reduced PrP levels.  Mutation of predicted ER stress response element (ERSE) sequences in the prion promoter reduced the PrP transcriptional response.</p>
<p><strong>copper response</strong></p>
<p>It&#8217;s been known for a while now that PrP<sup>C</sup> binds Cu<sup>2+</sup> ions at its PHGGGWGQ octapeptide repeats [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9414160">Brown 1997</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9585530">Stockel 1998</a>] and that Cu<sup>2+</sup> binding causes PrP<sup>C</sup> endocytosis [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9837873">Pauly &amp; Harris 1998</a>].</p>
<p>It was later shown that exposing primary cultured neurons to copper causes an increase in PrP mRNA and protein [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16148034">Varela-Nallar 2006</a> (<a href="http://ajpcell.physiology.org/content/290/1/C271.full">ft</a>)].  This led to a natural hypothesis as to what proteins might be regulating transcription.  A class of proteins called <a href="http://en.wikipedia.org/wiki/Metallothionein">metallothioneins</a>, which manage heavy metals, are regulated by <a href="http://en.wikipedia.org/wiki/MTF1">metal transcription factor 1</a> (MTF-1) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8026472/">Heuchel 1994</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395168/pdf/emboj00060-0142.pdf">ft</a>)], which binds to a a sort of loosely defined DNA sequence motif called a metal response element (MRE) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11554446">Giedroc 2001</a>].  Varela-Nallar describes the MRE motif as &#8220;a highly conserved 7-base pair functional core motif [5′-TGC(A/G)CNC] flanked by a less conserved 5-base pair GC-rich domain&#8221;.  Varela-Nallar identified a few sites that almost matched the MRE motif, located ~2000-2500 bases upstream of the PRNP TSS, and through deletion experiments (similar to Mahal&#8217;s method) found that one of these MREs (&#8220;−2,653 TGCGtCCCCTGC&#8221;; I was unable to find this sequence near PRNP in hg19), as well as some nearby non-MRE regions, appeared to be pretty important for copper-induced PrP expression.  Varela-Nallar then looked for MTF-1 binding to that particular 12bp sequence using an electrophoretic mobility shift assay (EMSA) [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc2757439/">Hellman &amp; Fried 2007</a>].  The EMSA did not reveal any MTF-1 binding to this particular 12bp sequence; also, zinc tends to activate MTF-1 yet had no effect on PrP levels.  For both of these reasons, Varela-Nallar concluded that MTF-1 was not involved in the transcriptional regulation of PrP.</p>
<p>A more recent study has disagreed [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)].  Bellingham&#8217;s study relied on cultured <a href="http://en.wikipedia.org/wiki/Fibroblast">fibroblasts</a> from a patient with <a href="http://en.wikipedia.org/wiki/Menkes_disease">Menkes disease</a>.  Menkes disease is an X-linked disease caused by loss-of-function mutations in the gene <a href="http://en.wikipedia.org/wiki/ATP7A">ATP7A</a>, which encodes Menkes protein, a P-type ATPase which pumps copper out of the cytosol and into the secretory pathway [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15634671">Barnes 2005</a> (<a href="http://www.jbc.org/content/280/10/9640.long">ft</a>)].  Patients with Menkes disease accumulate excess intracellular copper.  Bellingham compared the original unaltered patient fibroblasts, dubbed MNK(Del), to ones transfected with a vector overexpressing ATP7A, dubbed MNK(++), as well as an empty vector control, MNK(v/o).  The comparison of these lines was striking: PrP expression appeared to be completely abolished in the MNK(++) lines &#8211; neither the protein nor mRNA were detectable <em>at all</em> (<a href="http://www.jbc.org/content/284/2/1291/F3.expansion.html">Fig 3</a>).  Bellingham hypothesized that Sp1 and MTF-1 were involved in copper-dependent regulation of PrP and sought to confirm this experimentally.  In the MNK(Del) cells, knocking down Sp1, MTF-1 or both reduced PrP levels, and transfecting the cells with additional Sp1, MTF-1 or both increased PrP levels.  The MNK(++) cells stubbornly refused to express any PrP at all under any conditions studied.</p>
<p>The most striking aspect of Bellingham&#8217;s study is the suggestion that copper is absolutely <em>necessary</em> for PrP transcription &#8211; at least in fibroblasts, under the conditions studied, etc.  If it&#8217;s true, it begs the question of what transcription factors mediate this copper dependence.  MTF-1 is well known to respond to copper [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8026472/">Heuchel 1994</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395168/pdf/emboj00060-0142.pdf">ft</a>)], and interestingly, it appears to be on call for both copper surplus and copper deficit, which it manages through transcription of different genes, at least in <em>Drosophila</em> [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15833915">Selvaraj 2005</a> (<a href="http://genesdev.cshlp.org/content/19/8/891.long">ft</a>)].  Sp1 has also been reported to respond to copper [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18483225">Song 2008</a> (<a href="http://molpharm.aspetjournals.org/content/74/3/705.full">ft</a>)].  Bellingham didn&#8217;t do an EMSA or any other experiments to confirm that either of these proteins actually binds to any location in the prion promoter.  However, the finding that overexpression and knockdown both affected PrP levels is compelling functional evidence that they&#8217;re involved in some way, and seems to outweigh the earlier claim that MTF-1 is not involved, which was based on a lack of binding to just one predicted site [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16148034">Varela-Nallar 2006</a> (<a href="http://ajpcell.physiology.org/content/290/1/C271.full">ft</a>)].</p>
<p>Another study reported a different copper-dependent transcriptional mechanism for PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>)].  Qin examined the time course of PrP mRNA and protein elevation in response to Cu<sup>2+</sup> in some detail and concluded, as these other authors have, that the PrP response to copper is transcriptional in nature.  Qin proposed a pathway whereby copper causes phosphorylation of <a href="http://en.wikipedia.org/wiki/Ataxia_telangiectasia_mutated">ATM</a> at S1981, and activated ATM then activates Sp1 through the MEK/ERK pathway, and activates p53 by phosphorylation at S15.  Qin provides some pretty rigorous evidence for this in <a href="http://www.jbc.org/content/284/7/4582/F4.expansion.html">Fig 4</a>, by using siRNAs to knock down various elements of the pathway and show that this abolishes the response in all the downstream elements. Importantly, Qin also used an EMSA to show direct binding of both Sp1 and p53 to the prion promoter. Qin proposes that the upregulation of PrP in response to copper may constitute a stress response, since copper causes the creation of reactive oxygen species.</p>
<p>Qin&#8217;s finding that the MEK/ERK pathway is involved in PrP transcription has been subsequently validated and extended [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21586567/">Cisse 2011</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190726/">ft</a>)].  Cisse found that depletion of ERK1 or dominant negative inhibition of ERK1 or MEK reduced PrP mRNA levels, and transfection of cells with ERK1 or MEK increased PrP transcription.  Cisse notes that ERK1 activation can promote gene transcription either by phosphorylation of Sp1 or by phosphorylation of c-Fos, part of the AP-1 complex.   Mutation of the predicted AP-1 binding sites also reduced PrP transcription, but mutation of the predicted Sp1 binding sites had no effect, suggesting AP-1 was responsible.  Interestingly, ERK1 also  controls PrP proteolytic cleavage by ADAM10 [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3338960/">Checler 2012</a>].</p>
<p><strong>amyloid intracellular domain</strong></p>
<p>Here&#8217;s a lightning-quick review of the metabolism of amyloid precursor protein (APP).  Beta-secretase (BACE1) cleaves it on the exoplasmic side, and gamma-secretase (including PSEN1 &amp; PSEN2) cleaves it on the cytosolic side, producing an extracellular fragment amyloid beta (Aβ) with roles in Alzheimer&#8217;s disease, and an intracellular fragment called the amyloid intracellular domain (AICD) [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879048/">Vassar 2009</a>].</p>
<p>PrP and amyloid beta interact physically, leading to downstream signaling changes that are only now beginning to be understood &#8211; see <a title="Review of prion protein and amyloid beta interactions" href="/2013/05/02/review-of-prion-protein-and-amyloid-beta-interactions/">my latest PrP/Aβ post</a>.  PrP also regulates the production of Aβ through its inhibition of BACE1 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/17573534/">Parkin 2007</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/23386614">Rushworth 2013</a>].  It turns out that APP metabolism may also regulate PrP: it has been reported that the AICD activates transcription of PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19458243/">Vincent 2009</a> (<a href="http://www.jneurosci.org/content/29/20/6752.long">ft</a>)].</p>
<p>Vincent found that knockout of PSEN1 and PSEN2 in fibroblasts reduced PrP mRNA and protein by about half.  betaAPP overexpression increased PrP, betaAPP knockout reduced PrP, and direct expression of AICD fragments C50 or C59 increased PrP.  Vincent proposes that p53 mediates the AICD&#8217;s effects on PrP.  p53 knockout reduced PrP expression (rescued by p53 cDNA transfection) and abolished the stimulatory effects of C50 and C59; <a href="http://en.wikipedia.org/wiki/Chromatin_immunoprecipitation">ChIP</a> confirmed direct binding of p53 to the prion promoter.  Vincent&#8217;s group had previously reported that AICD activates transcription of p53 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16763046">Alves da Costa 2006</a>], so this is postulated as the mechanism for the AICD → p53 → PrP connection. <em><strong>update 2013-11-04: </strong>Vicki Lewis et al failed to replicate the AICD connection &#8211; see <a href="/2013/05/15/regulation-of-prion-protein-transcription/#comment-474">comment</a>.</em></p>
<p><strong>no unbiased screens so far</strong></p>
<p>All of these studies started from a particular hypothesis about what regulates PrP transcription and then sought to prove it.  Most started from completely different hypotheses and did not explore or directly seek to confirm one another&#8217;s findings.</p>
<p>No study has yet utilized an unbiased screen for proteins binding to the prion promoter.  One unbiased method for identifying regulators of a particular promoter is to prepare a biotinylated synthetic DNA oligomer of the desired promoter, dip it into a nuclear extract, and then perform mass spectrometry on the proteins that come out bound to the DNA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10471930">Nordhoff 1999</a>].  This method has its pitfalls &#8211; it by no means captures the range of physiological conditions under which transcription might be stimulated &#8211; but at least it is unbiased and has the potential to catch things that no one has thought to hypothesize.</p>
<p>Something big and unbiased has happened since 2009, though: the <a href="http://encodeproject.org/ENCODE/">ENCODE project</a>.</p>
<p><strong>ENCODE data</strong></p>
<p>ENCODE (Encyclopedia of DNA regulatory elements) aims to create a genome-wide map of everything that regulates gene expression: histone modifications, DNA methylation, and transcription factor binding among other things.  Most of the data comes from hundreds of genome-wide <a href="http://en.wikipedia.org/wiki/ChIP-sequencing">ChIP-seq</a> experiments.  DNA-binding proteins are cross-linked to DNA with <a href="http://en.wikipedia.org/wiki/Formaldehyde">formaldehyde</a>, one protein of interest is pulled down with antibodies, de-cross-linked, and then the DNA that had been attached to it is sequenced.  This gives you a map of where in the genome the protein had been bound.</p>
<p>Though ENCODE is unbiased in that it is genome-wide, it is limited to the transcription factors that were chosen to be studied.  The full <a href="http://encodeproject.org/ENCODE/dataSummary.html">list of ChIP-seq experiments</a> includes Sp1 but neither p53 nor MTF-1.  ENCODE currently has transcription factor binding data from 690 experiments on 161 transcription factors, but finding the data for all of them is confusing.  If you simply turn on the ENCODE Txn factor track in the UCSC genome browser, you&#8217;ll see only 26 of the 690.  If you instead visit the <a href="http://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=335850283&amp;g=wgEncodeAwgTfbsUniform">Uniform TFBS Track page</a> you&#8217;ll be confronted with the entire list of 690 and no way to display data from all of them except by checking each and every box.</p>
<p>The easiest way I&#8217;ve found of getting the data for all 161 transcription factors is through the <a href="http://genome.ucsc.edu/cgi-bin/hgTables">UCSC table browser</a>, settings as shown below. I used chr20:4665648-4668377, so ~2700bp, a 10x zoom-out from Mahal&#8217;s promoter region.</p>
<p><img class="alignnone size-large wp-image-2183" title="" alt="" src="/wp-content/uploads/2013/05/encodeTfbsPrionPromoter-ucsctablebrowser-1024x492.png"/></p>
<p>Results are here: <strong>[<a href="/wp-content/uploads/2013/05/encodeTfbsPrionPromoter.txt">TXT</a>]</strong>.  The table contains 48 entries, apparently only the transcription factors with scores &gt; 0.  Three transcription factors got the maximum possible score of 1000: <a href="http://en.wikipedia.org/wiki/JunD">JunD</a>, <a href="http://en.wikipedia.org/wiki/FOSL2">FOSL2</a> and <a href="http://en.wikipedia.org/wiki/C-Fos">c-Fos</a>.  All three of these are components of the <a href="http://en.wikipedia.org/wiki/AP-1_transcription_factor">AP-1 transcription factor complex</a>, thus confirming Mahal&#8217;s original bioinformatic prediction of AP-1 binding [<a href="http://www.ncbi.nlm.nih.gov/pubmed/11368906">Mahal 2001</a>].  The <code>thickStart</code> and <code>thickEnd</code> columns tell you where exactly the peaks are; FOSL2 and c-Fos both overlap Mahal&#8217;s 273bp promoter region of chr20:4666877-4667149.  Besides the AP-1 proteins, <a href="http://en.wikipedia.org/wiki/STAT3">STAT3</a> also comes close to the maximum, with a score of 950.  Sp1 is not on the list, implying it had a score of 0.  Some <a href="http://en.wikipedia.org/wiki/Activating_protein_2">AP-2</a> transcription factors are present lower on the list.</p>
<p><strong>summary</strong></p>
<p>This table attempts to summarize the findings from the above studies.</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>Tx factor</th>
<th>functional evidence</th>
<th>physical evidence</th>
<th>proposed regulatory pathways</th>
</tr>
</thead>
<tbody>
<tr><td>p53</td>
<td>Knockdown [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>),<a href="http://www.ncbi.nlm.nih.gov/pubmed/19458243/">Vincent 2009</a> (<a href="http://www.jneurosci.org/content/29/20/6752.long">ft</a>)], overexpression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>)]</td>
<td>EMSA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>)], ChIP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19458243/">Vincent 2009</a> (<a href="http://www.jneurosci.org/content/29/20/6752.long">ft</a>)]</td>
<td>PSEN1&amp;2+APP→AICD [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19458243/">Vincent 2009</a> (<a href="http://www.jneurosci.org/content/29/20/6752.long">ft</a>)], copper-induced via ATM→MEK→ERK pathway [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin</a>]</td>
</tr>

<tr><td>Sp1</td>
<td>Knockdown [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)], overexpression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)]</td>
<td>EMSA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>)]</td>
<td>Copper-induced via ATM→MEK→ERK [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>)] or unknown mechanism [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)]</td>
</tr>

<tr><td>AP-1</td>
<td></td>
<td>Mutation of predicted binding sites [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21586567/">Cisse 2011</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190726/">ft</a>)], ChIP-seq [<a href="/wp-content/uploads/2013/05/encodeTfbsPrionPromoter.txt">ENCODE</a>]</td>
<td>Activated by ERK1 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21586567/">Cisse 2011</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190726/">ft</a>)]</td>
</tr>

<tr><td>MTF-1</td>
<td>Knockdown, overexpression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)]</td>
<td></td>
<td>Copper-induced [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)]</td>
</tr>

<tr><td>XBP-1</td>
<td>Knockdown, overexpression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23497519">Dery 2013</a>(<a href="http://breast-cancer-research.com/content/pdf/bcr3398.pdf">ft</a>)]</td>
<td>Mutation of predicted binding sites [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23497519">Dery 2013</a>(<a href="http://breast-cancer-research.com/content/pdf/bcr3398.pdf">ft</a>)]</td>
<td>ER stress [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23497519">Dery 2013</a>(<a href="http://breast-cancer-research.com/content/pdf/bcr3398.pdf">ft</a>)]</td>
</tr>

<tr><td>HSF1</td>
<td>Luciferase reporter [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12392052">Shyu 2002</a>]</td>
<td>Deletion of predicted binding site, EMSA [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12392052">Shyu 2002</a>]</td>
<td>Heat shock response [<a href="http://www.ncbi.nlm.nih.gov/pubmed/12392052">Shyu 2002</a>]</td>
</tr>
</tbody></table></p>
<p>If we very generously (i.e. unskeptically) believe everything that&#8217;s been reported, the picture of prion protein transcription that has been developed so far looks as follows:</p>
<p><img alt="" src="https://docs.google.com/drawings/d/1i0hPgB-KRAS8ztKdC92LyJZqdOnggf03-wpsbR9WRis/pub?w=480&amp;h=360"/></p>
<p>If we&#8217;re more skeptical and only consider those things which (A) are supported by more than one study and (B) for which both physical and functional evidence are available, the picture thins considerably:</p>
<p><img alt="" src="https://docs.google.com/drawings/d/1UOlsIDBAphL_TbqRLRFg35zN9wwIeYNXanJX1-mTd1A/pub?w=480&amp;h=360"/></p>
<p>In either case, what we have at this point is certainly a very incomplete picture of how PrP transcription is regulated.  Most of these studies focused on transcriptional upregulation in response to a particular environmental stress, rather than characterizing what drives baseline transcription levels.  Only recently [<a href="http://www.ncbi.nlm.nih.gov/pubmed/21586567/">Cisse 2011</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3190726/">ft</a>)] was AP-1 activity implicated in the literature, even though according to ENCODE it is the most active transcription factor in the prion promoter.</p>
<p>The most important piece that is missing is what drives neuronal transcription of PrP.  PrP  mRNA is expressed ~10 times higher in the CNS than in most peripheral tissues [<a href="http://biogps.org/#goto=genereport&amp;id=5621">Novartis BioGPS microarray data</a>], implying there must be some neuronal-specific transcription factors, upstream regulatory mechanisms or epigenetic marks at work.  Of the above-cited studies, just one used primary cultured neurons [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16148034">Varela-Nallar 2006</a> (<a href="http://ajpcell.physiology.org/content/290/1/C271.full">ft</a>)] and two used CNS-derived cancer cell lines (N2a and NT-2) [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19064990">Qin 2009</a> (<a href="http://www.jbc.org/content/284/7/4582.long">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/12392052">Shyu 2002</a> respectively].  The rest used fibroblasts or peripheral cancer cell lines.  Browsing the <a href="http://encodeproject.org/ENCODE/dataSummary.html">ENCODE human cell experiment list</a>, I also don&#8217;t see any CNS cell types there.</p>
<p><strong>assessment of therapeutic potential</strong></p>
<p>It&#8217;s unclear if any of these pathways lend themselves to therapeutic intervention.  The most potentially interesting finding is that cytosolic copper depletion by overexpression of <a href="http://en.wikipedia.org/wiki/ATP7A">ATP7A</a> reduces PrP expression to undetectable levels [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18990686">Bellingham 2009</a> (<a href="http://www.jbc.org/content/284/2/1291.long">ft</a>)].  If this result (originally from fibroblasts) could be replicated in neurons, and could then be associated with the inactivation of one particular transcription factor or one particular protein upstream thereof, that could potentially constitute a drug target.  If Bellingham is right that MTF-1 is involved in some way, it is interesting to note that MTF-1 knockout is embryonic lethal, perhaps due to liver decay [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9582278/">Gunes 1998</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1170625/pdf/002846.pdf">ft</a>)] but that Cre-mediated adult knockout in the liver is survivable [<a href="http://www.ncbi.nlm.nih.gov/pubmed/15226267">Wang 2004</a> (<a href="http://www.fasebj.org/content/18/10/1071.full">ft</a>)].  As far as I can tell from the literature no one has done a Cre knockout of MTF-1 in neurons, and no one has checked whether PrP is expressed in MTF-1 knockout cells.</p>
<p>Transcription factors are not the ideal drug targets &#8211; they tend to promote hundreds of genes, so inhibiting them to turn off one substrate is truly razing a village to catch one man.   But blocking a transcription factor is not entirely unheard of &#8211; for example <a href="http://en.wikipedia.org/wiki/Disulfiram">disulfiram</a> inhibits NF-KB [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1314883">Schreck 1992</a> (<a href="http://jem.rupress.org/content/175/5/1181.full.pdf">ft</a>)] and has been explored for treatment of HIV and cancer, though its relevant mechanism of action in those diseases may be something different [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16206267">Lovborg 2006</a> (<a href="http://onlinelibrary.wiley.com/doi/10.1002/ijc.21534/pdf">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/22739395">Doyon 2013</a>].  Also, disrupting a particular DNA binding event or a recruitment interaction between two transcription factors can be more specific than just inhibiting a transcription factor altogether.</p>
