---
layout: post
title: "Allelic origin of PrPSc in heterozygotes"
author: ericminikel
date: 2013-08-15 09:31:27
---
<p><strong>introduction</strong></p>
<p>PrP molecules that differ in amino acid sequence have a hard time converting one another to prions, both <em>in vivo</em> [brief overview in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a>] and in cell-free conversion [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7732006/">Kocisko 1995</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC42074/pdf/pnas01493-0299.pdf">ft</a>)].</p>
<p>As a result, heterozygosity can be protective in prion disease, with polymorphisms at codons 129 [reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3226040/">Lukic &amp; Mead 2011</a>], 219 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9629853">Shibuya 1998</a>] and 127 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/19923577">Mead 2009</a> (<a href="http://www.nejm.org/doi/full/10.1056/NEJMoa0809716#t=article">ft</a>)] conferring some degree of resistance to prion disease.  For E219K and G127V it&#8217;s not clear the world has seen enough (or any?) homozygotes to know whether homozygous 219KK or 127VV individuals would be resistant, and therefore to know whether resistance is associated with heterozygosity per se or simply with the K and V alleles respectively.  Certainly for codon 129 in sporadic CJD, we know that 129MV individuals are at lower risk of sporadic CJD than either MM or VV, so heterozygosity is what matters.  In contrast, for variant CJD, neither MV nor VV individuals have gotten sick, suggesting that the V allele is what matters.  (In fact, there has been one subclinical case of a peripherally infected, subclinical 129MV vCJD patient, but none of VV [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3613713/">Bishop 2013</a>].)</p>
<p>Other polymorphisms are similarly protective in animal populations [e.g. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10211978">Tranulis 1999</a> (<a href="http://vir.sgmjournals.org/content/80/4/1073.full.pdf">ft</a>), <a href="http://www.ncbi.nlm.nih.gov/pubmed/22460693">Robinson 2012a</a> (<a href="http://naturalsciencecitizen.files.wordpress.com/2011/05/robinson-prnp-prion-review-reprint.pdf">ft</a>)].</p>
<p>Some of these phenomena could theoretically be explained away as matters of gene dosage of the prion conversion-competent allele.  For instance, in vCJD, a 129MV individual has only half as many conversion-competent 129M PrP molecules as a 129MM individual, and we know that hemizygosity more than doubles incubation time [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8790598">Bueler 1994</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2229922/pdf/molmed00043-0025.pdf">ft</a>)].  However, results from transgenic animal studies have established that non-identical PrP molecules are not just absent from prion conversion, they are actively dominant negative against it [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7553876/">Telling 1995</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC23307/">Kaneko 1997</a>, reviewed in <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC33918/">Prusiner 1998</a>] &#8211; a subject for a future post.</p>
<p>All of this raises the question of, in heterozygous individuals (I&#8217;ll focus on humans here), which allele(s) get converted to PrP<sup>Sc</sup>?  This question is of interest in modeling <a title="Facts about prion kinetics" href="/2013/06/08/facts-about-prion-kinetics/">prion kinetics</a>, understanding the nature of dominant negativity, and considering whether allele-specific gene silencing would be desirable if it were feasible.</p>
<p><strong>methods for determining allelic origin</strong></p>
<p>It&#8217;s not trivial to figure out which allele of a protein you&#8217;re looking at, which is probably why not that many studies have done it.  In these papers I saw three methods: size, immunoreactivity, and peptidase cleavage.</p>
<p>For the repeat insertion CJD case in [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a>], the size difference between the mutant and wild-type allele was sufficient to distinguish full-length PrPSc, though this still won&#8217;t distinguish <a title="Proteolysis of prion protein in the central region: alpha and beta cleavage" href="/2013/06/26/proteolysis-of-prion-protein-in-the-central-region-alpha-and-beta-cleavage/">beta-cleaved</a> or protease-digested PrPSc since then the octarepeat region is removed.  The size trick also worked for two of Chen&#8217;s FFI cases, who happened to have a 1 octarepeat deletion &#8211; a nonpathogenic polymorphism [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8100163">Palmer 1993</a>] &#8211; in <em>cis</em> with the FFI mutation.  After cleaving the protein with CNBr peptidase to get smaller peptides, <a href="http://www.ncbi.nlm.nih.gov/pubmed/9142120">Silvestrini 1997</a> used <a href="http://en.wikipedia.org/wiki/High-performance_liquid_chromatography">high performance liquid chromatography</a> (HPLC) as well as <a href="http://en.wikipedia.org/wiki/Fast_atom_bombardment">fast atom bombardment mass spectrometry</a> (FAB-MS) and these methods were able to detect the mass difference between 210V and 210I alleles apparently based solely on the different molecular mass of valine (V) and isoleucine (I).</p>
<p>Because Y145X truncates PrP, antibodies against the C terminus of PrP are specific for the wild-type allele [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC40125/pdf/pnas01506-0209.pdf">ft</a>)].  ICSM-35 binds only to 102P and not 102L PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/16597650">Wadsworth 2006</a> (<a href="http://brain.oxfordjournals.org/content/129/6/1557.long">ft</a>)].  To study E200K, <a href="http://www.ncbi.nlm.nih.gov/pubmed/8564843">Gabizon 1996</a> specifically raised antisera against peptides containing either 200E and 200K and managed to get antisera that were specific to each.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/10889337">Muramoto 2000</a> similarly raised antisera specific to E219K, which was found in cis with the P102L mutation.  As Gabizon points out, this method is qualitative and not quantitative.  It is possible to tell <em>whether</em> wtPrP is present in, say, a protease-resistant or detergent-insoluble fraction, but not possible to quantify it because the different antisera used for wt vs. mutant PrP may differ in the amount of PrP they&#8217;re able to pull down.</p>
<p>Finally, peptidases whose recognition site is either created or abolished by the disease mutation were used.  CNBr cleaves MX peptide bonds, so it will cleave only at 129M and not at 129V [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9142120">Silvestrini 1997</a>], which worked for one sporadic CJD MV patient.  <a href="https://www.neb.com/products/p8104-endoproteinase-aspn">Asp-N</a> cleaves XD peptide bonds, so it cleaves only 178D and not 178N [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a>].  Lys-C was used for recognizing P102L [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC20974/">Parchi 1998</a>].  In each case, after differential cleavage the fragments are subject to size separation, for instance by mass spectrometry.</p>
<p><strong>reported results in genetic prion diseases</strong></p>
<p>Note: OPRI = octapeptide repeat insertion, OPRD = octapeptide repeat deletion</p>
<p><table style="width:100%; " class="easy-table easy-table-default " border="0">
<thead>
<tr><th>citation</th>
<th>mutant allele</th>
<th>wild-type allele</th>
<th>n</th>
<th>does wild-type allele convert?</th>
</tr>
</thead>
<tbody>
<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/8564843">Gabizon 1996</a></td>
<td>E200K (129 genotype not stated)</td>
<td>129 genotype not stated</td>
<td>3</td>
<td>Maybe. wtPrP was "mainly" detergent-insoluble but no protease-resistant wtPrP was found.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a> (<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC40125/pdf/pnas01506-0209.pdf">ft</a>)</td>
<td>Y145X 129M</td>
<td>129M</td>
<td>1</td>
<td>Yes. wtPrP was present in amyloid plaques</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129M 1-OPRD</td>
<td>129M</td>
<td>1</td>
<td>No. No wtPrP was found in detergent-insoluble nor protease-resistant fractions.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129M 1-OPRD</td>
<td>129V</td>
<td>1</td>
<td>No. No wtPrP was found in detergent-insoluble nor protease-resistant fractions.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129M</td>
<td>129M</td>
<td>1</td>
<td>No. No wtPrP was found in protease-resistant fraction. (Detergent insolubility not tested).</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129M</td>
<td>129V</td>
<td>1</td>
<td>No. No wtPrP was found in protease-resistant fraction. (Detergent insolubility not tested).</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129V</td>
<td>129M</td>
<td>1</td>
<td>No. No wtPrP was found in protease-resistant fraction. (Detergent insolubility not tested).</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>D178N 129V</td>
<td>129V</td>
<td>1</td>
<td>No. No wtPrP was found in protease-resistant fraction. (Detergent insolubility not tested).</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>5-OPRI 129M</td>
<td>129M</td>
<td>1</td>
<td>Yes.  54% of wtPrP and 86% of mutant PrP were detergent insouble.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a></td>
<td>5-OPRI 129M</td>
<td>129V</td>
<td>1</td>
<td>Yes.  57% of wtPrP and 94% of mutant PrP were detergent insouble.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9142120">Silvestrini 1997</a></td>
<td>V210I 129M</td>
<td>129V</td>
<td>1</td>
<td>Yes. wtPrP was found in protease-resistant fraction.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/9142120">Silvestrini 1997</a></td>
<td>V210I 129M</td>
<td>129M</td>
<td>2</td>
<td>Yes. wtPrP was found in protease-resistant fraction.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC20974/">Parchi 1998</a></td>
<td>P102L 129M</td>
<td>129M</td>
<td>7</td>
<td>No. wtPrP not found in protease-resistant fraction.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/10889337">Muramoto 2000</a></td>
<td>P102L 219K*</td>
<td>219E</td>
<td>1</td>
<td>No. wtPrP not found in detergent-insoluble fraction.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597650">Wadsworth 2006</a> (<a href="http://brain.oxfordjournals.org/content/129/6/1557.long">ft</a>)</td>
<td>P102L 129M</td>
<td>129M</td>
<td>3</td>
<td>Yes. wtPrP comprised 1 to 10% of PrP in the protease-resistant 21-30 kDa fragments, depending on the patient.  The 8 kDa fragment particular to GSS was comprised exclusively of the mutant allele.  wtPrP also comprised ~10% of detergent-insoluble PrP.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597650">Wadsworth 2006</a> (<a href="http://brain.oxfordjournals.org/content/129/6/1557.long">ft</a>)</td>
<td>P102L, 129 unknown</td>
<td>unknown</td>
<td>2</td>
<td>It's complicated.  IHC did not reveal any wtPrP in amyloid plaques, but the blots performed for the 3 patients above were not done for these patients.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pubmed/16597650">Wadsworth 2006</a> (<a href="http://brain.oxfordjournals.org/content/129/6/1557.long">ft</a>)</td>
<td>P102L 129M</td>
<td>129V</td>
<td>1</td>
<td>Not clear. This patient is listed in <a href="http://brain.oxfordjournals.org/content/129/6/1557/T1.expansion.html">Table 1</a> but it is not clear which, if any, experiments included them.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285667/">Monaco 2012</a></td>
<td>P102L 129M</td>
<td>129M</td>
<td>3**</td>
<td>Yes.  wtPrP accounted for 5-10% of protease-resistant PrP depending on the patient.  It also accounted for some sPrPSc precipiated in a "cold PK" digestion plus NaPTA precipitation protocol, though this is never quantified.</td>
</tr>

<tr><td><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3285667/">Monaco 2012</a></td>
<td>P102L 129M</td>
<td>129V</td>
<td>3</td>
<td>Yes.  Same as above.</td>
</tr>
</tbody></table><br/>
<em>*129 genotype not stated but presumably 129MM since the V allele has only ~2% allele frequency in East Asia [<a href="http://browser.1000genomes.org/Homo_sapiens/Variation/Population?db=core;r=20:4679751-4680751;v=rs1799990;vdb=variation;vf=1365834">1000 Genomes</a>].<br/>
</em><em>**Monaco also found no protease-resistant PrP of <span style="text-decoration: underline;">either</span> allele in two patients, both of whom were 129MM.</em></p>
<p>Other interesting tidbits.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a> found that only 17.1 ± 2.6% (n=3) of FFI mutant PrP is detergent-insoluble.  Contrast this with the octarepeat cases above where most (~90%) of the mutant PrP was insoluble.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/16597650">Wadsworth 2006</a> (<a href="http://brain.oxfordjournals.org/content/129/6/1557.long">ft</a>) found that wtPrP (measured by ICSM-35) was 30-40% of total PrP (measured by 3F4) prior to protease digestion.</p>
<p><strong>reported results in sporadic prion disease</strong></p>
<p>There must surely be other studies that have examined this, but I could find only a single case.  <a href="http://www.ncbi.nlm.nih.gov/pubmed/9142120">Silvestrini 1997</a> found that in a single sporadic CJD MV case (type 1 vs. 2 not stated), both alleles contributed to the protease-resistant fraction.</p>
<p><strong>discussion</strong></p>
<p>The amount of wild-type PrP that converts to PrP<sup>Sc</sup> in genetic prion diseases appears to vary by mutation.   D178N was reported to have no involvement of the wild-type allele. E200K was reported to have some wild-type PrP that became detergent insoluble but none that became protease-resistant.  P102L has a little bit of wild-type involvement but it&#8217;s only ~10% of total PrP<sup>Sc</sup>.  Octapeptide repeat insertion patients had almost as much wild-type allele as mutant allele.  V210I and Y145X were both qualitatively reported to have wild-type PrP<sup>Sc</sup> but it was not quantified.</p>
<p>The fact that sporadic MV CJD PrP<sup>Sc</sup> was comprised of both alleles is consistent with the fact that both MV1 and MV2 disease were transmissible to mice expressing either 129MM HuPrP or 129VV HuPrP [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900653/">Bishop 2010</a>].</p>
<p>In contrast, it is surprising that no wild-type PrP<sup>Sc</sup> was detected in FFI [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9288728">Chen 1997</a>], given that FFI is transmissible to mice expressing wild-type chimeric PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8953038">Telling 1996</a>].  (The MHu2M chimera has 9 human amino acids and the rest mouse [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7937921">Telling 1994</a> (<a href="http://www.pnas.org/content/91/21/9936.full.pdf">ft</a>)], but no FFI mutation).  Is it possible that in heterozygous humans, the wild-type allele just replicates so much more slowly than the mutant allele that it doesn&#8217;t reach detectable levels before terminal illness?</p>
