---
layout: post
title: "How to, and how not to, knock out PRNP"
author: ericminikel
date: 2014-03-09 07:35:22
---
<p>About a year ago, I sat down to get my <a href="/2013/01/19/talens-and-zfns/">facts</a> straight on zinc finger nucleases (ZFNs) and TAL effector nucleases (TALENs), two genome-editing technologies.  But even as I wrote those words, a newer technology called CRISPR/Cas9 was beginning to make waves.  In the past few months, CRISPR has become <a href="http://www.nytimes.com/2014/03/04/health/a-powerful-new-way-to-edit-dna.html?_r=0">big news</a>, especially after <a href="http://editasmedicine.com/">Editas Medicine</a> here in Cambridge <a href="http://www.nature.com/news/crispr-technology-leaps-from-lab-to-industry-1.14299">raised $43M</a> to develop human therapeutics based on CRISPR genome editing.</p>
<p>Perhaps CRISPR&#8217;s biggest advantage over ZFNs and TALENs is its size: it uses RNA, rather than peptides, to recognize its target site in the genome.  The smaller size of the DNA required to code for the RNA plus the Cas9 nuclease makes it easier to, say, clone into your viral vector of choice.  It also appears to be highly efficient: for example, the landmark paper on editing human cell lines with CRISPR/Cas9 achieved non-homologous end-joining in 2-4% of clones in iPS and as many as 25% of clones in other cell lines, and successful homologous recombination in 3-8% of clones [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722">Mali 2013</a>].  My, how efficiency has improved since the bad old days: when the first PrP knockout mouse was created using plain homologous recombination (with no way to create double strand breaks to speed the process up), only 1 in 5000 clones had the knockout allele, even <em>after</em> selecting for <a href="http://en.wikipedia.org/wiki/G418" title="an antibiotic used to select genetically engineered cells">G418</a> resistance [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1373228">Bueler 1992</a>]. With CRISPR you don&#8217;t even need drug resistance cassettes.</p>
<p>To be sure, CRISPR/Cas9 does not, in and of itself, solve the problem of safe and effective delivery for use <em>in vivo</em>.  Getting it into the brain, and to a lesser extent other tissues of interest, will continue to be a challenge.  And even in a relatively low-immunogenic viral vector like AAV9, there is still the question of whether people&#8217;s immune systems will recognize the Cas9 nuclease as the non-self protein that it is.  These sorts of issues are the reason why the first clinical trial of genome editing with ZFNs used an <em>ex vivo</em> approach, deleting CCR5 in people&#8217;s T cells and then transplanting them back in [<a href="http://www.nejm.org/doi/full/10.1056/NEJMoa1300662">Tebas 2014</a>].</p>
<p>Still, the potential of CRISPR/Cas9 is enough to raise the question of what could be done with it therapeutically <em>if</em> we could get it into the brain. When people talk about &#8220;genome editing&#8221;, non-scientists often imagine this means correcting a missense mutation in someone&#8217;s genome.  But even with directed double-strand breaks, non-homologous end joining occurs at a much higher rate than homologous recombination, so a knock-in therapeutic strategy <em>in vivo</em> doesn&#8217;t seem very likely to me (though perhaps it could be done <em>ex vivo</em> by selecting cells before transplanting them back in).  A much easier target would be a gene where the goal is deletion, not correction.</p>
<p>In that sense, prion diseases are an ideal target for CRISPR/Cas9: a class of rapidly lethal gain-of-function disorders that depend upon expression of a single gene whose knockout appears well-tolerated.  As a way to deplete PrP and slow or stop prion diseases, knocking out PRNP with CRISPR/Cas9 would have several advantages over RNAi or ASOs.  It would outright eliminate, not merely reduce, PrP production in cells where you manage to hit both alleles, and sustained therapeutic benefit wouldn&#8217;t depend on sustained expression of the viral vector.  Even transient expression of the CRISPR/Cas9 construct would be enough to permanently knock out the gene, so that if the viral vector were later silenced the (beneficial) damage would already be done.</p>
<p>This got me thinking, if we could use a viral vector to deliver CRISPR/Cas9 to delete PRNP in the brain, where exactly would we want to cut?</p>
<p>PRNP is an exceptionally pathogenic gene: not only do missense mutations cause prion disease, but even the wild-type sequence can cause spontaneous disease if the protein is overexpressed [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8287472">Westaway 1994</a>], expressed ectopically under a foreign promoter [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9770528/">Tremblay 1998</a>], secreted instead of GPI-anchored [<a href="http://www.ncbi.nlm.nih.gov/pubmed/22160704">Stohr 2011</a>], or interstitially deleted [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1783448/">Li 2007</a>].  So when you cut, cut carefully.</p>
<p>There exist several lines of PrP knockout mice, so I looked back to the original papers to see how people targeted <em>Prnp</em> in the mouse embryo 20 years ago.  Mouse <em>Prnp</em> has three exons, and the entire coding sequence is in exon 3, as shown below.  </p>
<p><div id="attachment_3716" style="width: 610px" class="wp-caption aligncenter"><a href="/wp-content/uploads/2014/03/mouse-prnp-knockout-lines.png"><img src="/wp-content/uploads/2014/03/mouse-prnp-knockout-lines.png">Weissmann &#038; Flechsig 2003</a>.</p></div></p>
<p>The first knockout was the Zurich I line [<a href="http://www.ncbi.nlm.nih.gov/pubmed/1373228">Bueler 1992</a>], which has a stop codon, neomycin resistance cassette, and then two more stop codons, inserted into <em>Prnp</em>&#8216;s only coding exon in place of codons 4-176.  These mice do express PrP mRNA, it just doesn&#8217;t code for a viable version of PrP anymore.  These mice were quickly adopted by the Prusiner lab [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7902565/">Prusiner 1993</a>] and are the most widely used PrP knockout line today.</p>
<p>Next came the Edinburgh line [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7999308/">Manson 1994</a>], which inserts a neo cassette right at the beginning of exon 3 after the splice acceptor site.  The creators found that the knockout allele led to a complete loss of PrP mRNA expression.</p>
<p>Third came the Nagasaki line [<a href="http://www.ncbi.nlm.nih.gov/pubmed/7494265">Sakaguchi 1995</a>], and with it a lesson for the ages [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8606772">Sakaguchi 1996</a>]. I told a bit of this story in the <a href="/2013/09/01/pi-prps-hypothetical-backup-protein/">pi</a> post, but it merits a quick re-telling here.  Whereas the Zurich and Edinburgh lines had both been reported as developmentally normal, the Nagasaki line had a late (~70 weeks of age) onset ataxia caused by neurodegeneration in the Purkinje cells of the cerebellum.  </p>
<p>When the Nagasaki line&#8217;s phenotype was first published, it was big news.  By that time, the only PrP knockout phenotypes anyone had found were an electrophysiological difference observed in hippocampal slices [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8035877">Collinge 1994</a>] and a mild change in circadian rhythms [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8602267">Tobler 1996</a>].  A clear &#8211; and clearly deleterious &#8211; <em>in vivo</em> phenotype earned the Nagasaki line its place in <em>Nature</em>.  Not only did this supposed knockout phenotype point to a native function for PrP, it also raised the question of whether loss of function might be at least partly responsible for the neurodegeneration caused by prions.</p>
<p>But this phenotype was bound to prove contentious: after all, the Zurich line had been monitored out to 90 weeks and had no such phenotype.  Sakaguchi et al ventured a few possible explanations.  Perhaps a mouse genetic background effect controlled the age of onset of the ataxia, or perhaps the Zurich knockout construct was able to retain some of PrP&#8217;s native function. Notably, the authors had omitted the classic rescue experiment in which one checks whether a transgene can reverse the knockout phenotype.  So any real sticklers watching this saga probably thought that the late-onset ataxic phenotype was an artifact of the genetic manipulation, soon to be disproven by a failed rescue experiment.  </p>
<p>But the sticklers were in for a much more interesting show than expected!  It took the Nagasaki line&#8217;s creators three years to complete and publish the rescue experiment, but it worked: a PrP transgene did indeed prevent the late-onset ataxia [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10378511/">Nishida 1999</a>].  At almost the exact same time, though, another group found that the Nagasaki mice expressed a heretofore unknown gene in their brains [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10525406/">Moore 1999</a>]. This gene turned out to be a paralog of PrP itself, and located 16kb downstream of Prnp, so the second group christened their new protein Doppel, for <em>downstream prion protein like</em>. (Many later authors, including me in an earlier revision of this blog post, mistook it as standing for doppelgänger). Dpl, as it was known for short, was normally expressed during embryogenesis but not in the adult brain &#8211; except in the Nagasaki mice.</p>
<p>This new gene&#8217;s ectopic expression in the brain appeared, then, to explain the phenotype of the Nagasaki line, despite the successful rescue experiment.  Curiously, Prusiner was a co-author on both the transgene rescue paper and on the Doppel paper &#8211; one has to wonder what he thought about the whole saga.  People later generated two additional PrP knockout lines (Rcm0 and Zurich II) with the exon 3 splice acceptor deleted, and confirmed that ectopic CNS expression of Doppel indeed causes neurodegeneration which can be rescued by co-expression of PrP [<a href="http://www.ncbi.nlm.nih.gov/pubmed/10930132">Li 2000</a>, <a href="http://www.ncbi.nlm.nih.gov/pubmed/10842180">Silverman 2000</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC145426/">Rossi 2001</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC65022/">Moore 2001</a>].</p>
<p>The mechanism for the Doppel-induced neurodegeneration and the rescue by PrP remains unknown to this day (though there are some interesting theories discussed <a href="/2013/09/01/pi-prps-hypothetical-backup-protein/" title="pi: PrP’s hypothetical backup protein">here</a>).  However, the details of how Doppel (gene symbol <em>Prnd</em>) came to be expressed in the brain are now clear, and are perhaps best explained [<a href="http://www.ncbi.nlm.nih.gov/pubmed/14522848">Weissman &#038; Flechsig 2003</a> (<a href="http://bmb.oxfordjournals.org/content/66/1/43.long">ft</a>)], particularly in <a href="http://bmb.oxfordjournals.org/content/66/1/43/F1.large.jpg">Figure 1C</a>.  Absent the <em>Prnp</em> exon 3 splice acceptor site, the <em>Prnp</em> transcript reads straight through the end of the <em>Prnp</em> gene, splices to two additional exons in the 16 kb between <em>Prnp</em> and <em>Prnd</em> and then splices into all of the <em>Prnd</em> exons. The end result is a chimeric transcript under the control of the prion promoter and thus expressed in the brain, but with Doppel in its protein-coding sequence.  This chimeric transcript even made its way into RefSeq and can be seen in the UCSC Genome Browser:</p>
<p><div id="attachment_3718" style="width: 594px" class="wp-caption aligncenter"><a href="/wp-content/uploads/2014/03/prnp-prnd-fusion-ucsc.png"><img src="/wp-content/uploads/2014/03/prnp-prnd-fusion-ucsc-1024x45.png">in the UCSC Genome Browser</a>.  The two genes are separated by 16 kb (first line) but can be fused with an extra exon in between (second line).</p></div></p>
<p>In humans, <em>PRNP</em> only has two exons, and <em>PRND</em> slightly further away, 20kb downstream.  It&#8217;s not known whether a similar chimeric transcript could be created by deleting the <em>PRNP</em> exon 2 splice acceptor site, but it&#8217;s certainly a risk one would want be aware of and test out in cell lines before going <em>in vivo</em> with CRISPR/Cas9.</p>
<p><div id="attachment_3719" style="width: 716px" class="wp-caption aligncenter"><a href="/wp-content/uploads/2014/03/human-prnp-prnd-structure.png"><img src="/wp-content/uploads/2014/03/human-prnp-prnd-structure.png">these genes in the UCSC Genome Browser</a>.</p></div></p>
<p>Indeed, <em>any</em> CRISPR/Cas9-based strategy would need to be tested to figure out its effects on expression of <em>PRNP</em> and neighboring genes.  For any part of the gene you can think to target, it is at least possible to imagine a potential bad outcome, as diagrammed below.  We have no evidence that any of these things would actually occur, but still, any strategy would need validation in neuronal cell lines to see what happens.</p>
<p><div id="attachment_3720" style="width: 715px" class="wp-caption aligncenter"><a href="/wp-content/uploads/2014/03/crisprcas9-prnp-hazards.png"><img src="/wp-content/uploads/2014/03/crisprcas9-prnp-hazards.png">Depending where one targets <em>PRNP</em> with CRISPR/Cas9 (left), it will be important to double check that the potential hazards (right) do not occur.</p></div></p>
<p>Here&#8217;s a little more explanation, from left to right:</p>
<ul>
<li>Overexpression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8287472">Westaway 1994</a>], and ectopic expression [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9770528/">Tremblay 1998</a>] can both be toxic. Neither of these things seems particularly likely to occur if one is trying to delete the prion promoter, but it will be important to check to be sure.  Of course, to deplete PrP and stop prion disease, the most important thing will just be to see that PrP expression is abolished entirely.</li>
<li>It&#8217;s not known what would happen if exon 1 and/or its splice donor were deleted. Would the whole 10kb intron function as a giant 3&#8242;UTR and would the coding sequence still be translated? Probably not, but we&#8217;d need to do the experiment to find out.</li>
<li>Deleting the exon 2 splice acceptor might create a neurotoxic <em>PRND</em> fusion transcript, or this problem might only occur in mouse and not in human.</li>
<li>Disrupting the N terminus could create an interstitial deletion mutant which is neurotoxic [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9568713">Shmerling 1998</a>, <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1783448/">Li 2007</a>].  This doesn&#8217;t occur if the very N-terminal tip is deleted, but N-terminal deletion mutants are still competent of creating prions if the deletion extends up to the equivalent of human codon 90 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8635458/">Fischer 1996</a>]. Prion susceptibility is apparently abolished if the deletion goes up to codon 106 [<a href="http://www.ncbi.nlm.nih.gov/pubmed/14522848">Weissman &#038; Flechsig 2003</a> (<a href="http://bmb.oxfordjournals.org/content/66/1/43.long">ft</a>)]</li>
<li>Disrupting the C terminus might create a secreted frameshift mutant similar to D178fs, which was recently reported in a patient with prion disease [<a href="http://www.ncbi.nlm.nih.gov/pubmed/23577609">Matsuzono 2013</a>].  The most N-terminal stop codon associated with prion disease in humans is Y145X [<a href="http://www.ncbi.nlm.nih.gov/pubmed/8570627/">Ghetti 1996</a>], and a very early stop codon (homologous to human G29X) appears tolerated in goats [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3542104/">Benestad 2012</a>], so deletions more N-terminal might be fine, but then see the next bullet point above this one.</li>
</ul>
<p>Luckily, according to the <a href="http://www.genome-engineering.org/crispr/?page_id=159">CRISPR FAQ</a>, it&#8217;s pretty easy to find sites to target with CRISPR, so it should be an option to try all of these in a neuronal cell line and have a look at the distribution of outcomes.  Some frequency of bad outcomes is tolerable: if targeting the N terminus creates a neurotoxic interstitial deletion mutant 1% of the time but causes a total loss-of-function frameshift mutant 50% of the time, that would still be a clear therapeutic benefit on the net.  There may also be a possibility of deleting the whole gene: CRISPR/Cas9 has been used to create deletions as large as 1 Mb [<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3737551/">Xiao 2013</a>], though I haven&#8217;t yet been able to find any data on what size of deletion gives the highest efficiency.  Obviously, to go <em>in vivo</em>, you would want the deletion to occur in the highest possible percentage of cells.</p>
