---
layout: post
title:  "Thalidomide's renaissance"
date:   2015-08-24 12:01:00
author: ericminikel
location: Cambridge, MA
thumb120: http://www.cureffi.org/media/2015/08/thalidomide-post-thumbnail.png
summary200: "Long the go-to example for stereochemistry and tragic regulatory missteps, thalidomide's emerging mechanism of action now shows that small molecule drugs can cause targeted protein degradation."
---

In the second episode of *Breaking Bad*, Walter White [teaches](https://www.youtube.com/watch?v=AnmWTruibMY) his chemistry class about chirality:

> For instance, thalidomide. The right-handed isomer of the drug thalidomide is a perfectly fine, good medicine to give to a pregnant woman to prevent morning sickness, but make the mistake of giving that same pregnant woman the left-handed isomer of the drug thalidomide, and her child will be born with horrible birth defects. Which is precisely what happened in the 1950s. So: chiral, chirality, mirrored images - right? Active, inactive, good, bad. 

Walter White's summary isn't quite correct &mdash; (R)- and (S)-thalidomide readily interconvert *in vivo* [[Teo 2004]], such that (R)-thalidomide would not actually be a "perfectly fine, good medicine"  for pregnant women &mdash; but thalidomide is certainly *the* classic example of the starkly different bioactive properties that two entantiomers can possess. Aside from its role in introductory chemistry classes, the drug is also [invoked](http://blogs.fda.gov/fdavoice/index.php/2012/02/50-years-after-thalidomide-why-regulation-matters/) as a cautionary tale supporting careful regulation of pharmaceuticals: the FDA never approved it for morning sickness, ensuring that its European legacy of birth defects never extended to the U.S. But one way or the other, the tragedy of the 1950s is the only context in which most of us have ever heard of thalidomide.

Yet over the past few months, I find myself hearing the word thalidomide more and more, and this time people are talking about it for a different reason: its unique mechanism of action.

Thalidomide has already enjoyed a second life for almost two decades. In 1998, it [received](http://www.nytimes.com/1998/07/17/us/thalidomide-approved-to-treat-leprosy-with-other-uses-seen.html) FDA approval for the treatment of a complication of leprosy, and in 2006 it was [approved](http://www.cancer.gov/about-cancer/treatment/drugs/fda-thalidomide) for multiple myeloma as well. Now marketed as Thalomid&reg; in the U.S., thalidomide has been earning [$245 million per year](http://ir.celgene.com/releasedetail.cfm?releaseid=822091) in revenue for [Celgene](https://www.celgene.com/).

And all of that happened without us having basically any idea what its mechanism of action was, until a few years ago.

In the first breakthrough, thalidomide was conjugated to beads, the beads were incubated with cell lysates, and the proteins that stuck to the beads were subjected to mass spectrometry. Two proteins were specifically pulled down by thalidomide and were revealed to be cereblon (CRBN) and damaged DNA binding protein 1 (DDB1) [[Ito 2010]]. Cereblon was shown to be the direct target, and DDB1 indirect, as thalidomide pulled down recombinant cereblon on its own but only pulled down DDB1 if cereblon was also present. *CRBN* was at that time already known as a recessive intellectual disability gene [[Higgins 2004]], though the function of its protein product was unclear. DDB1, however, was known to be a component of E3 ubiquitin ligase complexes [[Angers 2006]], and the authors were able to show that cereblon, DDB1, and another protein called Cul4a formed a complex with autoubiquitination activity - an activity which was inhibited by thalidomide [[Ito 2010]]. Thus, thalidomide's target was cereblon, and the drug also affected the E3 ubiquitin ligase activities of its larger complex.

By screening a battery of deletion mutants and point mutants of cereblon, they identified a rough binding site for thalidomide and two point mutations &mdash; Y384A and W386A &mdash; that together virtually abolished thalidomide binding. Thalidomide was teratogenic in zebrafish and chick embryos, as it is in humans, but expression of Y384A/W386A double mutant cereblon rescued this effect in both species, thus demonstrating that thalidomide's binding to cereblon was the basis of its teratogenicity [[Ito 2010]]. The choice to use fish and chicks for these experiments was critical because thalidomide has long been known to lack teratogenic activity in mice [[Fratta 1965]], a subject I'll return to later in this post.

Within a couple of years, another group showed that the two thalidomide analogs lenalidomide and pomalidomide, FDA-approved for various blood cancers, also bind to cereblon [[Lopez-Girona 2012]]. Just as with thalidomide, these drugs inhibited the autoubiquitination of the cereblon-DDB1 complex, and the Y384A/W386A double mutant was resistant.

But did thalidomide and its analogs achieve their effects by abolishing some native function of cereblon, or by conferring upon it a new or heightened function? Ito et al were of two minds on this subject, at one point asserting that "thalidomide exerts teratogenic effects by binding to CRBN and inhibiting its function," but also noting that "Whereas CRBN is ubiquitously expressed in humans, thalidomide exerts tissue-specific effects. Evidently, CRBN is necessary, but not sufficient, for thalidomide teratogenicity" [[Ito 2010]]. Ito had shown that thalidomide could inhibit an autoubiquitination activity, and that thalidomide caused some phenotypes in zebrafish that were similar to those caused by knockdown of Cul4a, another member of the complex [[Ito 2010]]. Yet in the later study, when cancer cell lines were chronically exposed to lenalidomide or pomalidomide, they developed drug resistance through *decreased* cereblon expression [[Lopez-Girona 2012]], which would suggest the drugs conferred a gain of function upon cereblon. Human genetics didn't provide a smoking gun answer to the gain-vs-loss of function question either. The fact that mutations in *CRBN* seem to cause mild intellectual disability rather than birth defects [[Higgins 2004]] would have argued against simple inactivation of cereblon by the drug being responsible for teratogenicity, except that the causal mutation in the family Higgins studied was a stop codon very near the end of the protein (R419X in a protein of just 442 residues), where a total loss of function is by no means guaranteed. And as of this writing, in the 11 years since Higgins' paper, no other pathogenic mutations in *CRBN* have been reported in [ClinVar](https://github.com/macarthur-lab/clinvar).

Nonetheless, after a couple of years, three groups using different methods arrived almost simultaneously at the same answer for these drugs' mechanism of action [[Gandhi 2014], [Kronke 2014], [Lu 2014]]. When lenalidomide binds to cereblon, it increases the E3 ubiquitin ligase complex's affinity for two transcription factors, Ikaros (encoded by *IKZF1*) and Aiolos (encoded by *IKZF3*), leading to the ubiquitination and proteasomal degradation of those proteins. This was discovered through a variety of approaches. Kronke and Gandhi had each begun by performed quantitative mass spec proteomics, while Lu had created an arrayed library of 15,000 proteins fused to luciferase. All three found that these two proteins were among those most strongly reduced in abundance when cells were treated with lenalidomide. Lu and Kronke each then showed that Ikaros and Aiolos form a classic ubiquitination "ladder" on a Western blot when cells are treated with lenalidomide, while Gandhi used cycloheximide pulse-chase to show that lenalidomide reduced the half-life of the two proteins. All of the groups showed that siRNA knockdown of IKZF1 or IKZF3 could mimic some of the effects of lenalidomide in cell lines. By creating a series of deletion mutants of Aiolos (IKZF3), Kronke et al identified a 59-residue susbtring of amino acids that was sufficient to cause a protein to be ubiquitinated by cereblon in the presence of lenalidomide [[Kronke 2014]]. Within this sequence, dubbed a "degron", the sensitivity of IKZF1 and IKZF3 appears to be attributable to a single amino acid difference (Q instead of H) versus IKZF2 and IKZF4, which are *not* affected by lenalidomide.

So why does ubiquitination and degradation of Ikaros and Aiolos result in any therapeutic value against myeloma? It was known that thalidomide induces expression of the cytokine interleukin-2, thus potentially hastening an immune response to cancer [[Haslett 1998]], and that Ikaros and Aiolos are transcription factors that repress, among other things, interleukin-2 [[Thompson 2007], [Quintana 2012]]. Thus, Lu speculated, thalidomide and its analogs, by causing Ikaros and Aiolos to be degraded, might be freeing interleukin-2 from transcriptional repression, triggering a protective cytokine response. Kronke went a step further, at least partially confirming this hypothesis by showing that lenalidomide induces IL-2 mRNA, and that siRNA against IKZF1 or IKZF3 does the same, and that the combined effects of lenalidomide and the siRNA are less than additive, suggesting that the same mechanism is at play.

That same year, two groups solved the structure of thalidomide and/or lenalidomide bound to cereblon and DDB1 [[Fischer 2014], [Chamberlain 2014]]. One of these structures [[PDB# 4TZ4](http://www.rcsb.org/pdb/explore/explore.do?structureId=4TZ4)] is shown below:

~~~ 
fetch 4tz4
bg_color white
hide everything
show surface, chain A or chain C
color 0xFF9912, chain C
color 0x0000CD, chain A
show sticks, organic
color 0x000000, organic
~~~ 

![](/media/2015/08/4tz4-annotated.png)

Zooming in, lenalidomide almost appears as a tiny black key in a lock:

![](/media/2015/08/4tz4-zoom.png)

Indeed, the cleft that the drug binds on the protein's surface is so deep that when I play with the structure in PyMOL, it's actually hard to find an angle where you can get a clear view into it. Is the presence of such a binding pocket on this E3 ubiquitin ligase complex just a convenient coincidence, or did cereblon evolve to bind some endogenous small molecule? We don't yet know. Kronke points out one precedent for an endogenous small molecule regulator of ubiquitination &mdash; auxin, a growth hormone in plants, is known to bind to a ubiquitin ligase and change its affinity for various substrates too [[Tan 2007]]. A recent commentary [[King & Finley 2014]] points out that there is also a precedent in mammals: FBXL3, a ubiquitin ligase that targets cryptochromes, thus controlling the circadian clock, is regulated by flavin adenine dinucleotide [[Xing 2013]]. But if cereblon has an endogenous small molecule ligand, no one has found it yet.

Another curiosity about the structure is that one group reported a structure for mouse cereblon bound to thalidomide [[Chamberlain 2014], [PDB # 4ZTC](http://www.rcsb.org/pdb/explore/explore.do?structureId=4TZC)]. But wait, I thought thalidomide didn't cause birth defects in mice &mdash; otherwise it would surely never have been prescribed to pregnant women? Yet thalidomide *does* bind its target in mice? Yes, apparently. Indeed, not one of the amino acids that physically forms the thalidomide binding site differs between mouse and human. 

This inconsistency led another group to construct a battery of mouse/human cereblon chimeras to narrow down the region of the protein responsible for the species difference, and then to introduce each mouse-human amino acid difference in that region, one by one, to see which made the difference [[Kronke & Fink 2015]]. The answer was human cereblon residue V387, which is I in mouse cereblon. When a V387I mutation (which at least a few humans [actually have](http://exac.broadinstitute.org/variant/3-3192716-C-T), by the way) was introduced into human cereblon, it became unresponsive to lenalidomide; when the converse mutation, I391V in mouse numbering, was put into mouse cereblon, it became responsive to lenalidomide. The only structural difference that the V387I mutation introduces appears to be that the extra methyl group on the isoleuceine protrudes as a tiny bump on the surface of the protein just outside of the lenalidomide binding site [[Kronke & Fink 2015]] &mdash; shown here on structure [4ZTC](http://www.rcsb.org/pdb/explore/explore.do?structureId=4TZC):

![](/media/2015/08/mouse-cereblon-i391.png)

Though this bump apparently does not preclude lenalidomide (nor, presumably, thalidomide or pomalidomide) from binding, it must have some subtle yet profound effect on the efficiency of the protein-protein interaction interface between cereblon and its substrates. When I said in the last paragraph that the V387I mutation made human cereblon unresponsive to lenalidomide, that is true both for degradation of Ikaros and Aiolos, and also for degradation of another protein called casein kinase 1&alpha;, which is the main subject of that paper [[Kronke & Fink 2015]]. That group had just identified yet a third protein, known as CK1&alpha; for short (and encoded by *CSNK1A1*) which lenalidomide causes cereblon to target for degradation. The degradation of CK1&alpha; is apparently responsible for yet another clinically useful pharmacological activity of lenalidomide, which is its efficacy in myelodysplastic syndrome with a hemizygous deletion of chromosome 5q &mdash; yet another type of hematological cancer. And CK1&alpha;, the authors show, does not have the same "degron" amino acid sequence that Ikaros and Aiolos share, but instead must be recognized by cereblon through some other sequence.

This expansion of the set of lenalidomide's indirect targets to include CK1&alpha; offers an enticing hint that cereblon's activity is not limited to just a couple of transcription factors relevant to myeloma &mdash; it can be pharmacologically induced to target other proteins too, who knows how many. Indeed, it's already known that thalidomide reduces TNF&alpha; expression [[Sampaio 1991]], an activity which we must now consider might be mediated, however indirectly, by binding to cereblon as well. And for all the inroads into understanding the anticancer effects of thalidomide and its analogs, we still don't know which, if any, of these molecular outcomes actually account for why thalidomide causes birth defects, or why it alleviates morning sickness. There may be yet another substrate, or many substrates, of cereblon's ubiquitin ligase complex, that account for these other effects.

But the ability of thalidomide to cause cereblon to degrade several different proteins is only exciting if we get to *choose* which proteins it degrades. A drug that hits several targets is a drug that's likely to be &mdash; indeed, *was*, and *is* &mdash; quite toxic, at least in some settings. But it turns out that CC-122, an experimental analog of thalidomide now in clinical trials, causes degradation of Ikaros and Aiolos but *not* CK1&alpha; [[Kronke & Fink 2015]].

So if drugs can be designed to lend cereblon specificity for just one or a few proteins, then the door is wide open to imagining what could be done with this mechanism. And indeed, a lot more than imagining has been going on. A few months ago, in almost perfect sync, two different groups published their own bifunctional molecules that hijack cereblon by conjugating thalidomide to a small molecule ligand of BRD4, a bromodomain protein that reads activating acetylation marks on chromatin (see [these notes](/2014/10/15/molecular-biology-15/)) and is important for the growth of some cancers [[Winter 2015], [Lu 2015]]. 

One molecule, dubbed dBET1, successfully reduced BRD4 protein levels by up to 80 or 90% (see [Fig 1J](http://www.sciencemag.org/content/348/6241/1376/F1.expansion.html), with a sub-micromolar EC<sub>50</sub> [[Winter 2015]]. It worked *in vivo*, too, reducing tumor size in mice. And the molecule was fairly specific: according to their proteomic analyses the molecule resulted in reduced levels of three other bromodomain proteins (which of course is an issue of the bromodomain ligand's specificity, not cereblon's specificity), and just two other proteins, MYC and PIM1, at least one of which is in fact a desired downstream target of BRD4 inhibition [[Zuber 2011]]. The other group's molecule appeared to be at least as efficacious, with an EC<sub>50</sub> of ~1 nM and a pronounced effect on cancer cell line growth [[Lu 2015]].

![](/media/2015/08/hijacking-cereblon.png)

And notice that the BRD4-binding bifunctional molecules are a whole conceptual leap away from the original series of thalidomide anlaogs. Whereas the original molecules just slightly alter a protein-protein interaction surface, with apparent effects on its affinity for multiple substrates, these bifunctional molecules go out and just wholesale grab onto a different protein, possibly one which in nature would never have interacted with cereblon to begin with. So in principle, it seems possible to achieve a great deal of specificity with this approach. If you already have a protein you know how to bind with a small molecule, but you don't know how to degrade it, just stick it onto thalidomide:

![](/media/2015/08/your-ligand-here.png)

This, at last, must be why I've been hearing so much talk about thalidomide lately. The hope here is that maybe we've hit upon a modular way to target any protein for degradation, analogous to, say, targeting any mRNA for degradation using an siRNA of the right sequence.

As usual, the reality probably won't be quite as rosy as the hype. In an enthusiastic-but-also-critical [blog post](http://pipeline.corante.com/archives/2015/05/26/targeting_proteins_for_destruction.php), Derek Lowe points out three potential problems &mdash; finding a good ligand to your protein of interest in the first place, linking it to thalidomide without losing either one's activity, and then doing the medicinal chemistry to allow the resulting molecule to get into the cytosol.

And indeed, isn't another requirement of this strategy that your protein of interest be cytosolic in the first place? Consider trying to apply this strategy to PrP, which is a secretory pathway protein. It begins its life being co-translationally translocated into the ER, transits through the Golgi, and emerges, GPI-anchored, to divide its time between the cell surface and the endosomes. Never in this journey is it in the same subcellular compartment as cereblon, so even if you had a good PrP ligand (which I'm [not sure we do](/2015/07/22/reports-of-small-molecule-prpc-ligands/)), where is the opportunity to trigger degradation? Purely for the sake of illustration, let's suppose that the cationic porphyrin FeTMPyP binds PrP<sup>C</sup>, as reported [[Nicoll 2010]], and that it can be conjugated to thalidomide via a linker without losing activity. PrP binds one end of the bifunctional molecule, and cereblon binds the other, but never do the two bind the same copy of the molecule:

![](/media/2015/08/prp-cereblon-localization.png "ne'er the twain shall meet")

Perhaps I'm wrong about this &mdash; if so, I'd like to see what exactly this diagram looks like for a scenario where the molecule does bring PrP and cereblon into proximity. (Here's the [original](/media/2015/08/thalidomide-diagram.ppt) if anyone wants to edit).

Now, granted, some groups have reported seeing some PrP in the cytosol some of the time under some circumstances [[Zanusso 1999], [Jin 2000], [Yedidia 2001], [Ma & Lindquist 2001]] &mdash; see [this post](/2013/07/25/the-role-of-the-proteasome-in-prp-degradation-and-prion-disease/) for a review &mdash; and UPS reporter mice [[Dantuma 2000]], whose brains express GFP under conditions of proteasomal stress, glow in the terminal stages of prion disease [[Kristiansen 2007]]. So it's not as though PrP has never been sighted in the same compartment as cereblon. But to the extent that there exists such a thing as cytosolic PrP, is it a large enough fraction of total PrP to be worth targeting? And isn't it in the cytosol precisely because it has *already* been targeted for degradation via ERAD? And if so, is there any marginal benefit to ubiquitinating it even more?

For these reasons I am skeptical that the thalidomide conjugation approach could be relevant to PrP, which is my protein of interest. Of course, the thalidomide concept is just the latest technology in a series of bifunctional molecule concepts. Some of these concepts are very similar and suffer from the same limitations &mdash; for instance, over a decade ago, there was an effort to use bifunctional small molecules to deliver your-protein-of-interest to Skp1-Cullin-F box, another E3 ubiquitin ligase complex [[Sakamoto 2001]], and a couple of years ago, a patent was filed on an approach to do the same for VHL, the ubiquitin ligase that's mutated in [von Hippel Lindau](/2015/03/13/genetics-in-medicine-07/) [[CA2861066A1](https://patents.google.com/patent/CA2861066A1/en)]. Then there are the bifunctional molecules to dimerize a protein or bring your-protein-of-interest into proximity with something else of interest. For instance, a molecule consisting of two copies of FK506 fused together, dubbed "FK1012", can be used to trigger FKBP12, and other proteins bound to it, to dimerize [[Spencer 1993]]. Or, in the neurodegeneration space, there was SLF, another FKBP12 ligand, fused to Congo red, which commandeered FKBP12's steric bulk to block the elongation of A&beta; fibrils [[Gestwicki 2004]].

Perhaps the approach most specifically relevant to secretory pathways proteins is the idea of adding a bulky hydrophobic group to a ligand to attract the ER protein quality control chaperone BiP. BiP recognizes exposed hydrophobic patches, which usually indicate protein misfolding, and it can then help with folding or direct proteins for degradation [[Kleizen & Braakman 2004]]. The first proofs of principle for this strategy were done in pretty artificial systems: the target was a protein of interest genetically fused to a HaloTag protein, and the ligand was a covalent ligand of HaloTag, fused to some collection of aromatic rings or cyclohexanes or an adamantane as the hydrophobic moiety [[Neklesa 2011], [Tae 2012]]. It was later shown that similar results could be achieved with a non-covalent ligand [[Long 2012]; and see commentary in [Neklesa & Crews 2012]], though still in an artificial system, targeting a bacterial protein for degradation in mammalian cells. Just recently, success was reported in designing a molecule, dubbed SARD279, consisting of an androgen receptor (AR) ligand, fused via a linker to an adamantyl group, to cause AR degradation, with 50% degradation achieved at 1 &mu;M [[Gustafson 2015]].

![](/media/2015/08/sard279.png)

Whether any of these approaches will yield a drug remains to be seen. It's an area that deserves both excitement and, as Derek Lowe points out, some dose of cautious skepticism. So perhaps what's most exciting about the emerging thalidomide story, then, is simply the proof that using a small molecule to trigger a protein's degradation isn't an impossible concept for a drug. In fact, unwittingly, we've already been doing it for years.

[Gustafson 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26083457 "Gustafson JL, Neklesa TK, Cox CS, Roth AG, Buckley DL, Tae HS, Sundberg TB, Stagg DB, Hines J, McDonnell DP, Norris JD, Crews CM. Small-Molecule-Mediated Degradation of the Androgen Receptor through Hydrophobic Tagging. Angew Chem Int  Ed Engl. 2015 Aug 10;54(33):9659-62. doi: 10.1002/anie.201503720. Epub 2015 Jun 17. PubMed PMID: 26083457."

[Kleizen & Braakman 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15261665 "Kleizen B, Braakman I. Protein folding and quality control in the endoplasmic  reticulum. Curr Opin Cell Biol. 2004 Aug;16(4):343-9. Review. Erratum in: Curr Opin Cell Biol. 2004 Oct;16(5):597. PubMed PMID: 15261665."

[Gestwicki 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15514157 "Gestwicki JE, Crabtree GR, Graef IA. Harnessing chaperones to generate small-molecule inhibitors of amyloid beta aggregation. Science. 2004 Oct 29;306(5697):865-9. PubMed PMID: 15514157."

[Spencer 1993]: http://www.ncbi.nlm.nih.gov/pubmed/7694365 "Spencer DM, Wandless TJ, Schreiber SL, Crabtree GR. Controlling signal transduction with synthetic ligands. Science. 1993 Nov 12;262(5136):1019-24. PubMed PMID: 7694365."

[Sakamoto 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11438690/ "Sakamoto KM, Kim KB, Kumagai A, Mercurio F, Crews CM, Deshaies RJ. Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation. Proc Natl Acad Sci U S A. 2001 Jul 17;98(15):8554-9. Epub 2001 Jul 3. PubMed PMID: 11438690; PubMed Central PMCID: PMC37474."

[Dantuma 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10802622 "Dantuma NP, Lindsten K, Glas R, Jellne M, Masucci MG. Short-lived green fluorescent proteins for quantifying ubiquitin/proteasome-dependent proteolysis in living cells. Nat Biotechnol. 2000 May;18(5):538-43. PubMed PMID: 10802622."

[Zanusso 1999]: http://www.ncbi.nlm.nih.gov/pubmed/10438517 "Zanusso G, Petersen RB, Jin T, Jing Y, Kanoush R, Ferrari S, Gambetti P, Singh N. Proteasomal degradation and N-terminal protease resistance of the codon 145 mutant prion protein. J Biol Chem. 1999 Aug 13;274(33):23396-404. PubMed PMID: 10438517."

[Jin 2000]: http://www.ncbi.nlm.nih.gov/pubmed/10970892 "Jin T, Gu Y, Zanusso G, Sy M, Kumar A, Cohen M, Gambetti P, Singh N. The chaperone protein BiP binds to a mutant prion protein and mediates its degradation by the proteasome. J Biol Chem. 2000 Dec 8;275(49):38699-704. PubMed  PMID: 10970892."

[Yedidia 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11574470/ "Yedidia Y, Horonchik L, Tzaban S, Yanai A, Taraboulos A. Proteasomes and ubiquitin are involved in the turnover of the wild-type prion protein. EMBO J. 2001 Oct 1;20(19):5383-91. PubMed PMID: 11574470; PubMed Central PMCID: PMC125653."

[Ma & Lindquist 2001]: http://www.ncbi.nlm.nih.gov/pubmed/11742063/ "Ma J, Lindquist S. Wild-type PrP and a mutant associated with prion disease are subject to retrograde transport and proteasome degradation. Proc Natl Acad Sci U S A. 2001 Dec 18;98(26):14955-60. Epub 2001 Dec 11. PubMed PMID: 11742063;  PubMed Central PMCID: PMC64965."

[Kristiansen 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17466621 "Kristiansen M, Deriziotis P, Dimcheff DE, Jackson GS, Ovaa H, Naumann H, Clarke AR, van Leeuwen FW, Menéndez-Benito V, Dantuma NP, Portis JL, Collinge J,  Tabrizi SJ. Disease-associated prion protein oligomers inhibit the 26S proteasome. Mol Cell. 2007 Apr 27;26(2):175-88. PubMed PMID: 17466621."

[Nicoll 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20876144 "Nicoll AJ, Trevitt CR, Tattum MH, Risse E, Quarterman E, Ibarra AA, Wright C,  Jackson GS, Sessions RB, Farrow M, Waltho JP, Clarke AR, Collinge J. Pharmacological chaperone for the structured domain of human prion protein. Proc  Natl Acad Sci U S A. 2010 Oct 12;107(41):17610-5. doi: 10.1073/pnas.1009062107. Epub 2010 Sep 27. PubMed PMID: 20876144; PubMed Central PMCID: PMC2955083."

[Neklesa & Crews 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22810693 "Neklesa TK, Crews CM. Chemical biology: Greasy tags for protein removal. Nature. 2012 Jul 18;487(7407):308-9. doi: 10.1038/487308a. PubMed PMID: 22810693."

[Long 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22633414 "Long MJ, Gollapalli DR, Hedstrom L. Inhibitor mediated protein degradation. Chem Biol. 2012 May 25;19(5):629-37. doi: 10.1016/j.chembiol.2012.04.008. PubMed  PMID: 22633414; PubMed Central PMCID: PMC3361691."

[Tae 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22271667 "Tae HS, Sundberg TB, Neklesa TK, Noblin DJ, Gustafson JL, Roth AG, Raina K, Crews CM. Identification of hydrophobic tags for the degradation of stabilized proteins. Chembiochem. 2012 Mar 5;13(4):538-41. doi: 10.1002/cbic.201100793. Epub 2012 Jan 23. PubMed PMID: 22271667; PubMed Central PMCID: PMC3338156."

[Neklesa 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21725302 "Neklesa TK, Tae HS, Schneekloth AR, Stulberg MJ, Corson TW, Sundberg TB, Raina K, Holley SA, Crews CM. Small-molecule hydrophobic tagging-induced degradation of HaloTag fusion proteins. Nat Chem Biol. 2011 Jul 3;7(8):538-43. doi: 10.1038/nchembio.597. PubMed PMID: 21725302; PubMed Central PMCID: PMC3139752."

[Xing 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23503662 "Xing W, Busino L, Hinds TR, Marionni ST, Saifee NH, Bush MF, Pagano M, Zheng N. SCF(FBXL3) ubiquitin ligase targets cryptochromes at their cofactor pocket. Nature. 2013 Apr 4;496(7443):64-8. doi: 10.1038/nature11964. Epub 2013 Mar 17. PubMed PMID: 23503662; PubMed Central PMCID: PMC3618506."

[King & Finley 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25325686 "King RW, Finley D. Sculpting the proteome with small molecules. Nat Chem Biol. 2014 Nov;10(11):870-4. doi: 10.1038/nchembio.1671. PubMed PMID: 25325686."

[Gandhi 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24328678 "Gandhi AK, Kang J, Havens CG, Conklin T, Ning Y, Wu L, Ito T, Ando H, Waldman  MF, Thakurta A, Klippel A, Handa H, Daniel TO, Schafer PH, Chopra R. Immunomodulatory agents lenalidomide and pomalidomide co-stimulate T cells by inducing degradation of T cell repressors Ikaros and Aiolos via modulation of the E3 ubiquitin ligase complex CRL4(CRBN.). Br J Haematol. 2014 Mar;164(6):811-21. doi: 10.1111/bjh.12708. Epub 2013 Dec 13. PubMed PMID: 24328678; PubMed Central PMCID: PMC4232904."

[Fischer 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25043012 "Fischer ES, Böhm K, Lydeard JR, Yang H, Stadler MB, Cavadini S, Nagel J, Serluca F, Acker V, Lingaraju GM, Tichkule RB, Schebesta M, Forrester WC, Schirle M, Hassiepen U, Ottl J, Hild M, Beckwith RE, Harper JW, Jenkins JL, Thomä NH. Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide. Nature. 2014 Aug 7;512(7512):49-53. doi: 10.1038/nature13527. Epub 2014 Jul 16. PubMed PMID: 25043012; PubMed Central PMCID: PMC4423819."

[Fratta 1965]: http://www.ncbi.nlm.nih.gov/pubmed/14298018 "FRATTA ID, SIGG EB, MAIORANA K. TERATOGENIC EFFECTS OF THALIDOMIDE IN RABBITS, RATS, HAMSTERS, AND MICE. Toxicol Appl Pharmacol. 1965 Mar;7:268-86. PubMed PMID: 14298018."

[Angers 2006]: http://www.ncbi.nlm.nih.gov/pubmed/16964240 "Angers S, Li T, Yi X, MacCoss MJ, Moon RT, Zheng N. Molecular architecture and assembly of the DDB1-CUL4A ubiquitin ligase machinery. Nature. 2006 Oct 5;443(7111):590-3. PubMed PMID: 16964240."

[Higgins 2004]: http://www.ncbi.nlm.nih.gov/pubmed/15557513 "Higgins JJ, Pucilowska J, Lombardi RQ, Rooney JP. A mutation in a novel ATP-dependent Lon protease gene in a kindred with mild mental retardation. Neurology. 2004 Nov 23;63(10):1927-31. PubMed PMID: 15557513; PubMed Central PMCID: PMC1201536."

[Teo 2004]: https://www.ncbi.nlm.nih.gov/pubmed/15080764 "Teo SK, Colburn WA, Tracewell WG, Kook KA, Stirling DI, Jaworsky MS, Scheffler MA, Thomas SD, Laskin OL. Clinical pharmacokinetics of thalidomide. Clin Pharmacokinet. 2004;43(5):311-27. Review. PubMed PMID: 15080764."

[Ito 2010]: http://www.ncbi.nlm.nih.gov/pubmed/20223979 "Ito T, Ando H, Suzuki T, Ogura T, Hotta K, Imamura Y, Yamaguchi Y, Handa H. Identification of a primary target of thalidomide teratogenicity. Science. 2010 Mar 12;327(5971):1345-50. doi: 10.1126/science.1177319. PubMed PMID: 20223979."

[Lopez-Girona 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22552008 "Lopez-Girona A, Mendy D, Ito T, Miller K, Gandhi AK, Kang J, Karasawa S, Carmel G, Jackson P, Abbasian M, Mahmoudi A, Cathers B, Rychak E, Gaidarova S, Chen R, Schafer PH, Handa H, Daniel TO, Evans JF, Chopra R. Cereblon is a direct protein target for immunomodulatory and antiproliferative activities of lenalidomide and pomalidomide. Leukemia. 2012 Nov;26(11):2326-35. doi: 10.1038/leu.2012.119. Epub 2012 May 3. Erratum in: Leukemia. 2012 Nov;26(11):2445. PubMed PMID: 22552008; PubMed Central PMCID: PMC3496085."

[Chamberlain 2014]: http://www.ncbi.nlm.nih.gov/pubmed/25108355 "Chamberlain PP, Lopez-Girona A, Miller K, Carmel G, Pagarigan B, Chie-Leon B,  Rychak E, Corral LG, Ren YJ, Wang M, Riley M, Delker SL, Ito T, Ando H, Mori T, Hirano Y, Handa H, Hakoshima T, Daniel TO, Cathers BE. Structure of the human Cereblon-DDB1-lenalidomide complex reveals basis for responsiveness to thalidomide analogs. Nat Struct Mol Biol. 2014 Sep;21(9):803-9. doi: 10.1038/nsmb.2874. Epub 2014 Aug 10. PubMed PMID: 25108355."

[Lu 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24292623 "Lu G, Middleton RE, Sun H, Naniong M, Ott CJ, Mitsiades CS, Wong KK, Bradner JE, Kaelin WG Jr. The myeloma drug lenalidomide promotes the cereblon-dependent destruction of Ikaros proteins. Science. 2014 Jan 17;343(6168):305-9. doi: 10.1126/science.1244917. Epub 2013 Nov 29. PubMed PMID: 24292623; PubMed Central  PMCID: PMC4070318."

[Kronke 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24292625 "Krönke J, Udeshi ND, Narla A, Grauman P, Hurst SN, McConkey M, Svinkina T, Heckl D, Comer E, Li X, Ciarlo C, Hartman E, Munshi N, Schenone M, Schreiber SL,  Carr SA, Ebert BL. Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells. Science. 2014 Jan 17;343(6168):301-5. doi: 10.1126/science.1244851. Epub 2013 Nov 29. PubMed PMID: 24292625; PubMed Central  PMCID: PMC4077049."

[Stewart 2014]: http://www.ncbi.nlm.nih.gov/pubmed/24436409 "Stewart AK. Medicine. How thalidomide works against cancer. Science. 2014 Jan  17;343(6168):256-7. doi: 10.1126/science.1249543. PubMed PMID: 24436409; PubMed Central PMCID: PMC4084783."

[Tan 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17410169 "Tan X, Calderon-Villalobos LI, Sharon M, Zheng C, Robinson CV, Estelle M, Zheng N. Mechanism of auxin perception by the TIR1 ubiquitin ligase. Nature. 2007 Apr 5;446(7136):640-5. PubMed PMID: 17410169."

[Lu 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26051217 "Lu J, Qian Y, Altieri M, Dong H, Wang J, Raina K, Hines J, Winkler JD, Crew AP, Coleman K, Crews CM. Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4. Chem Biol. 2015 Jun 18;22(6):755-63. doi: 10.1016/j.chembiol.2015.05.009. Epub 2015 Jun 4. PubMed PMID: 26051217; PubMed Central PMCID: PMC4475452."

[Winter 2015]: http://www.ncbi.nlm.nih.gov/pubmed/25999370 "Winter GE, Buckley DL, Paulk J, Roberts JM, Souza A, Dhe-Paganon S, Bradner JE. DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation. Science. 2015 Jun 19;348(6241):1376-81. doi: 10.1126/science.aab1433. Epub 2015 May 21. PubMed PMID: 25999370."

[Kronke & Fink 2015]: http://www.ncbi.nlm.nih.gov/pubmed/26131937 "Krönke J, Fink EC, Hollenbach PW, MacBeth KJ, Hurst SN, Udeshi ND, Chamberlain PP, Mani DR, Man HW, Gandhi AK, Svinkina T, Schneider RK, McConkey M, Järås M, Griffiths E, Wetzler M, Bullinger L, Cathers BE, Carr SA, Chopra R, Ebert BL. Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS. Nature. 2015 Jul 1. doi: 10.1038/nature14610. [Epub ahead of print] PubMed PMID:  26131937."

[Haslett 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9607928/ "Haslett PA, Corral LG, Albert M, Kaplan G. Thalidomide costimulates primary human T lymphocytes, preferentially inducing proliferation, cytokine production,  and cytotoxic responses in the CD8+ subset. J Exp Med. 1998 Jun 1;187(11):1885-92. PubMed PMID: 9607928; PubMed Central PMCID: PMC2212313."

[Quintana 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22751139 "Quintana FJ, Jin H, Burns EJ, Nadeau M, Yeste A, Kumar D, Rangachari M, Zhu C, Xiao S, Seavitt J, Georgopoulos K, Kuchroo VK. Aiolos promotes TH17 differentiation by directly silencing Il2 expression. Nat Immunol. 2012 Jul 1;13(8):770-7. doi: 10.1038/ni.2363. Erratum in: Nat Immunol. 2014 Jan;15(1):109. PubMed PMID: 22751139; PubMed Central PMCID: PMC3541018."

[Thompson 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17363301 "Thompson EC, Cobb BS, Sabbattini P, Meixlsperger S, Parelho V, Liberg D, Taylor B, Dillon N, Georgopoulos K, Jumaa H, Smale ST, Fisher AG, Merkenschlager  M. Ikaros DNA-binding proteins as integral components of B cell developmental-stage-specific regulatory circuits. Immunity. 2007 Mar;26(3):335-44. Erratum in: Immunity. 2007 Apr;26(4):533. PubMed PMID: 17363301."

[Sampaio 1991]: http://www.ncbi.nlm.nih.gov/pubmed/1997652 "Sampaio EP, Sarno EN, Galilly R, Cohn ZA, Kaplan G. Thalidomide selectively inhibits tumor necrosis factor alpha production by stimulated human monocytes. J  Exp Med. 1991 Mar 1;173(3):699-703. PubMed PMID: 1997652; PubMed Central PMCID: PMC2118820."

[Zuber 2011]: http://www.ncbi.nlm.nih.gov/pubmed/21814200 "Zuber J, Shi J, Wang E, Rappaport AR, Herrmann H, Sison EA, Magoon D, Qi J, Blatt K, Wunderlich M, Taylor MJ, Johns C, Chicas A, Mulloy JC, Kogan SC, Brown P, Valent P, Bradner JE, Lowe SW, Vakoc CR. RNAi screen identifies Brd4 as a therapeutic target in acute myeloid leukaemia. Nature. 2011 Aug 3;478(7370):524-8. doi: 10.1038/nature10334. PubMed PMID: 21814200; PubMed Central PMCID: PMC3328300."


