---
layout: post
title:  "Does this mean I'll definitely get the disease?"
date:   2016-01-20 12:01:00
author: ericminikel
location: Cambridge, MA
thumb120: http://www.cureffi.org/media/2016/01/penetrance-thumbnail.png
summary200: "My wife and I had a question about her genetic test report. This is the four year long story of what we did to try to answer it."
---

I am excited to announce today the publication of our new study, "Quantifying prion disease penetrance using large population control cohorts" [[full text](http://stm.sciencemag.org/content/8/322/322ra9.full), [PDF](http://stm.sciencemag.org/content/8/322/322ra9.full.pdf), [perspective piece by Robert Green](http://stm.sciencemag.org/content/8/322/322fs3.full)]. I wanted to write a blog post to announce it and to offer up the comments section (bottom of this post) as a place for post-publication peer review. I also wanted to take this opportunity to tell the history of this study, a profoundly personal endeavor that has grown and taken shape over the entire four year course of my re-training as a scientist.

On December 9, 2011, my wife Sonia and I walked into a room in Adult Genetics at Brigham and Women's Hospital, and a medical geneticist looked us in the eyes and said:

> **The same change that was found in your mother was found in you.**

It was a conservative turn of phrase. He didn't say "you have the mutation." He didn't even call it a mutation. It was just a *change*, a single letter typo, one mistake out of 3 billion base pairs in Sonia's entire genome.

At the beginning of 2010, Sonia's healthy, middle-aged mother had a sudden and peculiar bout of blurry vision, a spike in blood pressure, a strange judgment lapse while driving that led to a minor accident. A few months later she was profoundly demented and unable to feed herself, walk, or recognize her family members. By summer she was on life support. Time inched by in her hospital room, even we watched her endure lifetimes of suffering. When she passed away at the end of the year, it was a relief.

Whatever her disease was &mdash; it had never been diagnosed &mdash; it was over, we thought. But months later, we found out from an autopsy report that her disease had been genetic. My mother-in-law had a genetic prion disease, a subtype called fatal familial insomnia. A rare and incurable neurodegenerative disease.

Sonia was at 50/50 risk.

The thought of just living with that 50/50 risk never occurred to us. Sonia and I both knew, immediately and without a trace of doubt, that we had to know. She had to get tested. If she was negative, our lives would go back to normal. This would all just have been a really weird few months. If she was positive, well, we couldn't feel worse than we already did. At least we'd know.

And now we knew. The same change that was found in Sonia's mother was found in Sonia. But what *exactly* did this mean? Was her fate sealed? Was she guaranteed to die young exactly as her mother had? Had her probability of dying this way just risen from 50% to 100%? In other words,

> **Does this mean I'll definitely get the disease?**

I should mention that at this time, Sonia and I had no scientific training, and a dim memory of high school biology. Sonia had just graduated law school. I was working in transportation technology consulting. We would eventually learn that the scientific term for what we were asking about was **penetrance**. Penetrance is the probability of developing a particular genetic disease if you have a particular genetic mutation.

We would also eventually learn that the questions we were asking were hard ones to answer. First of all, scientists don't give straight answers. They don't say "yes, it's high penetrance" or "no it's not", they say "the evidence we have suggests that..." In this case, what the evidence suggested was that Sonia's mutation was indeed highly penetrant. A few scientists and doctors told us they'd heard of one or two people who had that same mutation and had lived into their eighties and then died of something else, so maybe the penetrance was 90 or 95%, not 100%, but still very high. The evidence for this was that there have been several families with this mutation that have been studied closely, and in each generation, all or almost all of the people with the mutation eventually died of the disease.

To avoid creating false suspense followed by disappointment, I'll spoil the surprise now and tell you that Sonia's mutation *is* highly penetrant, and that everything we now know, including the results of the new study published today, supports this conclusion. I'll also tell you that even back in the final weeks of 2011, when we were beginning to learn all of this, we were already fairly convinced that the mutation was probably highly penetrant. Convinced enough that, rather than waiting for her diagnosis to simply blow away in the wind, we upended our lives, changed our careers, and became scientists to devote ourselves to trying to find a cure. 

But I'll also say, we carried with us at least some whiffs of doubt. There was no history of dementia in Sonia's family. In her entire massive family tree, her mother was the first in living memory to die of a neurodegenerative disease. It looked nothing like the classic picture of a dominant disease pedigree. Her mother was also the first South Asian ever to die of this particular subtype of prion disease, with this particular mutation. So even as we were taking night classes and starting our first jobs in biology labs, there was some part of us that was wondering: is it possible that this mutation just behaves a little differently on a South Asian genetic background? Or is it possible that we just *see* the families where everyone dies of this disease because that's who gets studied &mdash; and we don't see the families where no one gets the disease?

![](/media/2016/01/photos_of_sonias_mom.png)

*Sonia's mom in her youth (left) and helping Sonia get ready for our wedding in 2009 (right).*

As we learned enough to dig into the scientific literature ourselves, we saw that we weren't the first people to wonder about these sorts of questions. Geneticists were well aware that research studies on dominant disease families suffer from a disturbing circularity. The way that researchers discover the mutations that cause dominant genetic diseases in the first place is to find families where, generation after generation, about half the people get sick and die of the same disease. They sequence DNA from those people and look for a mutation that is found in all the people who did get sick and none (or very few) of the people who didn't. So when you then turn around and ask, "what percentage of people with the mutation get the disease?", the answer is tautologically 100% (or close to it). It is believed that this circularity, this ascertainment bias, has probably inflated the estimates of penetrance in various genetic diseases. People studying everything from *BRCA1* and *BRCA2* breast cancer to *LRRK2* Parkinson's disease have grappled with this problem and have sought to devise clever ways of avoiding such bias [[Wacholder 1998], [Goldwurm 2007]]. But many of those clever ideas (such as the [kin-cohort method](http://www.cureffi.org/2014/05/15/the-kin-cohort-method-for-estimating-disease-penetrance/)) don't really work for diseases as rare as prion disease. 

Prion disease is often referred to as a "one in a million" disease, although that's a bit misleading. The annual incidence of prion disease is often estimated at 1 new case per million population per year. There are certainly undiagnosed cases &mdash; if not for the autopsy, which many families refuse, Sonia's mother would have been one of them &mdash; and the countries that invest more heavily in tracking down such cases do find something more like 2 cases per million per year [[Klug 2013]]. And those are annual numbers, so if you integrate them over a human lifetime, or if you simply divide the number of prion disease deaths by the number of total deaths, you find that the lifetime risk of prion disease in the general population is actually about 1 in 5,000. Still, that's a lot rarer than Parkinson's disease or breast cancer. And the majority of cases of prion disease are what we call "sporadic" (they just *happen*), while only about 15% of cases even have a genetic mutation. And then there are many different mutations, so Sonia's exact mutation only accounts for about 2% of all prion disease cases.

So if you multiply the 2% by the 1 in 5,000, you can estimate that prion disease caused by Sonia's exact mutation probably affects about 1 in every 100,000 people. This sort of calculation led to an idea. If Sonia's mutation really was close to 100% penetrant, then her mutation would have to be just as rare in the general popuation as her disease is in the general population. Conversely, if the mutation was more common than expected, that would mean that either the disease was underdiagnosed, or that there were people walking around with this mutation their whole lives and not dying of it.

We started wondering how we could check this. As Sonia and I were first learning biology in the beginning of 2012, one of the first goals I set for myself was to learn enough bioinformatics to be able to check whether her mutation was present in any public databases of genetic variation. This was incredibly non-trivial. My [first-ever post](/2012/01/06/the-gene-sequence-for-ffi/) on this blog was me stumbling through just trying to figure out how to parse the name of the mutation that we'd been given, which was a seemingly random string of numbers and letters: *PRNP* D178N cis-129M. (Years later I wrote [this little guide](http://www.prionalliance.org/2013/12/09/what-does-it-mean-to-have-a-genetic-prion-disease/) to interpreting such strings). When I finally achieved my goal, I learned that no, Sonia's mutation was not found in the 1000 Genomes Project nor in anything else available at the time.

But that didn't actually mean much. At that time, the 1000 Genomes Project had genetic data on only 1,700 people. One can, and we did, imagine a scenario in which her mutation might be found in 1 in 10,000 people, which would make it far too rare to find in 1000 Genomes, yet still ten times more common than the corresponding disease is supposed to be. To rule out such a scenario, we would need genetic information from way, way, way, more people. 

This became the topic of conversation when I sat down to eat lunch with Daniel MacArthur in spring 2013. By that time, I had quit my job in consulting and had started a new gig as a bioinformatics analyst, writing code to analyze RNA and DNA sequences in a lab at Massachusetts General Hospital. Daniel was a newly minted principal investigator with his own lab on the floor directly above mine. We shared an interest in blogging, and in rare diseases. He told me that his lab was working on collecting DNA sequences from tens of thousands of people, to use as a reference dataset. The problem he was trying to solve was parallel to mine: he was looking at patients with very rare muscle diseases, and trying to figure out which DNA mutations caused them. If the disease was ultra-rare, then the mutation would have to be ultra-rare too. And to know whether a mutation was ultra-rare, he needed a huge reference dataset. So he'd been going around to all of his colleagues at MGH, at the Broad Institute, and all over the world, and asking them if they'd be willing to take whatever DNA sequence data they had from their own projects &mdash; studies of diabetes and mental illness and heart disease and so on &mdash; and let him merge it into one giant database of genetic variation. Amazingly, most of them said yes. (These people, and the people who shared their own DNA with researchers in the first place, all have hearts of gold, and I am forever grateful to them).

At the end of lunch, Daniel took me up to his office and introduced me to his postdoctoral researcher, Monkol Lek, who was leading the database effort. They ran a query of over 20,000 people's DNA and told me that no, none of them had Sonia's mutation. Daniel also said we should have a quick look for what he called **loss-of-function** mutations &mdash; mutations that inactivate a gene, making it produce a dysfunctional protein or no protein at all. Many of the muscle diseases that Daniel studies are actually caused by such mutations &mdash; patients just don't have a working copy of a protein that they desperately need. Prion disease is the opposite, a **gain-of-function** disease: Sonia's mutation makes a mutant gene that *does* do some new horrible thing that it shouldn't do. If you could wipe out the gene or the protein entirely, that would actually help matters. Prion protein knockout mice &mdash; mice engineered to lack a working copy of the gene where Sonia has a mutation &mdash; can't get prion disease [[Bueler 1993]].

Indeed, one idea for how to treat or cure Sonia's disease would be if we could find a drug that reduces the amount of prion protein her brain produces, or that eliminates it entirely. The real trick is how to find such a drug. But there were also scientists in the prion field, I had learned, who were skeptical about the whole strategy. This gene is the gem of millions of years of evolution, they would say, so there must be a good reason why we have it, it must do something absolutely essential, and don't go tinkering with what you don't understand. I never bought this argument, because the same could be said of all the genes that contain instructions for all of the proteins that are targeted by modern medicine, and yet modern medicine works. Mice lacking HMG-CoA reductase, the protein targeted by cholesterol-lowering statin drugs, die *in utero* [[Ohashi 2003]], yet people take statins, and not only does it not kill them, it saves their lives.

Still, it's certainly true that we have only a limited understanding of what prion protein is supposed to do, and I agree that it would be ideal to know a bit more before going and trying to turn the gene off in my wife's brain. So when Daniel and Monkol ran a query and found one person in their database who had 1 of his 2 copies of the prion protein gene effectively knocked out, that was interesting news. We immediately kicked off an email chain to ask Daniel's collaborator who had shared the DNA sequence data whether there was any way to follow up on this individual.

A few months later, Daniel hired me to come on full time in his lab. The database had grown, and with Daniel's mentorship, I started taking a closer look at what it could teach us about prion disease. Sonia's mutation stubbornly refused to show up, no matter how large the database got, but I started to notice that some other mutations in the prion protein gene did show up. These were mutations that were thought to cause prion disease, just like Sonia's mutation was. And here they were, popping up in a dataset that should be pretty representative of the general population. Sonia and I combed through the scientific literature to identify every mutation that had ever been reported to cause prion disease, and we came up with 63 of them. Collectively, these 63 mutations turned up in almost 1 in every 1,000 people in Daniel's database. Remember that prion disease, all told, only kills about 1 in every 5,000 people, and *genetic* prion disease is more on the order of 1 in 50,000. So something was profoundly wrong here. Sonia and I manually combed through all of the raw DNA sequence data to look for errors, but that wasn't it. These people really had these mutations, and in the aggregate, the mutations really were way more common than they were supposed to be.

One obvious explanation could be underdiagnosis. But while we were willing to believe that prion disease might be underdiagnosed by 25% or 50%, there was just no way that the disease was *fifty* times more common than we thought. The unique thing about prion diseases is that, under just the wrong circumstances, such as by re-use of contaminated neurosurgical instruments, they can be transmitted from person to person. Transmitted cases are exceptionally rare nowadays, but because of this threat, prion diseases are very closely surveilled. If you're a neurologist in the U.S. or pretty much any other developed country and you think you may have a patient with prion disease, you're required to report it to a national authority. Sure, some cases still get missed, but we have better statistics than for any other comparably rare disease, and it is simply not plausible to argue that we might be missing 98% of all cases.

The better explanation for what we'd found was that maybe some of those mutations were not fully penetrant. In favor of this interpretation, most of the mutations we were seeing in the database actually had relatively weak evidence for causing prion disease in the first place. Some of them had only been seen in one prion disease patient ever. Sometimes even that one patient didn't actually have a confirmed diagnosis of prion disease. In contrast, the handful of mutations (like Sonia's) that were very well studied were all absent from the database. Perhaps not all of these mutations were created equal, and there was a spectrum from the very bad to the moderately bad to the totally not bad at all.

Sonia and I flew to Trieste, Italy and presented a poster on our findings at the Prion2014 conference. A crowd gathered. All night people came by to ask us about it. We won a prize for best poster in the human prion disease category, and I cried. The research community had decided that we were legit. All of the sacrifices we'd made to re-train our lay selves as biologists were not a dead end. We were making it through to the other side. We were scientists now.

![](/media/2016/01/eric-sonia-trieste.png)

*Sonia and I on stage at Prion2014 in Trieste, Italy. Photo: Ekaterina Kochegurova*

When we got back to Boston, we set about trying to systematize our analysis. It was one thing to make an argument in the aggregate &mdash; *not all* of these mutations can be highly penetrant &mdash; but we wanted to be able to reach specific conclusions about individual mutations. Sonia and I started trying to comb through the literature on each mutation, but it turns out that the scientific literature is a mess. There are hundreds and hundreds of papers on genetic prion disease, and it is often impossible to tell whether a case written up in paper A is already included in a total number reported in paper B. So we started calling around to prion surveillance centers. These are the national authorities that perform autopsies, offer genetic testing, and collect statistics on prion disease, and in Trieste, we'd gotten to meet many of the people who run them. We asked if they'd be willing to share the total number of prion disease cases they'd observed in their country over the past decade or two, the number that had undergone genetic testing, and the number that had each different mutation. In another incredible act of generosity &mdash; both theirs and that of the patients and families who contributed their data &mdash; researchers from nine countries said yes. Within a few months they had given us data on 16,025 prion disease cases, a dataset about ten times larger than the largest case series we had found in the literature.

With this dataset in hand, we could finally start being rigorous about asking which mutations were high penetrance, which were low penetrance, and which were totally benign. Here's a slight oversimplification of how that works: if a mutation appears in 1% of controls but 5% of cases, then it probably increases disease risk by 5-fold. If this were a common disease that you're pretty likely to get anyway, like heart disease or Alzheimer's, then a 5-fold increase in risk would be a really big deal. But remember that in prion disease, the baseline, the lifetime risk in the general population, is only about 1 in 5,000, or 0.02%, so it takes a *huge* increase in risk to be anything clinically meaningful. For instance, one of the mutations we looked at, V180I, is about 50 times more common in cases than in controls, so we estimate that it increases prion disease risk by something like 50-fold. But 50 times 0.02% is still only 1%, so if you have that mutation, you're still far more likely to die of, well, heart disease or Alzheimer's, than you are to die of prion disease. For a mutation to have penetrance anywhere approaching 100% meant that it needed to increase risk thousands of times over.

With these new, firmer, conclusions came a deep and consuming fear. We were now doing stuff that could change people's genetic diagnosis, and yet our conclusions were based on just single-digit counts of mutations in controls. We thought we'd been as rigorous as we could, but the possibility of being wrong was haunting. That fall, at the Partnering for Cures conference in New York, Sonia spotted a group of people from the direct-to-consumer genetics company 23andMe and suggested that we approach them. At the time, 23andMe had sold genetic testing kits to over half a million customers, most of whom (including me) had volunteered their data to be used for research. In the conversations that ensued, after agreeing on careful plans for protecting their customers' privacy, 23andMe offered to partner with us, and so their and their customers' generosity became another crucial ingredient in the stone soup of this research study. I, who had never generated a lick of DNA sequence or genotyping data in my life, was now in the incredibly lucky position to analyze genetic data from over 600,000 people.

In terms of the fear of being wrong, we got to breathe a sigh of relief: the frequencies of different mutations in 23andMe's database were in close agreement with what we had seen in the data that Daniel and his colleagues had assembled (by this time, released publicly as the [Exome Aggregation Consortium](http://exac.broadinstitute.org/) database). An additional confirmation of our conclusions came from statistics on family history in patients with different mutations. Patients with the mutations that we estimated confer only, say, a 1% risk of prion disease, usually have no family history of the disease. The mutations that were common in patients and ultra-rare in controls, like Sonia's mutation, usually came with a family history of dementia.

My mother-in-law was the exception and, we now believe, probably had a *de novo* mutation that neither of her parents had. (This isn't as unlikely as it sounds &mdash; after all, all of these mutations have to start somewhere). Sonia's mutation has a frequency of less than 1 in 100,000 in 23andMe's database, and is absent from 60,706 people in the Exome Aggregation Consortium database. In other words, it turns out this mutation really *is* just as rare as it would have to be to cause such a rare disease. This was terrible news. But it was also what we'd already been assuming for years at this point. And meanwhile, a small but meaningful bit of good news had trickled back from this study.

That first day that Daniel and I had lunch, we had found one patient with a loss-of-function mutation in the prion protein gene, a person who only had 1 working copy of the gene instead of 2. As the database had grown, we eventually found two more such people. The hope was that these people could tell us something about what would happen if we could ever find a drug that would turn off, or turn down, this gene, in order to delay or prevent Sonia's disease. It was an experiment of nature: if these three people had all turned out to suffer from the same mysterious and profound illness, I might start to agree with the skeptics who said not to go tinkering with this gene. But if they were healthy, that would mean that a person can have 50% of the normal dosage of this gene and be just fine. Mice with only 50% gene dosage &mdash; 1 copy instead of 2 &mdash; are partially resistant to prion disease, in that they live about 2.5 times longer than normal mice if infected with prions [[Bueler 1994]]. So if we could reduce Sonia's prion protein levels by half, maybe that could be enough to delay her disease by years, or who knows, maybe even prevent it entirely. So far, no one has been able to develop a drug to reduce prion protein levels by half, but it's not out of the question. For instance, 2015 witnessed [the launch](http://en.hdbuzz.net/204) of a clinical trial of a gene silencing therapy (antisense oligonucleotides) for Huntington's disease, a therapy which, in preclinical studies on mice and monkeys, seemed able to reduce the level of huntingtin protein by about half [[Kordasiewicz 2012]]. Whether such a thing could be achieved for prion disease, we don't yet know, but we would certainly like to know whether it's a strategy worth pursuing. That's what hung in the balance for us.

And it turned out that these three people with loss-of-function mutations in the prion protein gene were healthy, or at least, as healthy as you'd expect three random people to be. One was a control in a schizophrenia study. One had mild type 2 diabetes. One was 78 years old, had no health problems, and was recruited in a study of healthy aging. It's just three people, so there exists a world of possibilities we can't yet rule in or out. Maybe these people do have some very subtle but consistent health problem, or maybe they are at slightly elevated risk for some disease &mdash; but whatever it is, it's not as bad as prion disease. All of this is one piece in the puzzle, one piece of validation from humans, indicating that this one therapeutic strategy may hold promise.

This bit of good news comes at a good time. Four years on, Sonia and I have finally found our scientific home. This past summer, now as PhD students in biology at Harvard Medical School, Sonia and I both joined the laboratory of Stuart Schreiber at the Broad Institute of MIT and Harvard, where we are now doing research with our own four hands to develop therapeutics for prion disease. It is a dream that neither of us had allowed ourselves to believe would really come true. We actually get out of bed each day and go to work on trying to cure her disease. Stuart is an expert on chemistry, chemical biology, discovering drugs, figuring out how drugs work, all of the things that Sonia and I need to understand and don't yet. Our colleagues have come together in spectacular fashion to mentor us, to advise us, to help in any way they can. The Broad Institute carved out a space for us to work, and awarded us a [BroadIgnite](https://www.broadinstitute.org/broadignite) grant, seed funds for equipping a lab and launching into a disease area that no one at the Broad has studied before. I don't know if anyone in history has taken such a leap of faith on two graduate students. 

![](/media/2016/01/eric-sonia-in-lab.png)

*Me and Sonia working in the lab at the Broad Institute. Photo: Maria Nemchuk*

One day last fall, while we were ramping up our work in the lab, Sonia bumped into Robert C. Green, a geneticist at Brigham and Women's Hospital who we've gotten to know through the research community in Boston. He had been following a young woman in seemingly exactly the same position as Sonia had been in four years ago. Her mother had died of prion disease and had been revealed to have a genetic mutation in the prion protein gene. This woman was at 50/50 risk. But her mother's specific mutation was a different one than Sonia's: E196A rather than D178N. We sent Dr. Green a copy of our paper, which was then in review. The E196A mutation was found in about 1 in every 500 Chinese controls, and had only been seen in 2 out of 790 Chinese prion disease patients, neither of whom had a family history of dementia. In other words, E196A is far too common to cause such a rare disease, is no more common in patients than in controls, and doesn't track with disease in families. It doesn't seem to cause prion disease at all. At worst, it might slightly increase risk, but as noted earlier, a slight increase in risk for a very rare disease isn't something worth worrying about.

If it were anyone else, I'd expect a rather cool reception for two upstart PhD students trying to advise an expert in his field on the prognosis of one of his patients. But Dr. Green is an amazing guy. He thanked us. He read the paper. He consulted with his colleagues. He picked up the phone. And so, one day a few weeks ago, a young woman got a call from Adult Genetics at Brigham and Women's Hospital, and learned that she wasn't at 50/50 risk of dying of her mother's disease after all. Instead, she is probably at about 0.02% risk, the same as you or I. Perhaps, for her, this was all just a really weird few months.

To be sure, she's in the minority. Over half of genetic prion disease cases are caused by just three very high penetrance mutations, and the majority of people with genetic prion disease mutations will have been told, correctly, that they are at high risk of developing the disease, just like Sonia. For certain other mutations, those which are very rare both in cases and in controls, it's hard to come by any solid estimate of their penetrance, and so, frustratingly, we can't give patients any firm answers. Meanwhile, at least some of the people with low penetrance mutations already know as much. My co-authors on the paper, who are experts in prion disease, were not entirely surprised by our results; some of them said that on the basis of the low rates of family history of dementia for some mutations, they had already been telling patients they might only be at low or moderate risk of the disease. But it's not clear how widespread this awareness has been outside of a few prion experts. Through our relationships in the patient community, Sonia and I have met three other patients who either have, or are at 50/50 risk for inheriting, what we now believe to be benign or low-risk mutations. In all three instances, they had been counseled that they were at very high risk of prion disease.

What we've found is probably not unique to prion disease. Ed Yong at *The Atlantic* has [argued](http://www.theatlantic.com/science/archive/2015/12/why-human-genetics-research-is-full-of-costly-mistakes/420693/) that there has to be a similar reckoning across a wide range of genetic disorders. For years, the standards for what counts as a pathogenic, disease-causing genetic mutation have been too loose, and there are probably a lot of mutations that will eventually turn out not to be as bad as we think they are. But just because we don't have perfect information doesn't mean that we shouldn't be disclosing genetic information to patients. People have a right to know what's in their DNA, even when our ability to interpret it is imperfect. If someone had waited for more definitive proof that Sonia's mutation was highly penetrant before telling us her genetic status, then who knows when that proof would have arrived. We would be, at a minimum, four years delayed in our quest for a cure.

Speaking of delays, it's time for me to get back to the lab. This story began with a question: does this mean I'll definitely get the disease?

The answer is no.


[Wacholder 1998]: http://www.ncbi.nlm.nih.gov/pubmed/9778168 "Wacholder S, Hartge P, Struewing JP, Pee D, McAdams M, Brody L, Tucker M. The kin-cohort study for estimating penetrance. Am J Epidemiol. 1998 Oct 1;148(7):623-30. PubMed PMID: 9778168."

[Goldwurm 2007]: http://www.ncbi.nlm.nih.gov/pubmed/17215492 "Goldwurm S, Zini M, Mariani L, Tesei S, Miceli R, Sironi F, Clementi M, Bonifati V, Pezzoli G. Evaluation of LRRK2 G2019S penetrance: relevance for genetic counseling in Parkinson disease. Neurology. 2007 Apr 3;68(14):1141-3. Epub 2007 Jan 10. PubMed PMID: 17215492."

[Klug 2013]: http://www.ncbi.nlm.nih.gov/pubmed/23965290 "Klug GM, Wand H, Simpson M, Boyd A, Law M, Masters CL, Matěj R, Howley R, Farrell M, Breithaupt M, Zerr I, van Duijn C, Ibrahim-Verbaas C, Mackenzie J, Will RG, Brandel JP, Alperovitch A, Budka H, Kovacs GG, Jansen GH, Coulthard M, Collins SJ. Intensity of human prion disease surveillance predicts observed disease incidence. J Neurol Neurosurg Psychiatry. 2013 Dec;84(12):1372-7. doi: 10.1136/jnnp-2012-304820. Epub 2013 Aug 21. PubMed PMID: 23965290."

[Bueler 1993]: http://www.ncbi.nlm.nih.gov/pubmed/8100741 "Büeler H, Aguzzi A, Sailer A, Greiner RA, Autenried P, Aguet M, Weissmann C. Mice devoid of PrP are resistant to scrapie. Cell. 1993 Jul 2;73(7):1339-47. PubMed PMID: 8100741."

[Ohashi 2003]: http://www.ncbi.nlm.nih.gov/pubmed/12920113 "Ohashi K, Osuga J, Tozawa R, Kitamine T, Yagyu H, Sekiya M, Tomita S, Okazaki  H, Tamura Y, Yahagi N, Iizuka Y, Harada K, Gotoda T, Shimano H, Yamada N, Ishibashi S. Early embryonic lethality caused by targeted disruption of the 3-hydroxy-3-methylglutaryl-CoA reductase gene. J Biol Chem. 2003 Oct 31;278(44):42936-41. Epub 2003 Aug 14. PubMed PMID: 12920113."

[Bueler 1994]: http://www.ncbi.nlm.nih.gov/pubmed/8790598 "Büeler H, Raeber A, Sailer A, Fischer M, Aguzzi A, Weissmann C. High prion and PrPSc levels but delayed onset of disease in scrapie-inoculated mice heterozygous for a disrupted PrP gene. Mol Med. 1994 Nov;1(1):19-30. PubMed PMID: 8790598; PubMed Central PMCID: PMC2229922."

[Kordasiewicz 2012]: http://www.ncbi.nlm.nih.gov/pubmed/22726834 "Kordasiewicz HB, Stanek LM, Wancewicz EV, Mazur C, McAlonis MM, Pytel KA, Artates JW, Weiss A, Cheng SH, Shihabuddin LS, Hung G, Bennett CF, Cleveland DW.  Sustained therapeutic reversal of Huntington's disease by transient repression of huntingtin synthesis. Neuron. 2012 Jun 21;74(6):1031-44. doi: 10.1016/j.neuron.2012.05.009. PubMed PMID: 22726834; PubMed Central PMCID: PMC3383626."




