---
layout: post
title:  "AGBT-PH 2019"
date:   2019-09-07 12:01:00
author: ericminikel
location: La Jolla, CA
thumb120: http://www.cureffi.org/media/2019/09/agbt-ph-la-jolla-sunset-thumbnail.png
summary200: "Notes from the AGBT2019 Precision Health Conference in La Jolla, CA September 5-7, 2019."
---

![](/media/2019/09/agbt-ph-la-jolla-sunset.png)

*These are my notes from a subset of talks at the [AGBT2019 Precision Health](http://web.archive.org/web/20190908000345/https://www.agbt.org/ph-agenda/) conference held in La Jolla, CA on September 5-7, 2019.*

### Heidi Rehm

Dr. Rehm's slides are [here](https://drive.google.com/file/d/15MaqLMEzA-Q257AFUtJXHHF9ASJ99NPW/view). She opened by telling [this story](https://www.theatlantic.com/science/archive/2015/12/why-human-genetics-research-is-full-of-costly-mistakes/420693/) about a misinterpretation of a clinical genome that could have been avoided through better data sharing. In a happier more recent story, a family contacted her for advice on a sequencing result based on her lab's ClinVar entry, and together they were able to classify it as likely benign. This illustrates that we are making progress in sharing data to enable clinical genome interpretation. But there is still a long way to go, and Dr. Rehm is leading ClinGen, an effort to create a centralized resource that annotates the clinical significance of variants and genes.

Their framework for scoring genetic association evidence is published [[Strande 2017]]. The level of evidence (from "definitive" down to "refuted/disputed") that should be required in order to justify acting on it clinically depends on the use context &mdash; for predictive testing, only the two most definitive categories of associations should be used, whereas for diagnostic testing, associations with moderate evidence can count too. The proportion of genes that were deemed "definitive" varied by disease area and was sometimes quite low. This has changed clinical practice. For example, InVitae now only reports the one high-confidence gene for Brugada syndrome, while allowing physicians to still order the other tests in case they become informative in the future.

### Dennis Lo

Dr. Lo spoke about testing of cell-free DNA in plasma in two contexts: fetal DNA in maternal plasma, and tumor DNA in cancer patients' plasma.

He first reported the existence of fetal DNA in maternal plasma 20 years ago [[Lo 1997]], using Y chromosome DNA as a marker of a male fetus. The proportion of cell-free DNA that is fetus-derived peaked at 15%, enough to support diagnostic testing. Since then, the use of cell-free DNA for fetal testing has exploded [[Chitty & Lo 2015]], for example, about 20% of all pregnancies in China now have this testing done.

The fetal DNA fragments (these are naturally occurring fragments &mdash; the DNA is not sonicated) have a different size distribution than maternal DNA. The size distribution arises from the fact that the mother and the fetus have different preferred fragment end positions, which they have mapped out using informative SNPs [[Chan 2016]]. Using the ratio of these different fetal-preferred ends to maternal-preferred ends, it is possible to calculate the proportion of fetal DNA. They are now cataloguing preferred end sites in different tissues. The different end positions might be due to different positioning of nucleosomes &mdash; they are still mapping out the molecular basis [[Cheng 2018], [Serpas 2019]].

Some of his lab's latest work has been on discriminating linear vs. circular mitochondrial DNA in plasma and correlating this to tissue of origin [[Ma 2019]].

### Katherine High

Dr. High was an HHMI investigator at UPenn / CHOP until five years ago when she left to spin out a startup, Spark Therapeutics. She described the history of gene therapy and her work in developing therapies for *RPE65* and *F9*, some of which is also covered in a recent review [[High & Roncarolo 2019]].

She described gene therapy as having a quiet childhood but difficult adolescence. The first clinical gene transfer occurred in 1990, but the death of Jesse Gelsinger in 1999 and development of leukemia in some children in a trial in 2003 dampened enthusiasm for the modality until more recently.

As I described in my [gene therapy post](/2019/07/23/what-gene-therapy-can-and-cant-do-today/), Spark's voretigene and Avexis's onasemnogene were the first two gene therapy drugs approved by FDA. But [alipogene tiparvovec](https://en.wikipedia.org/wiki/Alipogene_tiparvovec) (Glybera&reg;), back in 2012, was actually the first such drug approved by EMA in Europe. (That drug never got approval in the U.S., only 31 people were ever dosed, and the sponsor has allowed the approval to lapse. European patients with *LPL* mutations now have access to volanesorsen, an antisense oligo against *APOC3*). In just the past couple years, gene therapy IND applications to FDA are now increasing sharply. Most recent efforts involve adeno-associated virus (AAV). Each virus weighs in at about 5.1 MDa molecular weight, of which 74% is protein and 26% is DNA.

She transitioned to talking about Spark's voretigene neparvovec [[Maguire 2008], [Russell 2017]]. There were some big challenges in clinical development of this drug: they had an ultra-small patient population with a dearth of natural history data. Regulators were looking for a clinical endpoint showing improved vision. Because everyone has two eyes, there is a "nearly irresistible temptation" to inject one eye and use the other as a control, but this trial design was considered unacceptable because it is not how anyone would actually use the drug if approved &mdash; everyone would want both eyes treated. They therefore had to do a controlled trial with some participants randomized to placebo. They then offered an open label extension after 1 year, in which the placebo patients transitioned to active drug. This made the design more acceptable to the patient community.

Spark also has a program in hemophilia B, which presents different challenges. Here, the endpoints are well-established from other therapeutics in this class of diseases. But whereas voretigene was a small dose into a confined, immune-privileged tissue (the retina), this program requires dosing large amounts of virus into circulation to reach the liver. In this context, host immune response is a more formidable problem, and at first they achieved only transient transgene expression [[Manno 2006], [Mingozzi 2007]]. To address this problem they have encoded a more potent version of Factor IX (with a potentiating missense mutation), which achieves therapeutic effect with lower expression, into an AAV, and trials are ongoing [[NCT03587116](https://clinicaltrials.gov/ct2/show/NCT03587116)]. 

In Q&A, I asked if Dr. High could offer any advice for those of us developing drugs, as to why and how her *RPE65* gene therapy succeeded while other very similar programs that had commercial sponsors and entered the clinic ultimately failed. She said that success is the aggregate of getting many small things right, and that there is no one single thing that made the difference. Two things she thinks were important were that 1) in their AAV manufacturing, they added a step to get rid of empty AAVs, whose presence would make it more difficult for active AAV to enter the target cells, and 2) they used a more aggressive immunomodulatory regimen to keep patients from rejecting the vectors.

### Sonia Vallabh & Eric Minikel

We had a "fireside chat" interview with Mike Talkowski about our patient-scientist journey, and later in the afternoon we gave a scientific talk (which largely covered the same research from [Sonia's talk at Prion2019](/2019/05/23/prion2019-day-2-3-posters/) so I won't re-blog it here). I mostly wanted to note two interesting discussion points from the Q&A.

Dr. Katherine High noted that recessive loss-of-function mutations in *RPE65* have been referred to by over 20 different clinical names, and that one important struggle in the effort to get voretigene neparvovec approved as a gene therapy was trying to get a label for *RPE65* mutations, rather than for one clinically defined phenotype. Ultimately they prevailed &mdash; the [FDA-approved label](https://www.fda.gov/media/109906/download) is "for the treatment of patients with confirmed biallelic RPE65 mutation-associated retinal dystrophy". She noted that it appears the PrP lowering approach in general, and antisense oligos specifically, will be appropriate for anyone with *PRNP* mutations regardless of clinical subtype. She said therefore one thing we will need to think about is making sure we get the drug approved for (and prescribed for) prion disease and/or *PRNP* mutations as opposed to just one of the historical clinical names (CJD, FFI, etc.)

Dr. Heidi Rehm asked whether, based on our experience, we think there is a minimum threshold level of patient population size necessary in order for it to be feasible to engage an industry partner to develop a drug. We referred back to our ["pistachio analogy"](/2019/04/29/financial-modeling-in-rare-disease/). If technical and regulatory success in drug development were both certain and instant, then basically any disease, no matter how rare, would be profitable (the nut is always delicious); the question is how difficult the nut is to pluck out of the shell, and therefore much work someone else (such as patient groups) needs to do to help make it easy enough. In other words it seems like there is not a binary, but rather a continuum &mdash; the rarer the disease, the more that industry will need a target to be de-risked before they can get involved. (Though the nature of the drug development task can certainly change as you get closer to [N of 1](https://www.statnews.com/2018/10/22/a-tailor-made-therapy-may-have-halted-a-rare-disease/)).

### Susan Slaugenhaupt & Nicolais Naryshkin

Dr. Slaugehaupt has devoted her career to the disease caused by recessive partial loss of function of *ELP1* (formerly known as *IKAP* or *IKBKAP*), which is known clinically as familial dysautonomia (FD), Riley Day syndrome or hereditary sensory and autonomic neuropathy (HSANIII) [[Norcliffe-Kaufmann 2017]]. Most cases are caused by an extended splice site mutation that excludes exon 20, resulting in a tissue-specific reduction in (but not complete loss) of protein levels. This mutation has a carrier frequency of 1/27 in Ashkenazi Jews. After solving the genetic cause of the disease [[Slaugenhaupt 2001]], Dr. Slaugenhaupt partnered with a NINDS drug screening consortium and screened a thousand compounds and found one hit: kinetin [[Slaugenhaupt 2004]], which (after a long effort to develop an faithful mouse model) turned out to work *in vivo* [[Morini 2019]]. They tried testing kinetin in humans, but it turned out it wasn't very potent and required enormous doses that were not well-tolerated. With support from the NINDS Blueprint for Neuroscience Research they launched a three-year-long medicinal chemistry effort to improve upon kinetin, developing and screening 520 analogues in a dual luciferase assay [[Salani 2019]] and identifying 3 compounds to advance to *in vivo* testing. At the end of 2015, their efforts to date attracted the interest of PTC Therapeutics and they signed an agreement to collaborate on developing a new splice-modulating small molecle for *ELP1* patients.

PTC Therapeutics is committed to developing splice-modulating small molecule drugs, and Dr. Naryshkin has led their flagship program for spinal muscular atrophy [[Naryshkin 2014]]. *ELP1* is their second target, and they also have a program in Huntington disease (*HTT*). They were attracted to *ELP1* as a target by the genetically well-defined target, proof-of-concept compound (kinetin), established assay and medicinal chemistry, leading academic collaborator (Dr. Slaugenhaupt), and oustanding patient advocacy organization. Kinetin has a 10 &mu;M potency (concentration to double *ELP1* expression, indicated by E<sub>2X</sub>). The MGH/Blueprint collaboration yielded a 400 nM compound, and PTC has now achieved 0.3 nM. It has clear dose-responsiveness in patient cell lines and *in vivo* in the transgenic mouse brain after oral dosing. They anticipate entry into the clinic this year.

### Bruce Korf

Dr. Korf introduced the Alabama Genomic Health Initiative (AGHI) based at University of Alabama Birmingham in collaboration with HudsonAlpha Institute. They have two cohorts: a population cohort (healthy people) that undergoes a genotyping array and get results returned for any actionable variants, and a phenotypic cohort (affected patients) that undergoes whole genome sequencing with return of pathogenic variants.

In the population cohort, about 1.4% of the 5,000+ participants have had an actionable result returned based on ACMG v2.0 guidelines. The genotyping chip does not include all possible medically actionable variants. A majority (about two-thirds) of population cohort participants who had an actionable result had reported a family history that would be classified as moderate or high risk, but surprisingly, only about half the time was that family history suggestive for the phenotype associated to the actionable variant that was identified &mdash; the rest of the time it was apparently coincidental. For the phenotypes associated to their actionable variants, most of the people would have been assessed as being at a typical population risk level if not for the genotyping result.

In the phenotypic cohort, they have 200 families, some of which are sequnced as trios so in total 491 whole genomes have been sequenced. They've identified putative causal variants in tens of different genes.

They've recently had a few requests for return of raw data, which concerned them because they then have no control over how results are interpreted. They therefore submitted an IRB protocol specifically to educate people on the cautions and caveats of using their own raw data.

Looking forward they are aiming to engage with more clinical practices throughout the state, and start implementing polygenic risk scoring.

### Dan Geschwind

Dr. Geschwind is based at UCLA's Geffen School of Medicine. UC Health has 14-15 million patients, so it presents a big opportunity for genomic data aggregation. He is speaking today about the California Center for Rare Diseases and the UCLA Precision Health project ATLAS, and specifically how they are moving from whole exome to whole genome to RNA-seq for diagnosis. Early on they anticipated that personnel time to consent patients could be a major bottleneck so they developed a video-based consent system, translated into 8 languages. To date, over 50,000 patients have consented, of which 30,000 have opted in and 20,000 have been genotyped. Consistent with Los Angeles demographics, there is no majority ethnic or racial group in the cohort; the plurality are Latino (46%). 

They began offering exome sequencing for rare disease patients in 2012. Since then, they have had 337 physicians from 67 institutions order tests, totaling thousands of patients. The most common type of phenotype is neurological or neurodevelopmental though there are also a lot of neuromuscular diseases. After exome sequencing, they reach a conclusive diagnosis in 26% of cases, potential diagnosis in 22%, and identify a strong novel gene in 3% of cases. "Potential diagnosis" means the variant or gene does not perfectly match the phenotype, so that further phenotyping or segregation analysis is needed in order to decide whether it's causal. The novel disease genes discovered through this program include *KAT6A* [[Arboleda 2015]] and a number of autism genes. A number of different mutational mechanisms can contribute to autism including CNVs [[Luo 2012]], exonic SNPs or short indels, and intronic mutations that affect the transcriptome but may be missed on exome and uninterpretable on whole genome sequencing.

They are finding that moving from exomes to whole genomes and ultimately RNA-seq can improve diagnostic yield. For example, out of a cohort of unsolved patients, they were able to diagnose 11 using exomes alone, and another 7 using whole genomes. Out of 48 that then underwent RNA-seq, they were able to reach a diagnosis in 7. In one example, exome sequencing identified a single paternally-inherited hit in a known recessive disease gene that matched the phenotype; only through RNA-seq were they able to figure out that the second hit was a rare variant inherited from the mother, which was synonymous but turned out to affect splicing. In most cases like this, no deleterious effect on gene expression or function would be predicted by any bioinformatics software &mdash; only RNA-seq will solve it.

[Lo 1997]: https://www.ncbi.nlm.nih.gov/pubmed/9274585 "Lo YM, Corbetta N, Chamberlain PF, Rai V, Sargent IL, Redman CW, Wainscoat JS. Presence of fetal DNA in maternal plasma and serum. Lancet. 1997 Aug 16;350(9076):485-7. PubMed PMID: 9274585."

[Slaugenhaupt 2001]: https://www.ncbi.nlm.nih.gov/pubmed/11179008 "Slaugenhaupt SA, Blumenfeld A, Gill SP, Leyne M, Mull J, Cuajungco MP, Liebert CB, Chadwick B, Idelson M, Reznik L, Robbins C, Makalowska I, Brownstein M, Krappmann D, Scheidereit C, Maayan C, Axelrod FB, Gusella JF. Tissue-specific expression of a splicing mutation in the IKBKAP gene causes familial dysautonomia. Am J Hum Genet. 2001 Mar;68(3):598-605. Epub 2001 Jan 22. PubMed PMID: 11179008; PubMed Central PMCID: PMC1274473."

[Slaugenhaupt 2004]: https://www.ncbi.nlm.nih.gov/pubmed/14709595 "Slaugenhaupt SA, Mull J, Leyne M, Cuajungco MP, Gill SP, Hims MM, Quintero F,  Axelrod FB, Gusella JF. Rescue of a human mRNA splicing defect by the plant cytokinin kinetin. Hum Mol Genet. 2004 Feb 15;13(4):429-36. Epub 2004 Jan 6. PubMed PMID: 14709595."

[Manno 2006]: https://www.ncbi.nlm.nih.gov/pubmed/16474400 "Manno CS, Pierce GF, Arruda VR, Glader B, Ragni M, Rasko JJ, Ozelo MC, Hoots K, Blatt P, Konkle B, Dake M, Kaye R, Razavi M, Zajko A, Zehnder J, Rustagi PK, Nakai H, Chew A, Leonard D, Wright JF, Lessard RR, Sommer JM, Tigges M, Sabatino  D, Luk A, Jiang H, Mingozzi F, Couto L, Ertl HC, High KA, Kay MA. Successful transduction of liver in hemophilia by AAV-Factor IX and limitations imposed by the host immune response. Nat Med. 2006 Mar;12(3):342-7. Epub 2006 Feb 12. Erratum in: Nat Med. 2006 May;12(5):592. Rasko, John [corrected to Rasko, John JE]; Rustagi, Pradip K [added]. PubMed PMID: 16474400."

[Mingozzi 2007]: https://www.ncbi.nlm.nih.gov/pubmed/17369837 "Mingozzi F, Maus MV, Hui DJ, Sabatino DE, Murphy SL, Rasko JE, Ragni MV, Manno CS, Sommer J, Jiang H, Pierce GF, Ertl HC, High KA. CD8(+) T-cell responses to adeno-associated virus capsid in humans. Nat Med. 2007 Apr;13(4):419-22. Epub 2007 Mar 18. PubMed PMID: 17369837."

[Maguire 2008]: https://www.ncbi.nlm.nih.gov/pubmed/18441370 "Maguire AM, Simonelli F, Pierce EA, Pugh EN Jr, Mingozzi F, Bennicelli J, Banfi S, Marshall KA, Testa F, Surace EM, Rossi S, Lyubarsky A, Arruda VR, Konkle B, Stone E, Sun J, Jacobs J, Dell'Osso L, Hertle R, Ma JX, Redmond TM, Zhu X, Hauck B, Zelenaia O, Shindler KS, Maguire MG, Wright JF, Volpe NJ, McDonnell JW,  Auricchio A, High KA, Bennett J. Safety and efficacy of gene transfer for Leber's congenital amaurosis. N Engl J Med. 2008 May 22;358(21):2240-8. doi: 10.1056/NEJMoa0802315. Epub 2008 Apr 27. PubMed PMID: 18441370; PubMed Central PMCID: PMC2829748."

[Luo 2012]: https://www.ncbi.nlm.nih.gov/pubmed/22726847 "Luo R, Sanders SJ, Tian Y, Voineagu I, Huang N, Chu SH, Klei L, Cai C, Ou J, Lowe JK, Hurles ME, Devlin B, State MW, Geschwind DH. Genome-wide transcriptome profiling reveals the functional impact of rare de novo and recurrent CNVs in autism spectrum disorders. Am J Hum Genet. 2012 Jul 13;91(1):38-55. doi: 10.1016/j.ajhg.2012.05.011. Epub 2012 Jun 21. PubMed PMID: 22726847; PubMed Central PMCID: PMC3397271."

[Naryshkin 2014]: https://www.ncbi.nlm.nih.gov/pubmed/25104390 "Naryshkin NA, Weetall M, Dakka A, Narasimhan J, Zhao X, Feng Z, Ling KK, Karp  GM, Qi H, Woll MG, Chen G, Zhang N, Gabbeta V, Vazirani P, Bhattacharyya A, Furia B, Risher N, Sheedy J, Kong R, Ma J, Turpoff A, Lee CS, Zhang X, Moon YC, Trifillis P, Welch EM, Colacino JM, Babiak J, Almstead NG, Peltz SW, Eng LA, Chen KS, Mull JL, Lynes MS, Rubin LL, Fontoura P, Santarelli L, Haehnke D, McCarthy KD, Schmucki R, Ebeling M, Sivaramakrishnan M, Ko CP, Paushkin SV, Ratni H, Gerlach I, Ghosh A, Metzger F. Motor neuron disease. SMN2 splicing modifiers improve motor function and longevity in mice with spinal muscular atrophy. Science. 2014 Aug 8;345(6197):688-93. doi: 10.1126/science.1250127. PubMed PMID:  25104390."

[Chitty & Lo 2015]: https://www.ncbi.nlm.nih.gov/pubmed/26187875 "Chitty LS, Lo YM. Noninvasive Prenatal Screening for Genetic Diseases Using Massively Parallel Sequencing of Maternal Plasma DNA. Cold Spring Harb Perspect Med. 2015 Jul 17;5(9):a023085. doi: 10.1101/cshperspect.a023085. Review. PubMed PMID: 26187875; PubMed Central PMCID: PMC4561399."

[Arboleda 2015]: https://www.ncbi.nlm.nih.gov/pubmed/25728775 "Arboleda VA, Lee H, Dorrani N, Zadeh N, Willis M, Macmurdo CF, Manning MA, Kwan A, Hudgins L, Barthelemy F, Miceli MC, Quintero-Rivera F, Kantarci S, Strom  SP, Deignan JL; UCLA Clinical Genomics Center, Grody WW, Vilain E, Nelson SF. De  novo nonsense mutations in KAT6A, a lysine acetyl-transferase gene, cause a syndrome including microcephaly and global developmental delay. Am J Hum Genet. 2015 Mar 5;96(3):498-506. doi: 10.1016/j.ajhg.2015.01.017. Epub 2015 Feb 26. PubMed PMID: 25728775; PubMed Central PMCID: PMC4375619."

[Chan 2016]: https://www.ncbi.nlm.nih.gov/pubmed/27799561 "Chan KC, Jiang P, Sun K, Cheng YK, Tong YK, Cheng SH, Wong AI, Hudecova I, Leung TY, Chiu RW, Lo YM. Second generation noninvasive fetal genome analysis reveals de novo mutations, single-base parental inheritance, and preferred DNA ends. Proc Natl Acad Sci U S A. 2016 Dec 13;113(50):E8159-E8168. Epub 2016 Oct 31. PubMed PMID: 27799561; PubMed Central PMCID: PMC5167168."

[Strande 2017]: https://www.ncbi.nlm.nih.gov/pubmed/28552198 "Strande NT, Riggs ER, Buchanan AH, Ceyhan-Birsoy O, DiStefano M, Dwight SS, Goldstein J, Ghosh R, Seifert BA, Sneddon TP, Wright MW, Milko LV, Cherry JM, Giovanni MA, Murray MF, O'Daniel JM, Ramos EM, Santani AB, Scott AF, Plon SE, Rehm HL, Martin CL, Berg JS. Evaluating the Clinical Validity of Gene-Disease Associations: An Evidence-Based Framework Developed by the Clinical Genome Resource. Am J Hum Genet. 2017 Jun 1;100(6):895-906. doi: 10.1016/j.ajhg.2017.04.015. Epub 2017 May 25. PubMed PMID: 28552198; PubMed Central PMCID: PMC5473734."

[Russell 2017]: https://www.ncbi.nlm.nih.gov/pubmed/28712537 "Russell S, Bennett J, Wellman JA, Chung DC, Yu ZF, Tillman A, Wittes J, Pappas J, Elci O, McCague S, Cross D, Marshall KA, Walshire J, Kehoe TL, Reichert H, Davis M, Raffini L, George LA, Hudson FP, Dingfield L, Zhu X, Haller JA, Sohn EH, Mahajan VB, Pfeifer W, Weckmann M, Johnson C, Gewaily D, Drack A, Stone E, Wachtel K, Simonelli F, Leroy BP, Wright JF, High KA, Maguire AM. Efficacy and safety of voretigene neparvovec (AAV2-hRPE65v2) in patients with RPE65-mediated inherited retinal dystrophy: a randomised, controlled, open-label, phase 3 trial. Lancet. 2017 Aug 26;390(10097):849-860. doi: 10.1016/S0140-6736(17)31868-8. Epub  2017 Jul 14. Erratum in: Lancet. 2017 Aug 26;390(10097):848. PubMed PMID: 28712537; PubMed Central PMCID: PMC5726391."

[Norcliffe-Kaufmann 2017]: https://www.ncbi.nlm.nih.gov/pubmed/27317387 "Norcliffe-Kaufmann L, Slaugenhaupt SA, Kaufmann H. Familial dysautonomia: History, genotype, phenotype and translational research. Prog Neurobiol. 2017 May;152:131-148. doi: 10.1016/j.pneurobio.2016.06.003. Epub 2016 Jun 15. Review.  PubMed PMID: 27317387."

[Cheng 2018]: https://www.ncbi.nlm.nih.gov/pubmed/29097509 "Cheng THT, Lui KO, Peng XL, Cheng SH, Jiang P, Chan KCA, Chiu RWK, Lo YMD. DNase1 Does Not Appear to Play a Major Role in the Fragmentation of Plasma DNA in a Knockout Mouse Model. Clin Chem. 2018 Feb;64(2):406-408. doi: 10.1373/clinchem.2017.280446. Epub 2017 Nov 2. Erratum in: Clin Chem. 2018 May;64(5):869. PubMed PMID: 29097509."

[Serpas 2019]: https://www.ncbi.nlm.nih.gov/pubmed/30593563 "Serpas L, Chan RWY, Jiang P, Ni M, Sun K, Rashidfarrokhi A, Soni C, Sisirak V, Lee WS, Cheng SH, Peng W, Chan KCA, Chiu RWK, Reizis B, Lo YMD. Dnase1l3 deletion causes aberrations in length and end-motif frequencies in plasma DNA. Proc Natl Acad Sci U S A. 2019 Jan 8;116(2):641-649. doi: 10.1073/pnas.1815031116. Epub 2018 Dec 28. PubMed PMID: 30593563; PubMed Central PMCID: PMC6329986."

[Ma 2019]: https://www.ncbi.nlm.nih.gov/pubmed/31263037 "Ma ML, Zhang H, Jiang P, Sin STK, Lam WKJ, Cheng SH, Lee WS, Gai W, Tse OYO, Peng W, Wong J, Raghupathy R, Wong RSM, Sahota D, Leung TY, Chan KCA, Chiu RWK, Lo YMD. Topologic Analysis of Plasma Mitochondrial DNA Reveals the Coexistence of Both Linear and Circular Molecules. Clin Chem. 2019 Sep;65(9):1161-1170. doi: 10.1373/clinchem.2019.308122. Epub 2019 Jul 1. PubMed PMID: 31263037."

[Morini 2019]: https://www.ncbi.nlm.nih.gov/pubmed/30905397 "Morini E, Gao D, Montgomery CM, Salani M, Mazzasette C, Krussig TA, Swain B, Dietrich P, Narasimhan J, Gabbeta V, Dakka A, Hedrick J, Zhao X, Weetall M, Naryshkin NA, Wojtkiewicz GG, Ko CP, Talkowski ME, Dragatsis I, Slaugenhaupt SA.  ELP1 Splicing Correction Reverses Proprioceptive Sensory Loss in Familial Dysautonomia. Am J Hum Genet. 2019 Apr 4;104(4):638-650. doi: 10.1016/j.ajhg.2019.02.009. Epub 2019 Mar 21. PubMed PMID: 30905397; PubMed Central PMCID: PMC6451698."

[Salani 2019]: https://www.ncbi.nlm.nih.gov/pubmed/30085848 "Salani M, Urbina F, Brenner A, Morini E, Shetty R, Gallagher CS, Law EA, Sunshine S, Finneran DJ, Johnson G, Minor L, Slaugenhaupt SA. Development of a Screening Platform to Identify Small Molecules That Modify ELP1 Pre-mRNA Splicing in Familial Dysautonomia. SLAS Discov. 2019 Jan;24(1):57-67. doi: 10.1177/2472555218792264. Epub 2018 Aug 7. PubMed PMID: 30085848."

[High & Roncarolo 2019]: https://www.ncbi.nlm.nih.gov/pubmed/31365802 "High KA, Roncarolo MG. Gene Therapy. N Engl J Med. 2019 Aug 1;381(5):455-464.  doi: 10.1056/NEJMra1706910. Review. PubMed PMID: 31365802."

[Thauland 2019]: https://www.ncbi.nlm.nih.gov/pubmed/31031754 "Thauland TJ, Pellerin L, Ohgami RS, Bacchetta R, Butte MJ. Case Study: Mechanism for Increased Follicular Helper T Cell Development in Activated PI3K Delta Syndrome. Front Immunol. 2019 Apr 12;10:753. doi: 10.3389/fimmu.2019.00753. eCollection 2019. PubMed PMID: 31031754; PubMed Central PMCID: PMC6473200."
