---
layout: post
title:  "What's the deal with GPI-anchored proteins?"
date:   2019-05-14 12:01:00
author: ericminikel
location: Cambridge, MA
thumb120: http://www.cureffi.org/media/2019/05/gpi-thumbnail.png
summary200: "My protein of interest, PrP, is a GPI-anchored protein. What does that mean, and is it important?"
---

GPI-anchored proteins are the odd man out. In [introductory Cell Biology](/2013/02/24/cell-biology-04-the-secretory-pathway/), they taught us that there were five types of membrane proteins, named as follows: Type I, Type II, Type III, Type IV, and GPI-anchored. Why do we have this strange class of proteins fused to a sugar-and-fat chain? What do they do? Can we gain any insights into my protein of interest &mdash; PrP &mdash; by learning more about this class of proteins it is a member of?

Sonia and I and our teammate Andrew and have been doing some reading on this subject and I'm writing this blog post to share some of what we learned.

### reading

We started out by reading a few reviews [[Ferguson & Williams 1988], [Englund 1993], [Udenfriend & Kodukula 1995], [Mayor & Riezman 2004], [Fujita & Kinoshita 2010], [Kinoshita & Fujita 2016]]. These mostly covered the structure and biogenesis of the GPI anchor itself, about which an amazing about is known.

This anchor, whose full name is glycosylphosphatidylinositol, is not a monolith: it's a general description of a molecule whose details may vary. In general, starting from the &omega; (last post-translationally present) residue of the protein, you have ethanolamine, then a phosphate, then some sugars, then a phospholipid. The core sugar backbone is conserved, but the side chains branching off of it can vary, and the phosholipid head group and fatty acids can both vary as well. PrP's GPI anchor was characterized in [[Stahl 1992]], but even then it's not a monolith &mdash; they identified at least six different structures differing in sugar side chain composition.

Every chemical structure I have found of GPI anchors has at least some parts abbreviated or summarized, and the protein is usually just shown as a picture. I wanted to get a sense of what these anchors actually look like chemically, in the context of their attached proteins, so I set out to actually draw one complete structure in ChemDraw. Working from [Figure 1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2663890/figure/fig1/) of [[Paulick & Bertozzi 2008]] &mdash; the closest thing to a complete skeletal structure I could find &mdash; I added on the details of one of PrP's GPI anchors from the top panel of [[Stahl 1992]] Figure 6. The molecular weight came to 1,958 Da, so for context I drew in the final 16 residues of HuPrP23-230, which weigh in comparably, at 1,979 Da. This is about 8% of PrP's post-translationally modified sequence. I am not certain I got every bond right, but here's what I came up with:

![](/media/2019/05/prp-gpi-anchor-scale.png)

In many cases, a gene has multiple isoforms, with one splicing product giving rise to a GPI-anchored protein while others give rise to secreted or transmembrane forms. Examples include *NCAM1*, which has three major isoforms, one of which is GPI-anchored and the other two of which are transmembrane [[Owens 1987]], and *ACHE* (encoding acetylcholinesterase), whose GPI-anchored form is apparently found only on red blood cells ([NCBI Genes](https://www.ncbi.nlm.nih.gov/gene/43)). The most fascinating story here is that of the mouse gene *Ly6a* which, thanks to a genetic polymorphism, is GPI-anchored in some mouse strains and not others. Only in its GPI-anchored form, it acts as the receptor for the viral vector AAV PHP.eB [[Huang 2019]]. (This vector achieves amazingly efficient uptake into brain neurons for gene therapy [[Chan 2017]], but sadly, it's a mouse gene only &mdash; we humans don't even have *Ly6a*).

A lot is known about how GPI anchors get synthesized and attached to proteins [[Kinoshita & Fujita 2016]], with >20 proteins involved in the pathway, most of which start with the prefix "PIG" and are encoded by genes such as *PIGA*, *PIGK*, and so on &mdash; see [Figure 2](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4689344/figure/f2/) for a diagram. Most of the biosynthesis takes place with the anchor inserted in the membrane in the ER but not attached to any protein. In fact, the first few steps take place on the cytosolic leaflet of the membrane, and only later does the anchor flip to the lumenal side (inside the ER). The final step is when GPI transamidase, a complex comprised of at least five proteins, cleaves the GPI signal off of the protein's C terminus and attaches the GPI anchor to the so-called &omega; residue of the protein (the last residue in the post-translationally modified sequence). There is then further maturation of the GPI anchor as the protein migrates out of the ER toward the cell surface.

There exist a number of small molecule inhibitors of GPI biosynthesis in fungi, some of which people have tried to develop as antifungal drugs [[Kinoshita & Fujita 2016]], but as far as I could tell, the only known inhibitor of GPI biosynthesis in mammalian cells is mannosamine, a mannose analog that is chemically incompatible with incorporation into GPI [[Pan 1992], [Sevlever & Rosenberry 1993]].

I looked and looked for a sequence logo of what amino acid sequence motif GPI transamidase recognizes, but found none. Apparently the sequence motif is quite loose [[Eisenhaber 1999]], and apparently the GPI signals are not even homologous [[Englund 1993]], meaning they did not evolve from a common ancestral sequence, but rather evolved convergently, to the extent that there is even any convergence. The best description I've been able to find is that (reading N-to-C-terminal up to the end of the protein) you need 1) about 11 residues of an unstructured linker, 2) a few residues with small side chains including an &omega; residue which can be either S, N, D, G, A, or C, 3) a spacer of 5-10 polar amino acids, and finally 4) 15-20 hydrophobic amino acids [[Eisenhaber 1999], [Kinoshita & Fujita 2016]]. PrP loosely follows this motif. According to published structures [e.g. [Antonyuk 2009]], alpha helix 3 ends at residue Q223, which leaves the 'unstructured linker' as just AYYQR (somewhat shorter than the prescribed 11 residues). The 'small side chain' region would be GS\|SM (with the pipe denoting the transamidase cut site), the polar region would be VLFSSPP, and the hydrophobic C terminus as VILLISFLIFLIVG.

Some of the proteins in the GPI biosynthesis and attachment pathway are very important, and a number of severe diseases and syndromes of GPI anchor deficiency have been described, due to biallelic loss-of-function or apparently hypomorphic missense mutations in genes like *PIGO*, *PIGV*, *PIGW*, *PGAP2*, and *PGAP3* [[Kinoshita & Fujita 2016]].

Sonia found an excellent paper from a few years ago where they did a mutagenesis screen in haploid human cells to identify genes required for biogenesis of two GPI-anchored proteins: PrP and CD59 [[Davis 2015]]. They used repeated FACS sorting of cells based on cell surface PrP and CD59 in order to identify cells with dramatically reduced surface levels of these proteins, and then did sequencing to see which gene knockouts were enriched in those cells versus the parent population. As you'd expect, most of the PIG genes came up for both proteins ([Figure 4](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481156/figure/F4/)), but not all hits overlapped, which is a bit surprising, especially since at the RNA level, at least, PrP and CD59 are two of the proteins with the most similar expression profiles across tissues (see heat map at the bottom of this post). A bunch of enzymes involved in GPI anchor side chain modification turned up only for CD59, suggesting that CD59, but not PrP, needs these complex side chains in order to mature and reach the cell surface. Meanwhile, Sec62 and Sec63 turned up only for PrP &mdash; these are proteins somehow involved in co-translational translocation into the ER, but apparently, they are needed for PrP but not for CD59 nor for CD55 or CD109, two other control proteins they looked at. This is a fascinating new chapter in the answer to my question, ["is anything special about PrP's expression?"](/2015/10/21/is-there-anything-special-about-prps-expression/), where I was looking for anything unique about the biogenesis of PrP that could be potentially targetable with a small molecule. Of course, just because these proteins weren't important for three other control proteins in [[Davis 2015]] doesn't mean they are not important &mdash; one study found that Sec62 was needed for secretion of many small proteins [[Lakkaraju 2012]], and the [*SEC62*](http://exac.broadinstitute.org/gene/ENSG00000008952) gene is totally depleted for loss-of-function variants in the human population, enough to suggest haploinsufficiency. [*SEC63*](http://exac.broadinstitute.org/gene/ENSG00000025796) appears less constrained, though that could just mean it acts recessively.

None of the above answers the question of why GPI-anchored proteins exist. My old cell biology class omitted a detail, by the way: there is actually a sixth class of membrane proteins, called tail-anchored (TA) proteins [[Hegde & Keenan 2011]], which just have a hydrophobic C terminus that sticks into the membrane but does not protrude out the other side. Why couldn't all of these GPI anchored proteins just be TA proteins? Why did cells evolve such a complicated pathway to synthesize a sugar-fat anchor instead, and why did they evolve it so early in the game &mdash; GPI anchors are present throughout eukaryotes including in many single-celled pathogens that infect humans.

Most of the reviews did not spend much time on this question, probably because it's the hardest thing to answer. The GPI-anchored proteins themselves, to the extent that their native functions are known, have a huge range of functions &mdash; there are enzymes (such as AChE), cell adhesion molecules (such as NCAM1), proteins that regulate complement in the immune system (CD59), and so on [[Paulick & Bertozzi 2008]]. There is apparently at least one GPI-anchored protein involved in myelin maintenance in peripheral nerves [[Kuffer & Lakkaraju 2016]]. But what exactly can GPI-anchored proteins do that other proteins can't? One review [[Kinoshita & Fujita 2016]] cites a few ideas that have been proposed. One is that GPI-anchored proteins are good at transiently dimerizing [[Suzuki 2012]]. Some studies have explored the idea that homodimerization plays some role in prion biology [[Meier 2003], [Beland 2012]], though the relevance of the model systems used there to the *in vivo* situation is not yet clear. Another idea is that because GPI-anchored proteins can be shed from the cell surface, for instance by angiotensin-converting enzyme (ACE) [[Kondoh 2005]], their localization can be regulated in some dynamic way. Here too, we know that [PrP can be shed](/2013/06/29/proteolytic-shedding-of-prion-protein/), apparently by the enzyme ADAM10 [[Altmeppen 2011], [Linsenmeier 2018]], though any role in PrP native function is not yet clear. A third idea, and perhaps the one I have heard the most talk about, is that GPI-anchored proteins selectively congregate in "lipid rafts" [[Brown & Rose 1992], [Simons & Ikonen 1997]]. This is perhaps the most enticing explanation, because you could imagine all kinds of knock-on effects, where the increased effective local concentration of these proteins allows for more interactions, and so on. But one review pointed out that a caveat is that lipid rafts are still more of an abstract idea than a concrete thing [[Mayor & Riezman 2004]] &mdash; while they are functionally defined by detergent insolubility and most people describe them as being rich in sphingomyelin and cholesterol, there is no universally accepted definition of what is and is not a lipid raft, and the empirical evidence suggests they may be much smaller and more transient than most people think.

With that reading in hand, I set out to get a list of these proteins and do some analysis on them to see if I could get a better sense of what they're like.

### analyses

Uniprot has a list of [173 human GPI-anchored proteins](https://www.uniprot.org/uniprot/?query=keyword:%22GPI-anchor%20[KW-0336]%22&fil=organism%3A%22Homo+sapiens+%28Human%29+%5B9606%5D%22). These [mapped](https://www.uniprot.org/uploadlists/) to 140 gene symbols, which dropped to 135 after running [this script](https://github.com/macarthur-lab/gene_lists/blob/master/src/update_symbols.py) to update to currently HGNC-approved protein-coding gene symbols. The final list of 135 gene symbols is [here](https://github.com/macarthur-lab/gene_lists/blob/master/lists/gpi_anchored.tsv).

Uniprot doesn't offer any info on how their annotations were generated, though there must be a significant degree of manual curation. For comparison, Andrew also dug up a series of neat papers that used PI-PLD or PI-PLC, two enzymes that cleave GPI anchors, to empirically isolate GPI-anchored proteins from cells [[Elortza 2006], [Cortes 2014], [Masuishi 2016]]. Combining lists from these papers and mapping to current gene symbols yielded [107 genes](/media/2019/05/gpi_anchored_empirical.tsv). We spot-checked several of these at random. Among them were well-known GPI-anchored proteins such as glypican-1 (*GPC1*) and neural cell adhesion molecule (*NCAM1*), both of which are reported to have interactions with PrP [[Schmitt-Ulms 2001], [Taylor 2009]]. But also present were several genes for which no GPI anchoring seemed to be known in the literature, such as *VDAC3*, some of which may simply be very abundant proteins or false positives for other reasons. Meanwhile, there are obvious sources of false negative: genes that simply weren't expressed in the cell line studied, or weren't abundant enough to pick up by mass spec, and the PrP paralogs *SPRN* and *PRND* weren't in the lists. Overall, 51 genes were in both lists, a highly significant enrichment (OR = 217, *P* < 1 &times; 10<sup>-84</sup>), which helps reassure me that Uniprot's annotations are consistent with empirical data. But for further analyses we decided to go with the Uniprot list as it seems more sensitive and specific.

Armed with this list, I wanted to see how GPI-anchored proteins stack up. PrP is a single exon, short (208 amino acids in its mature form), non-essential, broadly-expressed protein. Are these features typical or atypical for a GPI-anchored protein?

It turns out that GPI-anchored proteins are all over the map, just as variable on every dimension I looked at as any other set of proteins are.

First, length. Below are overlaid histograms of the coding sequence length in base pairs for all genes, versus genes encoding GPI-anchored proteins. The GPI-anchored distribution is just barely left-shifted. The average GPI-anchored protein gene has 1,301 bp of coding sequence, while the average gene has 1,729, but this difference of means is small compared to the variation within each group. PrP, with just 762 bp of coding sequence, is definitely on the small side, though by no means an outlier in either group &mdash; *CD52*, with just 186 base pairs of sequence and apparently only 12 amino acids in its mature form [[Xia 1991]], is the smallest GPI-anchored protein.

![](/media/2019/05/gpi-vs-all-length.png)

What about number of exons? GPI-anchored proteins do have slightly fewer exons on average, compared to all genes (mean 7.8 vs. 10.1), consistent with the slight length distribution difference noted above, but most are multi-exon. Here, too, PrP is on the small side: there are only six GPI-anchored proteins that have just 1 coding exon, and three of them are PrP and its two paralogs, Sho and Dpl. (The other three genes are *GAS1*, *SPACA4*, and the fabulously named *OMG*). 

Next I looked at loss-of-function constraint. Constraint [[Samocha 2014], [Lek 2016]] is a measure of how strong of natural selection a gene is under, based on how depleted it is for, say, nonsense, frameshit, and splice site variation in the general population compared to expectation based on mutation rates. This metric is not very interpretable for short genes, both for statistical reasons (the number of expected mutations is low for short genes, so it is hard to quantify depletion) and biological reasons (single exon genes aren't subject to nonsense-mediated decay, so it is harder to know if protein-truncating variants are really "loss-of-function" or not). But since most GPI-anchored proteins are not as short as PrP, I thought it was worth taking a look. The result: on average, GPI-anchored proteins are just slightly less constrained, meaning they have more of their expected amount of loss-of-function variation, than the average gene. The average gene has 47% of its loss-of-function variation, and GPI-anchored proteins have 56%. But as with everything here, there's a wide distribution in both camps. For GPI-anchored proteins, you have the absolutely constrained [*ACHE*](http://exac.broadinstitute.org/gene/ENSG00000087085) (17 LoFs expected and none observed) at one end and, at the other end, several genes that appear to be under no selection against loss-of-function at all &mdash; *CNTN6*, *CD109*, *TREH*, and *MSLN* are a few examples. *PRNP* falls in the latter camp once you exclude residues &ge;145 where protein-truncating variants cause a gain of function [[Minikel 2016], [Minikel 2019]].

Finally, I wondered where GPI-anchored proteins are expressed. *PRNP* is highest in the brain but is expressed everywhere. Is that typical? I [downloaded](https://gtexportal.org/home/datasets) the full GTEx v7 "gene median tpm" summary file (Jan 15, 2016), where every row is a gene and every column is a tissue and the cells are RPKMs &mdash; RNA-seq reads per kilobase of exon per million mapped reads. Working with this dataset took some finessing. I've heard that some bioinformaticians consider <1 RPKM to be "not expressed" but the expression matrix is spare &mdash; most genes are not highly expressed in most tissues &mdash; so the noise below 1 RPKM can dominate if you just plot the raw RPKMs. Meanwhile, gene expression is something that you need to think of on a log scale, as genes in a tissue can vary from <1 RPKM to >10,000 RPKM, so if you consider everything on a linear scale, then the few really highly expressed gene/tissue combinations can also dominate, making the matrix look even sparser than it is. I therefore took log10 of the matrix and truncated the distribution at \[0,4\], thus, the purple scale I used runs 1 - 10 - 100 - 1,000 - 10,000 RPKM. Then I subsetted to the Uniprot GPI-anchored proteins. To visualize this, I made a heatmap for the first time in my life. I've often seen these in papers and they usually don't speak to me, but here my goal was just to get a sense of the pattern of expression, and after playing around a bit, this was what gave me the most insight. The principle of a heatmap is that the rows and columns are clustered so that similar things go together. Thus, for instance, all the brain tissue columns are lined up consecutively in a patch on the x axis, and all the highly brain-expressed genes are lined up consecutively in a patch on the y axis, so that their intersection forms a dense purple rectangle that can be interpreted as, "there exists a cluster of genes that are mostly brain-expressed".

Interested readers can view the [full-scale vector art PDF](/media/2019/05/gpi-gtex-heatmap.pdf) of the heatmap, but to make it more immediately accessible, here is a hand-annnotated version calling out the clusters of interest:

![](/media/2019/05/gpiap-gtex-heatmap-annotations.png)

The answer, then, is no &mdash; most GPI-anchored proteins do not have the same expression pattern as *PRNP* does. *PRNP* is one of the handful of more highly and broadly expressed ones, featuring near the top of this heatmap, along with *CD59*, *LY6E*, *GPC1*, and *BST2*. Most GPI-anchored proteins have lower or more tissue-restricted expression, with some almost uniquely expressed in the brain and others almost uniquely *not* expressed in the brain, and other smaller clusters belonging chiefly to specific tissues like the testes, such as PrP's paralog *PRND*, whose knockout causes male sterility [[Behrens 2002], [Paisley 2004]].

### conclusions

GPI-anchored proteins can be just about any size, expressed in just about any tissue, and apparently have just about any function, to the extent that their functions are known. Many GPI-anchored proteins have very clear native functions, but these functions are diverse and it is not clear why they require GPI anchoring, especially since many of these proteins exist in non-GPI-anchored isoforms as well. Meanwhile, for other GPI-anchored proteins, including PrP, we know little enough about native function to begin with, so it is difficult to even speculate why the native function requires GPI anchoring.  None of the analyses I did or reviews I read were able to draw a unifying principle as to why this anchoring mechanism exists or what makes these proteins require it. There are a number of hypotheses for why GPI-anchored proteins are unique, including lipid rafts, homodimers, and shedding. All of these hypotheses may hold some water. But at the end of the day, the answer seems unlikely to be a eureka moment, but rather, like so much of biology, a prosaic mix of different things.

*R code and raw data files for analyses in this post are [here](https://github.com/ericminikel/gpi)*.

[Owens 1987]: https://www.ncbi.nlm.nih.gov/pubmed/3025862 "Owens GC, Edelman GM, Cunningham BA. Organization of the neural cell adhesion  molecule (N-CAM) gene: alternative exon usage as the basis for different membrane-associated domains. Proc Natl Acad Sci U S A. 1987 Jan;84(1):294-8. PubMed PMID: 3025862; PubMed Central PMCID: PMC304190."

[Ferguson & Williams 1988]: https://www.ncbi.nlm.nih.gov/pubmed/3052274/ "Ferguson MA, Williams AF. Cell-surface anchoring of proteins via glycosyl-phosphatidylinositol structures. Annu Rev Biochem. 1988;57:285-320. Review. PubMed PMID: 3052274."

[Xia 1991]: https://www.ncbi.nlm.nih.gov/pubmed/1711975 "Xia MQ, Tone M, Packman L, Hale G, Waldmann H. Characterization of the CAMPATH-1 (CDw52) antigen: biochemical analysis and cDNA cloning reveal an unusually small peptide backbone. Eur J Immunol. 1991 Jul;21(7):1677-84. PubMed PMID: 1711975."

[Brown & Rose 1992]: https://www.ncbi.nlm.nih.gov/pubmed/1531449/ "Brown DA, Rose JK. Sorting of GPI-anchored proteins to glycolipid-enriched membrane subdomains during transport to the apical cell surface. Cell. 1992 Feb 7;68(3):533-44. PubMed PMID: 1531449."

[Deeg 1992]: https://www.ncbi.nlm.nih.gov/pubmed/1388156 "Deeg MA, Humphrey DR, Yang SH, Ferguson TR, Reinhold VN, Rosenberry TL. Glycan components in the glycoinositol phospholipid anchor of human erythrocyte acetylcholinesterase. Novel fragments produced by trifluoroacetic acid. J Biol Chem. 1992 Sep 15;267(26):18573-80. PubMed PMID: 1388156."

[Stahl 1992]: https://www.ncbi.nlm.nih.gov/pubmed/1350920 "Stahl N, Baldwin MA, Hecker R, Pan KM, Burlingame AL, Prusiner SB. Glycosylinositol phospholipid anchors of the scrapie and cellular prion proteins  contain sialic acid. Biochemistry. 1992 Jun 2;31(21):5043-53. PubMed PMID: 1350920."

[Pan 1992]: https://www.ncbi.nlm.nih.gov/pubmed/1400435 "Pan YT, Kamitani T, Bhuvaneswaran C, Hallaq Y, Warren CD, Yeh ET, Elbein AD. Inhibition of glycosylphosphatidylinositol anchor formation by mannosamine. J Biol Chem. 1992 Oct 15;267(29):21250-5. PubMed PMID: 1400435."

[Sevlever & Rosenberry 1993]: https://www.ncbi.nlm.nih.gov/pubmed/8496158 "Sevlever D, Rosenberry TL. Mannosamine inhibits the synthesis of putative glycoinositol phospholipid anchor precursors in mammalian cells without incorporating into an accumulated intermediate. J Biol Chem. 1993 May 25;268(15):10938-45. PubMed PMID: 8496158."

[Englund 1993]: https://www.ncbi.nlm.nih.gov/pubmed/8352586/ "Englund PT. The structure and biosynthesis of glycosyl phosphatidylinositol protein anchors. Annu Rev Biochem. 1993;62:121-38. Review. PubMed PMID: 8352586."

[Udenfriend & Kodukula 1995]: https://www.ncbi.nlm.nih.gov/pubmed/7574493/ "Udenfriend S, Kodukula K. How glycosylphosphatidylinositol-anchored membrane proteins are made. Annu Rev Biochem. 1995;64:563-91. Review. PubMed PMID: 7574493."

[Simons & Ikonen 1997]: https://www.ncbi.nlm.nih.gov/pubmed/9177342 "Simons K, Ikonen E. Functional rafts in cell membranes. Nature. 1997 Jun 5;387(6633):569-72. Review. PubMed PMID: 9177342."

[Eisenhaber 1999]: https://www.ncbi.nlm.nih.gov/pubmed/10497036/ "Eisenhaber B, Bork P, Eisenhaber F. Prediction of potential GPI-modification sites in proprotein sequences. J Mol Biol. 1999 Sep 24;292(3):741-58. PubMed PMID: 10497036."

[Schmitt-Ulms 2001]: https://www.ncbi.nlm.nih.gov/pubmed/11743735 "Schmitt-Ulms G, Legname G, Baldwin MA, Ball HL, Bradon N, Bosque PJ, Crossin KL, Edelman GM, DeArmond SJ, Cohen FE, Prusiner SB. Binding of neural cell adhesion molecules (N-CAMs) to the cellular prion protein. J Mol Biol. 2001 Dec 14;314(5):1209-25. PubMed PMID: 11743735."

[Behrens 2002]: https://www.ncbi.nlm.nih.gov/pubmed/12110578/ "Behrens A, Genoud N, Naumann H, Rülicke T, Janett F, Heppner FL, Ledermann B,  Aguzzi A. Absence of the prion protein homologue Doppel causes male sterility. EMBO J. 2002 Jul 15;21(14):3652-8. PubMed PMID: 12110578; PubMed Central PMCID: PMC125402."

[Meier 2003]: https://www.ncbi.nlm.nih.gov/pubmed/12679034 "Meier P, Genoud N, Prinz M, Maissen M, Rülicke T, Zurbriggen A, Raeber AJ, Aguzzi A. Soluble dimeric prion protein binds PrP(Sc) in vivo and antagonizes prion disease. Cell. 2003 Apr 4;113(1):49-60. PubMed PMID: 12679034."

[Paisley 2004]: https://www.ncbi.nlm.nih.gov/pubmed/15161660/ "Paisley D, Banks S, Selfridge J, McLennan NF, Ritchie AM, McEwan C, Irvine DS, Saunders PT, Manson JC, Melton DW. Male infertility and DNA damage in Doppel knockout and prion protein/Doppel double-knockout mice. Am J Pathol. 2004 Jun;164(6):2279-88. PubMed PMID: 15161660; PubMed Central PMCID: PMC1615753."

[Mayor & Riezman 2004]: https://www.ncbi.nlm.nih.gov/pubmed/15040444/ "Mayor S, Riezman H. Sorting GPI-anchored proteins. Nat Rev Mol Cell Biol. 2004 Feb;5(2):110-20. Review. PubMed PMID: 15040444."

[Kondoh 2005]: https://www.ncbi.nlm.nih.gov/pubmed/15665832/ "Kondoh G, Tojo H, Nakatani Y, Komazawa N, Murata C, Yamagata K, Maeda Y, Kinoshita T, Okabe M, Taguchi R, Takeda J. Angiotensin-converting enzyme is a GPI-anchored protein releasing factor crucial for fertilization. Nat Med. 2005 Feb;11(2):160-6. Epub 2005 Jan 23. PubMed PMID: 15665832."

[Elortza 2006]: https://www.ncbi.nlm.nih.gov/pubmed/16602701 "Elortza F, Mohammed S, Bunkenborg J, Foster LJ, Nühse TS, Brodbeck U, Peck SC, Jensen ON. Modification-specific proteomics of plasma membrane proteins: identification and characterization of glycosylphosphatidylinositol-anchored proteins released upon phospholipase D treatment. J Proteome Res. 2006 Apr;5(4):935-43. PubMed PMID: 16602701."

[Paulick & Bertozzi 2008]: https://www.ncbi.nlm.nih.gov/pubmed/18557633/ "Paulick MG, Bertozzi CR. The glycosylphosphatidylinositol anchor: a complex membrane-anchoring structure for proteins. Biochemistry. 2008 Jul 8;47(27):6991-7000. doi: 10.1021/bi8006324. Epub 2008 Jun 17. Review. PubMed PMID: 18557633; PubMed Central PMCID: PMC2663890."

[Antonyuk 2009]: http://www.ncbi.nlm.nih.gov/pubmed/19204296 "Antonyuk SV, Trevitt CR, Strange RW, Jackson GS, Sangar D, Batchelor M, Cooper S, Fraser C, Jones S, Georgiou T, Khalili-Shirazi A, Clarke AR, Hasnain SS, Collinge J. Crystal structure of human prion protein bound to a therapeutic antibody. Proc Natl Acad Sci U S A. 2009 Feb 24;106(8):2554-8. doi: 10.1073/pnas.0809170106. Epub 2009 Feb 9. PubMed PMID: 19204296; PubMed Central PMCID: PMC2637903."

[Taylor 2009]: https://www.ncbi.nlm.nih.gov/pubmed/19936054/ "Taylor DR, Whitehouse IJ, Hooper NM. Glypican-1 mediates both prion protein lipid raft association and disease isoform formation. PLoS Pathog. 2009 Nov;5(11):e1000666. doi: 10.1371/journal.ppat.1000666. Epub 2009 Nov 20. PubMed PMID: 19936054; PubMed Central PMCID: PMC2773931."

[Fujita & Kinoshita 2010]: https://www.ncbi.nlm.nih.gov/pubmed/19883648 "Fujita M, Kinoshita T. Structural remodeling of GPI anchors during biosynthesis and after attachment to proteins. FEBS Lett. 2010 May 3;584(9):1670-7. doi: 10.1016/j.febslet.2009.10.079. Epub 2009 Oct 31. Review. PubMed PMID: 19883648."

[Hegde & Keenan 2011]: https://www.ncbi.nlm.nih.gov/pubmed/22086371/ "Hegde RS, Keenan RJ. Tail-anchored membrane protein insertion into the endoplasmic reticulum. Nat Rev Mol Cell Biol. 2011 Nov 16;12(12):787-98. doi: 10.1038/nrm3226. Review. PubMed PMID: 22086371; PubMed Central PMCID: PMC3760496."

[Altmeppen 2011]: https://www.ncbi.nlm.nih.gov/pubmed/21619641 "Altmeppen HC, Prox J, Puig B, Kluth MA, Bernreuther C, Thurm D, Jorissen E, Petrowitz B, Bartsch U, De Strooper B, Saftig P, Glatzel M. Lack of a-disintegrin-and-metalloproteinase ADAM10 leads to intracellular accumulation and loss of shedding of the cellular prion protein in vivo. Mol Neurodegener. 2011 May 27;6:36. doi: 10.1186/1750-1326-6-36. PubMed PMID: 21619641; PubMed Central PMCID: PMC3224557."

[Beland 2012]: https://www.ncbi.nlm.nih.gov/pubmed/22993441 "Béland M, Motard J, Barbarin A, Roucou X. PrP(C) homodimerization stimulates the production of PrPC cleaved fragments PrPN1 and PrPC1. J Neurosci. 2012 Sep 19;32(38):13255-63. PubMed PMID: 22993441."

[Lakkaraju 2012]: https://www.ncbi.nlm.nih.gov/pubmed/22648169 "Lakkaraju AK, Thankappan R, Mary C, Garrison JL, Taunton J, Strub K. Efficient secretion of small proteins in mammalian cells relies on Sec62-dependent posttranslational translocation. Mol Biol Cell. 2012 Jul;23(14):2712-22. doi: 10.1091/mbc.E12-03-0228. Epub 2012 May 30. PubMed PMID: 22648169; PubMed Central  PMCID: PMC3395660."

[Suzuki 2012]: https://www.ncbi.nlm.nih.gov/pubmed/22820419/ "Suzuki KG, Kasai RS, Hirosawa KM, Nemoto YL, Ishibashi M, Miwa Y, Fujiwara TK, Kusumi A. Transient GPI-anchored protein homodimers are units for raft organization and function. Nat Chem Biol. 2012 Sep;8(9):774-83. doi: 10.1038/nchembio.1028. Epub 2012 Jul 22. PubMed PMID: 22820419."

[Samocha 2014]: https://www.ncbi.nlm.nih.gov/pubmed/25086666 "Samocha KE, Robinson EB, Sanders SJ, Stevens C, Sabo A, McGrath LM, Kosmicki JA, Rehnström K, Mallick S, Kirby A, Wall DP, MacArthur DG, Gabriel SB, DePristo  M, Purcell SM, Palotie A, Boerwinkle E, Buxbaum JD, Cook EH Jr, Gibbs RA, Schellenberg GD, Sutcliffe JS, Devlin B, Roeder K, Neale BM, Daly MJ. A framework for the interpretation of de novo mutation in human disease. Nat Genet. 2014 Sep;46(9):944-50. doi: 10.1038/ng.3050. Epub 2014 Aug 3. PubMed PMID: 25086666; PubMed Central PMCID: PMC4222185."

[Cortes 2014]: https://www.ncbi.nlm.nih.gov/pubmed/25262930 "Cortes LK, Vainauskas S, Dai N, McClung CM, Shah M, Benner JS, Corrêa IR Jr, VerBerkmoes NC, Taron CH. Proteomic identification of mammalian cell surface derived glycosylphosphatidylinositol-anchored proteins through selective glycan enrichment. Proteomics. 2014 Nov;14(21-22):2471-84. doi: 10.1002/pmic.201400148.  PubMed PMID: 25262930; PubMed Central PMCID: PMC4260145."

[Davis 2015]: https://www.ncbi.nlm.nih.gov/pubmed/26074080 "Davis EM, Kim J, Menasche BL, Sheppard J, Liu X, Tan AC, Shen J. Comparative Haploid Genetic Screens Reveal Divergent Pathways in the Biogenesis and Trafficking of Glycophosphatidylinositol-Anchored Proteins. Cell Rep. 2015 Jun 23;11(11):1727-36. doi: 10.1016/j.celrep.2015.05.026. Epub 2015 Jun 11. PubMed PMID: 26074080; PubMed Central PMCID: PMC4481156."

[Masuishi 2016]: https://www.ncbi.nlm.nih.gov/pubmed/26972028 "Masuishi Y, Kimura Y, Arakawa N, Hirano H. Identification of glycosylphosphatidylinositol-anchored proteins and ω-sites using TiO2-based affinity purification followed by hydrogen fluoride treatment. J Proteomics. 2016 Apr 29;139:77-83. doi: 10.1016/j.jprot.2016.03.008. Epub 2016 Mar 10. PubMed PMID: 26972028."

[Minikel 2016]: http://www.ncbi.nlm.nih.gov/pubmed/26791950 "Minikel EV, Vallabh SM, Lek M, Estrada K, Samocha KE, Sathirapongsasuti JF, McLean CY, Tung JY, Yu LP, Gambetti P, Blevins J, Zhang S, Cohen Y, Chen W, Yamada M, Hamaguchi T, Sanjo N, Mizusawa H, Nakamura Y, Kitamoto T, Collins SJ, Boyd A, Will RG, Knight R, Ponto C, Zerr I, Kraus TF, Eigenbrod S, Giese A, Calero M, de Pedro-Cuesta J, Haïk S, Laplanche JL, Bouaziz-Amar E, Brandel JP, Capellari S, Parchi P, Poleggi A, Ladogana A, O'Donnell-Luria AH, Karczewski KJ,  Marshall JL, Boehnke M, Laakso M, Mohlke KL, Kähler A, Chambert K, McCarroll S, Sullivan PF, Hultman CM, Purcell SM, Sklar P, van der Lee SJ, Rozemuller A, Jansen C, Hofman A, Kraaij R, van Rooij JG, Ikram MA, Uitterlinden AG, van Duijn  CM; Exome Aggregation Consortium (ExAC), Daly MJ, MacArthur DG. Quantifying prion disease penetrance using large population control cohorts. Sci Transl Med. 2016 Jan 20;8(322):322ra9. doi: 10.1126/scitranslmed.aad5169. PubMed PMID: 26791950."

[Lek 2016]: https://www.ncbi.nlm.nih.gov/pubmed/27535533 "Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, O'Donnell-Luria AH, Ware JS, Hill AJ, Cummings BB, Tukiainen T, Birnbaum DP, Kosmicki JA, Duncan LE, Estrada K, Zhao F, Zou J, Pierce-Hoffman E, Berghout J, Cooper DN, Deflaux N, DePristo M, Do R, Flannick J, Fromer M, Gauthier L, Goldstein J, Gupta N, Howrigan D, Kiezun A, Kurki MI, Moonshine AL, Natarajan P,  Orozco L, Peloso GM, Poplin R, Rivas MA, Ruano-Rubio V, Rose SA, Ruderfer DM, Shakir K, Stenson PD, Stevens C, Thomas BP, Tiao G, Tusie-Luna MT, Weisburd B, Won HH, Yu D, Altshuler DM, Ardissino D, Boehnke M, Danesh J, Donnelly S, Elosua  R, Florez JC, Gabriel SB, Getz G, Glatt SJ, Hultman CM, Kathiresan S, Laakso M, McCarroll S, McCarthy MI, McGovern D, McPherson R, Neale BM, Palotie A, Purcell SM, Saleheen D, Scharf JM, Sklar P, Sullivan PF, Tuomilehto J, Tsuang MT, Watkins HC, Wilson JG, Daly MJ, MacArthur DG; Exome Aggregation Consortium. Analysis of protein-coding genetic variation in 60,706 humans. Nature. 2016 Aug 17;536(7616):285-91. doi: 10.1038/nature19057. PubMed PMID: 27535533; PubMed Central PMCID: PMC5018207."

[Kuffer & Lakkaraju 2016]: https://www.ncbi.nlm.nih.gov/pubmed/27501152 "Küffer A, Lakkaraju AK, Mogha A, Petersen SC, Airich K, Doucerain C, Marpakwar R, Bakirci P, Senatore A, Monnard A, Schiavi C, Nuvolone M, Grosshans B, Hornemann S, Bassilana F, Monk KR, Aguzzi A. The prion protein is an agonistic ligand of the G protein-coupled receptor Adgrg6. Nature. 2016 Aug 8. doi: 10.1038/nature19312. [Epub ahead of print] PubMed PMID: 27501152."

[Kinoshita & Fujita 2016]: https://www.ncbi.nlm.nih.gov/pubmed/26563290 "Kinoshita T, Fujita M. Biosynthesis of GPI-anchored proteins: special emphasis on GPI lipid remodeling. J Lipid Res. 2016 Jan;57(1):6-24. doi: 10.1194/jlr.R063313. Epub 2015 Nov 12. Review. PubMed PMID: 26563290; PubMed Central PMCID: PMC4689344."

[Chan 2017]: https://www.ncbi.nlm.nih.gov/pubmed/28671695 "Chan KY, Jang MJ, Yoo BB, Greenbaum A, Ravi N, Wu WL, Sánchez-Guardado L, Lois C, Mazmanian SK, Deverman BE, Gradinaru V. Engineered AAVs for efficient noninvasive gene delivery to the central and peripheral nervous systems. Nat Neurosci. 2017 Aug;20(8):1172-1179. doi: 10.1038/nn.4593. Epub 2017 Jun 26. PubMed PMID: 28671695; PubMed Central PMCID: PMC5529245."

[Linsenmeier 2018]: https://www.ncbi.nlm.nih.gov/pubmed/29625583 "Linsenmeier L, Mohammadi B, Wetzel S, Puig B, Jackson WS, Hartmann A, Uchiyama K, Sakaguchi S, Endres K, Tatzelt J, Saftig P, Glatzel M, Altmeppen HC. Structural and mechanistic aspects influencing the ADAM10-mediated shedding of the prion protein. Mol Neurodegener. 2018 Apr 6;13(1):18. doi: 10.1186/s13024-018-0248-6. PubMed PMID: 29625583; PubMed Central PMCID: PMC5889536."

[Minikel 2019]: https://www.biorxiv.org/content/10.1101/530881v1 "Minikel EV, Karczewski KJ, Martin HC, Cummings BB, Whiffin N, Alfoldi J, Trembath RC, van Heel DA, Daly MJ, Schreiber SL, MacArthur DG. Evaluating potential drug targets through human loss-of-function genetic variation. bioRxiv. 2019 Jan 28;530881."

[Huang 2019]: https://doi.org/10.1101/538421 "Huang Q, Chan KY, Tobey IG, Chan YA, Poterba T, Boutros CL, Balazs AB, Daneman R, Bloom JM, Seed C, Deverman BE. Delivering genes across the blood-brain barrier: LY6A, a novel cellular receptor for AAV-PHP.B capsids. bioRxiv. 2019 Feb 1;538421."


