---
layout: post
title:  "STX6: validating and invalidating a drug target in the same breath"
date:   2023-02-21 12:01:00
author: ericminikel
location: Cambridge, MA
thumb120: http://www.cureffi.org/media/2023/02/stx6-thumbnail.png
summary200: "Sometimes a GWAS hit turns out to be an HMGCR, and sometimes it turns out to be a GPR151."
---

Here's a study I've been waiting to see for a while: prion infection of _Stx6_ knockout mice [[Jones & Hill 2023]]. To me, it beautifully illustrates the challenges in trying to go from GWAS hit to validated drug target.

The [backstory](/2020/05/04/glimpse-non-prnp-risk-factors/) is that 3 years ago, the largest-ever genome-wide association study (GWAS) in sporadic prion disease found, for the first time, two significant hits at loci other than _PRNP_ [[Jones 2020]]. One linkage peak included two missense variants in _GAL3ST1_ with an unknown direction of effect. The other mapped to an eQTL in _STX6_, where the allele associated with higher expression was associated with higher disease risk. This implied that pharmacologic inhibition of syntaxin-6 could potentially be beneficial. Fortuitously, the exact same higher-expressing haplotype was also a risk allele in a tauopathy, providing a tantalizing opportunity for a genetically validated target shared between two different neurodegenerative diseases.

The natural next step, reported in the new study, was to infect _Stx6_ knockout mice with prions and see whether their genotype affected their incubation time. Long story short:

![](/media/2023/02/jones-hill-2023-fig-3.png)

Against two different prion strains, RML and ME7, _Stx6_ knockout brought about a statistically significant but extremely small delay in disease endpoint &mdash; about 12 days (note the broken axis). As a rule, I generally don't let myself believe small effect sizes in mouse survival curves reported in the prion literature. But here, you have both a stronger prior than usual, since the gene is a GWAS hit, as well as slightly better evidence than usual, with independent experiments in two prion strains. On balance, I tend to believe that this effect could well be real. In that sense, this study validates _STX6_ as a potential drug target.

At the same time, the small effect size effectively invalidates it too. If even total, lifelong knockout of syntaxin-6 yields only a 12-day difference in disease tempo, what hope is there for late, incomplete, transient pharmacological inhibition to make enough of a difference that one ever could power a clinical trial in symptomatic patients? To be clear, for a disease with no standard of care, anything, even a small therapeutic benefit would be welcome &mdash; but it's hard to imagine anyone wanting to undertake the massive investment to develop a drug against syntaxin-6 for a benefit that might be too small to ever demonstrate. The original GWAS hit at _STX6_ likewise had only a small effect on sCJD risk (an odds ratio of 1.16), but that was for an [eQTL](https://gtexportal.org/home/snp/chr1_180984717_A_G_b38) that had but a modest effect on _STX6_ expression. The hope was that more potent modulation of syntaxin-6 function would yield more potent modification of disease. Apparently not.

Alternatively, we can also consider the pre-symptomatic community. It's possible &mdash; likely, even &mdash; that syntaxin-6 genotype has a larger effect on prion _initiation_ than it does on prion _replication_. After all, it was originally a GWAS hit for sporadic CJD _risk_, in a case/control design, and did not appear to modify disease _duration_ [[Jones 2020]]. But trying to develop a syntaxin-6 inhibitor for pre-symptomatic patients would be hard for all sorts of reasons. Syntaxin-6 knockdown/knockout does not affect PrP expression [[Jones 2020], [Jones & Hill 2023]], and indeed, the new paper shows that it doesn't have very much impact on prion neuropathology either [[Jones & Hill 2023]]. So a biomarker-based pathway in pre-symptomatics does not seem likely. Indeed, even trying to validate in a mouse model the effect of syntaxin-6 on prion initiation would be rather hard. As we learned the hard way, finding cold hard disease endpoints in a genetic prion disease mouse model [can be tough](/2022/10/12/lessons-learned-first-experiment/) [[Vallabh 2023]], and I worry that if you use a PrP-overexpressing model to drive a faster, more robust disease outcome, you could miss your chance to modify that outcome by knocking out a different risk gene.

My take, then, is that this is probably the end of the road for syntaxin-6 as a drug target in prion disease. We'll see if anyone ends up developing it for tauopathies instead.

It's one more interesting example of how you never know what you'll learn when you go back to functionally validate a GWAS hit. We know that GWAS hits can make successful targets despite their small effect sizes [[Nelson 2015]], but again, the hope is always that a drug is more potent than the original genetic variant, and thereby yields more impact on disease. The classic example is that [a SNP in _HMGCR_](https://genetics.opentargets.org/variant/5_75352778_A_T) only affects LDL cholesterol by 8% and coronary artery disease risk by 8%, but statins, which inhibit _HMGCR_'s protein product, can lower LDL by 19-37% and lower coronary heart disease-related deaths by 23% [[Wilt 2004]]. But then there are examples like _GPR151_ where GWAS found that heterozygous loss-of-function modestly reduces risk diabetes and body mass index, but a subsequent deep dive found that the effect of total knockout was null or maybe even paradoxically increased risk [[Gurtan 2022]]. When you see a new GWAS hit, you don't know if you've just hit upon an _HMGCR_ or a _GPR151_. But it's very much worth finding out, and I'm glad that they did this follow-up study on _STX6_. I wonder, is anyone doing _GAL3ST1_, the other new GWAS hit? For the time being, _PRNP_ continues its long reign as the sole drug target in prion disease.


[Wilt 2004]: https://pubmed.ncbi.nlm.nih.gov/15249352/ "Wilt TJ, Bloomfield HE, MacDonald R, Nelson D, Rutks I, Ho M, Larsen G, McCall A, Pineros S, Sales A. Effectiveness of statin therapy in adults with coronary heart disease. Arch Intern Med. 2004 Jul 12;164(13):1427-36. doi: 10.1001/archinte.164.13.1427. PMID: 15249352."

[Kathiresan 2008]: https://pubmed.ncbi.nlm.nih.gov/18193044/ "Kathiresan S, Melander O, Guiducci C, Surti A, Burtt NP, Rieder MJ, Cooper GM, Roos C, Voight BF, Havulinna AS, Wahlstrand B, Hedner T, Corella D, Tai ES, Ordovas JM, Berglund G, Vartiainen E, Jousilahti P, Hedblad B, Taskinen MR, Newton-Cheh C, Salomaa V, Peltonen L, Groop L, Altshuler DM, Orho-Melander M. Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. Nat Genet. 2008 Feb;40(2):189-97. doi: 10.1038/ng.75. Epub 2008 Jan 13. Erratum in: Nat Genet. 2008 Nov;40(11):1384. PMID: 18193044; PMCID: PMC2682493."

[Nelson 2015]: https://pubmed.ncbi.nlm.nih.gov/26121088/ "Nelson MR, Tipney H, Painter JL, Shen J, Nicoletti P, Shen Y, Floratos A, Sham PC, Li MJ, Wang J, Cardon LR, Whittaker JC, Sanseau P. The support of human genetic evidence for approved drug indications. Nat Genet. 2015 Aug;47(8):856-60. doi: 10.1038/ng.3314. Epub 2015 Jun 29. PMID: 26121088."

[Jones 2020]: https://pubmed.ncbi.nlm.nih.gov/32949544/ "Jones E, Hummerich H, Viré E, Uphill J, Dimitriadis A, Speedy H, Campbell T, Norsworthy P, Quinn L, Whitfield J, Linehan J, Jaunmuktane Z, Brandner S, Jat P, Nihat A, How Mok T, Ahmed P, Collins S, Stehmann C, Sarros S, Kovacs GG, Geschwind MD, Golubjatnikov A, Frontzek K, Budka H, Aguzzi A, Karamujić-Čomić H, van der Lee SJ, Ibrahim-Verbaas CA, van Duijn CM, Sikorska B, Golanska E, Liberski PP, Calero M, Calero O, Sanchez-Juan P, Salas A, Martinón-Torres F, Bouaziz-Amar E, Haïk S, Laplanche JL, Brandel JP, Amouyel P, Lambert JC, Parchi P, Bartoletti-Stella A, Capellari S, Poleggi A, Ladogana A, Pocchiari M, Aneli S, Matullo G, Knight R, Zafar S, Zerr I, Booth S, Coulthart MB, Jansen GH, Glisic K, Blevins J, Gambetti P, Safar J, Appleby B, Collinge J, Mead S. Identification of novel risk loci and causal insights for sporadic Creutzfeldt-Jakob disease: a genome-wide association study. Lancet Neurol. 2020 Oct;19(10):840-848. doi: 10.1016/S1474-4422(20)30273-8. Epub 2020 Sep 16. PMID: 32949544; PMCID: PMC8220892."

[Gurtan 2022]: https://pubmed.ncbi.nlm.nih.gov/35381001/ "Gurtan A, Dominy J, Khalid S, Vong L, Caplan S, Currie T, Richards S, Lamarche L, Denning D, Shpektor D, Gurinovich A, Rasheed A, Hameed S, Saeed S, Saleem I, Jalal A, Abbas S, Sultana R, Rasheed SZ, Memon FU, Shah N, Ishaq M, Khera AV, Danesh J, Frossard P, Saleheen D. Analyzing human knockouts to validate GPR151 as a therapeutic target for reduction of body mass index. PLoS Genet. 2022 Apr 5;18(4):e1010093. doi: 10.1371/journal.pgen.1010093. PMID: 35381001; PMCID: PMC9022822."

[Vallabh 2023]: https://pubmed.ncbi.nlm.nih.gov/36651748/ "Vallabh SM, Zou D, Pitstick R, O'Moore J, Peters J, Silvius D, Kriz J, Jackson WS, Carlson GA, Minikel EV, Cabin DE. Therapeutic Trial of anle138b in Mouse Models of Genetic Prion Disease. J Virol. 2023 Jan 18:e0167222. doi: 10.1128/jvi.01672-22. Epub ahead of print. PMID: 36651748."

[Jones & Hill 2023]: https://doi.org/10.1101/2023.01.10.523281 "Jones E, Hill E, Linehan J, Nazari T, Caulder A, Codner GF, Hutchison M, Mackenzie M, De Oliveira MW, Al-Doujaily H, Sandberg M, Viré E, Cunningham TJ, Asante EA, Brandner S, Collinge J, Mead S. Knockout of Sporadic Creutzfeldt-Jakob Disease Risk Gene <em>Stx6</em> in Mice Extends Prion Disease Incubation Time. bioRxiv. 2023 Jan 10;2023.01.10.523281."

