---
layout: post
title:  "Evidence that PrP GPI signal sequence matters"
date:   2024-02-29
author: ericminikel
location: Cambridge, MA
thumb120: 
summary200: "PrP codons 231-253 are not even in the mature protein, but it appears that they can affect the risk and tempo of prion disease nonetheless."
---

![](/media/2024/02/prp-diagram.png)

*Diagram of human PrP. Amino acids 23-230 are the mature protein. 231-253 are the GPI signal.*

Human PrP is 253 codons long, but there are only 208 amino acids in the mature, post-translationally modified protein. The first 22 residues are a signal peptide (SP) that directs PrP to the ER and ultimately the cell surface. The last 23 residues are a GPI signal that gets chopped off and replaced with a GPI anchor that is inserted into the ER membrane and ultimately the outer leaflet of the cell membrane. That which isn't there can't matter, right? If the 23 amino acids of the GPI signal are not even present in the mature protein, how can they affect the development of prion disease? And yet a fair amount of evidence now indicates that they do matter. This blog post will unpack what we know.

### background

GPI stands for glycosylphosphatidylinositol. It's a sugar-and-fat chain that gets attached to the C terminus of some proteins and anchors them into the membrane. I took a deeper dive on GPI anchors in [this post](https://www.cureffi.org/2019/05/14/the-deal-with-gpi-anchored-proteins/) if you want more background. That PrP is GPI-anchored was shown empirically in a series of papers from the Prusiner lab [[Stahl 1987], [Stahl 1990], [Stahl 1992]]. GPI anchors are not just one thing &mdash; there can be a variety of compositions. Through a series of enzymatic digestions and mass spectrometry experiments, they identified at least 6 GPI anchor compositions associated with PrP, comprising different combinations of mannose, galactose, GalNAc, and sialic acid, sitting on top of the phosphatidylinositol [[Stahl 1992]].

### evidence that the GPI anchor matters

First things first, the presence vs. absence of the GPI anchor matters a lot. As reviewed in [this post](/2013/06/29/proteolytic-shedding-of-prion-protein/), a series of mouse models have shown that expression of "anchorless" PrP causes an atypical prion disease (sometimes called PrP cerebral amyloid angiopathy) characterized by intense amyloid deposition including onto blood vessels [[Chesebro 2005], [Trifilo 2008], [Chesebro 2010], [Stohr 2011]]. This is mirrored by a rare subset of genetic prion disease in humans, where late truncating mutations that remove the GPI anchor, leading to secreted PrP, cause amyloid deposition in peripheral tissues as well as the brain, leading to inflammatory bowel disease-like symptoms, progressing to peripheral neuropathy and then slowly progressive dementia over 20 years [[Mead & Reilly 2015]]. The secreted PrP in these anchorless mouse models is (almost) entirely lacking in N-linked glycans at residues N181 and N197.

But of course the presence vs. absence of the GPI anchor matters, since you are talking about a difference in whether PrP is on the cell surface or secreted into extracellular space. What about the the amino acid sequence of the GPI signal and the composition of the GPI anchor itself?

The evidence from human genetics comes from M232R, a genetic variant that affects only the GPI signal and appears to increase the risk of prion disease by about 5-fold [[Nozaki 2010], [Minikel 2016]]. But _how_ does it do that?

One study from Tetsuyuki Kitamoto's lab expressed every possible mutant at codons 228, 230, 231, and 232 in N2a cells and tested their GPI anchoring efficiency and ability to convert to PrP<sup>Sc</sup> [[Hizume 2010]]. That study reported that 232R PrP was under-expressed on the cell surface, and that more of the 232R PrP was secreted into cell culture media. Whereas WT PrP was only released when cells were treted with PI-PLC &mdash; an enzyme that cleaves GPI anchors &mdash; it is shown in Figure 2C that some 232R PrP is either shed or secreted from the cells even in the absence of PI-PLC. They used Triton X-114 phase partitioning [[Bordier 1981]], a procedure which exploits the temperature-dependent solubility of the detergent Triton X-114. This detergent can dissolve at low temperatures, but at high temperatures it separates into an aqueous phase and a detergent phase. Membrane proteins and membrane-anchored (e.g. GPI) proteins will be in the detergent phase, while secreted proteins will be in the aqueous phase. Whereas WT PrP was found about equally in both, 232R PrP was found predominantly in the aqueous phase, suggesting more of it was not GPI-anchored. But an enormous caveat to all of this is that the system used was overexpression in transfected N2a cells, which may or may not reflect the real situation _in vivo_. And indeed, later work by this same group contradicted some of these findings (see below).

There have been various other cell culture studies, including one in primary neurons from knockout mice [[Bate 2016]]. PrP<sup>C</sup>purified from N9 glial cells was added to cell culture media and they watched if and where the PrP was taken up by the knockout neurons. They found that PrP with its GPI anchor intact was taken up and localized to synapses, whereas if they used enzymes to remove either the sialic acid or one of the acyl chain from the GPI anchor, the synaptic localization was lost. Although it was an artificial system, this study provided at least some evidence that a change in GPI anchor composition could lead to a change in PrP's behavior, in this case, its localization.

Perhaps the first in vivo evidence that changes in the GPI signal can affect prion disease came from a study replacing PrP's GPI signal with that of Thy1, a different GPI-anchored protein [[Puig 2019]]. Thy1's GPI anchor, unlike PrP's, does not contain sialic acid, so they hypothesized that as was shown in the primary neurons [[Bate 2016]], an unsialyated GPI anchor would lead PrP to behave differently. They created a transgenic mouse expressing PrP with the Thy1 GPI anchor. By Western blot, the transgenic mice appeared to express about the same amount of PrP as wild-type mice, or if anything, slightly more. And yet the incubation time when inoculated with prions was longer than wild-type mice: 195 vs. 155 dpi for RML, 156 vs. 144 dpi for 22L. Although the authors concluded that "the delay was independent of prion strain", and true, the direction of effect was the same, it actually looks to me like there may be some difference in the magnitude of effect caused by the Thy1 GPI anchor (+26% for RML vs. +8% for 22L), which may be interesting in light of later results (see below). They also reported several other functional differences in the PrP-Thy1GPI mice, including changed localization, different neuropathology after prion infection, and a striking reduction in the amount of PrP [shedding](/2013/06/29/proteolytic-shedding-of-prion-protein/) (cleavage at the GPI anchor, releasing PrP into extracellular space).

Last year, a new in vivo study from the Kitamoto group, this one using human prions in transgenic mice, provided more evidence for how the M232R variant in particular can influence prion disease [[Kobayashi 2023]]. They made knock-in mice expressing either wild-type (232M) human PrP, or 232R human PrP, and they performed detailed biochemical studies on these in addition to prion infection experiments. They found that 232R PrP was indeed GPI-anchored and in a Triton X-114 partitioning assay, it partitioned to the detergent side (where GPI-anchored proteins go) just as much as wild-type PrP did. This refuted the finding in N2a cells from [[Hizume 2010]] that 232R PrP is not properly anchored. Nonetheless, there were several notable biochemical differences, including that the 232R PrP had an increased proportion of mono- and un-glycosylated PrP, and mass spec revealed a reduced amount of sialic acid and GalNAc in the GPI anchors of the 232R mice. They inoculated both mouse lines with 7 different human prion isolates. 2 isolates yielded little or no transmission to mice with either wild-type or 232R human PrP; 1 isolate transmitted equivalently to both lines; and 4 isolates transmitted significantly faster to the 232R mice. In some cases, the difference was large &mdash; for instance for an sCJD MM1 isolate (called M1 I22 in the paper), the 232R mice died after 300+ days, while the mice with wild-type human PrP took 600 days. In each case, the neuropathology looked similar, it just onset much more rapidly. They concluded that M232R accelerates prion disease in a prion strain-dependent manner.

### conclusions so far

While there remain a lot of mysteries, I see enough data here to be fairly confident that differences in the GPI signal sequence, and therefore in the resulting GPI anchor, somehow affect prion disease risk. The human genetics are conclusive that M232R causes increased risk, the question is just by what mechanism. The mouse model work from [[Puig 2019]] and [[Kobayashi 2023]] offer the outlines of a mechanism by showing that changing the GPI signal sequence on PrP can result in biochemical changes including a reduction in sialyation. These may in turn lead to other differences such as a reduction in N-linked glycosylation. There may also be a change in subcellular localization, though such changes are usually just anecdotes &mdash; it's hard to gather really statistically robust data in an in vivo system to show how localization is affected. One or more of these changes ultimately affects how fast prions replicate, although it matters more for some strains than for others. 

This is all highly plausible. If anything, the problem is there are too many plausible mechanisms, and more than one may be at work, so it's hard to give one succinct answer as to _why_ the GPI signal matters. For instance, different prion strains seem to ["prefer"](/2013/05/05/prion-protein-n-linked-glycosylation-review-and-assessment-of-therapeutic-potential/) different N-linked glycosylation states of PrP, so maybe the glycosylation difference is ultimately what matters. But localization is at least as strong a contender: PrP is believed to be localized to cholesterol-rich lipid rafts thanks to its GPI anchor, and given the mechanism of prion replication &mdash; PrP<sup>Sc</sup> templates PrP<sup>C</sup>, differences in _local concentration_ could have a big impact on how often two PrP molecules get to touch each other. If one GPI anchor makes PrP more spread out than another, then prions will replicate more slowly &mdash; it's as if the year were 2020 and they were standing 2 meters apart. And I didn't even touch on all the hypotheses nominated in these articles, which also touched on things like trafficking into and through the ER. Cell biology of the secretory pathway is complicated, and a lot of the methods for studying it in vivo are low-throughput and imprecise. For now, it seems fairly safe to say that the GPI signal matters, and there are multiple possible reasons why.

[Bordier 1981]: https://pubmed.ncbi.nlm.nih.gov/6257680/ "Bordier C. Phase separation of integral membrane proteins in Triton X-114 solution. J Biol Chem. 1981 Feb 25;256(4):1604-7. PMID: 6257680."

[Stahl 1987]: https://pubmed.ncbi.nlm.nih.gov/2444340/ "Stahl N, Borchelt DR, Hsiao K, Prusiner SB. Scrapie prion protein contains a phosphatidylinositol glycolipid. Cell. 1987 Oct 23;51(2):229-40. doi: 10.1016/0092-8674(87)90150-4. PMID: 2444340."

[Stahl 1990]: https://pubmed.ncbi.nlm.nih.gov/1980209/ "Stahl N, Baldwin MA, Burlingame AL, Prusiner SB. Identification of glycoinositol phospholipid linked and truncated forms of the scrapie prion protein. Biochemistry. 1990 Sep 25;29(38):8879-84. doi: 10.1021/bi00490a001. PMID: 1980209."

[Stahl 1992]: https://pubmed.ncbi.nlm.nih.gov/1350920/ "Stahl N, Baldwin MA, Hecker R, Pan KM, Burlingame AL, Prusiner SB. Glycosylinositol phospholipid anchors of the scrapie and cellular prion proteins contain sialic acid. Biochemistry. 1992 Jun 2;31(21):5043-53. doi: 10.1021/bi00136a600. PMID: 1350920."

[Chesebro 2005]: https://pubmed.ncbi.nlm.nih.gov/15933194/ "Chesebro B, Trifilo M, Race R, Meade-White K, Teng C, LaCasse R, Raymond L, Favara C, Baron G, Priola S, Caughey B, Masliah E, Oldstone M. Anchorless prion protein results in infectious amyloid disease without clinical scrapie. Science. 2005 Jun 3;308(5727):1435-9. doi: 10.1126/science.1110837. PMID: 15933194."

[Trifilo 2008]: https://pubmed.ncbi.nlm.nih.gov/18667494/ "Trifilo MJ, Sanchez-Alavez M, Solforosi L, Bernard-Trifilo J, Kunz S, McGavern D, Oldstone MB. Scrapie-induced defects in learning and memory of transgenic mice expressing anchorless prion protein are associated with alterations in the gamma aminobutyric acid-ergic pathway. J Virol. 2008 Oct;82(20):9890-9. doi: 10.1128/JVI.00486-08. Epub 2008 Jul 30. PMID: 18667494; PMCID: PMC2566293."

[Chesebro 2010]: https://pubmed.ncbi.nlm.nih.gov/20221436/ "Chesebro B, Race B, Meade-White K, Lacasse R, Race R, Klingeborn M, Striebel J, Dorward D, McGovern G, Jeffrey M. Fatal transmissible amyloid encephalopathy: a new type of prion disease associated with lack of prion protein membrane anchoring. PLoS Pathog. 2010 Mar 5;6(3):e1000800. doi: 10.1371/journal.ppat.1000800. PMID: 20221436; PMCID: PMC2832701."

[Nozaki 2010]: https://pubmed.ncbi.nlm.nih.gov/20855418/ "Nozaki I, Hamaguchi T, Sanjo N, Noguchi-Shinohara M, Sakai K, Nakamura Y, Sato T, Kitamoto T, Mizusawa H, Moriwaka F, Shiga Y, Kuroiwa Y, Nishizawa M, Kuzuhara S, Inuzuka T, Takeda M, Kuroda S, Abe K, Murai H, Murayama S, Tateishi J, Takumi I, Shirabe S, Harada M, Sadakane A, Yamada M. Prospective 10-year surveillance of human prion diseases in Japan. Brain. 2010 Oct;133(10):3043-57. doi: 10.1093/brain/awq216. Epub 2010 Sep 20. PMID: 20855418."

[Hizume 2010]: https://pubmed.ncbi.nlm.nih.gov/20040362/ "Hizume M, Kobayashi A, Mizusawa H, Kitamoto T. Amino acid conditions near the GPI anchor attachment site of prion protein for the conversion and the GPI anchoring. Biochem Biophys Res Commun. 2010 Jan 22;391(4):1681-6. doi: 10.1016/j.bbrc.2009.12.128. Epub 2009 Dec 29. PMID: 20040362."

[Stohr 2011]: https://pubmed.ncbi.nlm.nih.gov/22160704/ "Stöhr J, Watts JC, Legname G, Oehler A, Lemus A, Nguyen HO, Sussman J, Wille H, DeArmond SJ, Prusiner SB, Giles K. Spontaneous generation of anchorless prions in transgenic mice. Proc Natl Acad Sci U S A. 2011 Dec 27;108(52):21223-8. doi: 10.1073/pnas.1117827108. Epub 2011 Dec 12. PMID: 22160704; PMCID: PMC3248514."

[Mead & Reilly 2015]: https://pubmed.ncbi.nlm.nih.gov/25623792/ "Mead S, Reilly MM. A new prion disease: relationship with central and peripheral amyloidoses. Nat Rev Neurol. 2015 Feb;11(2):90-7. doi: 10.1038/nrneurol.2014.263. Epub 2015 Jan 27. PMID: 25623792."

[Minikel 2016]: https://pubmed.ncbi.nlm.nih.gov/26791950/ "Minikel EV, Vallabh SM, Lek M, Estrada K, Samocha KE, Sathirapongsasuti JF, McLean CY, Tung JY, Yu LP, Gambetti P, Blevins J, Zhang S, Cohen Y, Chen W, Yamada M, Hamaguchi T, Sanjo N, Mizusawa H, Nakamura Y, Kitamoto T, Collins SJ, Boyd A, Will RG, Knight R, Ponto C, Zerr I, Kraus TF, Eigenbrod S, Giese A, Calero M, de Pedro-Cuesta J, Haïk S, Laplanche JL, Bouaziz-Amar E, Brandel JP, Capellari S, Parchi P, Poleggi A, Ladogana A, O'Donnell-Luria AH, Karczewski KJ, Marshall JL, Boehnke M, Laakso M, Mohlke KL, Kähler A, Chambert K, McCarroll S, Sullivan PF, Hultman CM, Purcell SM, Sklar P, van der Lee SJ, Rozemuller A, Jansen C, Hofman A, Kraaij R, van Rooij JG, Ikram MA, Uitterlinden AG, van Duijn CM; Exome Aggregation Consortium (ExAC); Daly MJ, MacArthur DG. Quantifying prion disease penetrance using large population control cohorts. Sci Transl Med. 2016 Jan 20;8(322):322ra9. doi: 10.1126/scitranslmed.aad5169. PMID: 26791950; PMCID: PMC4774245."

[Bate 2016]: https://pubmed.ncbi.nlm.nih.gov/27325697/ "Bate C, Nolan W, McHale-Owen H, Williams A. Sialic Acid within the Glycosylphosphatidylinositol Anchor Targets the Cellular Prion Protein to Synapses. J Biol Chem. 2016 Aug 12;291(33):17093-101. doi: 10.1074/jbc.M116.731117. Epub 2016 Jun 20. PMID: 27325697; PMCID: PMC5016113."

[Puig 2019]: https://pubmed.ncbi.nlm.nih.gov/30608982/ "Puig B, Altmeppen HC, Linsenmeier L, Chakroun K, Wegwitz F, Piontek UK, Tatzelt J, Bate C, Magnus T, Glatzel M. GPI-anchor signal sequence influences PrPC sorting, shedding and signalling, and impacts on different pathomechanistic aspects of prion disease in mice. PLoS Pathog. 2019 Jan 4;15(1):e1007520. doi: 10.1371/journal.ppat.1007520. PMID: 30608982; PMCID: PMC6334958."

[Kobayashi 2023]: https://pubmed.ncbi.nlm.nih.gov/36879070/ "Kobayashi A, Hirata T, Shimazaki T, Munesue Y, Aoshima K, Kimura T, Nio-Kobayashi J, Hasebe R, Takeuchi A, Matsuura Y, Kusumi S, Koga D, Iwasaki Y, Kinoshita T, Mohri S, Kitamoto T. A point mutation in GPI-attachment signal peptide accelerates the development of prion disease. Acta Neuropathol. 2023 May;145(5):637-650. doi: 10.1007/s00401-023-02553-5. Epub 2023 Mar 6. PMID: 36879070."

